---
layout: default
title: "Resources"
permalink: /resources/
---


<div class="container">
    <div class="col-md-8 offset-2">

        <div class="row border_top py-2 mt-4">
            <h5>Contact Information</h5>

            <p>455 Main St<br>
            Whitehead Institute Suite 661B<br>
            Cambridge, MA 02142</p>

            <p><a href="https://wi.mit.edu/">Whitehead Institute</a></p>
            <p><a href="https://biology.mit.edu/">MIT Biology</a></p>
            <p><a href="https://www.hhmi.org/">Howard Hughes Medical Institute</a></p>


            <p>Addgene: <a href="https://www.addgene.org/search/catalog/plasmids/?q=Weissman+Jonathan">Plasmids</a> | <a href="https://www.addgene.org/search/catalog/pooled-libraries/?q=Weissman+Jonathan">Pooled Libraries</a></p>

        </div>

        <div class="row border_top py-2 mt-4">
            <h4>New Resources</h4>

            <div class="col-6">
                <h5><a href="https://github.com/aristoteleo" target="_blank">Aristotle: a foundational software ecosystem</a></h5>
                <p>Aristotle is an open-source software ecosystem for predictive biology developed by the Weissman lab,
                encompassing a comprehensive collection of toolkits catering to various domains such as metabolic
                labeling data processing, RNA velocity field analysis, spatial transcriptomics, and quantitative modeling of single-cell genomics. </p>
            </div>
            <div class="col-3">
                <img src="/assets/img/ani/hsc_fate.gif" style="width: 220px; height: 225px;">
            </div>
            <div class="col-3">
                <img src="/assets/img/ani/hsc_fate_p.gif" style="width: 220px; height: 225px;">
            </div>

            <div class="col-6">
                <p>Current toolkits includes
                metabolic labeling data preprocessing pipeline <a href="https://github.com/aristoteleo/dynast-release" target="_blank">Dynast</a>
                (<a href="https://dynast-release.readthedocs.io/en/latest/" target="_blank">tutorial</a>),
                dynamics analytical and predictive framework <a href="https://github.com/aristoteleo/dynamo-release" target="_blank">Dynamo</a>
                (<a href="https://dynamo-release.readthedocs.io/en/latest/" target="_blank">tutorial</a>), 
                quantitative spatiotemporal modeling framework <a href="https://github.com/aristoteleo/spateo-release" target="_blank">Spateo</a>
                (<a href="https://spateo-release.readthedocs.io/en/latest/" target="_blank">tutorial</a>). </p>
            </div>

            <div class="col-6">
                <ul style="font-size: 14px;">
                  <li>Left visually depicts the trajectory predicted by the reconstructed vector field in the gene expression space.</li>
                  <li>Right demonstrates the cell-fate diversions after in silico perturbations.</li>
                </ul>
            </div>

            <div class="col">
                <h5><a href="https://gwps.wi.mit.edu/">Genome-Wide Perturb-Seq</a></h5>
                <p>Perturb-seq is an experimental approach to map the transcriptional effects of genetic perturbations by combining
                CRISPR-based genetic screening with information-rich, single-cell RNA-sequencing phenotypes.</p>

                <p>Applied at genome-scale, Perturb-seq enables systematic prediction of gene function and in-depth study of complex
                cellular phenotypes such as aneuploidy, differentiation, RNA processing, and stress-specific regulation of the mitochondrial genome.</p>
            </div>
            <div class="col">
                <img src="/assets/img/gwps.png" style="width: 400px;">
            </div>

            <div class="col-12 mt-5">
                <!-- <div style="width:100px;height: 2px;background-color: lightgray;" class="mt-2 mb-4"></div> -->

                <h5>CRISPRi screening protocols</h5>

                related to <a href="https://www.biorxiv.org/content/10.1101/2022.07.13.499814v2"> Maximizing CRISPRi efficacy and accessibility with dual-sgRNA libraries and optimal effectors</a>
                <p><ul>
                <li>Protocol 1: <a href="/resources/2022_crispri_protocols/Protocol_1_dual_sgRNA_lib_cloning.pdf">design and cloning of pooled libraries of programmed dual-sgRNAs</a></li>
                <li>Protocol 2: <a href="/resources/2022_crispri_protocols/Protocol_2_sequencing_library_preparation.pdf">preparation of sgRNA sequencing libraries after CRISPR screens</a></li>
                <li>Protocol 3: <a href="/resources/2022_crispri_protocols/Protocol_3_arrayed_dual_sgRNA_cloning.pdf">cloning of arrayed dual-sgRNAs</a></li>
                <li>Protocol 4: <a href="/resources/2022_crispri_protocols/Protocol_4_CRISPRi_cell_line_generation.pdf">generation and validation of CRISPRi cell lines</a></li>
                </ul></p>

            </div>

        </div>


        <div class="row border_top py-2 mt-4">
            <h4>Software from the Weissman Lab</h4>

            <p>
                <a href="https://github.com/aristoteleo" target="_blank"><b><i>Aristotle</i></b> foundational software ecosystem for predictive biology, including:</a><br>
                &emsp;<a href="https://github.com/aristoteleo/dynast-release" target="_blank"><b><i>Dynast: </i></b>metabolic labeling data preprocessing pipeline</a><br>
                &emsp;<a href="https://github.com/aristoteleo/dynamo-release" target="_blank"><b><i>Dynamo: </i></b>single cell expression dynamics analytical and predictive framework</a><br>
                &emsp;<a href="https://github.com/aristoteleo/spateo-release" target="_blank"><b><i>Spateo: </i></b>single cell quantitative spatiotemporal modeling framework</a>
            </p>

            <p><a href="http://gimap.ucsf.edu/"><b><i>GImap</i></b> software for analysis of pooled screens and genetic interaction maps in mammalian cells</a></p>

            <p><a href="https://github.com/alexfields/ORF-RATER/"><b><i>ORF-RATER</i></b> package for coding sequence annotation based on ribosome profiling data</a></p>

            <p><a href="https://github.com/joshuagryphon/plastid/"><b><i>Plastid</i></b> python library for analysis of high-throughput sequencing and other genomic data</a></p>

            <p><a href="https://github.com/mhorlbeck/ScreenProcessing"><b><i>ScreenProcessing pipeline</i></b> for analysis of pooled genetic screens</a></p>

        </div>

        <div class="row border_top py-2 mt-4">
            <h4>Other links</h4>

            <p><a href="/crispr/">CRISPRi/a screening materials</a></p>

            <p><a href="https://yeastgfp.yeastgenome.org/">Yeast GFP-fusion localization database</a></p>

            <p><a href="https://www.yeastgenome.org/">Saccharomyces Genome Database</a></p>

            <p><a href="https://www.yeastgenome.org/">Saccharomyces Genome Database</a></p>

        </div>
    </div>
</div>