{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "69c1fd79-af03-4e01-80de-3f71722276e7",
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "/usr/local/miniconda3/envs/py-env/lib/python3.11/site-packages/requests/__init__.py:102: RequestsDependencyWarning: urllib3 (1.26.8) or chardet (5.2.0)/charset_normalizer (2.0.4) doesn't match a supported version!\n",
      "  warnings.warn(\"urllib3 ({}) or chardet ({})/charset_normalizer ({}) doesn't match a supported \"\n"
     ]
    }
   ],
   "source": [
    "import yaml\n",
    "import requests\n",
    "from bs4 import BeautifulSoup\n",
    "from pymed import PubMed\n",
    "import urllib.request\n",
    "import requests\n",
    "import gdown\n",
    "import os\n",
    "import pandas as pd\n",
    "from PIL import Image"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a7486e04-b58e-49bd-97bd-e91e2862b221",
   "metadata": {
    "tags": []
   },
   "source": [
    "### Publications"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "f74dd27d-f5ba-4a0d-8760-72212ccce249",
   "metadata": {},
   "outputs": [],
   "source": [
    "# url = 'https://weissmanlab.ucsf.edu/publications/publications.html'\n",
    "# page = requests.get(url)\n",
    "\n",
    "# soup = BeautifulSoup(page.text, 'html.parser')\n",
    "\n",
    "# old_website_pubs = [(a.get('href'), a.text) for pub_div in soup.find_all(id='publication') for a in pub_div.find_all('a')]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "77ef5dac-0008-4d28-afb9-06f4adb555d8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "280"
      ]
     },
     "execution_count": 2,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pubmed = PubMed(tool=\"MyTool\", email=\"lenail@mit.edu\")\n",
    "results = list(pubmed.query(\"Jonathan Weissman\", max_results=500))\n",
    "len(results)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "0e8cd5af-8fdc-4df9-a7ba-0deb957efde8",
   "metadata": {},
   "outputs": [],
   "source": [
    "pubmed_pubs = []\n",
    "\n",
    "for result in results:\n",
    "    try:\n",
    "        pubmed_pubs.append(\n",
    "            dict(title=result.title,\n",
    "                authors=[str(author['firstname'])+' '+str(author['lastname']) for author in result.authors],\n",
    "                publication_date=result.publication_date.strftime(\"%d/%m/%y\"),\n",
    "                publication_year=result.publication_date.strftime(\"%Y\"),\n",
    "                pubmed_id=result.pubmed_id.split('\\n')[0],\n",
    "                abstract=result.abstract,\n",
    "                doi=result.doi.split('\\n')[0],\n",
    "                journal=result.journal))\n",
    "\n",
    "    except:\n",
    "        # print(result.toDict())\n",
    "\n",
    "        print(result.title)\n",
    "        print(result.authors)\n",
    "        print(result.publication_date)\n",
    "        print(result.pubmed_id)\n",
    "        print(result.abstract)\n",
    "        print(result.doi)\n",
    "        print(result.journal)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "ca05e025-c9cc-4718-bdc7-a332a0829f18",
   "metadata": {},
   "outputs": [],
   "source": [
    "replacements = {\n",
    "    'bioRxiv : the preprint server for biology': 'biorxiv',\n",
    "    'Methods in molecular biology (Clifton, N.J.)': 'Methods in molecular biology',\n",
    "    'Current biology : CB': 'Current biology',\n",
    "    'Science (New York, N.Y.)': 'Science',\n",
    "    'Proceedings of the National Academy of Sciences of the United States of America': 'PNAS',\n",
    "    'Molecular & cellular proteomics : MCP': 'Molecular & cellular proteomics'\n",
    "}"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "b61a87ea-ecc2-4ea9-b827-701e9d116ddf",
   "metadata": {},
   "outputs": [],
   "source": [
    "for p in pubmed_pubs:\n",
    "    if p['journal'] in replacements:\n",
    "        p['journal'] = replacements[p['journal']]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "64cd4451-34b8-41c9-8919-0016dcdebcce",
   "metadata": {},
   "outputs": [],
   "source": [
    "pubmed_pubs = pd.DataFrame.from_records(pubmed_pubs)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "582e47d3-5ee5-410b-9723-d0f48414f5ad",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('_data/publications.yml', 'r') as f:\n",
    "    old_pubs = pd.json_normalize(yaml.safe_load(f))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "44124882",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>publication_year</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>journal</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>High-resolution spatial mapping of cell state ...</td>\n",
       "      <td>[Luke W Koblan, Kathryn E Yost, Pu Zheng, Will...</td>\n",
       "      <td>24/07/25</td>\n",
       "      <td>2025</td>\n",
       "      <td>40705858</td>\n",
       "      <td>Charting the spatiotemporal dynamics of cell f...</td>\n",
       "      <td>10.1126/science.adx3800</td>\n",
       "      <td>Science</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>High-resolution spatial mapping of cell state ...</td>\n",
       "      <td>[Luke W Koblan, Kathryn E Yost, Pu Zheng, Will...</td>\n",
       "      <td>15/07/25</td>\n",
       "      <td>2025</td>\n",
       "      <td>40661400</td>\n",
       "      <td>Charting the spatiotemporal dynamics of cell f...</td>\n",
       "      <td>10.1101/2025.06.15.659774</td>\n",
       "      <td>biorxiv</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Shaping the composition of the mitochondrial o...</td>\n",
       "      <td>[Gayathri Muthukumar, Jonathan S Weissman]</td>\n",
       "      <td>17/06/25</td>\n",
       "      <td>2025</td>\n",
       "      <td>40523936</td>\n",
       "      <td>Mitochondria are critical double-membraned org...</td>\n",
       "      <td>10.1038/s41556-025-01683-0</td>\n",
       "      <td>Nature cell biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Perturb-Multimodal: A platform for pooled gene...</td>\n",
       "      <td>[Reuben A Saunders, William E Allen, Xingjie P...</td>\n",
       "      <td>14/06/25</td>\n",
       "      <td>2025</td>\n",
       "      <td>40513557</td>\n",
       "      <td>Metazoan life requires the coordinated activit...</td>\n",
       "      <td>10.1016/j.cell.2025.05.022</td>\n",
       "      <td>Cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Transcriptome-wide analysis of differential ex...</td>\n",
       "      <td>[Ajay Nadig, Joseph M Replogle, Angela N Pogso...</td>\n",
       "      <td>22/04/25</td>\n",
       "      <td>2025</td>\n",
       "      <td>40259084</td>\n",
       "      <td>Single-cell CRISPR screens such as Perturb-seq...</td>\n",
       "      <td>10.1038/s41588-025-02169-3</td>\n",
       "      <td>Nature genetics</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>274</th>\n",
       "      <td>Generation of prion transmission barriers by m...</td>\n",
       "      <td>[Peter Chien, Angela H DePace, Sean R Collins,...</td>\n",
       "      <td>22/08/03</td>\n",
       "      <td>2003</td>\n",
       "      <td>12931190</td>\n",
       "      <td>Self-propagating beta-sheet-rich protein aggre...</td>\n",
       "      <td>10.1038/nature01894</td>\n",
       "      <td>Nature</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>275</th>\n",
       "      <td>Directed evolution of substrate-optimized GroE...</td>\n",
       "      <td>[Jue D Wang, Christophe Herman, Kimberly A Tip...</td>\n",
       "      <td>01/01/03</td>\n",
       "      <td>2003</td>\n",
       "      <td>12507429</td>\n",
       "      <td>GroEL/S chaperonin ring complexes fold many un...</td>\n",
       "      <td>10.1016/s0092-8674(02)01198-4</td>\n",
       "      <td>Cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>276</th>\n",
       "      <td>The FAD- and O(2)-dependent reaction cycle of ...</td>\n",
       "      <td>[Benjamin P Tu, Jonathan S Weissman]</td>\n",
       "      <td>28/11/02</td>\n",
       "      <td>2002</td>\n",
       "      <td>12453408</td>\n",
       "      <td>The endoplasmic reticulum (ER) supports disulf...</td>\n",
       "      <td>10.1016/s1097-2765(02)00696-2</td>\n",
       "      <td>Molecular cell</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>277</th>\n",
       "      <td>Origins and kinetic consequences of diversity ...</td>\n",
       "      <td>[Angela H DePace, Jonathan S Weissman]</td>\n",
       "      <td>09/04/02</td>\n",
       "      <td>2002</td>\n",
       "      <td>11938354</td>\n",
       "      <td>A remarkable feature of prions is that infecti...</td>\n",
       "      <td>10.1038/nsb786</td>\n",
       "      <td>Nature structural biology</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>278</th>\n",
       "      <td>The utility of prions.</td>\n",
       "      <td>[Lev Z Osherovich, Jonathan S Weissman]</td>\n",
       "      <td>08/02/02</td>\n",
       "      <td>2002</td>\n",
       "      <td>11832240</td>\n",
       "      <td>Infectious, self-propagating protein aggregate...</td>\n",
       "      <td>10.1016/s1534-5807(02)00118-1</td>\n",
       "      <td>Developmental cell</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>279 rows × 8 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 title  \\\n",
       "0    High-resolution spatial mapping of cell state ...   \n",
       "1    High-resolution spatial mapping of cell state ...   \n",
       "2    Shaping the composition of the mitochondrial o...   \n",
       "3    Perturb-Multimodal: A platform for pooled gene...   \n",
       "4    Transcriptome-wide analysis of differential ex...   \n",
       "..                                                 ...   \n",
       "274  Generation of prion transmission barriers by m...   \n",
       "275  Directed evolution of substrate-optimized GroE...   \n",
       "276  The FAD- and O(2)-dependent reaction cycle of ...   \n",
       "277  Origins and kinetic consequences of diversity ...   \n",
       "278                             The utility of prions.   \n",
       "\n",
       "                                               authors publication_date  \\\n",
       "0    [Luke W Koblan, Kathryn E Yost, Pu Zheng, Will...         24/07/25   \n",
       "1    [Luke W Koblan, Kathryn E Yost, Pu Zheng, Will...         15/07/25   \n",
       "2           [Gayathri Muthukumar, Jonathan S Weissman]         17/06/25   \n",
       "3    [Reuben A Saunders, William E Allen, Xingjie P...         14/06/25   \n",
       "4    [Ajay Nadig, Joseph M Replogle, Angela N Pogso...         22/04/25   \n",
       "..                                                 ...              ...   \n",
       "274  [Peter Chien, Angela H DePace, Sean R Collins,...         22/08/03   \n",
       "275  [Jue D Wang, Christophe Herman, Kimberly A Tip...         01/01/03   \n",
       "276               [Benjamin P Tu, Jonathan S Weissman]         28/11/02   \n",
       "277             [Angela H DePace, Jonathan S Weissman]         09/04/02   \n",
       "278            [Lev Z Osherovich, Jonathan S Weissman]         08/02/02   \n",
       "\n",
       "    publication_year pubmed_id  \\\n",
       "0               2025  40705858   \n",
       "1               2025  40661400   \n",
       "2               2025  40523936   \n",
       "3               2025  40513557   \n",
       "4               2025  40259084   \n",
       "..               ...       ...   \n",
       "274             2003  12931190   \n",
       "275             2003  12507429   \n",
       "276             2002  12453408   \n",
       "277             2002  11938354   \n",
       "278             2002  11832240   \n",
       "\n",
       "                                              abstract  \\\n",
       "0    Charting the spatiotemporal dynamics of cell f...   \n",
       "1    Charting the spatiotemporal dynamics of cell f...   \n",
       "2    Mitochondria are critical double-membraned org...   \n",
       "3    Metazoan life requires the coordinated activit...   \n",
       "4    Single-cell CRISPR screens such as Perturb-seq...   \n",
       "..                                                 ...   \n",
       "274  Self-propagating beta-sheet-rich protein aggre...   \n",
       "275  GroEL/S chaperonin ring complexes fold many un...   \n",
       "276  The endoplasmic reticulum (ER) supports disulf...   \n",
       "277  A remarkable feature of prions is that infecti...   \n",
       "278  Infectious, self-propagating protein aggregate...   \n",
       "\n",
       "                               doi                    journal  \n",
       "0          10.1126/science.adx3800                    Science  \n",
       "1        10.1101/2025.06.15.659774                    biorxiv  \n",
       "2       10.1038/s41556-025-01683-0        Nature cell biology  \n",
       "3       10.1016/j.cell.2025.05.022                       Cell  \n",
       "4       10.1038/s41588-025-02169-3            Nature genetics  \n",
       "..                             ...                        ...  \n",
       "274            10.1038/nature01894                     Nature  \n",
       "275  10.1016/s0092-8674(02)01198-4                       Cell  \n",
       "276  10.1016/s1097-2765(02)00696-2             Molecular cell  \n",
       "277                 10.1038/nsb786  Nature structural biology  \n",
       "278  10.1016/s1534-5807(02)00118-1         Developmental cell  \n",
       "\n",
       "[279 rows x 8 columns]"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "old_pubs"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a52f3014-15c3-49ec-a164-bf1e8b76bcee",
   "metadata": {
    "tags": []
   },
   "source": [
    "#### New DOIs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "0f8f7fd2-aab4-4d20-968f-3ac819db2eb1",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>publication_year</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>journal</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Author Correction: CRISPR technologies for gen...</td>\n",
       "      <td>[Lukas Villiger, Julia Joung, Luke Koblan, Jon...</td>\n",
       "      <td>14/05/24</td>\n",
       "      <td>2024</td>\n",
       "      <td>38740926</td>\n",
       "      <td>None</td>\n",
       "      <td>10.1038/s41580-024-00745-9</td>\n",
       "      <td>Nature reviews. Molecular cell biology</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                title  \\\n",
       "19  Author Correction: CRISPR technologies for gen...   \n",
       "\n",
       "                                              authors publication_date  \\\n",
       "19  [Lukas Villiger, Julia Joung, Luke Koblan, Jon...         14/05/24   \n",
       "\n",
       "   publication_year pubmed_id abstract                         doi  \\\n",
       "19             2024  38740926     None  10.1038/s41580-024-00745-9   \n",
       "\n",
       "                                   journal  \n",
       "19  Nature reviews. Molecular cell biology  "
      ]
     },
     "execution_count": 9,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pubmed_pubs[ ~pubmed_pubs.doi.isin(old_pubs.doi)]\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "remove dois"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [],
   "source": [
    "remove_pubs = [\"37853259\",\"38740926\"] # author correction\n",
    "pubmed_pubs = pubmed_pubs[ ~pubmed_pubs.pubmed_id.isin(remove_pubs)]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e3f94ecf-b555-4695-b0a5-ecb49fd49192",
   "metadata": {},
   "source": [
    "#### Updates to existing IDs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "e7fb4c41-7bee-4c83-8264-6d62c7fb89cc",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>publication_year</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>journal</th>\n",
       "      <th>update</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>Combinatorial genetics in liver repopulation a...</td>\n",
       "      <td>[Kirk J Wangensteen, Yue J Wang, Zhixun Dou, A...</td>\n",
       "      <td>02/11/17</td>\n",
       "      <td>2017</td>\n",
       "      <td>29091290</td>\n",
       "      <td>Clustered regularly interspaced short palindro...</td>\n",
       "      <td>10.1002/hep.29626</td>\n",
       "      <td>Hepatology (Baltimore, Md.)</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>387</th>\n",
       "      <td>Combinatorial genetics in liver repopulation a...</td>\n",
       "      <td>[Kirk J Wangensteen, Yue J Wang, Zhixun Dou, A...</td>\n",
       "      <td>02/11/17</td>\n",
       "      <td>2017</td>\n",
       "      <td>29091290</td>\n",
       "      <td>Clustered regularly interspaced short palindro...</td>\n",
       "      <td>10.1002/hep.29626</td>\n",
       "      <td>Hepatology</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>416</th>\n",
       "      <td>Targeting the AAA ATPase p97 as an Approach to...</td>\n",
       "      <td>[Daniel J Anderson, Ronan Le Moigne, Stevan Dj...</td>\n",
       "      <td>12/11/15</td>\n",
       "      <td>2015</td>\n",
       "      <td>26555175</td>\n",
       "      <td>p97 is a AAA-ATPase with multiple cellular fun...</td>\n",
       "      <td>10.1016/j.ccell.2015.10.002</td>\n",
       "      <td>Cancer cell</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>Targeting the AAA ATPase p97 as an Approach to...</td>\n",
       "      <td>[Daniel J Anderson, Ronan Le Moigne, Stevan Dj...</td>\n",
       "      <td>12/11/15</td>\n",
       "      <td>2015</td>\n",
       "      <td>26555175</td>\n",
       "      <td>p97 is a AAA-ATPase with multiple cellular fun...</td>\n",
       "      <td>10.1016/j.ccell.2015.10.002</td>\n",
       "      <td>Cancer cell</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>460</th>\n",
       "      <td>Ribosome profiling provides evidence that larg...</td>\n",
       "      <td>[Mitchell Guttman, Pamela Russell, Nicholas T ...</td>\n",
       "      <td>03/07/13</td>\n",
       "      <td>2013</td>\n",
       "      <td>23810193</td>\n",
       "      <td>Large noncoding RNAs are emerging as an import...</td>\n",
       "      <td>10.1016/j.cell.2013.06.009</td>\n",
       "      <td>Cell</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>422</th>\n",
       "      <td>Pharmacological dimerization and activation of...</td>\n",
       "      <td>[Carmela Sidrauski, Jordan C Tsai, Martin Kamp...</td>\n",
       "      <td>16/04/15</td>\n",
       "      <td>2015</td>\n",
       "      <td>25875391</td>\n",
       "      <td>The general translation initiation factor eIF2...</td>\n",
       "      <td>10.7554/eLife.07314</td>\n",
       "      <td>eLife</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>Paradoxical resistance of multiple myeloma to ...</td>\n",
       "      <td>[Diego Acosta-Alvear, Min Y Cho, Thomas Wild, ...</td>\n",
       "      <td>04/09/15</td>\n",
       "      <td>2015</td>\n",
       "      <td>26327694</td>\n",
       "      <td>Hallmarks of cancer, including rapid growth an...</td>\n",
       "      <td>10.7554/eLife.08153</td>\n",
       "      <td>eLife</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>418</th>\n",
       "      <td>Paradoxical resistance of multiple myeloma to ...</td>\n",
       "      <td>[Diego Acosta-Alvear, Min Y Cho, Thomas Wild, ...</td>\n",
       "      <td>04/09/15</td>\n",
       "      <td>2015</td>\n",
       "      <td>26327694</td>\n",
       "      <td>Hallmarks of cancer, including rapid growth an...</td>\n",
       "      <td>10.7554/eLife.08153</td>\n",
       "      <td>eLife</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>The Human Cell Atlas.</td>\n",
       "      <td>[Aviv Regev, Sarah A Teichmann, Eric S Lander,...</td>\n",
       "      <td>06/12/17</td>\n",
       "      <td>2017</td>\n",
       "      <td>29206104</td>\n",
       "      <td>The recent advent of methods for high-throughp...</td>\n",
       "      <td>10.7554/eLife.27041</td>\n",
       "      <td>eLife</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>386</th>\n",
       "      <td>The Human Cell Atlas.</td>\n",
       "      <td>[Aviv Regev, Sarah A Teichmann, Eric S Lander,...</td>\n",
       "      <td>06/12/17</td>\n",
       "      <td>2017</td>\n",
       "      <td>29206104</td>\n",
       "      <td>The recent advent of methods for high-throughp...</td>\n",
       "      <td>10.7554/eLife.27041</td>\n",
       "      <td>eLife</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>110 rows × 9 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                 title  \\\n",
       "109  Combinatorial genetics in liver repopulation a...   \n",
       "387  Combinatorial genetics in liver repopulation a...   \n",
       "416  Targeting the AAA ATPase p97 as an Approach to...   \n",
       "138  Targeting the AAA ATPase p97 as an Approach to...   \n",
       "460  Ribosome profiling provides evidence that larg...   \n",
       "..                                                 ...   \n",
       "422  Pharmacological dimerization and activation of...   \n",
       "140  Paradoxical resistance of multiple myeloma to ...   \n",
       "418  Paradoxical resistance of multiple myeloma to ...   \n",
       "108                              The Human Cell Atlas.   \n",
       "386                              The Human Cell Atlas.   \n",
       "\n",
       "                                               authors publication_date  \\\n",
       "109  [Kirk J Wangensteen, Yue J Wang, Zhixun Dou, A...         02/11/17   \n",
       "387  [Kirk J Wangensteen, Yue J Wang, Zhixun Dou, A...         02/11/17   \n",
       "416  [Daniel J Anderson, Ronan Le Moigne, Stevan Dj...         12/11/15   \n",
       "138  [Daniel J Anderson, Ronan Le Moigne, Stevan Dj...         12/11/15   \n",
       "460  [Mitchell Guttman, Pamela Russell, Nicholas T ...         03/07/13   \n",
       "..                                                 ...              ...   \n",
       "422  [Carmela Sidrauski, Jordan C Tsai, Martin Kamp...         16/04/15   \n",
       "140  [Diego Acosta-Alvear, Min Y Cho, Thomas Wild, ...         04/09/15   \n",
       "418  [Diego Acosta-Alvear, Min Y Cho, Thomas Wild, ...         04/09/15   \n",
       "108  [Aviv Regev, Sarah A Teichmann, Eric S Lander,...         06/12/17   \n",
       "386  [Aviv Regev, Sarah A Teichmann, Eric S Lander,...         06/12/17   \n",
       "\n",
       "    publication_year pubmed_id  \\\n",
       "109             2017  29091290   \n",
       "387             2017  29091290   \n",
       "416             2015  26555175   \n",
       "138             2015  26555175   \n",
       "460             2013  23810193   \n",
       "..               ...       ...   \n",
       "422             2015  25875391   \n",
       "140             2015  26327694   \n",
       "418             2015  26327694   \n",
       "108             2017  29206104   \n",
       "386             2017  29206104   \n",
       "\n",
       "                                              abstract  \\\n",
       "109  Clustered regularly interspaced short palindro...   \n",
       "387  Clustered regularly interspaced short palindro...   \n",
       "416  p97 is a AAA-ATPase with multiple cellular fun...   \n",
       "138  p97 is a AAA-ATPase with multiple cellular fun...   \n",
       "460  Large noncoding RNAs are emerging as an import...   \n",
       "..                                                 ...   \n",
       "422  The general translation initiation factor eIF2...   \n",
       "140  Hallmarks of cancer, including rapid growth an...   \n",
       "418  Hallmarks of cancer, including rapid growth an...   \n",
       "108  The recent advent of methods for high-throughp...   \n",
       "386  The recent advent of methods for high-throughp...   \n",
       "\n",
       "                             doi                      journal    update  \n",
       "109            10.1002/hep.29626  Hepatology (Baltimore, Md.)   current  \n",
       "387            10.1002/hep.29626                   Hepatology  previous  \n",
       "416  10.1016/j.ccell.2015.10.002                  Cancer cell  previous  \n",
       "138  10.1016/j.ccell.2015.10.002                  Cancer cell   current  \n",
       "460   10.1016/j.cell.2013.06.009                         Cell  previous  \n",
       "..                           ...                          ...       ...  \n",
       "422          10.7554/eLife.07314                        eLife  previous  \n",
       "140          10.7554/eLife.08153                        eLife   current  \n",
       "418          10.7554/eLife.08153                        eLife  previous  \n",
       "108          10.7554/eLife.27041                        eLife   current  \n",
       "386          10.7554/eLife.27041                        eLife  previous  \n",
       "\n",
       "[110 rows x 9 columns]"
      ]
     },
     "execution_count": 11,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pubmed_pubs['update'] = 'current'\n",
    "old_pubs['update'] = 'previous'\n",
    "\n",
    "df = pd.concat([pubmed_pubs[pubmed_pubs.doi.isin(old_pubs.doi)],old_pubs[old_pubs.doi.isin(pubmed_pubs.doi)]]).reset_index(drop=True)\n",
    "df.loc[df.astype(str).drop_duplicates(subset=['title', 'authors', 'publication_date', \n",
    "                                             'publication_year', 'pubmed_id', 'abstract', 'doi', 'journal'], keep=False).index].sort_values(by='doi')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a0d06e66-374b-483d-8e45-7c4144c47864",
   "metadata": {},
   "source": [
    "#### Removal of IDs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "cf90bde3-f373-4b45-b3f8-103162792023",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>title</th>\n",
       "      <th>authors</th>\n",
       "      <th>publication_date</th>\n",
       "      <th>publication_year</th>\n",
       "      <th>pubmed_id</th>\n",
       "      <th>abstract</th>\n",
       "      <th>doi</th>\n",
       "      <th>journal</th>\n",
       "      <th>update</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Author Correction: Efficient C•G-to-G•C base e...</td>\n",
       "      <td>[Luke W Koblan, Mandana Arbab, Max W Shen, Jef...</td>\n",
       "      <td>19/10/23</td>\n",
       "      <td>2023</td>\n",
       "      <td>37853259</td>\n",
       "      <td>None</td>\n",
       "      <td>10.1038/s41587-023-02028-8</td>\n",
       "      <td>Nature biotechnology</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                                                title  \\\n",
       "24  Author Correction: Efficient C•G-to-G•C base e...   \n",
       "\n",
       "                                              authors publication_date  \\\n",
       "24  [Luke W Koblan, Mandana Arbab, Max W Shen, Jef...         19/10/23   \n",
       "\n",
       "   publication_year pubmed_id abstract                         doi  \\\n",
       "24             2023  37853259     None  10.1038/s41587-023-02028-8   \n",
       "\n",
       "                 journal    update  \n",
       "24  Nature biotechnology  previous  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "old_pubs[ ~old_pubs.doi.isin(pubmed_pubs.doi)]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ebcec338-fbdc-4dd6-9358-b17a84d4d034",
   "metadata": {},
   "source": [
    "#### For now: "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "id": "3dbcd339-0e1d-41c8-8916-998b0372b325",
   "metadata": {},
   "outputs": [],
   "source": [
    "pubmed_pubs = pd.concat([pubmed_pubs[ ~pubmed_pubs.doi.isin(old_pubs.doi)], old_pubs])\n",
    "\n",
    "pubmed_pubs = pubmed_pubs.drop('update', axis=1)\n",
    "pubmed_pubs = pubmed_pubs.sort_values([\"publication_year\",\"pubmed_id\"],ascending=False).reset_index(drop=True)\n",
    "pubs_records = [{k:v for k,v in m.items() if str(v) != 'nan'} for m in pubmed_pubs.to_dict(orient='records')]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'title': 'High-resolution spatial mapping of cell state and lineage dynamics in vivo with PEtracer.',\n",
       "  'authors': ['Luke W Koblan',\n",
       "   'Kathryn E Yost',\n",
       "   'Pu Zheng',\n",
       "   'William N Colgan',\n",
       "   'Matthew G Jones',\n",
       "   'Dian Yang',\n",
       "   'Arhan Kumar',\n",
       "   'Jaspreet Sandhu',\n",
       "   'Alexandra Schnell',\n",
       "   'Dawei Sun',\n",
       "   'Can Ergen',\n",
       "   'Reuben A Saunders',\n",
       "   'Xiaowei Zhuang',\n",
       "   'William E Allen',\n",
       "   'Nir Yosef',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '24/07/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40705858',\n",
       "  'abstract': 'Charting the spatiotemporal dynamics of cell fate determination in development and disease is a long-standing objective in biology. Here we present the design, development, and extensive validation of PEtracer, a prime editing-based, evolving lineage tracing technology compatible with both single-cell sequencing and multimodal imaging methodologies to jointly profile cell state and lineage in dissociated cells or while preserving cellular context in tissues with high spatial resolution. Using PEtracer coupled with MERFISH spatial transcriptomic profiling in a syngeneic mouse model of tumor metastasis, we reconstruct the growth of individually-seeded tumors in vivo and uncover distinct modules of cell-intrinsic and cell-extrinsic factors that coordinate tumor growth. More generally, PEtracer enables systematic characterization of cell state and lineage relationships in intact tissues over biologically-relevant temporal and spatial scales.',\n",
       "  'doi': '10.1126/science.adx3800',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'High-resolution spatial mapping of cell state and lineage dynamics ',\n",
       "  'authors': ['Luke W Koblan',\n",
       "   'Kathryn E Yost',\n",
       "   'Pu Zheng',\n",
       "   'William N Colgan',\n",
       "   'Matthew G Jones',\n",
       "   'Dian Yang',\n",
       "   'Arhan Kumar',\n",
       "   'Jaspreet Sandhu',\n",
       "   'Alexandra Schnell',\n",
       "   'Dawei Sun',\n",
       "   'Can Ergen',\n",
       "   'Reuben A Saunders',\n",
       "   'Xiaowei Zhuang',\n",
       "   'William E Allen',\n",
       "   'Nir Yosef',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '15/07/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40661400',\n",
       "  'abstract': 'Charting the spatiotemporal dynamics of cell fate determination in development and disease is a long-standing objective in biology. Here we present the design, development, and extensive validation of PEtracer, a prime editing-based, evolving lineage tracing technology compatible with both single-cell sequencing and multimodal imaging methodologies to jointly profile cell state and lineage in dissociated cells or while preserving cellular context in tissues with high spatial resolution. Using PEtracer coupled with MERFISH spatial transcriptomic profiling in a syngeneic mouse model of tumor metastasis, we reconstruct the growth of individually-seeded tumors ',\n",
       "  'doi': '10.1101/2025.06.15.659774',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Shaping the composition of the mitochondrial outer membrane.',\n",
       "  'authors': ['Gayathri Muthukumar', 'Jonathan S Weissman'],\n",
       "  'publication_date': '17/06/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40523936',\n",
       "  'abstract': 'Mitochondria are critical double-membraned organelles that act as biosynthetic and bioenergetic cellular factories, with the outer membrane providing an interface with the rest of the cell. Mitochondrial outer membrane proteins regulate a variety of processes, including metabolism, innate immunity and apoptosis. Although the biophysical and functional diversity of these proteins is highly documented, the mechanisms of their biogenesis and the integration of that into cellular homeostasis are just starting to take shape. Here, focusing on α-helical outer membrane proteins, we review recent insights into the mechanisms of synthesis and cytosolic chaperoning, insertion and assembly in the lipid bilayer, and quality control of unassembled or mislocalized transmembrane domains. We further discuss the role convergent evolution played in this process, comparing key biogenesis players from lower eukaryotes, including yeast and trypanosomes, with multicellular metazoan systems, and draw comparisons with the endoplasmic reticulum biogenesis system, in which membrane proteins face similar challenges.',\n",
       "  'doi': '10.1038/s41556-025-01683-0',\n",
       "  'journal': 'Nature cell biology'},\n",
       " {'title': 'Perturb-Multimodal: A platform for pooled genetic screens with imaging and sequencing in intact mammalian tissue.',\n",
       "  'authors': ['Reuben A Saunders',\n",
       "   'William E Allen',\n",
       "   'Xingjie Pan',\n",
       "   'Jaspreet Sandhu',\n",
       "   'Jiaqi Lu',\n",
       "   'Thomas K Lau',\n",
       "   'Karina Smolyar',\n",
       "   'Zuri A Sullivan',\n",
       "   'Catherine Dulac',\n",
       "   'Jonathan S Weissman',\n",
       "   'Xiaowei Zhuang'],\n",
       "  'publication_date': '14/06/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40513557',\n",
       "  'abstract': 'Metazoan life requires the coordinated activities of thousands of genes in spatially organized cell types. Understanding the basis of tissue function requires approaches to dissect the genetic control of diverse cellular and tissue phenotypes in vivo. Here, we present Perturb-Multimodal (Perturb-Multi), a paired imaging and sequencing method to construct large-scale, multimodal genotype-phenotype maps in tissues with pooled genetic perturbations. Using imaging, we identify perturbations in individual cells while simultaneously measuring their gene expression profiles and subcellular morphology. Using single-cell sequencing, we measure full transcriptomic responses to the same perturbations. We apply Perturb-Multi to study hundreds of genetic perturbations in the mouse liver. Our data suggest the genetic regulators and mechanisms underlying the dynamic control of hepatocyte zonation, the unfolded protein response, and steatosis. Perturb-Multi accelerates discoveries of the genetic basis of complex cell and tissue physiology and provides critical training data for emerging machine learning models of cellular function.',\n",
       "  'doi': '10.1016/j.cell.2025.05.022',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Transcriptome-wide analysis of differential expression in perturbation atlases.',\n",
       "  'authors': ['Ajay Nadig',\n",
       "   'Joseph M Replogle',\n",
       "   'Angela N Pogson',\n",
       "   'Mukundh Murthy',\n",
       "   'Steven A McCarroll',\n",
       "   'Jonathan S Weissman',\n",
       "   'Elise B Robinson',\n",
       "   \"Luke J O'Connor\"],\n",
       "  'publication_date': '22/04/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40259084',\n",
       "  'abstract': \"Single-cell CRISPR screens such as Perturb-seq enable transcriptomic profiling of genetic perturbations at scale. However, the data produced by these screens are noisy, and many effects may go undetected. Here we introduce transcriptome-wide analysis of differential expression (TRADE)-a statistical model for the distribution of true differential expression effects that accounts for estimation error appropriately. TRADE estimates the 'transcriptome-wide impact', which quantifies the total effect of a perturbation across the transcriptome. Analyzing several large Perturb-seq datasets, we show that many transcriptional effects remain undetected in standard analyses but emerge in aggregate using TRADE. A typical gene perturbation affects an estimated 45 genes, whereas a typical essential gene affects over 500. We find moderate consistency of perturbation effects across cell types, identify perturbations where transcriptional responses vary qualitatively across dosage levels and clarify the relationship between genetic and transcriptomic correlations across neuropsychiatric disorders.\",\n",
       "  'doi': '10.1038/s41588-025-02169-3',\n",
       "  'journal': 'Nature genetics'},\n",
       " {'title': 'Spatiotemporal modeling of molecular holograms.',\n",
       "  'authors': ['Xiaojie Qiu',\n",
       "   'Daniel Y Zhu',\n",
       "   'Yifan Lu',\n",
       "   'Jiajun Yao',\n",
       "   'Zehua Jing',\n",
       "   'Kyung Hoi Min',\n",
       "   'Mengnan Cheng',\n",
       "   'Hailin Pan',\n",
       "   'Lulu Zuo',\n",
       "   'Samuel King',\n",
       "   'Qi Fang',\n",
       "   'Huiwen Zheng',\n",
       "   'Mingyue Wang',\n",
       "   'Shuai Wang',\n",
       "   'Qingquan Zhang',\n",
       "   'Sichao Yu',\n",
       "   'Sha Liao',\n",
       "   'Chao Liu',\n",
       "   'Xinchao Wu',\n",
       "   'Yiwei Lai',\n",
       "   'Shijie Hao',\n",
       "   'Zhewei Zhang',\n",
       "   'Liang Wu',\n",
       "   'Yong Zhang',\n",
       "   'Mei Li',\n",
       "   'Zhencheng Tu',\n",
       "   'Jinpei Lin',\n",
       "   'Zhuoxuan Yang',\n",
       "   'Yuxiang Li',\n",
       "   'Ying Gu',\n",
       "   'David Ellison',\n",
       "   'Yuancheng Ryan Lu',\n",
       "   'Qinan Hu',\n",
       "   'Yuhui Hu',\n",
       "   'Ao Chen',\n",
       "   'Longqi Liu',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jiayi Ma',\n",
       "   'Xun Xu',\n",
       "   'Shiping Liu',\n",
       "   'Yinqi Bai'],\n",
       "  'publication_date': '03/03/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '40024251',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1016/j.cell.2025.02.026',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Multiplexed mosaic tumor models reveal natural phenotypic variations in drug response within and between populations.',\n",
       "  'authors': ['Johnny X Yu',\n",
       "   'Jung Min Suh',\n",
       "   'Katerina D Popova',\n",
       "   'Kristle Garcia',\n",
       "   'Tanvi Joshi',\n",
       "   'Bruce Culbertson',\n",
       "   'Jessica B Spinelli',\n",
       "   'Vishvak Subramanyam',\n",
       "   'Kevin Lou',\n",
       "   'Kevan M Shokat',\n",
       "   'Jonathan Weissman',\n",
       "   'Hani Goodarzi'],\n",
       "  'publication_date': '07/01/25',\n",
       "  'publication_year': '2025',\n",
       "  'pubmed_id': '39763922',\n",
       "  'abstract': 'Many agents that show promise in preclinical cancer models lack efficacy in patients due to patient heterogeneity that is not captured in traditional assays. To address this problem, we have developed GENEVA, a platform that measures the molecular and phenotypic consequences of drug perturbations within diverse populations of cancer cells at single-cell resolution, both ',\n",
       "  'doi': '10.1101/2024.12.13.628239',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'CD47 predominates over CD24 as a macrophage immune checkpoint in cancer.',\n",
       "  'authors': ['Juliet Allen',\n",
       "   'Anna Meglan',\n",
       "   'Kyle Vaccaro',\n",
       "   'José Velarde',\n",
       "   'Victor Chen',\n",
       "   'Juliano Ribeiro',\n",
       "   'Jasmine Blandin',\n",
       "   'Sumeet Gupta',\n",
       "   'Ranjan Mishra',\n",
       "   'Raymond Ho',\n",
       "   'Jennifer Love',\n",
       "   'Ferenc Reinhardt',\n",
       "   'George W Bell',\n",
       "   'Jin Chen',\n",
       "   'Robert Weinberg',\n",
       "   'Dian Yang',\n",
       "   'Jonathan Weissman',\n",
       "   'Kipp Weiskopf'],\n",
       "  'publication_date': '09/12/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39651307',\n",
       "  'abstract': 'Macrophages hold tremendous promise as effectors of cancer immunotherapy, but the best strategies to provoke these cells to attack tumors remain unknown. Here, we evaluated the therapeutic potential of targeting two distinct macrophage immune checkpoints: CD47 and CD24. We found that antibodies targeting these antigens could elicit maximal levels of phagocytosis when combined together in vitro. However, to our surprise, via unbiased genome-wide CRISPR screens, we found that CD24 primarily acts as a target of opsonization rather than an immune checkpoint. In a series of in vitro and in vivo genetic validation studies, we found that CD24 was neither necessary nor sufficient to protect cancer cells from macrophage phagocytosis in most mouse and human tumor models. Instead, anti-CD24 antibodies exhibit robust Fc-dependent activity, and as a consequence, they cause significant on-target hematologic toxicity in mice. To overcome these challenges and leverage our findings for therapeutic purposes, we engineered a collection of 77 novel bispecific antibodies that bind to a tumor antigen with one arm and engage macrophages with the second arm. We discovered multiple novel bispecifics that maximally activate macrophage-mediated cytotoxicity and reduce binding to healthy blood cells, including bispecifics targeting macrophage immune checkpoint molecules in combination with EGFR, TROP2, and CD71. Overall, our findings indicate that CD47 predominates over CD24 as a macrophage immune checkpoint in cancer, and that the novel bispecifics we created may be optimal immunotherapies to direct myeloid cells to eradicate solid tumors.',\n",
       "  'doi': '10.1101/2024.11.25.625185',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'A platform for multimodal ',\n",
       "  'authors': ['Reuben A Saunders',\n",
       "   'William E Allen',\n",
       "   'Xingjie Pan',\n",
       "   'Jaspreet Sandhu',\n",
       "   'Jiaqi Lu',\n",
       "   'Thomas K Lau',\n",
       "   'Karina Smolyar',\n",
       "   'Zuri A Sullivan',\n",
       "   'Catherine Dulac',\n",
       "   'Jonathan S Weissman',\n",
       "   'Xiaowei Zhuang'],\n",
       "  'publication_date': '28/11/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39605605',\n",
       "  'abstract': 'Organ function requires coordinated activities of thousands of genes in distinct, spatially organized cell types. Understanding the basis of emergent tissue function requires approaches to dissect the genetic control of diverse cellular and tissue phenotypes ',\n",
       "  'doi': '10.1101/2024.11.18.624217',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Combinatorial effector targeting (COMET) for transcriptional modulation and locus-specific biochemistry.',\n",
       "  'authors': ['Caroline M Wilson',\n",
       "   'Greg C Pommier',\n",
       "   'Daniel D Richman',\n",
       "   'Nicholas Sambold',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Jonathan S Weissman',\n",
       "   'Luke A Gilbert'],\n",
       "  'publication_date': '18/11/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39554033',\n",
       "  'abstract': 'Understanding how human gene expression is coordinately regulated by functional units of proteins across the genome remains a major biological goal. Here, we present COMET, a high-throughput screening platform for com binatorial e ffector targeting for the identification of transcriptional modulators. We generate libraries of combinatorial dCas9-based fusion proteins, containing two to six effector domains, allowing us to systematically investigate more than 110,000 combinations of effector proteins at endogenous human loci for their influence on transcription. Importantly, we keep full proteins or domains intact, maintaining catalytic cores and surfaces for protein-protein interactions. We observe more than 5800 significant hits that modulate transcription, we demonstrate cell type specific transcriptional modulation, and we further investigate epistatic relationships between our effector combinations. We validate unexpected combinations as synergistic or buffering, emphasizing COMET as both a method for transcriptional effector discovery, and as a functional genomics tool for identifying novel domain interactions and directing locus-specific biochemistry.',\n",
       "  'doi': '10.1101/2024.10.28.620517',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Spatiotemporal modeling of molecular holograms.',\n",
       "  'authors': ['Xiaojie Qiu',\n",
       "   'Daniel Y Zhu',\n",
       "   'Yifan Lu',\n",
       "   'Jiajun Yao',\n",
       "   'Zehua Jing',\n",
       "   'Kyung Hoi Min',\n",
       "   'Mengnan Cheng',\n",
       "   'Hailin Pan',\n",
       "   'Lulu Zuo',\n",
       "   'Samuel King',\n",
       "   'Qi Fang',\n",
       "   'Huiwen Zheng',\n",
       "   'Mingyue Wang',\n",
       "   'Shuai Wang',\n",
       "   'Qingquan Zhang',\n",
       "   'Sichao Yu',\n",
       "   'Sha Liao',\n",
       "   'Chao Liu',\n",
       "   'Xinchao Wu',\n",
       "   'Yiwei Lai',\n",
       "   'Shijie Hao',\n",
       "   'Zhewei Zhang',\n",
       "   'Liang Wu',\n",
       "   'Yong Zhang',\n",
       "   'Mei Li',\n",
       "   'Zhencheng Tu',\n",
       "   'Jinpei Lin',\n",
       "   'Zhuoxuan Yang',\n",
       "   'Yuxiang Li',\n",
       "   'Ying Gu',\n",
       "   'David Ellison',\n",
       "   'Ao Chen',\n",
       "   'Longqi Liu',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jiayi Ma',\n",
       "   'Xun Xu',\n",
       "   'Shiping Liu',\n",
       "   'Yinqi Bai'],\n",
       "  'publication_date': '13/11/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39532097',\n",
       "  'abstract': 'Quantifying spatiotemporal dynamics during embryogenesis is crucial for understanding congenital diseases. We developed Spateo (https://github.com/aristoteleo/spateo-release), a 3D spatiotemporal modeling framework, and applied it to a 3D mouse embryogenesis atlas at E9.5 and E11.5, capturing eight million cells. Spateo enables scalable, partial, non-rigid alignment, multi-slice refinement, and mesh correction to create molecular holograms of whole embryos. It introduces digitization methods to uncover multi-level biology from subcellular to whole organ, identifying expression gradients along orthogonal axes of emergent 3D structures, e.g., secondary organizers such as midbrain-hindbrain boundary (MHB). Spateo further jointly models intercellular and intracellular interaction to dissect signaling landscapes in 3D structures, including the zona limitans intrathalamica (ZLI). Lastly, Spateo introduces \"morphometric vector fields\" of cell migration and integrates spatial differential geometry to unveil molecular programs underlying asymmetrical murine heart organogenesis and others, bridging macroscopic changes with molecular dynamics. Thus, Spateo enables the study of organ ecology at a molecular level in 3D space over time.',\n",
       "  'doi': '10.1016/j.cell.2024.10.011',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Spatiotemporal lineage tracing reveals the dynamic spatial architecture of tumor growth and metastasis.',\n",
       "  'authors': ['Matthew G Jones',\n",
       "   'Dawei Sun',\n",
       "   'Kyung Hoi Joseph Min',\n",
       "   'William N Colgan',\n",
       "   'Luyi Tian',\n",
       "   'Jackson A Weir',\n",
       "   'Victor Z Chen',\n",
       "   'Luke W Koblan',\n",
       "   'Kathryn E Yost',\n",
       "   'Nicolas Mathey-Andrews',\n",
       "   'Andrew J C Russell',\n",
       "   'Robert R Stickels',\n",
       "   'Karol S Balderrama',\n",
       "   'William M Rideout',\n",
       "   'Howard Y Chang',\n",
       "   'Tyler Jacks',\n",
       "   'Fei Chen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nir Yosef',\n",
       "   'Dian Yang'],\n",
       "  'publication_date': '04/11/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39484491',\n",
       "  'abstract': 'Tumor progression is driven by dynamic interactions between cancer cells and their surrounding microenvironment. Investigating the spatiotemporal evolution of tumors can provide crucial insights into how intrinsic changes within cancer cells and extrinsic alterations in the microenvironment cooperate to drive different stages of tumor progression. Here, we integrate high-resolution spatial transcriptomics and evolving lineage tracing technologies to elucidate how tumor expansion, plasticity, and metastasis co-evolve with microenvironmental remodeling in a ',\n",
       "  'doi': '10.1101/2024.10.21.619529',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'High-quality peptide evidence for annotating non-canonical open reading frames as human proteins.',\n",
       "  'authors': ['Eric W Deutsch',\n",
       "   'Leron W Kok',\n",
       "   'Jonathan M Mudge',\n",
       "   'Jorge Ruiz-Orera',\n",
       "   'Ivo Fierro-Monti',\n",
       "   'Zhi Sun',\n",
       "   'Jennifer G Abelin',\n",
       "   'M Mar Alba',\n",
       "   'Julie L Aspden',\n",
       "   'Ariel A Bazzini',\n",
       "   'Elspeth A Bruford',\n",
       "   'Marie A Brunet',\n",
       "   'Lorenzo Calviello',\n",
       "   'Steven A Carr',\n",
       "   'Anne-Ruxandra Carvunis',\n",
       "   'Sonia Chothani',\n",
       "   'Jim Clauwaert',\n",
       "   'Kellie Dean',\n",
       "   'Pouya Faridi',\n",
       "   'Adam Frankish',\n",
       "   'Norbert Hubner',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Michele Magrane',\n",
       "   'Maria Jesus Martin',\n",
       "   'Thomas F Martinez',\n",
       "   'Gerben Menschaert',\n",
       "   'Uwe Ohler',\n",
       "   'Sandra Orchard',\n",
       "   'Owen Rackham',\n",
       "   'Xavier Roucou',\n",
       "   'Sarah A Slavoff',\n",
       "   'Eivind Valen',\n",
       "   'Aaron Wacholder',\n",
       "   'Jonathan S Weissman',\n",
       "   'Wei Wu',\n",
       "   'Zhi Xie',\n",
       "   'Jyoti Choudhary',\n",
       "   'Michal Bassani-Sternberg',\n",
       "   'Juan Antonio Vizcaíno',\n",
       "   'Nicola Ternette',\n",
       "   'Robert L Moritz',\n",
       "   'John R Prensner',\n",
       "   'Sebastiaan van Heesch'],\n",
       "  'publication_date': '24/09/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39314370',\n",
       "  'abstract': 'A major scientific drive is to characterize the protein-coding genome as it provides the primary basis for the study of human health. But the fundamental question remains: what has been missed in prior genomic analyses? Over the past decade, the translation of non-canonical open reading frames (ncORFs) has been observed across human cell types and disease states, with major implications for proteomics, genomics, and clinical science. However, the impact of ncORFs has been limited by the absence of a large-scale understanding of their contribution to the human proteome. Here, we report the collaborative efforts of stakeholders in proteomics, immunopeptidomics, Ribo-seq ORF discovery, and gene annotation, to produce a consensus landscape of protein-level evidence for ncORFs. We show that at least 25% of a set of 7,264 ncORFs give rise to translated gene products, yielding over 3,000 peptides in a pan-proteome analysis encompassing 3.8 billion mass spectra from 95,520 experiments. With these data, we developed an annotation framework for ncORFs and created public tools for researchers through GENCODE and PeptideAtlas. This work will provide a platform to advance ncORF-derived proteins in biomedical discovery and, beyond humans, diverse animals and plants where ncORFs are similarly observed.',\n",
       "  'doi': '10.1101/2024.09.09.612016',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Robustness and reliability of single-cell regulatory multi-omics with deep mitochondrial mutation profiling.',\n",
       "  'authors': ['Chen Weng', 'Jonathan S Weissman', 'Vijay G Sankaran'],\n",
       "  'publication_date': '04/09/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39229039',\n",
       "  'abstract': 'The detection of mitochondrial DNA (mtDNA) mutations in single cells holds considerable potential to define clonal relationships coupled with information on cell state in humans. Previous methods focused on higher heteroplasmy mutations that are limited in number and can be influenced by functional selection, introducing biases for lineage tracing. Although more challenging to detect, intermediate to low heteroplasmy mtDNA mutations are valuable due to their high diversity, abundance, and lower propensity to selection. To enhance mtDNA mutation detection and facilitate fine-scale lineage tracing, we developed the single-cell Regulatory multi-omics with Deep Mitochondrial mutation profiling (ReDeeM) approach, an integrated experimental and computational framework. Recently, some concerns have been raised about the analytical workflow in the ReDeeM framework. Specifically, it was noted that the mutations detected in a single molecule per cell are enriched on edges of mtDNA molecules, suggesting they resemble artifacts reported in other sequencing approaches. It was then proposed that all mutations found in one molecule per cell should be removed. We detail our error correction method, demonstrating that the observed edge mutations are distinct from previously reported sequencing artifacts. We further show that the proposed removal leads to massive elimination of bona fide and informative mutations. Indeed, mutations accumulating on edges impact a minority of all mutation calls (for example, in hematopoietic stem cells, the excess mutations on the edge account for only 4.3%-7.6% of the total). Recognizing the value of addressing edge mutations even after applying consensus correction, we provide an additional filtering option in the ReDeeM-R package. This approach effectively eliminates the position biases, leads to a mutational signature indistinguishable from bona fide mitochondrial mutations, and removes excess low molecule high connectedness mutations. Importantly, this option preserves the large majority of unique mutations identified by ReDeeM, maintaining the ability of ReDeeM to provide a more than 10-fold increase in variant detection compared to previous methods. Additionally, the cells remain well-connected. While there is room for further refinement in mutation calling strategies, the significant advances and biological insights provided by the ReDeeM framework are unique and remain intact. We hope that this detailed discussion and analysis enables the community to employ this approach and contribute to its further development.',\n",
       "  'doi': '10.1101/2024.08.23.609473',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Multi-species genome-wide CRISPR screens identify GPX4 as a conserved suppressor of cold-induced cell death.',\n",
       "  'authors': ['Breanna Lam',\n",
       "   'Kathrin M Kajderowicz',\n",
       "   'Heather R Keys',\n",
       "   'Julian M Roessler',\n",
       "   'Evgeni M Frenkel',\n",
       "   'Adina Kirkland',\n",
       "   'Punam Bisht',\n",
       "   'Mohamed A El-Brolosy',\n",
       "   'Rudolf Jaenisch',\n",
       "   'George W Bell',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eric C Griffith',\n",
       "   'Sinisa Hrvatin'],\n",
       "  'publication_date': '02/08/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39091747',\n",
       "  'abstract': 'Cells must adapt to environmental changes to maintain homeostasis. One of the most striking environmental adaptations is entry into hibernation during which core body temperature can decrease from 37°C to as low at 4°C. How mammalian cells, which evolved to optimally function within a narrow range of temperatures, adapt to this profound decrease in temperature remains poorly understood. In this study, we conducted the first genome-scale CRISPR-Cas9 screen in cells derived from Syrian hamster, a facultative hibernator, as well as human cells to investigate the genetic basis of cold tolerance in a hibernator and a non-hibernator in an unbiased manner. Both screens independently revealed glutathione peroxidase 4 (GPX4), a selenium-containing enzyme, and associated proteins as critical for cold tolerance. We utilized genetic and pharmacological approaches to demonstrate that GPX4 is active in the cold and its catalytic activity is required for cold tolerance. Furthermore, we show that the role of GPX4 as a suppressor of cold-induced cell death extends across hibernating species, including 13-lined ground squirrels and greater horseshoe bats, highlighting the evolutionary conservation of this mechanism of cold tolerance. This study identifies GPX4 as a central modulator of mammalian cold tolerance and advances our understanding of the evolved mechanisms by which cells mitigate cold-associated damage-one of the most common challenges faced by cells and organisms in nature.',\n",
       "  'doi': '10.1101/2024.07.25.605098',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Transcriptome-wide characterization of genetic perturbations.',\n",
       "  'authors': ['Ajay Nadig',\n",
       "   'Joseph M Replogle',\n",
       "   'Angela N Pogson',\n",
       "   'Steven A McCarroll',\n",
       "   'Jonathan S Weissman',\n",
       "   'Elise B Robinson',\n",
       "   \"Luke J O'Connor\"],\n",
       "  'publication_date': '15/07/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '39005298',\n",
       "  'abstract': 'Single cell CRISPR screens such as Perturb-seq enable transcriptomic profiling of genetic perturbations at scale. However, the data produced by these screens are often noisy due to cost and technical constraints, limiting power to detect true effects with conventional differential expression analyses. Here, we introduce TRanscriptome-wide Analysis of Differential Expression (TRADE), a statistical framework which estimates the transcriptome-wide distribution of true differential expression effects from noisy gene-level measurements. Within TRADE, we derive multiple novel, interpretable statistical metrics, including the \"transcriptome-wide impact\", an estimator of the overall transcriptional effect of a perturbation which is stable across sampling depths. We analyze new and published large-scale Perturb-seq datasets to show that many true transcriptional effects are not statistically significant, but detectable in aggregate with TRADE. In a genome-scale Perturb-seq screen, we find that a typical gene perturbation affects an estimated 45 genes, whereas a typical essential gene perturbation affects over 500 genes. An advantage of our approach is its ability to compare the transcriptomic effects of genetic perturbations across contexts and dosages despite differences in power. We use this ability to identify perturbations with cell-type dependent effects and to find examples of perturbations where transcriptional responses are not only larger in magnitude, but also qualitatively different, as a function of dosage. Lastly, we expand our analysis to case/control comparison of gene expression for neuropsychiatric conditions, finding that transcriptomic effect correlations are greater than genetic correlations for these diagnoses. TRADE lays an analytic foundation for the systematic comparison of genetic perturbation atlases, as well as differential expression experiments more broadly.',\n",
       "  'doi': '10.1101/2024.07.03.601903',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Brainwide silencing of prion protein by AAV-mediated delivery of an engineered compact epigenetic editor.',\n",
       "  'authors': ['Edwin N Neumann',\n",
       "   'Tessa M Bertozzi',\n",
       "   'Elaine Wu',\n",
       "   'Fiona Serack',\n",
       "   'John W Harvey',\n",
       "   'Pamela P Brauer',\n",
       "   'Catherine P Pirtle',\n",
       "   'Alissa Coffey',\n",
       "   'Michael Howard',\n",
       "   'Nikita Kamath',\n",
       "   'Kenney Lenz',\n",
       "   'Kenia Guzman',\n",
       "   'Michael H Raymond',\n",
       "   'Ahmad S Khalil',\n",
       "   'Benjamin E Deverman',\n",
       "   'Eric Vallabh Minikel',\n",
       "   'Sonia M Vallabh',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '27/06/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38935715',\n",
       "  'abstract': 'Prion disease is caused by misfolding of the prion protein (PrP) into pathogenic self-propagating conformations, leading to rapid-onset dementia and death. However, elimination of endogenous PrP halts prion disease progression. In this study, we describe Coupled Histone tail for Autoinhibition Release of Methyltransferase (CHARM), a compact, enzyme-free epigenetic editor capable of silencing transcription through programmable DNA methylation. Using a histone H3 tail-Dnmt3l fusion, CHARM recruits and activates endogenous DNA methyltransferases, thereby reducing transgene size and cytotoxicity. When delivered to the mouse brain by systemic injection of adeno-associated virus (AAV), ',\n",
       "  'doi': '10.1126/science.ado7082',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Nucleotide depletion promotes cell fate transitions by inducing DNA replication stress.',\n",
       "  'authors': ['Brian T Do',\n",
       "   'Peggy P Hsu',\n",
       "   'Sidney Y Vermeulen',\n",
       "   'Zhishan Wang',\n",
       "   'Taghreed Hirz',\n",
       "   'Keene L Abbott',\n",
       "   'Najihah Aziz',\n",
       "   'Joseph M Replogle',\n",
       "   'Stefan Bjelosevic',\n",
       "   'Jonathan Paolino',\n",
       "   'Samantha A Nelson',\n",
       "   'Samuel Block',\n",
       "   'Alicia M Darnell',\n",
       "   'Raphael Ferreira',\n",
       "   'Hanyu Zhang',\n",
       "   'Jelena Milosevic',\n",
       "   'Daniel R Schmidt',\n",
       "   'Christopher Chidley',\n",
       "   'Isaac S Harris',\n",
       "   'Jonathan S Weissman',\n",
       "   'Yana Pikman',\n",
       "   'Kimberly Stegmaier',\n",
       "   'Sihem Cheloufi',\n",
       "   'Xiaofeng A Su',\n",
       "   'David B Sykes',\n",
       "   'Matthew G Vander Heiden'],\n",
       "  'publication_date': '02/06/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38823395',\n",
       "  'abstract': 'Control of cellular identity requires coordination of developmental programs with environmental factors such as nutrient availability, suggesting that perturbing metabolism can alter cell state. Here, we find that nucleotide depletion and DNA replication stress drive differentiation in human and murine normal and transformed hematopoietic systems, including patient-derived acute myeloid leukemia (AML) xenografts. These cell state transitions begin during S phase and are independent of ATR/ATM checkpoint signaling, double-stranded DNA break formation, and changes in cell cycle length. In systems where differentiation is blocked by oncogenic transcription factor expression, replication stress activates primed regulatory loci and induces lineage-appropriate maturation genes despite the persistence of progenitor programs. Altering the baseline cell state by manipulating transcription factor expression causes replication stress to induce genes specific for alternative lineages. The ability of replication stress to selectively activate primed maturation programs across different contexts suggests a general mechanism by which changes in metabolism can promote lineage-appropriate cell state transitions.',\n",
       "  'doi': '10.1016/j.devcel.2024.05.010',\n",
       "  'journal': 'Developmental cell'},\n",
       " {'title': 'CRISPRi screens identify the lncRNA, LOUP, as a multifunctional locus regulating macrophage differentiation and inflammatory signaling',\n",
       "  'authors': ['Haley Halasz',\n",
       "   'Eric Malekos',\n",
       "   'Sergio Covarrubias',\n",
       "   'Samira Yitiz',\n",
       "   'Christy Montano',\n",
       "   'Lisa Sudek',\n",
       "   'Sol Katzman',\n",
       "   'S John Liu',\n",
       "   'Max A Horlbeck',\n",
       "   'Leila Namvar',\n",
       "   'Jonathan S Weissman',\n",
       "   'Susan Carpenter'],\n",
       "  'publication_date': '23/05/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38781216',\n",
       "  'abstract': 'Long noncoding RNAs (lncRNAs) account for the largest portion of RNA from the transcriptome, yet most of their functions remain unknown. Here, we performed two independent high-throughput CRISPRi screens to understand the role of lncRNAs in monocyte function and differentiation. The first was a reporter-based screen to identify lncRNAs that regulate TLR4-NFkB signaling in human monocytes and the second screen identified lncRNAs involved in monocyte to macrophage differentiation. We successfully identified numerous noncoding and protein-coding genes that can positively or negatively regulate inflammation and differentiation. To understand the functional roles of lncRNAs in both processes, we chose to further study the lncRNA ',\n",
       "  'doi': '10.1073/pnas.2322524121',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Neuronal activity rapidly reprograms dendritic translation via eIF4G2:uORF binding.',\n",
       "  'authors': ['Ezgi Hacisuleyman',\n",
       "   'Caryn R Hale',\n",
       "   'Natalie Noble',\n",
       "   'Ji-Dung Luo',\n",
       "   'John J Fak',\n",
       "   'Misa Saito',\n",
       "   'Jin Chen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Robert B Darnell'],\n",
       "  'publication_date': '09/04/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38589584',\n",
       "  'abstract': 'Learning and memory require activity-induced changes in dendritic translation, but which mRNAs are involved and how they are regulated are unclear. In this study, to monitor how depolarization impacts local dendritic biology, we employed a dendritically targeted proximity labeling approach followed by crosslinking immunoprecipitation, ribosome profiling and mass spectrometry. Depolarization of primary cortical neurons with KCl or the glutamate agonist DHPG caused rapid reprogramming of dendritic protein expression, where changes in dendritic mRNAs and proteins are weakly correlated. For a subset of pre-localized messages, depolarization increased the translation of upstream open reading frames (uORFs) and their downstream coding sequences, enabling localized production of proteins involved in long-term potentiation, cell signaling and energy metabolism. This activity-dependent translation was accompanied by the phosphorylation and recruitment of the non-canonical translation initiation factor eIF4G2, and the translated uORFs were sufficient to confer depolarization-induced, eIF4G2-dependent translational control. These studies uncovered an unanticipated mechanism by which activity-dependent uORF translational control by eIF4G2 couples activity to local dendritic remodeling.',\n",
       "  'doi': '10.1038/s41593-024-01615-5',\n",
       "  'journal': 'Nature neuroscience'},\n",
       " {'title': 'Triaging of α-helical proteins to the mitochondrial outer membrane by distinct chaperone machinery based on substrate topology.',\n",
       "  'authors': ['Gayathri Muthukumar',\n",
       "   'Taylor A Stevens',\n",
       "   'Alison J Inglis',\n",
       "   'Theodore K Esantsi',\n",
       "   'Reuben A Saunders',\n",
       "   'Fabian Schulte',\n",
       "   'Rebecca M Voorhees',\n",
       "   'Alina Guna',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '02/03/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38428433',\n",
       "  'abstract': 'Mitochondrial outer membrane ⍺-helical proteins play critical roles in mitochondrial-cytoplasmic communication, but the rules governing the targeting and insertion of these biophysically diverse proteins remain unknown. Here, we first defined the complement of required mammalian biogenesis machinery through genome-wide CRISPRi screens using topologically distinct membrane proteins. Systematic analysis of nine identified factors across 21 diverse ⍺-helical substrates reveals that these components are organized into distinct targeting pathways that act on substrates based on their topology. NAC is required for the efficient targeting of polytopic proteins, whereas signal-anchored proteins require TTC1, a cytosolic chaperone that physically engages substrates. Biochemical and mutational studies reveal that TTC1 employs a conserved TPR domain and a hydrophobic groove in its C-terminal domain to support substrate solubilization and insertion into mitochondria. Thus, the targeting of diverse mitochondrial membrane proteins is achieved through topological triaging in the cytosol using principles with similarities to ER membrane protein biogenesis systems.',\n",
       "  'doi': '10.1016/j.molcel.2024.01.028',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'CRISPR technologies for genome, epigenome and transcriptome editing.',\n",
       "  'authors': ['Lukas Villiger',\n",
       "   'Julia Joung',\n",
       "   'Luke Koblan',\n",
       "   'Jonathan Weissman',\n",
       "   'Omar O Abudayyeh',\n",
       "   'Jonathan S Gootenberg'],\n",
       "  'publication_date': '03/02/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38308006',\n",
       "  'abstract': 'Our ability to edit genomes lags behind our capacity to sequence them, but the growing understanding of CRISPR biology and its application to genome, epigenome and transcriptome engineering is narrowing this gap. In this Review, we discuss recent developments of various CRISPR-based systems that can transiently or permanently modify the genome and the transcriptome. The discovery of further CRISPR enzymes and systems through functional metagenomics has meaningfully broadened the applicability of CRISPR-based editing. Engineered Cas variants offer diverse capabilities such as base editing, prime editing, gene insertion and gene regulation, thereby providing a panoply of tools for the scientific community. We highlight the strengths and weaknesses of current CRISPR tools, considering their efficiency, precision, specificity, reliance on cellular DNA repair mechanisms and their applications in both fundamental biology and therapeutics. Finally, we discuss ongoing clinical trials that illustrate the potential impact of CRISPR systems on human health.',\n",
       "  'doi': '10.1038/s41580-023-00697-6',\n",
       "  'journal': 'Nature reviews. Molecular cell biology'},\n",
       " {'title': 'Deciphering cell states and genealogies of human haematopoiesis.',\n",
       "  'authors': ['Chen Weng',\n",
       "   'Fulong Yu',\n",
       "   'Dian Yang',\n",
       "   'Michael Poeschla',\n",
       "   'L Alexander Liggett',\n",
       "   'Matthew G Jones',\n",
       "   'Xiaojie Qiu',\n",
       "   'Lara Wahlster',\n",
       "   'Alexis Caulier',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Alexandra Schnell',\n",
       "   'Kathryn E Yost',\n",
       "   'Luke W Koblan',\n",
       "   'Jorge D Martin-Rufino',\n",
       "   'Joseph Min',\n",
       "   'Alessandro Hammond',\n",
       "   'Daniel Ssozi',\n",
       "   'Raphael Bueno',\n",
       "   'Hari Mallidi',\n",
       "   'Antonia Kreso',\n",
       "   'Javier Escabi',\n",
       "   'William M Rideout',\n",
       "   'Tyler Jacks',\n",
       "   'Sahand Hormoz',\n",
       "   'Peter van Galen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Vijay G Sankaran'],\n",
       "  'publication_date': '23/01/24',\n",
       "  'publication_year': '2024',\n",
       "  'pubmed_id': '38253266',\n",
       "  'abstract': 'The human blood system is maintained through the differentiation and massive amplification of a limited number of long-lived haematopoietic stem cells (HSCs)',\n",
       "  'doi': '10.1038/s41586-024-07066-z',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'A dual sgRNA library design to probe genetic modifiers using genome-wide CRISPRi screens.',\n",
       "  'authors': ['Alina Guna',\n",
       "   'Katharine R Page',\n",
       "   'Joseph M Replogle',\n",
       "   'Theodore K Esantsi',\n",
       "   'Maxine L Wang',\n",
       "   'Jonathan S Weissman',\n",
       "   'Rebecca M Voorhees'],\n",
       "  'publication_date': '31/10/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37904134',\n",
       "  'abstract': 'Mapping genetic interactions is essential for determining gene function and defining novel biological pathways. We report a simple to use CRISPR interference (CRISPRi) based platform, compatible with Fluorescence Activated Cell Sorting (FACS)-based reporter screens, to query epistatic relationships at scale. This is enabled by a flexible dual-sgRNA library design that allows for the simultaneous delivery and selection of a fixed sgRNA and a second randomized guide, comprised of a genome-wide library, with a single transduction. We use this approach to identify epistatic relationships for a defined biological pathway, showing both increased sensitivity and specificity than traditional growth screening approaches.',\n",
       "  'doi': '10.1186/s12864-023-09754-y',\n",
       "  'journal': 'BMC genomics'},\n",
       " {'title': 'Author Correction: Efficient C•G-to-G•C base editors developed using CRISPRi screens, target-library analysis, and machine learning.',\n",
       "  'authors': ['Luke W Koblan',\n",
       "   'Mandana Arbab',\n",
       "   'Max W Shen',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Andrew V Anzalone',\n",
       "   'Jordan L Doman',\n",
       "   'Gregory A Newby',\n",
       "   'Dian Yang',\n",
       "   'Beverly Mok',\n",
       "   'Joseph M Replogle',\n",
       "   'Albert Xu',\n",
       "   'Tyler A Sisley',\n",
       "   'Jonathan S Weissman',\n",
       "   'Britt Adamson',\n",
       "   'David R Liu'],\n",
       "  'publication_date': '19/10/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37853259',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/s41587-023-02028-8',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Systematic functional interrogation of SARS-CoV-2 host factors using Perturb-seq.',\n",
       "  'authors': ['Sara Sunshine',\n",
       "   'Andreas S Puschnik',\n",
       "   'Joseph M Replogle',\n",
       "   'Matthew T Laurie',\n",
       "   'Jamin Liu',\n",
       "   'Beth Shoshana Zha',\n",
       "   'James K Nuñez',\n",
       "   'Janie R Byrum',\n",
       "   'Aidan H McMorrow',\n",
       "   'Matthew B Frieman',\n",
       "   'Juliane Winkler',\n",
       "   'Xiaojie Qiu',\n",
       "   'Oren S Rosenberg',\n",
       "   'Manuel D Leonetti',\n",
       "   'Chun Jimmie Ye',\n",
       "   'Jonathan S Weissman',\n",
       "   'Joseph L DeRisi',\n",
       "   'Marco Y Hein'],\n",
       "  'publication_date': '07/10/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37803001',\n",
       "  'abstract': 'Genomic and proteomic screens have identified numerous host factors of SARS-CoV-2, but efficient delineation of their molecular roles during infection remains a challenge. Here we use Perturb-seq, combining genetic perturbations with a single-cell readout, to investigate how inactivation of host factors changes the course of SARS-CoV-2 infection and the host response in human lung epithelial cells. Our high-dimensional data resolve complex phenotypes such as shifts in the stages of infection and modulations of the interferon response. However, only a small percentage of host factors showed such phenotypes upon perturbation. We further identified the NF-κB inhibitor IκBα (NFKBIA), as well as the translation factors EIF4E2 and EIF4H as strong host dependency factors acting early in infection. Overall, our study provides massively parallel functional characterization of host factors of SARS-CoV-2 and quantitatively defines their roles both in virus-infected and bystander cells.',\n",
       "  'doi': '10.1038/s41467-023-41788-4',\n",
       "  'journal': 'Nature communications'},\n",
       " {'title': 'Triaging of ⍺-helical proteins to the mitochondrial outer membrane by distinct chaperone machinery based on substrate topology.',\n",
       "  'authors': ['Gayathri Muthukumar',\n",
       "   'Taylor A Stevens',\n",
       "   'Alison J Inglis',\n",
       "   'Theodore K Esantsi',\n",
       "   'Reuben A Saunders',\n",
       "   'Fabian Schulte',\n",
       "   'Rebecca M Voorhees',\n",
       "   'Alina Guna',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '30/08/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37645817',\n",
       "  'abstract': 'Mitochondrial outer membrane ⍺-helical proteins play critical roles in mitochondrial-cytoplasmic communication, but the rules governing the targeting and insertion of these biophysically diverse substrates remain unknown. Here, we first defined the complement of required mammalian biogenesis machinery through genome-wide CRISPRi screens using topologically distinct membrane proteins. Systematic analysis of nine identified factors across 21 diverse ⍺-helical substrates reveals that these components are organized into distinct targeting pathways which act on substrates based on their topology. NAC is required for efficient targeting of polytopic proteins whereas signal-anchored proteins require TTC1, a novel cytosolic chaperone which physically engages substrates. Biochemical and mutational studies reveal that TTC1 employs a conserved TPR domain and a hydrophobic groove in its C-terminal domain to support substrate solubilization and insertion into mitochondria. Thus, targeting of diverse mitochondrial membrane proteins is achieved through topological triaging in the cytosol using principles with similarities to ER membrane protein biogenesis systems.',\n",
       "  'doi': '10.1101/2023.08.16.553624',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'FET fusion oncoproteins disrupt physiologic DNA repair networks and induce ATR synthetic lethality in cancer.',\n",
       "  'authors': ['Asmin Tulpule',\n",
       "   'Shruti Menon',\n",
       "   'Marcus Breese',\n",
       "   'Yone Lin',\n",
       "   'Hannah Allegakoen',\n",
       "   'Shruthi Perati',\n",
       "   'Ann Heslin',\n",
       "   'Max Horlbeck',\n",
       "   'Jonathan Weissman',\n",
       "   'Alejandro Sweet-Cordero',\n",
       "   'Trever Bivona'],\n",
       "  'publication_date': '03/07/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37398210',\n",
       "  'abstract': 'The genetic principle of synthetic lethality is clinically validated in cancers with loss of specific DNA damage response (DDR) pathway genes (i.e. BRCA1/2 tumor suppressor mutations). The broader question of whether and how oncogenes create tumor-specific vulnerabilities within DDR networks remains unanswered. Native FET protein family members are among the earliest proteins recruited to DNA double-strand breaks (DSBs) during the DDR, though the function of both native FET proteins and FET fusion oncoproteins in DSB repair remains poorly defined. Here we focus on Ewing sarcoma (ES), an EWS-FLI1 fusion oncoprotein-driven pediatric bone tumor, as a model for FET rearranged cancers. We discover that the EWS-FLI1 fusion oncoprotein is recruited to DNA DSBs and interferes with native EWS function in activating the DNA damage sensor ATM. Using preclinical mechanistic approaches and clinical datasets, we establish functional ATM deficiency as a principal DNA repair defect in ES and the compensatory ATR signaling axis as a collateral dependency and therapeutic target in FET rearranged cancers. Thus, aberrant recruitment of a fusion oncoprotein to sites of DNA damage can disrupt normal DSB repair, revealing a mechanism for how oncogenes can create cancer-specific synthetic lethality within DDR networks.',\n",
       "  'doi': '10.21203/rs.3.rs-2869150/v1',\n",
       "  'journal': 'Research square'},\n",
       " {'title': 'Comparative landscape of genetic dependencies in human and chimpanzee stem cells.',\n",
       "  'authors': ['Richard She',\n",
       "   'Tyler Fair',\n",
       "   'Nathan K Schaefer',\n",
       "   'Reuben A Saunders',\n",
       "   'Bryan J Pavlovic',\n",
       "   'Jonathan S Weissman',\n",
       "   'Alex A Pollen'],\n",
       "  'publication_date': '22/06/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37343560',\n",
       "  'abstract': 'Comparative studies of great apes provide a window into our evolutionary past, but the extent and identity of cellular differences that emerged during hominin evolution remain largely unexplored. We established a comparative loss-of-function approach to evaluate whether human cells exhibit distinct genetic dependencies. By performing genome-wide CRISPR interference screens in human and chimpanzee pluripotent stem cells, we identified 75 genes with species-specific effects on cellular proliferation. These genes comprised coherent processes, including cell-cycle progression and lysosomal signaling, which we determined to be human-derived by comparison with orangutan cells. Human-specific robustness to CDK2 and CCNE1 depletion persisted in neural progenitor cells and cerebral organoids, supporting the G1-phase length hypothesis as a potential evolutionary mechanism in human brain expansion. Our findings demonstrate that evolutionary changes in human cells reshaped the landscape of essential genes and establish a platform for systematically uncovering latent cellular and molecular differences between species.',\n",
       "  'doi': '10.1016/j.cell.2023.05.043',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Coupled protein quality control during nonsense-mediated mRNA decay.',\n",
       "  'authors': ['Alison J Inglis',\n",
       "   'Alina Guna',\n",
       "   'Ángel Gálvez-Merchán',\n",
       "   'Akshaye Pal',\n",
       "   'Theodore K Esantsi',\n",
       "   'Heather R Keys',\n",
       "   'Evgeni M Frenkel',\n",
       "   'Robert Oania',\n",
       "   'Jonathan S Weissman',\n",
       "   'Rebecca M Voorhees'],\n",
       "  'publication_date': '23/05/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37218462',\n",
       "  'abstract': 'Translation of mRNAs containing premature termination codons (PTCs) results in truncated protein products with deleterious effects. Nonsense-mediated decay (NMD) is a surveillance pathway responsible for detecting PTC containing transcripts. Although the molecular mechanisms governing mRNA degradation have been extensively studied, the fate of the nascent protein product remains largely uncharacterized. Here, we use a fluorescent reporter system in mammalian cells to reveal a selective degradation pathway specifically targeting the protein product of an NMD mRNA. We show that this process is post-translational and dependent on the ubiquitin proteasome system. To systematically uncover factors involved in NMD-linked protein quality control, we conducted genome-wide flow cytometry-based screens. Our screens recovered known NMD factors but suggested that protein degradation did not depend on the canonical ribosome-quality control (RQC) pathway. A subsequent arrayed screen demonstrated that protein and mRNA branches of NMD rely on a shared recognition event. Our results establish the existence of a targeted pathway for nascent protein degradation from PTC containing mRNAs, and provide a reference for the field to identify and characterize required factors.',\n",
       "  'doi': '10.1242/jcs.261216',\n",
       "  'journal': 'Journal of cell science'},\n",
       " {'title': 'FET fusion oncoproteins disrupt physiologic DNA repair networks in cancer.',\n",
       "  'authors': ['Shruti Menon',\n",
       "   'Marcus R Breese',\n",
       "   'Yone Phar Lin',\n",
       "   'Hannah Allegakoen',\n",
       "   'Shruthi Perati',\n",
       "   'Ann Heslin',\n",
       "   'Max A Horlbeck',\n",
       "   'Jonathan Weissman',\n",
       "   'E Alejandro Sweet-Cordero',\n",
       "   'Trever G Bivona',\n",
       "   'Asmin Tulpule'],\n",
       "  'publication_date': '19/05/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37205599',\n",
       "  'abstract': 'While oncogenes promote cancer cell growth, unrestrained proliferation represents a significant stressor to cellular homeostasis networks such as the DNA damage response (DDR). To enable oncogene tolerance, many cancers disable tumor suppressive DDR signaling through genetic loss of DDR pathways and downstream effectors (e.g., ATM or p53 tumor suppressor mutations). Whether and how oncogenes can help \"self-tolerize\" by creating analogous functional deficiencies in physiologic DDR networks is not known. Here we focus on Ewing sarcoma, a FET fusion oncoprotein (EWS-FLI1) driven pediatric bone tumor, as a model for the class of FET rearranged cancers. Native FET protein family members are among the earliest factors recruited to DNA double-strand breaks (DSBs) during the DDR, though the function of both native FET proteins and FET fusion oncoproteins in DNA repair remains to be defined. Using preclinical mechanistic studies of the DDR and clinical genomic datasets from patient tumors, we discover that the EWS-FLI1 fusion oncoprotein is recruited to DNA DSBs and interferes with native FET (EWS) protein function in activating the DNA damage sensor ATM. As a consequence of FET fusion-mediated interference with the DDR, we establish functional ATM deficiency as the principal DNA repair defect in Ewing sarcoma and the compensatory ATR signaling axis as a collateral dependency and therapeutic target in multiple FET rearranged cancers. More generally, we find that aberrant recruitment of a fusion oncoprotein to sites of DNA damage can disrupt physiologic DSB repair, revealing a mechanism for how growth-promoting oncogenes can also create a functional deficiency within tumor suppressive DDR networks.',\n",
       "  'doi': '10.1101/2023.04.30.538578',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Translational fidelity screens in mammalian cells reveal eIF3 and eIF4G2 as regulators of start codon selectivity.',\n",
       "  'authors': ['Richard She', 'Jingchuan Luo', 'Jonathan S Weissman'],\n",
       "  'publication_date': '05/05/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '37144468',\n",
       "  'abstract': 'The translation initiation machinery and the ribosome orchestrate a highly dynamic scanning process to distinguish proper start codons from surrounding nucleotide sequences. Here, we performed genome-wide CRISPRi screens in human K562 cells to systematically identify modulators of the frequency of translation initiation at near-cognate start codons. We observed that depletion of any eIF3 core subunit promoted near-cognate start codon usage, though sensitivity thresholds of each subunit to sgRNA-mediated depletion varied considerably. Double sgRNA depletion experiments suggested that enhanced near-cognate usage in eIF3D depleted cells required canonical eIF4E cap-binding and was not driven by eIF2A or eIF2D-dependent leucine tRNA initiation. We further characterized the effects of eIF3D depletion and found that the N-terminus of eIF3D was strictly required for accurate start codon selection, whereas disruption of the cap-binding properties of eIF3D had no effect. Lastly, depletion of eIF3D activated TNFα signaling via NF-κB and the interferon gamma response. Similar transcriptional profiles were observed upon knockdown of eIF1A and eIF4G2, which also promoted near-cognate start codon usage, suggesting that enhanced near-cognate usage could potentially contribute to NF-κB activation. Our study thus provides new avenues to study the mechanisms and consequences of alternative start codon usage.',\n",
       "  'doi': '10.1093/nar/gkad329',\n",
       "  'journal': 'Nucleic acids research'},\n",
       " {'title': 'Comparative landscape of genetic dependencies in human and chimpanzee stem cells.',\n",
       "  'authors': ['Richard She',\n",
       "   'Tyler Fair',\n",
       "   'Nathan K Schaefer',\n",
       "   'Reuben A Saunders',\n",
       "   'Bryan J Pavlovic',\n",
       "   'Jonathan S Weissman',\n",
       "   'Alex A Pollen'],\n",
       "  'publication_date': '31/03/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '36993685',\n",
       "  'abstract': 'Comparative studies of great apes provide a window into our evolutionary past, but the extent and identity of cellular differences that emerged during hominin evolution remain largely unexplored. We established a comparative loss-of-function approach to evaluate whether changes in human cells alter requirements for essential genes. By performing genome-wide CRISPR interference screens in human and chimpanzee pluripotent stem cells, we identified 75 genes with species-specific effects on cellular proliferation. These genes comprised coherent processes, including cell cycle progression and lysosomal signaling, which we determined to be human-derived by comparison with orangutan cells. Human-specific robustness to ',\n",
       "  'doi': '10.1101/2023.03.19.533346',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Microfluidics-free single-cell genomics with templated emulsification.',\n",
       "  'authors': ['Iain C Clark',\n",
       "   'Kristina M Fontanez',\n",
       "   'Robert H Meltzer',\n",
       "   'Yi Xue',\n",
       "   'Corey Hayford',\n",
       "   'Aaron May-Zhang',\n",
       "   \"Chris D'Amato\",\n",
       "   'Ahmad Osman',\n",
       "   'Jesse Q Zhang',\n",
       "   'Pabodha Hettige',\n",
       "   'Jacob S A Ishibashi',\n",
       "   'Cyrille L Delley',\n",
       "   'Daniel W Weisgerber',\n",
       "   'Joseph M Replogle',\n",
       "   'Marco Jost',\n",
       "   'Kiet T Phong',\n",
       "   'Vanessa E Kennedy',\n",
       "   'Cheryl A C Peretz',\n",
       "   'Esther A Kim',\n",
       "   'Siyou Song',\n",
       "   'William Karlon',\n",
       "   'Jonathan S Weissman',\n",
       "   'Catherine C Smith',\n",
       "   'Zev J Gartner',\n",
       "   'Adam R Abate'],\n",
       "  'publication_date': '07/03/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '36879006',\n",
       "  'abstract': 'Current single-cell RNA-sequencing approaches have limitations that stem from the microfluidic devices or fluid handling steps required for sample processing. We develop a method that does not require specialized microfluidic devices, expertise or hardware. Our approach is based on particle-templated emulsification, which allows single-cell encapsulation and barcoding of cDNA in uniform droplet emulsions with only a vortexer. Particle-templated instant partition sequencing (PIP-seq) accommodates a wide range of emulsification formats, including microwell plates and large-volume conical tubes, enabling thousands of samples or millions of cells to be processed in minutes. We demonstrate that PIP-seq produces high-purity transcriptomes in mouse-human mixing studies, is compatible with multiomics measurements and can accurately characterize cell types in human breast tissue compared to a commercial microfluidic platform. Single-cell transcriptional profiling of mixed phenotype acute leukemia using PIP-seq reveals the emergence of heterogeneity within chemotherapy-resistant cell subsets that were hidden by standard immunophenotyping. PIP-seq is a simple, flexible and scalable next-generation workflow that extends single-cell sequencing to new applications.',\n",
       "  'doi': '10.1038/s41587-023-01685-z',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'A dual sgRNA library design to probe genetic modifiers using genome-wide CRISPRi screens.',\n",
       "  'authors': ['Alina Guna',\n",
       "   'Katharine R Page',\n",
       "   'Joseph R Replogle',\n",
       "   'Theodore K Esantsi',\n",
       "   'Maxine L Wang',\n",
       "   'Jonathan S Weissman',\n",
       "   'Rebecca M Voorhees'],\n",
       "  'publication_date': '31/01/23',\n",
       "  'publication_year': '2023',\n",
       "  'pubmed_id': '36711738',\n",
       "  'abstract': 'The ability to map genetic interactions has been essential for determining gene function and defining biological pathways. Therefore, a system to readily perform genome-wide genetic modifier screens in human cells is a powerful platform for dissecting complex processes in mammalian cells, where redundancy and adaptation commonly mask the phenotype of a single genetic perturbation. Here, we report a CRISPR interference (CRISPRi) based platform, compatible with Fluorescence Activated Cell Sorting (FACS)-based reporter screens, that can be used to query epistatic relationships at scale. This is enabled by a flexible dual-sgRNA library design that allows for the simultaneous delivery and selection of a fixed sgRNA and a second randomized guide, comprised of a genome-wide library, with a single transduction. As a proof of principle, we apply our approach to study the pathways that mediate tail-anchored (TA) protein insertion at the endoplasmic reticulum (ER). We show that this dual-guide library approach can be successfully coupled with FACS-based reporter screening, to identify genetic epistasis and thereby place TA biogenesis factors in their respective parallel pathways. We demonstrate that this dual-guide approach is both more sensitive and specific than traditional growth screening approaches, and is ideally suited for dissecting the complex interplay between factors in human cells.',\n",
       "  'doi': '10.1101/2023.01.22.525086',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'High-content CRISPR screening.',\n",
       "  'authors': ['Christoph Bock',\n",
       "   'Paul Datlinger',\n",
       "   'Florence Chardon',\n",
       "   'Matthew A Coelho',\n",
       "   'Matthew B Dong',\n",
       "   'Keith A Lawson',\n",
       "   'Tian Lu',\n",
       "   'Laetitia Maroc',\n",
       "   'Thomas M Norman',\n",
       "   'Bicna Song',\n",
       "   'Geoff Stanley',\n",
       "   'Sidi Chen',\n",
       "   'Mathew Garnett',\n",
       "   'Wei Li',\n",
       "   'Jason Moffat',\n",
       "   'Lei S Qi',\n",
       "   'Rebecca S Shapiro',\n",
       "   'Jay Shendure',\n",
       "   'Jonathan S Weissman',\n",
       "   'Xiaowei Zhuang'],\n",
       "  'publication_date': '01/01/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '37214176',\n",
       "  'abstract': 'CRISPR screens are a powerful source of biological discovery, enabling the unbiased interrogation of gene function in a wide range of applications and species. In pooled CRISPR screens, various genetically encoded perturbations are introduced into pools of cells. The targeted cells proliferate under a biological challenge such as cell competition, drug treatment or viral infection. Subsequently, the perturbation-induced effects are evaluated by sequencing-based counting of the guide RNAs that specify each perturbation. The typical results of such screens are ranked lists of genes that confer sensitivity or resistance to the biological challenge of interest. Contributing to the broad utility of CRISPR screens, adaptations of the core CRISPR technology make it possible to activate, silence or otherwise manipulate the target genes. Moreover, high-content read-outs such as single-cell RNA sequencing and spatial imaging help characterize screened cells with unprecedented detail. Dedicated software tools facilitate bioinformatic analysis and enhance reproducibility. CRISPR screening has unravelled various molecular mechanisms in basic biology, medical genetics, cancer research, immunology, infectious diseases, microbiology and other fields. This Primer describes the basic and advanced concepts of CRISPR screening and its application as a flexible and reliable method for biological discovery, biomedical research and drug development - with a special emphasis on high-content methods that make it possible to obtain detailed biological insights directly as part of the screen.',\n",
       "  'doi': '10.1038/s43586-022-00098-7',\n",
       "  'journal': 'Nature reviews. Methods primers'},\n",
       " {'title': 'Maximizing CRISPRi efficacy and accessibility with dual-sgRNA libraries and optimal effectors.',\n",
       "  'authors': ['Joseph M Replogle',\n",
       "   'Jessica L Bonnar',\n",
       "   'Angela N Pogson',\n",
       "   'Christina R Liem',\n",
       "   'Nolan K Maier',\n",
       "   'Yufang Ding',\n",
       "   'Baylee J Russell',\n",
       "   'Xingren Wang',\n",
       "   'Kun Leng',\n",
       "   'Alina Guna',\n",
       "   'Thomas M Norman',\n",
       "   'Ryan A Pak',\n",
       "   'Daniel M Ramos',\n",
       "   'Michael E Ward',\n",
       "   'Luke A Gilbert',\n",
       "   'Martin Kampmann',\n",
       "   'Jonathan S Weissman',\n",
       "   'Marco Jost'],\n",
       "  'publication_date': '29/12/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36576240',\n",
       "  'abstract': 'CRISPR interference (CRISPRi) enables programmable, reversible, and titratable repression of gene expression (knockdown) in mammalian cells. Initial CRISPRi-mediated genetic screens have showcased the potential to address basic questions in cell biology, genetics, and biotechnology, but wider deployment of CRISPRi screening has been constrained by the large size of single guide RNA (sgRNA) libraries and challenges in generating cell models with consistent CRISPRi-mediated knockdown. Here, we present next-generation CRISPRi sgRNA libraries and effector expression constructs that enable strong and consistent knockdown across mammalian cell models. First, we combine empirical sgRNA selection with a dual-sgRNA library design to generate an ultra-compact (1-3 elements per gene), highly active CRISPRi sgRNA library. Next, we compare CRISPRi effectors to show that the recently published Zim3-dCas9 provides an excellent balance between strong on-target knockdown and minimal non-specific effects on cell growth or the transcriptome. Finally, we engineer a suite of cell lines with stable expression of Zim3-dCas9 and robust on-target knockdown. Our results and publicly available reagents establish best practices for CRISPRi genetic screening.',\n",
       "  'doi': '10.7554/eLife.81856',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Dual genome-wide coding and lncRNA screens in neural induction of induced pluripotent stem cells.',\n",
       "  'authors': ['David Wu',\n",
       "   'Aunoy Poddar',\n",
       "   'Elpiniki Ninou',\n",
       "   'Elizabeth Hwang',\n",
       "   'Mitchel A Cole',\n",
       "   'S John Liu',\n",
       "   'Max A Horlbeck',\n",
       "   'Jin Chen',\n",
       "   'Joseph M Replogle',\n",
       "   'Giovanni A Carosso',\n",
       "   'Nicolas W L Eng',\n",
       "   'Jonghoon Chang',\n",
       "   'Yin Shen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Daniel A Lim'],\n",
       "  'publication_date': '17/11/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36381608',\n",
       "  'abstract': 'Human chromosomes are pervasively transcribed, but systematic understanding of coding and lncRNA genome function in cell differentiation is lacking. Using CRISPR interference (CRISPRi) in human induced pluripotent stem cells, we performed dual genome-wide screens - assessing 18,905 protein-coding and 10,678 lncRNA loci - and identified 419 coding and 201 lncRNA genes that regulate neural induction. Integrative analyses revealed distinct properties of coding and lncRNA genome function, including a 10-fold enrichment of lncRNA genes for roles in differentiation compared to proliferation. Further, we applied Perturb-seq to obtain granular insights into neural induction phenotypes. While most coding hits stalled or aborted differentiation, lncRNA hits were enriched for the genesis of diverse cellular states, including those outside the neural lineage. In addition to providing a rich resource (danlimlab.shinyapps.io/dualgenomewide) for understanding coding and lncRNA gene function in development, these results indicate that the lncRNA genome regulates lineage commitment in a manner fundamentally distinct from coding genes.',\n",
       "  'doi': '10.1016/j.xgen.2022.100177',\n",
       "  'journal': 'Cell genomics'},\n",
       " {'title': 'GATOR2-dependent mTORC1 activity is a therapeutic vulnerability in FOXO1 fusion positive rhabdomyosarcoma.',\n",
       "  'authors': ['Jacqueline Morales',\n",
       "   'David V Allegakoen',\n",
       "   'José A Garcia',\n",
       "   'Kristen Kwong',\n",
       "   'Pushpendra K Sahu',\n",
       "   'Drew A Fajardo',\n",
       "   'Yue Pan',\n",
       "   'Max A Horlbeck',\n",
       "   'Jonathan S Weissman',\n",
       "   'W Clay Gustafson',\n",
       "   'Trever G Bivona',\n",
       "   'Amit J Sabnis'],\n",
       "  'publication_date': '26/10/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36282590',\n",
       "  'abstract': 'Oncogenic FOXO1 gene fusions drive a subset of rhabdomyosarcoma (RMS) with poor survival and to date these cancer drivers are therapeutically intractable. To identify new therapies for this disease, we undertook an isogenic CRISPR-interference screen to define PAX3-FOXO1 specific genetic dependencies and identified genes in the GATOR2 complex. GATOR2 loss in RMS abrogated amino acid-induced lysosomal localization of mTORC1 and consequent downstream signaling, slowing G1-S cell cycle transition. In vivo suppression of GATOR2 impaired the growth of tumor xenografts and favored the outgrowth of cells lacking PAX3-FOXO1. Loss of a subset of GATOR2 members can be compensated by direct genetic activation of mTORC1. RAS mutations are also sufficient to decouple mTORC1 activation from GATOR2, and indeed fusion negative RMS harboring such mutations exhibit amino acid-independent mTORC1 activity. A bi-steric, mTORC1-selective small molecule induced tumor regressions in fusion positive patient-derived tumor xenografts. These findings highlight a vulnerability in FOXO1 fusion positive RMS and provide rationale for the clinical evaluation of bi-steric mTORC1 inhibitors, currently in phase 1 testing, to treat this disease. Isogenic genetic screens can thus identify potentially exploitable vulnerabilities in fusion driven pediatric cancers which otherwise remain mostly undruggable.',\n",
       "  'doi': '10.1172/jci.insight.162207',\n",
       "  'journal': 'JCI insight'},\n",
       " {'title': 'MTCH2 is a mitochondrial outer membrane protein insertase.',\n",
       "  'authors': ['Alina Guna',\n",
       "   'Taylor A Stevens',\n",
       "   'Alison J Inglis',\n",
       "   'Joseph M Replogle',\n",
       "   'Theodore K Esantsi',\n",
       "   'Gayathri Muthukumar',\n",
       "   'Kelly C L Shaffer',\n",
       "   'Maxine L Wang',\n",
       "   'Angela N Pogson',\n",
       "   'Jeff J Jones',\n",
       "   'Brett Lomenick',\n",
       "   'Tsui-Fen Chou',\n",
       "   'Jonathan S Weissman',\n",
       "   'Rebecca M Voorhees'],\n",
       "  'publication_date': '21/10/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36264797',\n",
       "  'abstract': 'In the mitochondrial outer membrane, α-helical transmembrane proteins play critical roles in cytoplasmic-mitochondrial communication. Using genome-wide CRISPR screens, we identified mitochondrial carrier homolog 2 (MTCH2), and its paralog MTCH1, and showed that it is required for insertion of biophysically diverse tail-anchored (TA), signal-anchored, and multipass proteins, but not outer membrane β-barrel proteins. Purified MTCH2 was sufficient to mediate insertion into reconstituted proteoliposomes. Functional and mutational studies suggested that MTCH2 has evolved from a solute carrier transporter. MTCH2 uses membrane-embedded hydrophilic residues to function as a gatekeeper for the outer membrane, controlling mislocalization of TAs into the endoplasmic reticulum and modulating the sensitivity of leukemia cells to apoptosis. Our identification of MTCH2 as an insertase provides a mechanistic explanation for the diverse phenotypes and disease states associated with MTCH2 dysfunction.',\n",
       "  'doi': '10.1126/science.add1856',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Cellular barcoding to decipher clonal dynamics in disease.',\n",
       "  'authors': ['Vijay G Sankaran', 'Jonathan S Weissman', 'Leonard I Zon'],\n",
       "  'publication_date': '14/10/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36227997',\n",
       "  'abstract': 'Cellular barcodes are distinct DNA sequences that enable one to track specific cells across time or space. Recent advances in our ability to detect natural or synthetic cellular barcodes, paired with single-cell readouts of cell state, have markedly increased our knowledge of clonal dynamics and genealogies of the cells that compose a variety of tissues and organs. These advances hold promise to redefine our view of human disease. Here, we provide an overview of cellular barcoding approaches, discuss applications to gain new insights into disease mechanisms, and provide an outlook on future applications. We discuss unanticipated insights gained through barcoding in studies of cancer and blood cell production and describe how barcoding can be applied to a growing array of medical fields, particularly with the increasing recognition of clonal contributions in human diseases.',\n",
       "  'doi': '10.1126/science.abm5874',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Adaptive exchange sustains cullin-RING ubiquitin ligase networks and proper licensing of DNA replication.',\n",
       "  'authors': ['Yaru Zhang',\n",
       "   'Marco Jost',\n",
       "   'Ryan A Pak',\n",
       "   'Daniel Lu',\n",
       "   'Jing Li',\n",
       "   'Brett Lomenick',\n",
       "   'Spiros D Garbis',\n",
       "   'Chi-Ming Li',\n",
       "   'Jonathan S Weissman',\n",
       "   'James Russell Lipford',\n",
       "   'Raymond J Deshaies'],\n",
       "  'publication_date': '30/08/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '36037385',\n",
       "  'abstract': 'Cop9 signalosome (CSN) regulates the function of cullin-RING E3 ubiquitin ligases (CRLs) by deconjugating the ubiquitin-like protein NEDD8 from the cullin subunit. To understand the physiological impact of CSN function on the CRL network and cell proliferation, we combined quantitative mass spectrometry and genome-wide CRISPR interference (CRISPRi) and CRISPR activation (CRISPRa) screens to identify factors that modulate cell viability upon inhibition of CSN by the small molecule CSN5i-3. CRL components and regulators strongly modulated the antiproliferative effects of CSN5i-3, and in addition we found two pathways involved in genome integrity, SCF',\n",
       "  'doi': '10.1073/pnas.2205608119',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Ras-mutant cancers are sensitive to small molecule inhibition of V-type ATPases in mice.',\n",
       "  'authors': ['Bhairavi Tolani',\n",
       "   'Anna Celli',\n",
       "   'Yanmin Yao',\n",
       "   'Yong Zi Tan',\n",
       "   'Richard Fetter',\n",
       "   'Christina R Liem',\n",
       "   'Adam J de Smith',\n",
       "   'Thamiya Vasanthakumar',\n",
       "   'Paola Bisignano',\n",
       "   'Adam D Cotton',\n",
       "   'Ian B Seiple',\n",
       "   'John L Rubinstein',\n",
       "   'Marco Jost',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '26/07/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35879364',\n",
       "  'abstract': 'Mutations in Ras family proteins are implicated in 33% of human cancers, but direct pharmacological inhibition of Ras mutants remains challenging. As an alternative to direct inhibition, we screened for sensitivities in Ras-mutant cells and discovered 249C as a Ras-mutant selective cytotoxic agent with nanomolar potency against a spectrum of Ras-mutant cancers. 249C binds to vacuolar (V)-ATPase with nanomolar affinity and inhibits its activity, preventing lysosomal acidification and inhibiting autophagy and macropinocytosis pathways that several Ras-driven cancers rely on for survival. Unexpectedly, potency of 249C varies with the identity of the Ras driver mutation, with the highest potency for KRASG13D and G12V both in vitro and in vivo, highlighting a mutant-specific dependence on macropinocytosis and lysosomal pH. Indeed, 249C potently inhibits tumor growth without adverse side effects in mouse xenografts of KRAS-driven lung and colon cancers. A comparison of isogenic SW48 xenografts with different KRAS mutations confirmed that KRASG13D/+ (followed by G12V/+) mutations are especially sensitive to 249C treatment. These data establish proof-of-concept for targeting V-ATPase in cancers driven by specific KRAS mutations such as KRASG13D and G12V.',\n",
       "  'doi': '10.1038/s41587-022-01386-z',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Standardized annotation of translated open reading frames.',\n",
       "  'authors': ['Jonathan M Mudge',\n",
       "   'Jorge Ruiz-Orera',\n",
       "   'John R Prensner',\n",
       "   'Marie A Brunet',\n",
       "   'Ferriol Calvet',\n",
       "   'Irwin Jungreis',\n",
       "   'Jose Manuel Gonzalez',\n",
       "   'Michele Magrane',\n",
       "   'Thomas F Martinez',\n",
       "   'Jana Felicitas Schulz',\n",
       "   'Yucheng T Yang',\n",
       "   'M Mar Albà',\n",
       "   'Julie L Aspden',\n",
       "   'Pavel V Baranov',\n",
       "   'Ariel A Bazzini',\n",
       "   'Elspeth Bruford',\n",
       "   'Maria Jesus Martin',\n",
       "   'Lorenzo Calviello',\n",
       "   'Anne-Ruxandra Carvunis',\n",
       "   'Jin Chen',\n",
       "   'Juan Pablo Couso',\n",
       "   'Eric W Deutsch',\n",
       "   'Paul Flicek',\n",
       "   'Adam Frankish',\n",
       "   'Mark Gerstein',\n",
       "   'Norbert Hubner',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Manolis Kellis',\n",
       "   'Gerben Menschaert',\n",
       "   'Robert L Moritz',\n",
       "   'Uwe Ohler',\n",
       "   'Xavier Roucou',\n",
       "   'Alan Saghatelian',\n",
       "   'Jonathan S Weissman',\n",
       "   'Sebastiaan van Heesch'],\n",
       "  'publication_date': '14/07/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35831657',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/s41587-022-01369-0',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'A campaign targeting a conserved Hsp70 binding site uncovers how subcellular localization is linked to distinct biological activities.',\n",
       "  'authors': ['Hao Shao',\n",
       "   'Shuhei Taguwa',\n",
       "   'Luke Gilbert',\n",
       "   'Arielle Shkedi',\n",
       "   'Sara Sannino',\n",
       "   'Christopher J Guerriero',\n",
       "   'Zachary J Gale-Day',\n",
       "   'Zapporah T Young',\n",
       "   'Jeffrey L Brodsky',\n",
       "   'Jonathan Weissman',\n",
       "   'Jason E Gestwicki',\n",
       "   'Judith Frydman'],\n",
       "  'publication_date': '14/07/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35830852',\n",
       "  'abstract': 'The potential of small molecules to localize within subcellular compartments is rarely explored. To probe this question, we measured the localization of Hsp70 inhibitors using fluorescence microscopy. We found that even closely related analogs had dramatically different distributions, with some residing predominantly in the mitochondria and others in the ER. CRISPRi screens supported this idea, showing that different compounds had distinct chemogenetic interactions with Hsp70s of the ER (HSPA5/BiP) and mitochondria (HSPA9/mortalin) and their co-chaperones. Moreover, localization seemed to determine function, even for molecules with conserved binding sites. Compounds with distinct partitioning have distinct anti-proliferative activity in breast cancer cells compared with anti-viral activity in cellular models of Dengue virus replication, likely because different sets of Hsp70s are required in these processes. These findings highlight the contributions of subcellular partitioning and chemogenetic interactions to small molecule activity, features that are rarely explored during medicinal chemistry campaigns.',\n",
       "  'doi': '10.1016/j.chembiol.2022.06.006',\n",
       "  'journal': 'Cell chemical biology'},\n",
       " {'title': 'Mapping information-rich genotype-phenotype landscapes with genome-scale Perturb-seq.',\n",
       "  'authors': ['Joseph M Replogle',\n",
       "   'Reuben A Saunders',\n",
       "   'Angela N Pogson',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Alexander Lenail',\n",
       "   'Alina Guna',\n",
       "   'Lauren Mascibroda',\n",
       "   'Eric J Wagner',\n",
       "   'Karen Adelman',\n",
       "   'Gila Lithwick-Yanai',\n",
       "   'Nika Iremadze',\n",
       "   'Florian Oberstrass',\n",
       "   'Doron Lipson',\n",
       "   'Jessica L Bonnar',\n",
       "   'Marco Jost',\n",
       "   'Thomas M Norman',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '11/06/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35688146',\n",
       "  'abstract': 'A central goal of genetics is to define the relationships between genotypes and phenotypes. High-content phenotypic screens such as Perturb-seq (CRISPR-based screens with single-cell RNA-sequencing readouts) enable massively parallel functional genomic mapping but, to date, have been used at limited scales. Here, we perform genome-scale Perturb-seq targeting all expressed genes with CRISPR interference (CRISPRi) across >2.5 million human cells. We use transcriptional phenotypes to predict the function of poorly characterized genes, uncovering new regulators of ribosome biogenesis (including CCDC86, ZNF236, and SPATA5L1), transcription (C7orf26), and mitochondrial respiration (TMEM242). In addition to assigning gene function, single-cell transcriptional phenotypes allow for in-depth dissection of complex cellular phenomena-from RNA processing to differentiation. We leverage this ability to systematically identify genetic drivers and consequences of aneuploidy and to discover an unanticipated layer of stress-specific regulation of the mitochondrial genome. Our information-rich genotype-phenotype map reveals a multidimensional portrait of gene and cellular function.',\n",
       "  'doi': '10.1016/j.cell.2022.05.013',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Variant to function mapping at single-cell resolution through network propagation.',\n",
       "  'authors': ['Fulong Yu',\n",
       "   'Liam D Cato',\n",
       "   'Chen Weng',\n",
       "   'L Alexander Liggett',\n",
       "   'Soyoung Jeon',\n",
       "   'Keren Xu',\n",
       "   'Charleston W K Chiang',\n",
       "   'Joseph L Wiemels',\n",
       "   'Jonathan S Weissman',\n",
       "   'Adam J de Smith',\n",
       "   'Vijay G Sankaran'],\n",
       "  'publication_date': '07/06/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35668323',\n",
       "  'abstract': 'Genome-wide association studies in combination with single-cell genomic atlases can provide insights into the mechanisms of disease-causal genetic variation. However, identification of disease-relevant or trait-relevant cell types, states and trajectories is often hampered by sparsity and noise, particularly in the analysis of single-cell epigenomic data. To overcome these challenges, we present SCAVENGE, a computational algorithm that uses network propagation to map causal variants to their relevant cellular context at single-cell resolution. We demonstrate how SCAVENGE can help identify key biological mechanisms underlying human genetic variation, applying the method to blood traits at distinct stages of human hematopoiesis, to monocyte subsets that increase the risk for severe Coronavirus Disease 2019 (COVID-19) and to intermediate lymphocyte developmental states that predispose to acute leukemia. Our approach not only provides a framework for enabling variant-to-function insights at single-cell resolution but also suggests a more general strategy for maximizing the inferences that can be made using single-cell genomic data.',\n",
       "  'doi': '10.1038/s41587-022-01341-y',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Lineage tracing reveals the phylodynamics, plasticity, and paths of tumor evolution.',\n",
       "  'authors': ['Dian Yang',\n",
       "   'Matthew G Jones',\n",
       "   'Santiago Naranjo',\n",
       "   'William M Rideout',\n",
       "   'Kyung Hoi Joseph Min',\n",
       "   'Raymond Ho',\n",
       "   'Wei Wu',\n",
       "   'Joseph M Replogle',\n",
       "   'Jennifer L Page',\n",
       "   'Jeffrey J Quinn',\n",
       "   'Felix Horns',\n",
       "   'Xiaojie Qiu',\n",
       "   'Michael Z Chen',\n",
       "   'William A Freed-Pastor',\n",
       "   'Christopher S McGinnis',\n",
       "   'David M Patterson',\n",
       "   'Zev J Gartner',\n",
       "   'Eric D Chow',\n",
       "   'Trever G Bivona',\n",
       "   'Michelle M Chan',\n",
       "   'Nir Yosef',\n",
       "   'Tyler Jacks',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '07/05/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35523183',\n",
       "  'abstract': 'Tumor evolution is driven by the progressive acquisition of genetic and epigenetic alterations that enable uncontrolled growth and expansion to neighboring and distal tissues. The study of phylogenetic relationships between cancer cells provides key insights into these processes. Here, we introduced an evolving lineage-tracing system with a single-cell RNA-seq readout into a mouse model of Kras;Trp53(KP)-driven lung adenocarcinoma and tracked tumor evolution from single-transformed cells to metastatic tumors at unprecedented resolution. We found that the loss of the initial, stable alveolar-type2-like state was accompanied by a transient increase in plasticity. This was followed by the adoption of distinct transcriptional programs that enable rapid expansion and, ultimately, clonal sweep of stable subclones capable of metastasizing. Finally, tumors develop through stereotypical evolutionary trajectories, and perturbing additional tumor suppressors accelerates progression by creating novel trajectories. Our study elucidates the hierarchical nature of tumor evolution and, more broadly, enables in-depth studies of tumor progression.',\n",
       "  'doi': '10.1016/j.cell.2022.04.015',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'OpenCell: Endogenous tagging for the cartography of human cellular organization.',\n",
       "  'authors': ['Nathan H Cho',\n",
       "   'Keith C Cheveralls',\n",
       "   'Andreas-David Brunner',\n",
       "   'Kibeom Kim',\n",
       "   'André C Michaelis',\n",
       "   'Preethi Raghavan',\n",
       "   'Hirofumi Kobayashi',\n",
       "   'Laura Savy',\n",
       "   'Jason Y Li',\n",
       "   'Hera Canaj',\n",
       "   'James Y S Kim',\n",
       "   'Edna M Stewart',\n",
       "   'Christian Gnann',\n",
       "   'Frank McCarthy',\n",
       "   'Joana P Cabrera',\n",
       "   'Rachel M Brunetti',\n",
       "   'Bryant B Chhun',\n",
       "   'Greg Dingle',\n",
       "   'Marco Y Hein',\n",
       "   'Bo Huang',\n",
       "   'Shalin B Mehta',\n",
       "   'Jonathan S Weissman',\n",
       "   'Rafael Gómez-Sjöberg',\n",
       "   'Daniel N Itzhak',\n",
       "   'Loïc A Royer',\n",
       "   'Matthias Mann',\n",
       "   'Manuel D Leonetti'],\n",
       "  'publication_date': '11/03/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35271311',\n",
       "  'abstract': 'Elucidating the wiring diagram of the human cell is a central goal of the postgenomic era. We combined genome engineering, confocal live-cell imaging, mass spectrometry, and data science to systematically map the localization and interactions of human proteins. Our approach provides a data-driven description of the molecular and spatial networks that organize the proteome. Unsupervised clustering of these networks delineates functional communities that facilitate biological discovery. We found that remarkably precise functional information can be derived from protein localization patterns, which often contain enough information to identify molecular interactions, and that RNA binding proteins form a specific subgroup defined by unique interaction and localization properties. Paired with a fully interactive website (opencell.czbiohub.org), our work constitutes a resource for the quantitative cartography of human cellular organization.',\n",
       "  'doi': '10.1126/science.abi6983',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Variant to function mapping at single-cell resolution through network propagation.',\n",
       "  'authors': ['Fulong Yu',\n",
       "   'Liam D Cato',\n",
       "   'Chen Weng',\n",
       "   'L Alexander Liggett',\n",
       "   'Soyoung Jeon',\n",
       "   'Keren Xu',\n",
       "   'Charleston W K Chiang',\n",
       "   'Joseph L Wiemels',\n",
       "   'Jonathan S Weissman',\n",
       "   'Adam J de Smith',\n",
       "   'Vijay G Sankaran'],\n",
       "  'publication_date': '05/02/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35118467',\n",
       "  'abstract': 'With burgeoning human disease genetic associations and single-cell genomic atlases covering a range of tissues, there are unprecedented opportunities to systematically gain insights into the mechanisms of disease-causal variation. However, sparsity and noise, particularly in the context of single-cell epigenomic data, hamper the identification of disease- or trait-relevant cell types, states, and trajectories. To overcome these challenges, we have developed the SCAVENGE method, which maps causal variants to their relevant cellular context at single-cell resolution by employing the strategy of network propagation. We demonstrate how SCAVENGE can help identify key biological mechanisms underlying human genetic variation including enrichment of blood traits at distinct stages of human hematopoiesis, defining monocyte subsets that increase the risk for severe coronavirus disease 2019 (COVID-19), and identifying intermediate lymphocyte developmental states that are critical for predisposition to acute leukemia. Our approach not only provides a framework for enabling variant-to-function insights at single-cell resolution, but also suggests a more general strategy for maximizing the inferences that can be made using single-cell genomic data.',\n",
       "  'doi': '10.1101/2022.01.23.477426',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Mapping transcriptomic vector fields of single cells.',\n",
       "  'authors': ['Xiaojie Qiu',\n",
       "   'Yan Zhang',\n",
       "   'Jorge D Martin-Rufino',\n",
       "   'Chen Weng',\n",
       "   'Shayan Hosseinzadeh',\n",
       "   'Dian Yang',\n",
       "   'Angela N Pogson',\n",
       "   'Marco Y Hein',\n",
       "   'Kyung Hoi Joseph Min',\n",
       "   'Li Wang',\n",
       "   'Emanuelle I Grody',\n",
       "   'Matthew J Shurtleff',\n",
       "   'Ruoshi Yuan',\n",
       "   'Song Xu',\n",
       "   'Yian Ma',\n",
       "   'Joseph M Replogle',\n",
       "   'Eric S Lander',\n",
       "   'Spyros Darmanis',\n",
       "   'Ivet Bahar',\n",
       "   'Vijay G Sankaran',\n",
       "   'Jianhua Xing',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '03/02/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35108499',\n",
       "  'abstract': \"Single-cell (sc)RNA-seq, together with RNA velocity and metabolic labeling, reveals cellular states and transitions at unprecedented resolution. Fully exploiting these data, however, requires kinetic models capable of unveiling governing regulatory functions. Here, we introduce an analytical framework dynamo (https://github.com/aristoteleo/dynamo-release), which infers absolute RNA velocity, reconstructs continuous vector fields that predict cell fates, employs differential geometry to extract underlying regulations, and ultimately predicts optimal reprogramming paths and perturbation outcomes. We highlight dynamo's power to overcome fundamental limitations of conventional splicing-based RNA velocity analyses to enable accurate velocity estimations on a metabolically labeled human hematopoiesis scRNA-seq dataset. Furthermore, differential geometry analyses reveal mechanisms driving early megakaryocyte appearance and elucidate asymmetrical regulation within the PU.1-GATA1 circuit. Leveraging the least-action-path method, dynamo accurately predicts drivers of numerous hematopoietic transitions. Finally, in silico perturbations predict cell-fate diversions induced by gene perturbations. Dynamo, thus, represents an important step in advancing quantitative and predictive theories of cell-state transitions.\",\n",
       "  'doi': '10.1016/j.cell.2021.12.045',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'BRD2 inhibition blocks SARS-CoV-2 infection by reducing transcription of the host cell receptor ACE2.',\n",
       "  'authors': ['Avi J Samelson',\n",
       "   'Quang Dinh Tran',\n",
       "   'Rémy Robinot',\n",
       "   'Lucia Carrau',\n",
       "   'Veronica V Rezelj',\n",
       "   'Alice Mac Kain',\n",
       "   'Merissa Chen',\n",
       "   'Gokul N Ramadoss',\n",
       "   'Xiaoyan Guo',\n",
       "   'Shion A Lim',\n",
       "   'Irene Lui',\n",
       "   'James K Nuñez',\n",
       "   'Sarah J Rockwood',\n",
       "   'Jianhui Wang',\n",
       "   'Na Liu',\n",
       "   'Jared Carlson-Stevermer',\n",
       "   'Jennifer Oki',\n",
       "   'Travis Maures',\n",
       "   'Kevin Holden',\n",
       "   'Jonathan S Weissman',\n",
       "   'James A Wells',\n",
       "   'Bruce R Conklin',\n",
       "   'Benjamin R TenOever',\n",
       "   'Lisa A Chakrabarti',\n",
       "   'Marco Vignuzzi',\n",
       "   'Ruilin Tian',\n",
       "   'Martin Kampmann'],\n",
       "  'publication_date': '15/01/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35027731',\n",
       "  'abstract': 'SARS-CoV-2 infection of human cells is initiated by the binding of the viral Spike protein to its cell-surface receptor ACE2. We conducted a targeted CRISPRi screen to uncover druggable pathways controlling Spike protein binding to human cells. Here we show that the protein BRD2 is required for ACE2 transcription in human lung epithelial cells and cardiomyocytes, and BRD2 inhibitors currently evaluated in clinical trials potently block endogenous ACE2 expression and SARS-CoV-2 infection of human cells, including those of human nasal epithelia. Moreover, pharmacological BRD2 inhibition with the drug ABBV-744 inhibited SARS-CoV-2 replication in Syrian hamsters. We also found that BRD2 controls transcription of several other genes induced upon SARS-CoV-2 infection, including the interferon response, which in turn regulates the antiviral response. Together, our results pinpoint BRD2 as a potent and essential regulator of the host response to SARS-CoV-2 infection and highlight the potential of BRD2 as a therapeutic target for COVID-19.',\n",
       "  'doi': '10.1038/s41556-021-00821-8',\n",
       "  'journal': 'Nature cell biology'},\n",
       " {'title': 'Peroxisome function relies on organelle-associated mRNA translation.',\n",
       "  'authors': ['Noa Dahan',\n",
       "   'Yury S Bykov',\n",
       "   'Elizabeth A Boydston',\n",
       "   'Amir Fadel',\n",
       "   'Zohar Gazi',\n",
       "   'Hodaya Hochberg-Laufer',\n",
       "   'James Martenson',\n",
       "   'Vlad Denic',\n",
       "   'Yaron Shav-Tal',\n",
       "   'Jonathan S Weissman',\n",
       "   'Naama Aviram',\n",
       "   'Einat Zalckvar',\n",
       "   'Maya Schuldiner'],\n",
       "  'publication_date': '13/01/22',\n",
       "  'publication_year': '2022',\n",
       "  'pubmed_id': '35020435',\n",
       "  'abstract': '[Figure: see text].',\n",
       "  'doi': '10.1126/sciadv.abk2141',\n",
       "  'journal': 'Science advances'},\n",
       " {'title': 'The dark proteome: translation from noncanonical open reading frames.',\n",
       "  'authors': ['Bradley W Wright',\n",
       "   'Zixin Yi',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jin Chen'],\n",
       "  'publication_date': '01/12/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34844857',\n",
       "  'abstract': 'Omics-based technologies have revolutionized our understanding of the coding potential of the genome. In particular, these studies revealed widespread unannotated open reading frames (ORFs) throughout genomes and that these regions have the potential to encode novel functional (micro-)proteins and/or hold regulatory roles. However, despite their genomic prevalence, relatively few of these noncanonical ORFs have been functionally characterized, likely in part due to their under-recognition by the broader scientific community. The few that have been investigated in detail have demonstrated their essentiality in critical and divergent biological processes. As such, here we aim to discuss recent advances in understanding the diversity of noncanonical ORFs and their roles, as well as detail biologically important examples within the context of the mammalian genome.',\n",
       "  'doi': '10.1016/j.tcb.2021.10.010',\n",
       "  'journal': 'Trends in cell biology'},\n",
       " {'title': 'Functional single-cell genomics of human cytomegalovirus infection.',\n",
       "  'authors': ['Marco Y Hein', 'Jonathan S Weissman'],\n",
       "  'publication_date': '27/10/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34697476',\n",
       "  'abstract': 'Understanding how viral and host factors interact and how perturbations impact infection is the basis for designing antiviral interventions. Here we define the functional contribution of each viral and host factor involved in human cytomegalovirus infection in primary human fibroblasts through pooled CRISPR interference and nuclease screening. To determine how genetic perturbation of critical host and viral factors alters the timing, course and progression of infection, we applied Perturb-seq to record the transcriptomes of tens of thousands of CRISPR-modified single cells and found that, normally, most cells follow a stereotypical transcriptional trajectory. Perturbing critical host factors does not change the stereotypical transcriptional trajectory per se but can stall, delay or accelerate progression along the trajectory, allowing one to pinpoint the stage of infection at which host factors act. Conversely, perturbation of viral factors can create distinct, abortive trajectories. Our results reveal the roles of host and viral factors and provide a roadmap for the dissection of host-pathogen interactions.',\n",
       "  'doi': '10.1038/s41587-021-01059-3',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Mapping the genetic landscape of DNA double-strand break repair.',\n",
       "  'authors': ['Jeffrey A Hussmann',\n",
       "   'Jia Ling',\n",
       "   'Purnima Ravisankar',\n",
       "   'Jun Yan',\n",
       "   'Ann Cirincione',\n",
       "   'Albert Xu',\n",
       "   'Danny Simpson',\n",
       "   'Dian Yang',\n",
       "   'Anne Bothmer',\n",
       "   'Cecilia Cotta-Ramusino',\n",
       "   'Jonathan S Weissman',\n",
       "   'Britt Adamson'],\n",
       "  'publication_date': '22/10/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34672952',\n",
       "  'abstract': 'Cells repair DNA double-strand breaks (DSBs) through a complex set of pathways critical for maintaining genomic integrity. To systematically map these pathways, we developed a high-throughput screening approach called Repair-seq that measures the effects of thousands of genetic perturbations on mutations introduced at targeted DNA lesions. Using Repair-seq, we profiled DSB repair products induced by two programmable nucleases (Cas9 and Cas12a) in the presence or absence of oligonucleotides for homology-directed repair (HDR) after knockdown of 476 genes involved in DSB repair or associated processes. The resulting data enabled principled, data-driven inference of DSB end joining and HDR pathways. Systematic interrogation of this data uncovered unexpected relationships among DSB repair genes and demonstrated that repair outcomes with superficially similar sequence architectures can have markedly different genetic dependencies. This work provides a foundation for mapping DNA repair pathways and for optimizing genome editing across diverse modalities.',\n",
       "  'doi': '10.1016/j.cell.2021.10.002',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Enhanced prime editing systems by manipulating cellular determinants of editing outcomes.',\n",
       "  'authors': ['Peter J Chen',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Jun Yan',\n",
       "   'Friederike Knipping',\n",
       "   'Purnima Ravisankar',\n",
       "   'Pin-Fang Chen',\n",
       "   'Cidi Chen',\n",
       "   'James W Nelson',\n",
       "   'Gregory A Newby',\n",
       "   'Mustafa Sahin',\n",
       "   'Mark J Osborn',\n",
       "   'Jonathan S Weissman',\n",
       "   'Britt Adamson',\n",
       "   'David R Liu'],\n",
       "  'publication_date': '16/10/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34653350',\n",
       "  'abstract': 'While prime editing enables precise sequence changes in DNA, cellular determinants of prime editing remain poorly understood. Using pooled CRISPRi screens, we discovered that DNA mismatch repair (MMR) impedes prime editing and promotes undesired indel byproducts. We developed PE4 and PE5 prime editing systems in which transient expression of an engineered MMR-inhibiting protein enhances the efficiency of substitution, small insertion, and small deletion prime edits by an average 7.7-fold and 2.0-fold compared to PE2 and PE3 systems, respectively, while improving edit/indel ratios by 3.4-fold in MMR-proficient cell types. Strategic installation of silent mutations near the intended edit can enhance prime editing outcomes by evading MMR. Prime editor protein optimization resulted in a PEmax architecture that enhances editing efficacy by 2.8-fold on average in HeLa cells. These findings enrich our understanding of prime editing and establish prime editing systems that show substantial improvement across 191 edits in seven mammalian cell types.',\n",
       "  'doi': '10.1016/j.cell.2021.09.018',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Structured elements drive extensive circular RNA translation.',\n",
       "  'authors': ['Chun-Kan Chen',\n",
       "   'Ran Cheng',\n",
       "   'Janos Demeter',\n",
       "   'Jin Chen',\n",
       "   'Shira Weingarten-Gabbay',\n",
       "   'Lihua Jiang',\n",
       "   'Michael P Snyder',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eran Segal',\n",
       "   'Peter K Jackson',\n",
       "   'Howard Y Chang'],\n",
       "  'publication_date': '27/08/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34437836',\n",
       "  'abstract': \"The human genome encodes tens of thousands circular RNAs (circRNAs) with mostly unknown functions. Circular RNAs require internal ribosome entry sites (IRES) if they are to undergo translation without a 5' cap. Here, we develop a high-throughput screen to systematically discover RNA sequences that can direct circRNA translation in human cells. We identify more than 17,000 endogenous and synthetic sequences as candidate circRNA IRES. 18S rRNA complementarity and a structured RNA element positioned on the IRES are important for driving circRNA translation. Ribosome profiling and peptidomic analyses show extensive IRES-ribosome association, hundreds of circRNA-encoded proteins with tissue-specific distribution, and antigen presentation. We find that circFGFR1p, a protein encoded by circFGFR1 that is downregulated in cancer, functions as a negative regulator of FGFR1 oncoprotein to suppress cell growth during stress. Systematic identification of circRNA IRES elements may provide important links among circRNA regulation, biological function, and disease.\",\n",
       "  'doi': '10.1016/j.molcel.2021.07.042',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'An engineered transcriptional reporter of protein localization identifies regulators of mitochondrial and ER membrane protein trafficking in high-throughput CRISPRi screens.',\n",
       "  'authors': ['Robert Coukos',\n",
       "   'David Yao',\n",
       "   'Mateo I Sanchez',\n",
       "   'Eric T Strand',\n",
       "   'Meagan E Olive',\n",
       "   'Namrata D Udeshi',\n",
       "   'Jonathan S Weissman',\n",
       "   'Steven A Carr',\n",
       "   'Michael C Bassik',\n",
       "   'Alice Y Ting'],\n",
       "  'publication_date': '21/08/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34414886',\n",
       "  'abstract': 'The trafficking of specific protein cohorts to correct subcellular locations at correct times is essential for every signaling and regulatory process in biology. Gene perturbation screens could provide a powerful approach to probe the molecular mechanisms of protein trafficking, but only if protein localization or mislocalization can be tied to a simple and robust phenotype for cell selection, such as cell proliferation or fluorescence-activated cell sorting (FACS). To empower the study of protein trafficking processes with gene perturbation, we developed a genetically encoded molecular tool named HiLITR (High-throughput Localization Indicator with Transcriptional Readout). HiLITR converts protein colocalization into proteolytic release of a membrane-anchored transcription factor, which drives the expression of a chosen reporter gene. Using HiLITR in combination with FACS-based CRISPRi screening in human cell lines, we identified genes that influence the trafficking of mitochondrial and ER tail-anchored proteins. We show that loss of the SUMO E1 component SAE1 results in mislocalization and destabilization of many mitochondrial tail-anchored proteins. We also demonstrate a distinct regulatory role for EMC10 in the ER membrane complex, opposing the transmembrane-domain insertion activity of the complex. Through transcriptional integration of complex cellular functions, HiLITR expands the scope of biological processes that can be studied by genetic perturbation screening technologies.',\n",
       "  'doi': '10.7554/eLife.69142',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Efficient C•G-to-G•C base editors developed using CRISPRi screens, target-library analysis, and machine learning.',\n",
       "  'authors': ['Luke W Koblan',\n",
       "   'Mandana Arbab',\n",
       "   'Max W Shen',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Andrew V Anzalone',\n",
       "   'Jordan L Doman',\n",
       "   'Gregory A Newby',\n",
       "   'Dian Yang',\n",
       "   'Beverly Mok',\n",
       "   'Joseph M Replogle',\n",
       "   'Albert Xu',\n",
       "   'Tyler A Sisley',\n",
       "   'Jonathan S Weissman',\n",
       "   'Britt Adamson',\n",
       "   'David R Liu'],\n",
       "  'publication_date': '30/06/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34183861',\n",
       "  'abstract': 'Programmable C•G-to-G•C base editors (CGBEs) have broad scientific and therapeutic potential, but their editing outcomes have proved difficult to predict and their editing efficiency and product purity are often low. We describe a suite of engineered CGBEs paired with machine learning models to enable efficient, high-purity C•G-to-G•C base editing. We performed a CRISPR interference (CRISPRi) screen targeting DNA repair genes to identify factors that affect C•G-to-G•C editing outcomes and used these insights to develop CGBEs with diverse editing profiles. We characterized ten promising CGBEs on a library of 10,638\\u2009genomically integrated target sites in mammalian cells and trained machine learning models that accurately predict the purity and yield of editing outcomes (R\\u2009=\\u20090.90) using these data. These CGBEs enable correction to the wild-type coding sequence of 546\\u2009disease-related transversion single-nucleotide variants (SNVs) with\\u2009>90% precision (mean 96%) and up to 70% efficiency (mean 14%). Computational prediction of optimal CGBE-single-guide RNA pairs enables high-purity transversion base editing at over fourfold more target sites than achieved using any single CGBE variant.',\n",
       "  'doi': '10.1038/s41587-021-00938-z',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Genome-wide CRISPRi screening identifies OCIAD1 as a prohibitin client and regulatory determinant of mitochondrial Complex III assembly in human cells.',\n",
       "  'authors': ['Maxence Le Vasseur',\n",
       "   'Jonathan Friedman',\n",
       "   'Marco Jost',\n",
       "   'Jiawei Xu',\n",
       "   'Justin Yamada',\n",
       "   'Martin Kampmann',\n",
       "   'Max A Horlbeck',\n",
       "   'Michelle R Salemi',\n",
       "   'Brett S Phinney',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jodi Nunnari'],\n",
       "  'publication_date': '27/05/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '34034859',\n",
       "  'abstract': 'Dysfunction of the mitochondrial electron transport chain (mETC) is a major cause of human mitochondrial diseases. To identify determinants of mETC function, we screened a genome-wide human CRISPRi library under oxidative metabolic conditions with selective inhibition of mitochondrial Complex III and identified ovarian carcinoma immunoreactive antigen (OCIA) domain-containing protein 1 (OCIAD1) as a Complex III assembly factor. We find that OCIAD1 is an inner mitochondrial membrane protein that forms a complex with supramolecular prohibitin assemblies. Our data indicate that OCIAD1 is required for maintenance of normal steady-state levels of Complex III and the proteolytic processing of the catalytic subunit cytochrome ',\n",
       "  'doi': '10.7554/eLife.67624',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'CRISPR-based functional genomics in human dendritic cells.',\n",
       "  'authors': ['Marco Jost',\n",
       "   'Amy N Jacobson',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Giana Cirolia',\n",
       "   'Michael A Fischbach',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/04/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33904395',\n",
       "  'abstract': 'Dendritic cells (DCs) regulate processes ranging from antitumor and antiviral immunity to host-microbe communication at mucosal surfaces. It remains difficult, however, to genetically manipulate human DCs, limiting our ability to probe how DCs elicit specific immune responses. Here, we develop a CRISPR-Cas9 genome editing method for human monocyte-derived DCs (moDCs) that mediates knockouts with a median efficiency of >94% across >300 genes. Using this method, we perform genetic screens in moDCs, identifying mechanisms by which DCs tune responses to lipopolysaccharides from the human microbiome. In addition, we reveal donor-specific responses to lipopolysaccharides, underscoring the importance of assessing immune phenotypes in donor-derived cells, and identify candidate genes that control this specificity, highlighting the potential of our method to pinpoint determinants of inter-individual variation in immunity. Our work sets the stage for a systematic dissection of the immune signaling at the host-microbiome interface and for targeted engineering of DCs for neoantigen vaccination.',\n",
       "  'doi': '10.7554/eLife.65856',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Genome-wide programmable transcriptional memory by CRISPR-based epigenome editing.',\n",
       "  'authors': ['James K Nuñez',\n",
       "   'Jin Chen',\n",
       "   'Greg C Pommier',\n",
       "   'J Zachery Cogan',\n",
       "   'Joseph M Replogle',\n",
       "   'Carmen Adriaens',\n",
       "   'Gokul N Ramadoss',\n",
       "   'Quanming Shi',\n",
       "   'King L Hung',\n",
       "   'Avi J Samelson',\n",
       "   'Angela N Pogson',\n",
       "   'James Y S Kim',\n",
       "   'Amanda Chung',\n",
       "   'Manuel D Leonetti',\n",
       "   'Howard Y Chang',\n",
       "   'Martin Kampmann',\n",
       "   'Bradley E Bernstein',\n",
       "   'Volker Hovestadt',\n",
       "   'Luke A Gilbert',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '11/04/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33838111',\n",
       "  'abstract': 'A general approach for heritably altering gene expression has the potential to enable many discovery and therapeutic efforts. Here, we present CRISPRoff-a programmable epigenetic memory writer consisting of a single dead Cas9 fusion protein that establishes DNA methylation and repressive histone modifications. Transient CRISPRoff expression initiates highly specific DNA methylation and gene repression that is maintained through cell division and differentiation of stem cells to neurons. Pairing CRISPRoff with genome-wide screens and analysis of chromatin marks establishes rules for heritable gene silencing. We identify single guide RNAs (sgRNAs) capable of silencing the large majority of genes including those lacking canonical CpG islands (CGIs) and reveal a wide targeting window extending beyond annotated CGIs. The broad ability of CRISPRoff to initiate heritable gene silencing even outside of CGIs expands the canonical model of methylation-based silencing and enables diverse applications including genome-wide screens, multiplexed cell engineering, enhancer silencing, and mechanistic exploration of epigenetic inheritance.',\n",
       "  'doi': '10.1016/j.cell.2021.03.025',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Regulation of translation by methylation multiplicity of 18S rRNA.',\n",
       "  'authors': ['Kuanqing Liu',\n",
       "   'Daniel A Santos',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Yun Wang',\n",
       "   'Benjamin M Sutter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Benjamin P Tu'],\n",
       "  'publication_date': '11/03/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33691096',\n",
       "  'abstract': 'N',\n",
       "  'doi': '10.1016/j.celrep.2021.108825',\n",
       "  'journal': 'Cell reports'},\n",
       " {'title': 'Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression.',\n",
       "  'authors': ['Lei S Qi',\n",
       "   'Matthew H Larson',\n",
       "   'Luke A Gilbert',\n",
       "   'Jennifer A Doudna',\n",
       "   'Jonathan S Weissman',\n",
       "   'Adam P Arkin',\n",
       "   'Wendell A Lim'],\n",
       "  'publication_date': '06/02/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33545038',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1016/j.cell.2021.01.019',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'BRD2 inhibition blocks SARS-CoV-2 infection in vitro by reducing transcription of the host cell receptor ACE2.',\n",
       "  'authors': ['Ruilin Tian',\n",
       "   'Avi J Samelson',\n",
       "   'Veronica V Rezelj',\n",
       "   'Merissa Chen',\n",
       "   'Gokul N Ramadoss',\n",
       "   'Xiaoyan Guo',\n",
       "   'Alice Mac Kain',\n",
       "   'Quang Dinh Tran',\n",
       "   'Shion A Lim',\n",
       "   'Irene Lui',\n",
       "   'James Nunez',\n",
       "   'Sarah J Rockwood',\n",
       "   'Na Liu',\n",
       "   'Jared Carlson-Stevermer',\n",
       "   'Jennifer Oki',\n",
       "   'Travis Maures',\n",
       "   'Kevin Holden',\n",
       "   'Jonathan S Weissman',\n",
       "   'James A Wells',\n",
       "   'Bruce Conklin',\n",
       "   'Marco Vignuzzi',\n",
       "   'Martin Kampmann'],\n",
       "  'publication_date': '28/01/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33501440',\n",
       "  'abstract': 'SARS-CoV-2 infection of human cells is initiated by the binding of the viral Spike protein to its cell-surface receptor ACE2. We conducted an unbiased CRISPRi screen to uncover druggable pathways controlling Spike protein binding to human cells. We found that the protein BRD2 is an essential node in the cellular response to SARS-CoV-2 infection. BRD2 is required for ACE2 transcription in human lung epithelial cells and cardiomyocytes, and BRD2 inhibitors currently evaluated in clinical trials potently block endogenous ACE2 expression and SARS-CoV-2 infection of human cells. BRD2 also controls transcription of several other genes induced upon SARS-CoV-2 infection, including the interferon response, which in turn regulates ACE2 levels. It is possible that the previously reported interaction between the viral E protein and BRD2 evolved to manipulate the transcriptional host response during SARS-CoV-2 infection. Together, our results pinpoint BRD2 as a potent and essential regulator of the host response to SARS-CoV-2 infection and highlight the potential of BRD2 as a novel therapeutic target for COVID-19.',\n",
       "  'doi': '10.1101/2021.01.19.427194',\n",
       "  'journal': 'biorxiv'},\n",
       " {'title': 'Single-cell lineages reveal the rates, routes, and drivers of metastasis in cancer xenografts.',\n",
       "  'authors': ['Jeffrey J Quinn',\n",
       "   'Matthew G Jones',\n",
       "   'Ross A Okimoto',\n",
       "   'Shigeki Nanjo',\n",
       "   'Michelle M Chan',\n",
       "   'Nir Yosef',\n",
       "   'Trever G Bivona',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '23/01/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33479121',\n",
       "  'abstract': 'Detailed phylogenies of tumor populations can recount the history and chronology of critical events during cancer progression, such as metastatic dissemination. We applied a Cas9-based, single-cell lineage tracer to study the rates, routes, and drivers of metastasis in a lung cancer xenograft mouse model. We report deeply resolved phylogenies for tens of thousands of cancer cells traced over months of growth and dissemination. This revealed stark heterogeneity in metastatic capacity, arising from preexisting and heritable differences in gene expression. We demonstrate that these identified genes can drive invasiveness and uncovered an unanticipated suppressive role for ',\n",
       "  'doi': '10.1126/science.abc1944',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'High-content imaging-based pooled CRISPR screens in mammalian cells.',\n",
       "  'authors': ['Xiaowei Yan',\n",
       "   'Nico Stuurman',\n",
       "   'Susana A Ribeiro',\n",
       "   'Marvin E Tanenbaum',\n",
       "   'Max A Horlbeck',\n",
       "   'Christina R Liem',\n",
       "   'Marco Jost',\n",
       "   'Jonathan S Weissman',\n",
       "   'Ronald D Vale'],\n",
       "  'publication_date': '20/01/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33465779',\n",
       "  'abstract': 'CRISPR (clustered regularly interspaced short palindromic repeats)-based gene inactivation provides a powerful means for linking genes to particular cellular phenotypes. CRISPR-based screening typically uses large genomic pools of single guide RNAs (sgRNAs). However, this approach is limited to phenotypes that can be enriched by chemical selection or FACS sorting. Here, we developed a microscopy-based approach, which we name optical enrichment, to select cells displaying a particular CRISPR-induced phenotype by automated imaging-based computation, mark them by photoactivation of an expressed photoactivatable fluorescent protein, and then isolate the fluorescent cells using fluorescence-activated cell sorting (FACS). A plugin was developed for the open source software μManager to automate the phenotypic identification and photoactivation of cells, allowing ∼1.5 million individual cells to be screened in 8 h. We used this approach to screen 6,092 sgRNAs targeting 544 genes for their effects on nuclear size regulation and identified 14 bona fide hits. These results present a scalable approach to facilitate imaging-based pooled CRISPR screens.',\n",
       "  'doi': '10.1083/jcb.202008158',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'A CRISPR/Cas9-Engineered ARID1A-Deficient Human Gastric Cancer Organoid Model Reveals Essential and Nonessential Modes of Oncogenic Transformation',\n",
       "  'authors': ['Yuan-Hung Lo',\n",
       "   'Kevin S Kolahi',\n",
       "   'Yuhong Du',\n",
       "   'Chiung-Ying Chang',\n",
       "   'Andrey Krokhotin',\n",
       "   'Ajay Nair',\n",
       "   'Walter D Sobba',\n",
       "   'Kasper Karlsson',\n",
       "   'Sunny J Jones',\n",
       "   'Teri A Longacre',\n",
       "   'Amanda T Mah',\n",
       "   'Bahar Tercan',\n",
       "   'Alexandra Sockell',\n",
       "   'Hang Xu',\n",
       "   'Jose A Seoane',\n",
       "   'Jin Chen',\n",
       "   'Ilya Shmulevich',\n",
       "   'Jonathan S Weissman',\n",
       "   'Christina Curtis',\n",
       "   'Andrea Califano',\n",
       "   'Haian Fu',\n",
       "   'Gerald R Crabtree',\n",
       "   'Calvin J Kuo'],\n",
       "  'publication_date': '17/01/21',\n",
       "  'publication_year': '2021',\n",
       "  'pubmed_id': '33451982',\n",
       "  'abstract': 'Mutations in ',\n",
       "  'doi': '10.1158/2159-8290.CD-20-1109',\n",
       "  'journal': 'Cancer discovery'},\n",
       " {'title': 'Genome-Scale Perturbation of Long Noncoding RNA Expression Using CRISPR Interference.',\n",
       "  'authors': ['S John Liu',\n",
       "   'Max A Horlbeck',\n",
       "   'Jonathan S Weissman',\n",
       "   'Daniel A Lim'],\n",
       "  'publication_date': '17/12/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '33326085',\n",
       "  'abstract': 'CRISPR-mediated interference (CRISPRi), a robust and specific system for programmably repressing transcription, provides a versatile tool for systematically characterizing the function of long noncoding RNAs (lncRNAs). When used with highly parallel, lentiviral pooled screening approaches, CRISPRi enables the targeted knockdown of tens of thousands of lncRNA-expressing loci in a single screen. Here we describe the use of CRISPRi to target lncRNA loci in a pooled screen, using cell growth and proliferation as an example of a phenotypic readout. Considerations for custom lncRNA-targeting libraries, alternative phenotypic readouts, and orthogonal validation approaches are also discussed.',\n",
       "  'doi': '10.1007/978-1-0716-1158-6_20',\n",
       "  'journal': 'Methods in molecular biology'},\n",
       " {'title': 'Structural and mechanistic basis of the EMC-dependent biogenesis of distinct transmembrane clients.',\n",
       "  'authors': ['Lakshmi E Miller-Vedam',\n",
       "   'Bastian Bräuning',\n",
       "   'Katerina D Popova',\n",
       "   'Nicole T Schirle Oakdale',\n",
       "   'Jessica L Bonnar',\n",
       "   'Jesuraj R Prabu',\n",
       "   'Elizabeth A Boydston',\n",
       "   'Natalia Sevillano',\n",
       "   'Matthew J Shurtleff',\n",
       "   'Robert M Stroud',\n",
       "   'Charles S Craik',\n",
       "   'Brenda A Schulman',\n",
       "   'Adam Frost',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '26/11/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '33236988',\n",
       "  'abstract': \"Membrane protein biogenesis in the endoplasmic reticulum (ER) is complex and failure-prone. The ER membrane protein complex (EMC), comprising eight conserved subunits, has emerged as a central player in this process. Yet, we have limited understanding of how EMC enables insertion and integrity of diverse clients, from tail-anchored to polytopic transmembrane proteins. Here, yeast and human EMC cryo-EM structures reveal conserved intricate assemblies and human-specific features associated with pathologies. Structure-based functional studies distinguish between two separable EMC activities, as an insertase regulating tail-anchored protein levels and a broader role in polytopic membrane protein biogenesis. These depend on mechanistically coupled yet spatially distinct regions including two lipid-accessible membrane cavities which confer client-specific regulation, and a non-insertase EMC function mediated by the EMC lumenal domain. Our studies illuminate the structural and mechanistic basis of EMC's multifunctionality and point to its role in differentially regulating the biogenesis of distinct client protein classes.\\nCells are surrounded and contained by a plasma membrane consisting of a double layer of fats and proteins. These proteins monitor and facilitate the movement of food, oxygen and messages in and out of the cell, and help neighboring cells communicate. Membrane proteins are manufactured in a cell compartment called the endoplasmic reticulum. Cellular machines called ribosomes visit this compartment’s membrane to manufacture proteins that need to be secreted or embedded into the cell’s membranes. As these proteins are made, they are pulled into the endoplasmic reticulum so they can be folded correctly and inserted in the membrane. A cellular machine in this compartment’s membrane that aids this process is the endoplasmic reticulum membrane protein complex (EMC). Many steps can go wrong during protein assembly, so to control protein quality, the EMC has to accommodate the variety of complex physical features that proteins can have. To explore the activity of the EMC, Miller-Vedam, Bräuning, Popova et al. studied the normal structure of the EMC in both yeast and human cells grown in the lab. These snapshots of the complex in different species had a lot in common, including how the complex was arranged within and around the membrane. Next, Miller-Vedam, Bräuning, Popova et al. generated 50 mutant versions of the EMC in human cells to determine how changing different parts of the complex affected the production of three proteins that rely on the EMC to fold correctly. These proteins were an enzyme called squalene synthase, a signaling protein called the beta adrenergic receptor and sigma intracellular receptor 2, a protein involved in the regulation of cholesterol levels. Mutations in the section of the EMC outside of the endoplasmic reticulum, within the main cellular compartment, negatively impacted the stability of squalene synthase. This section of the EMC provides a platform where proteins can associate before entering the membrane. The part of EMC that spans the membrane contains both a fat-filled cavity and a cavity with a ‘door’ that is either open or closed. Mutations in this section disrupted the insertion of both squalene synthase and the beta adrenergic receptor into the membrane, a role performed by the cavity with the door. The specific role of the fat-filled cavity is still not fully understood, but a mutation affecting this cavity disrupts the correct production of all three proteins studied. The largest section of the complex, which sits inside the endoplasmic reticulum, protected proteins as they folded, ensuring they were not destroyed for being folded incorrectly before they were fully formed. Mutations in this part of the EMC negatively impacted the stability of sigma intracellular receptor 2 without negatively affecting the other proteins. This molecular dissection of the activity of the EMC provides insights into how membrane proteins are manufactured, stabilized, coordinated, and monitored for quality. These findings could contribute towards the development of new treatments for certain congenital diseases. For example, cystic fibrosis, retinitis pigmentosa, and Charcot-Marie-Tooth disease are all thought to be caused by mutations within membrane proteins that require the EMC during their production.\",\n",
       "  'doi': '10.7554/eLife.62611',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Mismatch-CRISPRi Reveals the Co-varying Expression-Fitness Relationships of Essential Genes in Escherichia coli and Bacillus subtilis.',\n",
       "  'authors': ['John S Hawkins',\n",
       "   'Melanie R Silvis',\n",
       "   'Byoung-Mo Koo',\n",
       "   'Jason M Peters',\n",
       "   'Hendrik Osadnik',\n",
       "   'Marco Jost',\n",
       "   'Cameron C Hearne',\n",
       "   'Jonathan S Weissman',\n",
       "   'Horia Todor',\n",
       "   'Carol A Gross'],\n",
       "  'publication_date': '21/10/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '33080209',\n",
       "  'abstract': 'Essential genes are the hubs of cellular networks, but lack of high-throughput methods for titrating gene expression has limited our understanding of the fitness landscapes against which their expression levels are optimized. We developed a modified CRISPRi system leveraging the predictable reduction in efficacy of imperfectly matched sgRNAs to generate defined levels of CRISPRi activity and demonstrated its broad applicability. Using libraries of mismatched sgRNAs predicted to span the full range of knockdown levels, we characterized the expression-fitness relationships of most essential genes in Escherichia coli and Bacillus subtilis. We find that these relationships vary widely from linear to bimodal but are similar within pathways. Notably, despite ∼2 billion years of evolutionary separation between E.\\xa0coli and B.\\xa0subtilis, most essential homologs have similar expression-fitness relationships with rare but informative differences. Thus, the expression levels of essential genes may reflect homeostatic or evolutionary constraints shared between the two organisms.',\n",
       "  'doi': '10.1016/j.cels.2020.09.009',\n",
       "  'journal': 'Cell systems'},\n",
       " {'title': 'Conserved Functions of Ether Lipids and Sphingolipids in the Early Secretory Pathway.',\n",
       "  'authors': ['Noemi Jiménez-Rojo',\n",
       "   'Manuel D Leonetti',\n",
       "   'Valeria Zoni',\n",
       "   'Adai Colom',\n",
       "   'Suihan Feng',\n",
       "   'Namrata R Iyengar',\n",
       "   'Stefan Matile',\n",
       "   'Aurélien Roux',\n",
       "   'Stefano Vanni',\n",
       "   'Jonathan S Weissman',\n",
       "   'Howard Riezman'],\n",
       "  'publication_date': '29/08/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32857977',\n",
       "  'abstract': 'Sphingolipids play important roles in physiology and cell biology, but a systematic examination of their functions is lacking. We performed a genome-wide CRISPRi screen in sphingolipid-depleted human cells and identified hypersensitive mutants in genes of membrane trafficking and lipid biosynthesis, including ether lipid synthesis. Systematic lipidomic analysis showed a coordinate regulation of ether lipids with sphingolipids, suggesting an adaptation and functional compensation. Biophysical experiments on model membranes show common properties of these structurally diverse lipids that also share a known function as glycosylphosphatidylinositol (GPI) anchors in different kingdoms of life. Molecular dynamics simulations show a selective enrichment of ether phosphatidylcholine around p24 proteins, which are receptors for the export of GPI-anchored proteins and have been shown to bind a specific sphingomyelin species. Our results support a model of convergent evolution of proteins and lipids, based on their physico-chemical properties, to regulate GPI-anchored protein transport and maintain homeostasis in the early secretory pathway.',\n",
       "  'doi': '10.1016/j.cub.2020.07.059',\n",
       "  'journal': 'Current biology'},\n",
       " {'title': 'Therapy-Induced Evolution of Human Lung Cancer Revealed by Single-Cell RNA Sequencing.',\n",
       "  'authors': ['Ashley Maynard',\n",
       "   'Caroline E McCoach',\n",
       "   'Julia K Rotow',\n",
       "   'Lincoln Harris',\n",
       "   'Franziska Haderk',\n",
       "   'D Lucas Kerr',\n",
       "   'Elizabeth A Yu',\n",
       "   'Erin L Schenk',\n",
       "   'Weilun Tan',\n",
       "   'Alexander Zee',\n",
       "   'Michelle Tan',\n",
       "   'Philippe Gui',\n",
       "   'Tasha Lea',\n",
       "   'Wei Wu',\n",
       "   'Anatoly Urisman',\n",
       "   'Kirk Jones',\n",
       "   'Rene Sit',\n",
       "   'Pallav K Kolli',\n",
       "   'Eric Seeley',\n",
       "   'Yaron Gesthalter',\n",
       "   'Daniel D Le',\n",
       "   'Kevin A Yamauchi',\n",
       "   'David M Naeger',\n",
       "   'Sourav Bandyopadhyay',\n",
       "   'Khyati Shah',\n",
       "   'Lauren Cech',\n",
       "   'Nicholas J Thomas',\n",
       "   'Anshal Gupta',\n",
       "   'Mayra Gonzalez',\n",
       "   'Hien Do',\n",
       "   'Lisa Tan',\n",
       "   'Bianca Bacaltos',\n",
       "   'Rafael Gomez-Sjoberg',\n",
       "   'Matthew Gubens',\n",
       "   'Thierry Jahan',\n",
       "   'Johannes R Kratz',\n",
       "   'David Jablons',\n",
       "   'Norma Neff',\n",
       "   'Robert C Doebele',\n",
       "   'Jonathan Weissman',\n",
       "   'Collin M Blakely',\n",
       "   'Spyros Darmanis',\n",
       "   'Trever G Bivona'],\n",
       "  'publication_date': '22/08/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32822576',\n",
       "  'abstract': 'Lung cancer, the leading cause of cancer mortality, exhibits heterogeneity that enables adaptability, limits therapeutic success, and remains incompletely understood. Single-cell RNA sequencing (scRNA-seq) of metastatic lung cancer was performed using 49 clinical biopsies obtained from 30 patients before and during targeted therapy. Over 20,000 cancer and tumor microenvironment (TME) single-cell profiles exposed a rich and dynamic tumor ecosystem. scRNA-seq of cancer cells illuminated targetable oncogenes beyond those detected clinically. Cancer cells surviving therapy as residual disease (RD) expressed an alveolar-regenerative cell signature suggesting a therapy-induced primitive cell-state transition, whereas those present at on-therapy progressive disease (PD) upregulated kynurenine, plasminogen, and gap-junction pathways. Active T-lymphocytes and decreased macrophages were present at RD and immunosuppressive cell states characterized PD. Biological features revealed by scRNA-seq were biomarkers of clinical outcomes in independent cohorts. This study highlights how therapy-induced adaptation of the multi-cellular ecosystem of metastatic cancer shapes clinical outcomes.',\n",
       "  'doi': '10.1016/j.cell.2020.07.017',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Life span extension by glucose restriction is abrogated by methionine supplementation: Cross-talk between glucose and methionine and implication of methionine as a key regulator of life span.',\n",
       "  'authors': ['Ke Zou',\n",
       "   'Silvia Rouskin',\n",
       "   'Kevin Dervishi',\n",
       "   'Mark A McCormick',\n",
       "   'Arjun Sasikumar',\n",
       "   'Changhui Deng',\n",
       "   'Zhibing Chen',\n",
       "   'Matt Kaeberlein',\n",
       "   'Rachel B Brem',\n",
       "   'Michael Polymenis',\n",
       "   'Brian K Kennedy',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jiashun Zheng',\n",
       "   'Qi Ouyang',\n",
       "   'Hao Li'],\n",
       "  'publication_date': '22/08/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32821821',\n",
       "  'abstract': 'Caloric restriction (CR) is known to extend life span across species; however, the molecular mechanisms are not well understood. We investigate the mechanism by which glucose restriction (GR) extends yeast replicative life span, by combining ribosome profiling and RNA-seq with microfluidic-based single-cell analysis. We discovered a cross-talk between glucose sensing and the regulation of intracellular methionine: GR down-regulated the transcription and translation of methionine biosynthetic enzymes and transporters, leading to a decreased intracellular methionine concentration; external supplementation of methionine cancels the life span extension by GR. Furthermore, genetic perturbations that decrease methionine synthesis/uptake extend life span. These observations suggest that intracellular methionine mediates the life span effects of various nutrient and genetic perturbations, and that the glucose-methionine cross-talk is a general mechanism for coordinating the nutrient status and the translation/growth of a cell. Our work also implicates proteasome as a downstream effector of the life span extension by GR.',\n",
       "  'doi': '10.1126/sciadv.aba1306',\n",
       "  'journal': 'Science advances'},\n",
       " {'title': 'GIGYF2 and 4EHP Inhibit Translation Initiation of Defective Messenger RNAs to Assist Ribosome-Associated Quality Control.',\n",
       "  'authors': ['Kelsey L Hickey',\n",
       "   'Kimberley Dickson',\n",
       "   'J Zachery Cogan',\n",
       "   'Joseph M Replogle',\n",
       "   'Michael Schoof',\n",
       "   \"Karole N D'Orazio\",\n",
       "   'Niladri K Sinha',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Marco Jost',\n",
       "   'Adam Frost',\n",
       "   'Rachel Green',\n",
       "   'Jonathan S Weissman',\n",
       "   'Kamena K Kostova'],\n",
       "  'publication_date': '30/07/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32726578',\n",
       "  'abstract': 'Ribosome-associated quality control (RQC) pathways protect cells from toxicity caused by incomplete protein products resulting from translation of damaged or problematic mRNAs. Extensive work in yeast has identified highly conserved mechanisms that lead to degradation of faulty mRNA and partially synthesized polypeptides. Here we used CRISPR-Cas9-based screening to search for additional RQC strategies in mammals. We found that failed translation leads to specific inhibition of translation initiation on that message. This negative feedback loop is mediated by two translation inhibitors, GIGYF2 and 4EHP. Model substrates and growth-based assays established that inhibition of additional rounds of translation acts in concert with known RQC pathways to prevent buildup of toxic proteins. Inability to block translation of faulty mRNAs and subsequent accumulation of partially synthesized polypeptides could explain the neurodevelopmental and neuropsychiatric disorders observed in mice and humans with compromised GIGYF2 function.',\n",
       "  'doi': '10.1016/j.molcel.2020.07.007',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Pharmaceutical-Grade Rigosertib Is a Microtubule-Destabilizing Agent.',\n",
       "  'authors': ['Marco Jost',\n",
       "   'Yuwen Chen',\n",
       "   'Luke A Gilbert',\n",
       "   'Max A Horlbeck',\n",
       "   'Lenno Krenning',\n",
       "   'Grégory Menchon',\n",
       "   'Ankit Rai',\n",
       "   'Min Y Cho',\n",
       "   'Jacob J Stern',\n",
       "   'Andrea E Prota',\n",
       "   'Martin Kampmann',\n",
       "   'Anna Akhmanova',\n",
       "   'Michel O Steinmetz',\n",
       "   'Marvin E Tanenbaum',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '04/07/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32619469',\n",
       "  'abstract': 'We recently used CRISPRi/a-based chemical-genetic screens and cell biological, biochemical, and structural assays to determine that rigosertib, an anti-cancer agent in phase III clinical trials, kills cancer cells by destabilizing microtubules. Reddy and co-workers (Baker et\\xa0al., 2020, this issue of Molecular Cell) suggest that a contaminating degradation product in commercial formulations of rigosertib is responsible for the microtubule-destabilizing activity. Here, we demonstrate that cells treated with pharmaceutical-grade rigosertib (>99.9% purity) or commercially obtained rigosertib have qualitatively indistinguishable phenotypes across multiple assays. The two formulations have indistinguishable chemical-genetic interactions with genes that modulate microtubule stability, both destabilize microtubules in cells and in\\xa0vitro, and expression of a rationally designed tubulin mutant with a mutation in the rigosertib binding site (L240F TUBB) allows cells to proliferate in the presence of either formulation. Importantly, the specificity of the L240F TUBB mutant for microtubule-destabilizing agents has been confirmed independently. Thus, rigosertib kills cancer cells by destabilizing microtubules, in agreement with our original findings.',\n",
       "  'doi': '10.1016/j.molcel.2020.06.008',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': \"Polysomes Bypass a 50-Nucleotide Coding Gap Less Efficiently Than Monosomes Due to Attenuation of a 5' mRNA Stem-Loop and Enhanced Drop-off.\",\n",
       "  'authors': [\"Sinéad O'Loughlin\",\n",
       "   'Mark C Capece',\n",
       "   'Mariia Klimova',\n",
       "   'Norma M Wills',\n",
       "   'Arthur Coakley',\n",
       "   'Ekaterina Samatova',\n",
       "   \"Patrick B F O'Connor\",\n",
       "   'Gary Loughran',\n",
       "   'Jonathan S Weissman',\n",
       "   'Pavel V Baranov',\n",
       "   'Marina V Rodnina',\n",
       "   'Joseph D Puglisi',\n",
       "   'John F Atkins'],\n",
       "  'publication_date': '27/05/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32454154',\n",
       "  'abstract': \"Efficient translational bypassing of a 50-nt non-coding gap in a phage T4 topoisomerase subunit gene (gp60) requires several recoding signals. Here we investigate the function of the mRNA stem-loop 5' of the take-off codon, as well as the importance of ribosome loading density on the mRNA for efficient bypassing. We show that polysomes are less efficient at mediating bypassing than monosomes, both in vitro and in vivo, due to their preventing formation of a stem-loop 5' of the take-off codon and allowing greater peptidyl-tRNA drop off. A ribosome profiling analysis of phage T4-infected Escherichia coli yielded protected mRNA fragments within the normal size range derived from ribosomes stalled at the take-off codon. However, ribosomes at this position also yielded some 53-nucleotide fragments, 16 longer. These were due to protection of the nucleotides that form the 5' stem-loop. NMR shows that the 5' stem-loop is highly dynamic. The importance of different nucleotides in the 5' stem-loop is revealed by mutagenesis studies. These data highlight the significance of the 5' stem-loop for the 50-nt bypassing and further enhance appreciation of relevance of the extent of ribosome loading for recoding.\",\n",
       "  'doi': '10.1016/j.jmb.2020.05.010',\n",
       "  'journal': 'Journal of molecular biology'},\n",
       " {'title': 'Inference of single-cell phylogenies from lineage tracing data using Cassiopeia.',\n",
       "  'authors': ['Matthew G Jones',\n",
       "   'Alex Khodaverdian',\n",
       "   'Jeffrey J Quinn',\n",
       "   'Michelle M Chan',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Robert Wang',\n",
       "   'Chenling Xu',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nir Yosef'],\n",
       "  'publication_date': '16/04/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32290857',\n",
       "  'abstract': 'The pairing of CRISPR/Cas9-based gene editing with massively parallel single-cell readouts now enables large-scale lineage tracing. However, the rapid growth in complexity of data from these assays has outpaced our ability to accurately infer phylogenetic relationships. First, we introduce Cassiopeia-a suite of scalable maximum parsimony approaches for tree reconstruction. Second, we provide a simulation framework for evaluating algorithms and exploring lineage tracer design principles. Finally, we generate the most complex experimental lineage tracing dataset to date, 34,557 human cells continuously traced over 15 generations, and use it for benchmarking phylogenetic inference approaches. We show that Cassiopeia outperforms traditional methods by several metrics and under a wide variety of parameter regimes, and provide insight into the principles for the design of improved Cas9-enabled recorders. Together, these should broadly enable large-scale mammalian lineage tracing efforts. Cassiopeia and its benchmarking resources are publicly available at www.github.com/YosefLab/Cassiopeia.',\n",
       "  'doi': '10.1186/s13059-020-02000-8',\n",
       "  'journal': 'Genome biology'},\n",
       " {'title': 'CRISPRi-based radiation modifier screen identifies long non-coding RNA therapeutic targets in glioma.',\n",
       "  'authors': ['S John Liu',\n",
       "   'Martina Malatesta',\n",
       "   'Brian V Lien',\n",
       "   'Parna Saha',\n",
       "   'Shivani S Thombare',\n",
       "   'Sung Jun Hong',\n",
       "   'Leslie Pedraza',\n",
       "   'Mark Koontz',\n",
       "   'Kyounghee Seo',\n",
       "   'Max A Horlbeck',\n",
       "   'Daniel He',\n",
       "   'Harjus S Birk',\n",
       "   'Miten Jain',\n",
       "   'Hugh E Olsen',\n",
       "   'Mark Akeson',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michelle Monje',\n",
       "   'Nalin Gupta',\n",
       "   'David R Raleigh',\n",
       "   'Erik M Ullian',\n",
       "   'Daniel A Lim'],\n",
       "  'publication_date': '03/04/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32234056',\n",
       "  'abstract': 'Long non-coding RNAs (lncRNAs) exhibit highly cell type-specific expression and function, making this class of transcript attractive for targeted cancer therapy. However, the vast majority of lncRNAs have not been tested as potential therapeutic targets, particularly in the context of currently used cancer treatments. Malignant glioma is rapidly fatal, and ionizing radiation is part of the current standard-of-care used to slow tumor growth in both adult and pediatric patients.\\nWe use CRISPR interference (CRISPRi) to screen 5689 lncRNA loci in human glioblastoma (GBM) cells, identifying 467 hits that modify cell growth in the presence of clinically relevant doses of fractionated radiation. Thirty-three of these lncRNA hits sensitize cells to radiation, and based on their expression in adult and pediatric gliomas, nine of these hits are prioritized as lncRNA Glioma Radiation Sensitizers (lncGRS). Knockdown of lncGRS-1, a primate-conserved, nuclear-enriched lncRNA, inhibits the growth and proliferation of primary adult and pediatric glioma cells, but not the viability of normal brain cells. Using human brain organoids comprised of mature neural cell types as a three-dimensional tissue substrate to model the invasive growth of glioma, we find that antisense oligonucleotides targeting lncGRS-1 selectively decrease tumor growth and sensitize glioma cells to radiation therapy.\\nThese studies identify lncGRS-1 as a glioma-specific therapeutic target and establish a generalizable approach to rapidly identify novel therapeutic targets in the vast non-coding genome to enhance radiation therapy.',\n",
       "  'doi': '10.1186/s13059-020-01995-4',\n",
       "  'journal': 'Genome biology'},\n",
       " {'title': 'Combinatorial single-cell CRISPR screens by direct guide RNA capture and targeted sequencing.',\n",
       "  'authors': ['Joseph M Replogle',\n",
       "   'Thomas M Norman',\n",
       "   'Albert Xu',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Jin Chen',\n",
       "   'J Zachery Cogan',\n",
       "   'Elliott J Meer',\n",
       "   'Jessica M Terry',\n",
       "   'Daniel P Riordan',\n",
       "   'Niranjan Srinivas',\n",
       "   'Ian T Fiddes',\n",
       "   'Joseph G Arthur',\n",
       "   'Luigi J Alvarado',\n",
       "   'Katherine A Pfeiffer',\n",
       "   'Tarjei S Mikkelsen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Britt Adamson'],\n",
       "  'publication_date': '02/04/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32231336',\n",
       "  'abstract': 'Single-cell CRISPR screens enable the exploration of mammalian gene function and genetic regulatory networks. However, use of this technology has been limited by reliance on indirect indexing of single-guide RNAs (sgRNAs). Here we present direct-capture Perturb-seq, a versatile screening approach in which expressed sgRNAs are sequenced alongside single-cell transcriptomes. Direct-capture Perturb-seq enables detection of multiple distinct sgRNA sequences from individual cells and thus allows pooled single-cell CRISPR screens to be easily paired with combinatorial perturbation libraries that contain dual-guide expression vectors. We demonstrate the utility of this approach for high-throughput investigations of genetic interactions and, leveraging this ability, dissect epistatic interactions between cholesterol biogenesis and DNA repair. Using direct capture Perturb-seq, we also show that targeting individual genes with multiple sgRNAs per cell improves efficacy of CRISPR interference and activation, facilitating the use of compact, highly active CRISPR libraries for single-cell screens. Last, we show that hybridization-based target enrichment permits sensitive, specific sequencing of informative transcripts from single-cell RNA-seq experiments.',\n",
       "  'doi': '10.1038/s41587-020-0470-y',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Pervasive functional translation of noncanonical human open reading frames.',\n",
       "  'authors': ['Jin Chen',\n",
       "   'Andreas-David Brunner',\n",
       "   'J Zachery Cogan',\n",
       "   'James K Nuñez',\n",
       "   'Alexander P Fields',\n",
       "   'Britt Adamson',\n",
       "   'Daniel N Itzhak',\n",
       "   'Jason Y Li',\n",
       "   'Matthias Mann',\n",
       "   'Manuel D Leonetti',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '07/03/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32139545',\n",
       "  'abstract': 'Ribosome profiling has revealed pervasive but largely uncharacterized translation outside of canonical coding sequences (CDSs). In this work, we exploit a systematic CRISPR-based screening strategy to identify hundreds of noncanonical CDSs that are essential for cellular growth and whose disruption elicits specific, robust transcriptomic and phenotypic changes in human cells. Functional characterization of the encoded microproteins reveals distinct cellular localizations, specific protein binding partners, and hundreds of microproteins that are presented by the human leukocyte antigen system. We find multiple microproteins encoded in upstream open reading frames, which form stable complexes with the main, canonical protein encoded on the same messenger RNA, thereby revealing the use of functional bicistronic operons in mammals. Together, our results point to a family of functional human microproteins that play critical and diverse cellular roles.',\n",
       "  'doi': '10.1126/science.aay0262',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Fitness effects of CRISPR/Cas9-targeting of long noncoding RNA genes.',\n",
       "  'authors': ['Max A Horlbeck',\n",
       "   'S John Liu',\n",
       "   'Howard Y Chang',\n",
       "   'Daniel A Lim',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '26/02/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '32094656',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/s41587-020-0428-0',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Titrating gene expression using libraries of systematically attenuated CRISPR guide RNAs.',\n",
       "  'authors': ['Marco Jost',\n",
       "   'Daniel A Santos',\n",
       "   'Reuben A Saunders',\n",
       "   'Max A Horlbeck',\n",
       "   'John S Hawkins',\n",
       "   'Sonia M Scaria',\n",
       "   'Thomas M Norman',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Christina R Liem',\n",
       "   'Carol A Gross',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '15/01/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '31932729',\n",
       "  'abstract': 'A lack of tools to precisely control gene expression has limited our ability to evaluate relationships between expression levels and phenotypes. Here, we describe an approach to titrate expression of human genes using CRISPR interference and series of single-guide RNAs (sgRNAs) with systematically modulated activities. We used large-scale measurements across multiple cell models to characterize activities of sgRNAs containing mismatches to their target sites and derived rules governing mismatched sgRNA activity using deep learning. These rules enabled us to synthesize a compact sgRNA library to titrate expression of ~2,400 genes essential for robust cell growth and to construct an in silico sgRNA library spanning the human genome. Staging cells along a continuum of gene expression levels combined with single-cell RNA-seq readout revealed sharp transitions in cellular behaviors at gene-specific expression thresholds. Our work provides a general tool to control gene expression, with applications ranging from tuning biochemical pathways to identifying suppressors for diseases of dysregulated gene expression.',\n",
       "  'doi': '10.1038/s41587-019-0387-5',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'The microtubule regulator ringer functions downstream from the RNA repair/splicing pathway to promote axon regeneration',\n",
       "  'authors': ['Ernest J Monahan Vargas',\n",
       "   'Andrew J Matamoros',\n",
       "   'Jingyun Qiu',\n",
       "   'Calvin H Jan',\n",
       "   'Qin Wang',\n",
       "   'David Gorczyca',\n",
       "   'Tina W Han',\n",
       "   'Jonathan S Weissman',\n",
       "   'Yuh Nung Jan',\n",
       "   'Swati Banerjee',\n",
       "   'Yuanquan Song'],\n",
       "  'publication_date': '11/01/20',\n",
       "  'publication_year': '2020',\n",
       "  'pubmed_id': '31919191',\n",
       "  'abstract': 'Promoting axon regeneration in the central and peripheral nervous system is of clinical importance in neural injury and neurodegenerative diseases. Both pro- and antiregeneration factors are being identified. We previously reported that the Rtca mediated RNA repair/splicing pathway restricts axon regeneration by inhibiting the nonconventional splicing of ',\n",
       "  'doi': '10.1101/gad.331330.119',\n",
       "  'journal': 'Genes & development'},\n",
       " {'title': 'The ER membrane protein complex is required to ensure correct topology and stable expression of flavivirus polyproteins.',\n",
       "  'authors': ['Ashley M Ngo',\n",
       "   'Matthew J Shurtleff',\n",
       "   'Katerina D Popova',\n",
       "   'Jessie Kulsuptrakul',\n",
       "   'Jonathan S Weissman',\n",
       "   'Andreas S Puschnik'],\n",
       "  'publication_date': '14/09/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31516121',\n",
       "  'abstract': 'Flaviviruses translate their genomes as multi-pass transmembrane proteins at the endoplasmic reticulum (ER) membrane. Here, we show that the ER membrane protein complex (EMC) is indispensable for the expression of viral polyproteins. We demonstrated that EMC was essential for accurate folding and post-translational stability rather than translation efficiency. Specifically, we revealed degradation of NS4A-NS4B, a region rich in transmembrane domains, in absence of EMC. Orthogonally, by serial passaging of virus on EMC-deficient cells, we identified two non-synonymous point mutations in NS4A and NS4B, which rescued viral replication. Finally, we showed a physical interaction between EMC and viral NS4B and that the NS4A-4B region adopts an aberrant topology in the absence of the EMC leading to degradation. Together, our data highlight how flaviviruses hijack the EMC for transmembrane protein biogenesis to achieve optimal expression of their polyproteins, which reinforces a role for the EMC in stabilizing challenging transmembrane proteins during synthesis.',\n",
       "  'doi': '10.7554/eLife.48469',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'New factors for protein transport identified by a genome-wide CRISPRi screen in mammalian cells.',\n",
       "  'authors': ['Laia Bassaganyas',\n",
       "   'Stephanie J Popa',\n",
       "   'Max Horlbeck',\n",
       "   'Claudia Puri',\n",
       "   'Sarah E Stewart',\n",
       "   'Felix Campelo',\n",
       "   'Anupama Ashok',\n",
       "   'Cristian M Butnaru',\n",
       "   'Nathalie Brouwers',\n",
       "   'Kartoosh Heydari',\n",
       "   'Jean Ripoche',\n",
       "   'Jonathan Weissman',\n",
       "   'David C Rubinsztein',\n",
       "   'Randy Schekman',\n",
       "   'Vivek Malhotra',\n",
       "   'Kevin Moreau',\n",
       "   'Julien Villeneuve'],\n",
       "  'publication_date': '07/09/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31488582',\n",
       "  'abstract': 'Protein and membrane trafficking pathways are critical for cell and tissue homeostasis. Traditional genetic and biochemical approaches have shed light on basic principles underlying these processes. However, the list of factors required for secretory pathway function remains incomplete, and mechanisms involved in their adaptation poorly understood. Here, we present a powerful strategy based on a pooled genome-wide CRISPRi screen that allowed the identification of new factors involved in protein transport. Two newly identified factors, TTC17 and CCDC157, localized along the secretory pathway and were found to interact with resident proteins of ER-Golgi membranes. In addition, we uncovered that upon TTC17 knockdown, the polarized organization of Golgi cisternae was altered, creating glycosylation defects, and that CCDC157 is an important factor for the fusion of transport carriers to Golgi membranes. In conclusion, our work identified and characterized new actors in the mechanisms of protein transport and secretion and opens stimulating perspectives for the use of our platform in physiological and pathological contexts.',\n",
       "  'doi': '10.1083/jcb.201902028',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'Exploring genetic interaction manifolds constructed from rich single-cell phenotypes.',\n",
       "  'authors': ['Thomas M Norman',\n",
       "   'Max A Horlbeck',\n",
       "   'Joseph M Replogle',\n",
       "   'Alex Y Ge',\n",
       "   'Albert Xu',\n",
       "   'Marco Jost',\n",
       "   'Luke A Gilbert',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '10/08/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31395745',\n",
       "  'abstract': 'How cellular and organismal complexity emerges from combinatorial expression of genes is a central question in biology. High-content phenotyping approaches such as Perturb-seq (single-cell RNA-sequencing pooled CRISPR screens) present an opportunity for exploring such genetic interactions (GIs) at scale. Here, we present an analytical framework for interpreting high-dimensional landscapes of cell states (manifolds) constructed from transcriptional phenotypes. We applied this approach to Perturb-seq profiling of strong GIs mined from a growth-based, gain-of-function GI map. Exploration of this manifold enabled ordering of regulatory pathways, principled classification of GIs (e.g., identifying suppressors), and mechanistic elucidation of synergistic interactions, including an unexpected synergy between ',\n",
       "  'doi': '10.1126/science.aax4438',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'MULTI-seq: sample multiplexing for single-cell RNA sequencing using lipid-tagged indices.',\n",
       "  'authors': ['Christopher S McGinnis',\n",
       "   'David M Patterson',\n",
       "   'Juliane Winkler',\n",
       "   'Daniel N Conrad',\n",
       "   'Marco Y Hein',\n",
       "   'Vasudha Srivastava',\n",
       "   'Jennifer L Hu',\n",
       "   'Lyndsay M Murrow',\n",
       "   'Jonathan S Weissman',\n",
       "   'Zena Werb',\n",
       "   'Eric D Chow',\n",
       "   'Zev J Gartner'],\n",
       "  'publication_date': '19/06/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31209384',\n",
       "  'abstract': 'Sample multiplexing facilitates scRNA-seq by reducing costs and identifying artifacts such as cell doublets. However, universal and scalable sample barcoding strategies have not been described. We therefore developed MULTI-seq: multiplexing using lipid-tagged indices for single-cell and single-nucleus RNA sequencing. MULTI-seq reagents can barcode any cell type or nucleus from any species with an accessible plasma membrane. The method involves minimal sample processing, thereby preserving cell viability and endogenous gene expression patterns. When cells are classified into sample groups using MULTI-seq barcode abundances, data quality is improved through doublet identification and recovery of cells with low RNA content that would otherwise be discarded by standard quality-control workflows. We use MULTI-seq to track the dynamics of T-cell activation, perform a 96-plex perturbation experiment with primary human mammary epithelial cells and multiplex cryopreserved tumors and metastatic sites isolated from a patient-derived xenograft mouse model of triple-negative breast cancer.',\n",
       "  'doi': '10.1038/s41592-019-0433-8',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Ceapins block the unfolded protein response sensor ATF6α by inducing a neomorphic inter-organelle tether.',\n",
       "  'authors': ['Sandra Elizabeth Torres',\n",
       "   'Ciara M Gallagher',\n",
       "   'Lars Plate',\n",
       "   'Meghna Gupta',\n",
       "   'Christina R Liem',\n",
       "   'Xiaoyan Guo',\n",
       "   'Ruilin Tian',\n",
       "   'Robert M Stroud',\n",
       "   'Martin Kampmann',\n",
       "   'Jonathan S Weissman',\n",
       "   'Peter Walter'],\n",
       "  'publication_date': '01/06/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31149896',\n",
       "  'abstract': \"The unfolded protein response (UPR) detects and restores deficits in the endoplasmic reticulum (ER) protein folding capacity. Ceapins specifically inhibit the UPR sensor ATF6α, an ER-tethered transcription factor, by retaining it at the ER through an unknown mechanism. Our genome-wide CRISPR interference (CRISPRi) screen reveals that Ceapins function is completely dependent on the ABCD3 peroxisomal transporter. Proteomics studies establish that ABCD3 physically associates with ER-resident ATF6α in cells and in vitro in a Ceapin-dependent manner. Ceapins induce the neomorphic association of ER and peroxisomes by directly tethering the cytosolic domain of ATF6α to ABCD3's transmembrane regions without inhibiting or depending on ABCD3 transporter activity. Thus, our studies reveal that Ceapins function by chemical-induced misdirection which explains their remarkable specificity and opens up new mechanistic routes for drug development and synthetic biology.\",\n",
       "  'doi': '10.7554/eLife.46595',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Chaperone-mediated reflux of secretory proteins to the cytosol during endoplasmic reticulum stress.',\n",
       "  'authors': ['Aeid Igbaria',\n",
       "   'Philip I Merksamer',\n",
       "   'Ala Trusina',\n",
       "   'Firehiwot Tilahun',\n",
       "   'Jeffrey R Johnson',\n",
       "   'Onn Brandman',\n",
       "   'Nevan J Krogan',\n",
       "   'Jonathan S Weissman',\n",
       "   'Feroz R Papa'],\n",
       "  'publication_date': '19/05/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31101715',\n",
       "  'abstract': 'Diverse perturbations to endoplasmic reticulum (ER) functions compromise the proper folding and structural maturation of secretory proteins. To study secretory pathway physiology during such \"ER stress,\" we employed an ER-targeted, redox-responsive, green fluorescent protein-eroGFP-that reports on ambient changes in oxidizing potential. Here we find that diverse ER stress regimes cause properly folded, ER-resident eroGFP (and other ER luminal proteins) to \"reflux\" back to the reducing environment of the cytosol as intact, folded proteins. By utilizing eroGFP in a comprehensive genetic screen in ',\n",
       "  'doi': '10.1073/pnas.1904516116',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Molecular recording of mammalian embryogenesis.',\n",
       "  'authors': ['Michelle M Chan',\n",
       "   'Zachary D Smith',\n",
       "   'Stefanie Grosswendt',\n",
       "   'Helene Kretzmer',\n",
       "   'Thomas M Norman',\n",
       "   'Britt Adamson',\n",
       "   'Marco Jost',\n",
       "   'Jeffrey J Quinn',\n",
       "   'Dian Yang',\n",
       "   'Matthew G Jones',\n",
       "   'Alex Khodaverdian',\n",
       "   'Nir Yosef',\n",
       "   'Alexander Meissner',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '16/05/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '31086336',\n",
       "  'abstract': 'Ontogeny describes the emergence of complex multicellular organisms from single totipotent cells. This field is particularly challenging in mammals, owing to the indeterminate relationship between self-renewal and differentiation, variation in progenitor field sizes, and internal gestation in these animals. Here we present a flexible, high-information, multi-channel molecular recorder with a single-cell readout and apply it as an evolving lineage tracer to assemble mouse cell-fate maps from fertilization through gastrulation. By combining lineage information with single-cell RNA sequencing profiles, we recapitulate canonical developmental relationships between different tissue types and reveal the nearly complete transcriptional convergence of endodermal cells of extra-embryonic and embryonic origins. Finally, we apply our cell-fate maps to estimate the number of embryonic progenitor cells and their degree of asymmetric partitioning during specification. Our approach enables massively parallel, high-resolution recording of lineage and other information in mammalian systems, which will facilitate the construction of a quantitative framework for understanding developmental processes.',\n",
       "  'doi': '10.1038/s41586-019-1184-5',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Cycloheximide can distort measurements of mRNA levels and translation efficiency.',\n",
       "  'authors': ['Daniel A Santos',\n",
       "   'Lei Shi',\n",
       "   'Benjamin P Tu',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/03/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '30916348',\n",
       "  'abstract': 'Regulation of the efficiency with which an mRNA is translated into proteins represents a key mechanism for controlling gene expression. Such regulation impacts the number of actively translating ribosomes per mRNA molecule, referred to as translation efficiency (TE), which can be monitored using ribosome profiling and RNA-seq, or by evaluating the position of an mRNA in a polysome gradient. Here we show that in budding yeast, under nutrient limiting conditions, the commonly used translation inhibitor cycloheximide induces rapid transcriptional upregulation of hundreds of genes involved in ribosome biogenesis. Cycloheximide also prevents translation of these newly transcribed messages, leading to an apparent drop in TE of these genes under conditions that include key transitions during the yeast metabolic cycle, meiosis, and amino acid starvation; however, this effect is abolished when cycloheximide pretreatment is omitted. This response requires TORC1 signaling, and is modulated by the genetic background as well as the vehicle used to deliver the drug. The present work highlights an important caveat to the use of translation inhibitors when measuring TE or mRNA levels, and will hopefully aid in future experimental design as well as interpretation of prior results.',\n",
       "  'doi': '10.1093/nar/gkz205',\n",
       "  'journal': 'Nucleic acids research'},\n",
       " {'title': 'Cellular response to small molecules that selectively stall protein synthesis by the ribosome.',\n",
       "  'authors': ['Nadège Liaud',\n",
       "   'Max A Horlbeck',\n",
       "   'Luke A Gilbert',\n",
       "   'Ketrin Gjoni',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jamie H D Cate'],\n",
       "  'publication_date': '16/03/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '30875366',\n",
       "  'abstract': 'Identifying small molecules that inhibit protein synthesis by selectively stalling the ribosome constitutes a new strategy for therapeutic development. Compounds that inhibit the translation of PCSK9, a major regulator of low-density lipoprotein cholesterol, have been identified that reduce LDL cholesterol in preclinical models and that affect the translation of only a few off-target proteins. Although some of these compounds hold potential for future therapeutic development, it is not known how they impact the physiology of cells or ribosome quality control pathways. Here we used a genome-wide CRISPRi screen to identify proteins and pathways that modulate cell growth in the presence of high doses of a selective PCSK9 translational inhibitor, PF-06378503 (PF8503). The two most potent genetic modifiers of cell fitness in the presence of PF8503, the ubiquitin binding protein ASCC2 and helicase ASCC3, bind to the ribosome and protect cells from toxic effects of high concentrations of the compound. Surprisingly, translation quality control proteins Pelota (PELO) and HBS1L sensitize cells to PF8503 treatment. In genetic interaction experiments, ASCC3 acts together with ASCC2, and functions downstream of HBS1L. Taken together, these results identify new connections between ribosome quality control pathways, and provide new insights into the selectivity of compounds that stall human translation that will aid the development of next-generation selective translation stalling compounds to treat disease.',\n",
       "  'doi': '10.1371/journal.pgen.1008057',\n",
       "  'journal': 'PLoS genetics'},\n",
       " {'title': 'Probing the Global Cellular Responses to Lipotoxicity Caused by Saturated Fatty Acids.',\n",
       "  'authors': ['Manuele Piccolis',\n",
       "   'Laura M Bond',\n",
       "   'Martin Kampmann',\n",
       "   'Pamela Pulimeno',\n",
       "   'Chandramohan Chitraju',\n",
       "   'Christina B K Jayson',\n",
       "   'Laura P Vaites',\n",
       "   'Sebastian Boland',\n",
       "   'Zon Weng Lai',\n",
       "   'Katlyn R Gabriel',\n",
       "   'Shane D Elliott',\n",
       "   'Joao A Paulo',\n",
       "   'J Wade Harper',\n",
       "   'Jonathan S Weissman',\n",
       "   'Tobias C Walther',\n",
       "   'Robert V Farese'],\n",
       "  'publication_date': '09/03/19',\n",
       "  'publication_year': '2019',\n",
       "  'pubmed_id': '30846318',\n",
       "  'abstract': 'Excessive levels of saturated fatty acids are toxic to cells, although the basis for this lipotoxicity remains incompletely understood. Here, we analyzed the transcriptome, lipidome, and genetic interactions of human leukemia cells exposed to palmitate. Palmitate treatment increased saturated glycerolipids, accompanied by a transcriptional stress response, including upregulation of the endoplasmic reticulum (ER) stress response. A comprehensive genome-wide short hairpin RNA (shRNA) screen identified >350 genes modulating lipotoxicity. Among previously unknown genetic modifiers of lipotoxicity, depletion of RNF213, a putative ubiquitin ligase mutated in Moyamoya vascular disease, protected cells from lipotoxicity. On a broader level, integration of our comprehensive datasets revealed that changes in di-saturated glycerolipids, but not other lipid classes, are central to lipotoxicity in this model. Consistent with this, inhibition of ER-localized glycerol-3-phosphate acyltransferase activity protected from all aspects of lipotoxicity. Identification of genes modulating the response to saturated fatty acids may reveal novel therapeutic strategies for treating metabolic diseases linked to lipotoxicity.',\n",
       "  'doi': '10.1016/j.molcel.2019.01.036',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Ribosome Profiling: Global Views of Translation.',\n",
       "  'authors': ['Nicholas T Ingolia',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '25/07/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '30037969',\n",
       "  'abstract': 'The translation of messenger RNA (mRNA) into protein and the folding of the resulting protein into an active form are prerequisites for virtually every cellular process and represent the single largest investment of energy by cells. Ribosome profiling-based approaches have revolutionized our ability to monitor every step of protein synthesis in vivo, allowing one to measure the rate of protein synthesis across the proteome, annotate the protein coding capacity of genomes, monitor localized protein synthesis, and explore cotranslational folding and targeting. The rich and quantitative nature of ribosome profiling data provides an unprecedented opportunity to explore and model complex cellular processes. New analytical techniques and improved experimental protocols will provide a deeper understanding of the factors controlling translation speed and its impact on protein function and cell physiology as well as the role of ribosomal RNA and mRNA modifications in regulating translation.',\n",
       "  'doi': '10.1101/cshperspect.a032698',\n",
       "  'journal': 'Cold Spring Harbor perspectives in biology'},\n",
       " {'title': 'Mapping the Genetic Landscape of Human Cells.',\n",
       "  'authors': ['Max A Horlbeck',\n",
       "   'Albert Xu',\n",
       "   'Min Wang',\n",
       "   'Neal K Bennett',\n",
       "   'Chong Y Park',\n",
       "   'Derek Bogdanoff',\n",
       "   'Britt Adamson',\n",
       "   'Eric D Chow',\n",
       "   'Martin Kampmann',\n",
       "   'Tim R Peterson',\n",
       "   'Ken Nakamura',\n",
       "   'Michael A Fischbach',\n",
       "   'Jonathan S Weissman',\n",
       "   'Luke A Gilbert'],\n",
       "  'publication_date': '24/07/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '30033366',\n",
       "  'abstract': 'Seminal yeast studies have established the value of comprehensively mapping genetic interactions (GIs) for inferring gene function. Efforts in human cells using focused gene sets underscore the utility of this approach, but the feasibility of generating large-scale, diverse human GI maps remains unresolved. We developed a CRISPR interference platform for large-scale quantitative mapping of human GIs. We systematically perturbed 222,784 gene pairs in two cancer cell lines. The resultant maps cluster functionally related genes, assigning function to poorly characterized genes, including TMEM261, a new electron transport chain component. Individual GIs pinpoint unexpected relationships between pathways, exemplified by a specific cholesterol biosynthesis intermediate whose accumulation induces deoxynucleotide depletion, causing replicative DNA damage and a synthetic-lethal interaction with the ATR/9-1-1 DNA repair pathway. Our map provides a broad resource, establishes GI maps as a high-resolution tool for dissecting gene function, and serves as a blueprint for mapping the genetic landscape of human cells.',\n",
       "  'doi': '10.1016/j.cell.2018.06.010',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Reprogramming human T cell function and specificity with non-viral genome targeting.',\n",
       "  'authors': ['Theodore L Roth',\n",
       "   'Cristina Puig-Saus',\n",
       "   'Ruby Yu',\n",
       "   'Eric Shifrut',\n",
       "   'Julia Carnevale',\n",
       "   'P Jonathan Li',\n",
       "   'Joseph Hiatt',\n",
       "   'Justin Saco',\n",
       "   'Paige Krystofinski',\n",
       "   'Han Li',\n",
       "   'Victoria Tobin',\n",
       "   'David N Nguyen',\n",
       "   'Michael R Lee',\n",
       "   'Amy L Putnam',\n",
       "   'Andrea L Ferris',\n",
       "   'Jeff W Chen',\n",
       "   'Jean-Nicolas Schickel',\n",
       "   'Laurence Pellerin',\n",
       "   'David Carmody',\n",
       "   'Gorka Alkorta-Aranburu',\n",
       "   'Daniela Del Gaudio',\n",
       "   'Hiroyuki Matsumoto',\n",
       "   'Montse Morell',\n",
       "   'Ying Mao',\n",
       "   'Min Cho',\n",
       "   'Rolen M Quadros',\n",
       "   'Channabasavaiah B Gurumurthy',\n",
       "   'Baz Smith',\n",
       "   'Michael Haugwitz',\n",
       "   'Stephen H Hughes',\n",
       "   'Jonathan S Weissman',\n",
       "   'Kathrin Schumann',\n",
       "   'Jonathan H Esensten',\n",
       "   'Andrew P May',\n",
       "   'Alan Ashworth',\n",
       "   'Gary M Kupfer',\n",
       "   'Siri Atma W Greeley',\n",
       "   'Rosa Bacchetta',\n",
       "   'Eric Meffre',\n",
       "   'Maria Grazia Roncarolo',\n",
       "   'Neil Romberg',\n",
       "   'Kevan C Herold',\n",
       "   'Antoni Ribas',\n",
       "   'Manuel D Leonetti',\n",
       "   'Alexander Marson'],\n",
       "  'publication_date': '12/07/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29995861',\n",
       "  'abstract': 'Decades of work have aimed to genetically reprogram T cells for therapeutic purposes',\n",
       "  'doi': '10.1038/s41586-018-0326-5',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Engineering ER-stress dependent non-conventional mRNA splicing.',\n",
       "  'authors': ['Weihan Li',\n",
       "   'Voytek Okreglak',\n",
       "   'Jirka Peschek',\n",
       "   'Philipp Kimmig',\n",
       "   'Meghan Zubradt',\n",
       "   'Jonathan S Weissman',\n",
       "   'Peter Walter'],\n",
       "  'publication_date': '10/07/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29985129',\n",
       "  'abstract': 'The endoplasmic reticulum (ER) protein folding capacity is balanced with the protein folding burden to prevent accumulation of un- or misfolded proteins. The ER membrane-resident kinase/RNase Ire1 maintains ER protein homeostasis through two fundamentally distinct processes. First, Ire1 can initiate a transcriptional response through a non-conventional mRNA splicing reaction to increase the ER folding capacity. Second, Ire1 can decrease the ER folding burden through selective mRNA decay. In ',\n",
       "  'doi': '10.7554/eLife.35388',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Exploration of Benzothiazole Rhodacyanines as Allosteric Inhibitors of Protein-Protein Interactions with Heat Shock Protein 70 (Hsp70).',\n",
       "  'authors': ['Hao Shao',\n",
       "   'Xiaokai Li',\n",
       "   'Michael A Moses',\n",
       "   'Luke A Gilbert',\n",
       "   'Chakrapani Kalyanaraman',\n",
       "   'Zapporah T Young',\n",
       "   'Margarita Chernova',\n",
       "   'Sara N Journey',\n",
       "   'Jonathan S Weissman',\n",
       "   'Byron Hann',\n",
       "   'Matthew P Jacobson',\n",
       "   'Len Neckers',\n",
       "   'Jason E Gestwicki'],\n",
       "  'publication_date': '29/06/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29953808',\n",
       "  'abstract': 'Cancer cells rely on the chaperone heat shock protein 70 (Hsp70) for survival and proliferation. Recently, benzothiazole rhodacyanines have been shown to bind an allosteric site on Hsp70, interrupting its binding to nucleotide-exchange factors (NEFs) and promoting cell death in breast cancer cell lines. However, proof-of-concept molecules, such as JG-98, have relatively modest potency (EC',\n",
       "  'doi': '10.1021/acs.jmedchem.8b00583',\n",
       "  'journal': 'Journal of medicinal chemistry'},\n",
       " {'title': \"The helicase Ded1p controls use of near-cognate translation initiation codons in 5' UTRs.\",\n",
       "  'authors': ['Ulf-Peter Guenther',\n",
       "   'David E Weinberg',\n",
       "   'Meghan M Zubradt',\n",
       "   'Frank A Tedeschi',\n",
       "   'Brittany N Stawicki',\n",
       "   'Leah L Zagore',\n",
       "   'Gloria A Brar',\n",
       "   'Donny D Licatalosi',\n",
       "   'David P Bartel',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eckhard Jankowsky'],\n",
       "  'publication_date': '29/06/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29950728',\n",
       "  'abstract': 'The conserved and essential DEAD-box RNA helicase Ded1p from yeast and its mammalian orthologue DDX3 are critical for the initiation of translation',\n",
       "  'doi': '10.1038/s41586-018-0258-0',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'The ER membrane protein complex interacts cotranslationally to enable biogenesis of multipass membrane proteins.',\n",
       "  'authors': ['Matthew J Shurtleff',\n",
       "   'Daniel N Itzhak',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Nicole T Schirle Oakdale',\n",
       "   'Elizabeth A Costa',\n",
       "   'Martin Jonikas',\n",
       "   'Jimena Weibezahn',\n",
       "   'Katerina D Popova',\n",
       "   'Calvin H Jan',\n",
       "   'Pavel Sinitcyn',\n",
       "   'Shruthi S Vembar',\n",
       "   'Hilda Hernandez',\n",
       "   'Jürgen Cox',\n",
       "   'Alma L Burlingame',\n",
       "   'Jeffrey L Brodsky',\n",
       "   'Adam Frost',\n",
       "   'Georg Hh Borner',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '29/05/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29809151',\n",
       "  'abstract': 'The endoplasmic reticulum (ER) supports biosynthesis of proteins with diverse transmembrane domain (TMD) lengths and hydrophobicity. Features in transmembrane domains such as charged residues in ion channels are often functionally important, but could pose a challenge during cotranslational membrane insertion and folding. Our systematic proteomic approaches in both yeast and human cells revealed that the ER membrane protein complex (EMC) binds to and promotes the biogenesis of a range of multipass transmembrane proteins, with a particular enrichment for transporters. Proximity-specific ribosome profiling demonstrates that the EMC engages clients cotranslationally and immediately following clusters of TMDs enriched for charged residues. The EMC can remain associated after completion of translation, which both protects clients from premature degradation and allows recruitment of substrate-specific and general chaperones. Thus, the EMC broadly enables the biogenesis of multipass transmembrane proteins containing destabilizing features, thereby mitigating the trade-off between function and stability.',\n",
       "  'doi': '10.7554/eLife.37018',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Identification of a transporter complex responsible for the cytosolic entry of nitrogen-containing bisphosphonates.',\n",
       "  'authors': ['Zhou Yu',\n",
       "   'Lauren E Surface',\n",
       "   'Chong Yon Park',\n",
       "   'Max A Horlbeck',\n",
       "   'Gregory A Wyant',\n",
       "   'Monther Abu-Remaileh',\n",
       "   'Timothy R Peterson',\n",
       "   'David M Sabatini',\n",
       "   'Jonathan S Weissman',\n",
       "   \"Erin K O'Shea\"],\n",
       "  'publication_date': '11/05/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29745899',\n",
       "  'abstract': 'Nitrogen-containing-bisphosphonates (N-BPs) are a class of drugs widely prescribed to treat osteoporosis and other bone-related diseases. Although previous studies have established that N-BPs function by inhibiting the mevalonate pathway in osteoclasts, the mechanism by which N-BPs enter the cytosol from the extracellular space to reach their molecular target is not understood. Here, we implemented a CRISPRi-mediated genome-wide screen and identified ',\n",
       "  'doi': '10.7554/eLife.36620',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Promoter of lncRNA Gene PVT1 Is a Tumor-Suppressor DNA Boundary Element.',\n",
       "  'authors': ['Seung Woo Cho',\n",
       "   'Jin Xu',\n",
       "   'Ruping Sun',\n",
       "   'Maxwell R Mumbach',\n",
       "   'Ava C Carter',\n",
       "   'Y Grace Chen',\n",
       "   'Kathryn E Yost',\n",
       "   'Jeewon Kim',\n",
       "   'Jing He',\n",
       "   'Stephanie A Nevins',\n",
       "   'Suet-Feung Chin',\n",
       "   'Carlos Caldas',\n",
       "   'S John Liu',\n",
       "   'Max A Horlbeck',\n",
       "   'Daniel A Lim',\n",
       "   'Jonathan S Weissman',\n",
       "   'Christina Curtis',\n",
       "   'Howard Y Chang'],\n",
       "  'publication_date': '08/05/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29731168',\n",
       "  'abstract': 'Noncoding mutations in cancer genomes are frequent\\xa0but challenging to interpret. PVT1 encodes an oncogenic lncRNA, but recurrent translocations and deletions in human cancers suggest alternative mechanisms. Here, we show that the PVT1 promoter has a tumor-suppressor function that is independent of PVT1 lncRNA. CRISPR interference of PVT1 promoter enhances breast cancer cell competition and growth in\\xa0vivo. The promoters of the PVT1 and the MYC oncogenes, located 55 kb apart on chromosome 8q24, compete for engagement with four intragenic enhancers in the PVT1 locus, thereby allowing the PVT1 promoter to regulate pause release of MYC transcription. PVT1 undergoes developmentally regulated monoallelic expression, and the PVT1 promoter inhibits MYC expression only from the same chromosome via promoter competition. Cancer genome sequencing identifies recurrent mutations encompassing the human PVT1 promoter, and genome editing verified that PVT1 promoter mutation promotes cancer cell growth. These results highlight regulatory sequences of lncRNA genes as potential disease-associated DNA elements.',\n",
       "  'doi': '10.1016/j.cell.2018.03.068',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'A Stress Response that Monitors and Regulates mRNA Structure Is Central to Cold Shock Adaptation.',\n",
       "  'authors': ['Yan Zhang',\n",
       "   'David H Burkhardt',\n",
       "   'Silvi Rouskin',\n",
       "   'Gene-Wei Li',\n",
       "   'Jonathan S Weissman',\n",
       "   'Carol A Gross'],\n",
       "  'publication_date': '10/04/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29628307',\n",
       "  'abstract': 'Temperature influences the structural and functional properties of cellular components, necessitating stress responses to restore homeostasis following temperature shift. Whereas the circuitry controlling the heat shock response is well understood, that controlling the E.\\xa0coli cold shock adaptation program is not. We found that during the growth arrest phase (acclimation) that follows shift to low temperature, protein synthesis increases, and open reading frame (ORF)-wide mRNA secondary structure decreases. To identify the regulatory system controlling this process, we screened for players required for increased translation. We identified a two-member mRNA surveillance system that enables recovery of translation during acclimation: RNase R assures appropriate mRNA degradation and the Csps dynamically adjust mRNA secondary structure to globally modulate protein expression level. An autoregulatory switch in which Csps tune their own expression to cellular demand enables dynamic control of global translation. The universality of Csps in bacteria suggests broad utilization of this control mechanism.',\n",
       "  'doi': '10.1016/j.molcel.2018.02.035',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Targeting RAS-driven human cancer cells with antibodies to upregulated and essential cell-surface proteins.',\n",
       "  'authors': ['Alexander J Martinko',\n",
       "   'Charles Truillet',\n",
       "   'Olivier Julien',\n",
       "   'Juan E Diaz',\n",
       "   'Max A Horlbeck',\n",
       "   'Gordon Whiteley',\n",
       "   'Josip Blonder',\n",
       "   'Jonathan S Weissman',\n",
       "   'Sourav Bandyopadhyay',\n",
       "   'Michael J Evans',\n",
       "   'James A Wells'],\n",
       "  'publication_date': '24/01/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29359686',\n",
       "  'abstract': 'While there have been tremendous efforts to target oncogenic RAS signaling from inside the cell, little effort has focused on the cell-surface. Here, we used quantitative surface proteomics to reveal a signature of proteins that are upregulated on cells transformed with KRAS',\n",
       "  'doi': '10.7554/eLife.31098',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Defining the physiological role of SRP in protein-targeting efficiency and specificity.',\n",
       "  'authors': ['Elizabeth A Costa',\n",
       "   'Kelly Subramanian',\n",
       "   'Jodi Nunnari',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '20/01/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29348368',\n",
       "  'abstract': \"The signal recognition particle (SRP) enables cotranslational delivery of proteins for translocation into the endoplasmic reticulum (ER), but its full in vivo role remains incompletely explored. We combined rapid auxin-induced SRP degradation with proximity-specific ribosome profiling to define SRP's in vivo function in yeast. Despite the classic view that SRP recognizes amino-terminal signal sequences, we show that SRP was generally essential for targeting transmembrane domains regardless of their position relative to the amino terminus. By contrast, many proteins containing cleavable amino-terminal signal peptides were efficiently cotranslationally targeted in SRP's absence. We also reveal an unanticipated consequence of SRP loss: Transcripts normally targeted to the ER were mistargeted to mitochondria, leading to mitochondrial defects. These results elucidate SRP's essential roles in maintaining the efficiency and specificity of protein targeting.\",\n",
       "  'doi': '10.1126/science.aar3607',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Lipid Homeostasis Is Maintained by Dual Targeting of the Mitochondrial PE Biosynthesis Enzyme to the ER.',\n",
       "  'authors': ['Jonathan R Friedman',\n",
       "   'Muthukumar Kannan',\n",
       "   'Alexandre Toulmay',\n",
       "   'Calvin H Jan',\n",
       "   'Jonathan S Weissman',\n",
       "   'William A Prinz',\n",
       "   'Jodi Nunnari'],\n",
       "  'publication_date': '02/01/18',\n",
       "  'publication_year': '2018',\n",
       "  'pubmed_id': '29290583',\n",
       "  'abstract': 'Spatial organization of phospholipid synthesis in eukaryotes is critical for cellular homeostasis. The synthesis of phosphatidylcholine (PC), the most abundant cellular phospholipid, occurs redundantly via the ER-localized Kennedy pathway and a pathway that traverses the ER and mitochondria via membrane contact sites. The basis of the ER-mitochondrial PC synthesis pathway is the exclusive mitochondrial localization of a key pathway enzyme, phosphatidylserine decarboxylase Psd1, which generates phosphatidylethanolamine (PE). We find that Psd1 is localized to both mitochondria and the ER. Our data indicate that Psd1-dependent PE made at mitochondria and the ER has separable cellular functions. In addition, the relative organellar localization of Psd1 is dynamically modulated based on metabolic needs. These data reveal a critical role for ER-localized Psd1 in cellular phospholipid homeostasis, question the significance of an ER-mitochondrial PC synthesis pathway to cellular phospholipid homeostasis, and establish the importance of fine spatial regulation of lipid biosynthesis for cellular functions.',\n",
       "  'doi': '10.1016/j.devcel.2017.11.023',\n",
       "  'journal': 'Developmental cell'},\n",
       " {'title': 'CRISPR Approaches to Small Molecule Target Identification.',\n",
       "  'authors': ['Marco Jost', 'Jonathan S Weissman'],\n",
       "  'publication_date': '21/12/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '29261286',\n",
       "  'abstract': 'A long-standing challenge in drug development is the identification of the mechanisms of action of small molecules with therapeutic potential. A number of methods have been developed to address this challenge, each with inherent strengths and limitations. We here provide a brief review of these methods with a focus on chemical-genetic methods that are based on systematically profiling the effects of genetic perturbations on drug sensitivity. In particular, application of these methods to mammalian systems has been facilitated by the recent advent of CRISPR-based approaches, which enable one to readily repress, induce, or delete a given gene and determine the resulting effects on drug sensitivity.',\n",
       "  'doi': '10.1021/acschembio.7b00965',\n",
       "  'journal': 'ACS chemical biology'},\n",
       " {'title': 'The Human Cell Atlas.',\n",
       "  'authors': ['Aviv Regev',\n",
       "   'Sarah A Teichmann',\n",
       "   'Eric S Lander',\n",
       "   'Ido Amit',\n",
       "   'Christophe Benoist',\n",
       "   'Ewan Birney',\n",
       "   'Bernd Bodenmiller',\n",
       "   'Peter Campbell',\n",
       "   'Piero Carninci',\n",
       "   'Menna Clatworthy',\n",
       "   'Hans Clevers',\n",
       "   'Bart Deplancke',\n",
       "   'Ian Dunham',\n",
       "   'James Eberwine',\n",
       "   'Roland Eils',\n",
       "   'Wolfgang Enard',\n",
       "   'Andrew Farmer',\n",
       "   'Lars Fugger',\n",
       "   'Berthold Göttgens',\n",
       "   'Nir Hacohen',\n",
       "   'Muzlifah Haniffa',\n",
       "   'Martin Hemberg',\n",
       "   'Seung Kim',\n",
       "   'Paul Klenerman',\n",
       "   'Arnold Kriegstein',\n",
       "   'Ed Lein',\n",
       "   'Sten Linnarsson',\n",
       "   'Emma Lundberg',\n",
       "   'Joakim Lundeberg',\n",
       "   'Partha Majumder',\n",
       "   'John C Marioni',\n",
       "   'Miriam Merad',\n",
       "   'Musa Mhlanga',\n",
       "   'Martijn Nawijn',\n",
       "   'Mihai Netea',\n",
       "   'Garry Nolan',\n",
       "   \"Dana Pe'er\",\n",
       "   'Anthony Phillipakis',\n",
       "   'Chris P Ponting',\n",
       "   'Stephen Quake',\n",
       "   'Wolf Reik',\n",
       "   'Orit Rozenblatt-Rosen',\n",
       "   'Joshua Sanes',\n",
       "   'Rahul Satija',\n",
       "   'Ton N Schumacher',\n",
       "   'Alex Shalek',\n",
       "   'Ehud Shapiro',\n",
       "   'Padmanee Sharma',\n",
       "   'Jay W Shin',\n",
       "   'Oliver Stegle',\n",
       "   'Michael Stratton',\n",
       "   'Michael J T Stubbington',\n",
       "   'Fabian J Theis',\n",
       "   'Matthias Uhlen',\n",
       "   'Alexander van Oudenaarden',\n",
       "   'Allon Wagner',\n",
       "   'Fiona Watt',\n",
       "   'Jonathan Weissman',\n",
       "   'Barbara Wold',\n",
       "   'Ramnik Xavier',\n",
       "   'Nir Yosef',\n",
       "   'Human Cell Atlas Meeting Participants'],\n",
       "  'publication_date': '06/12/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '29206104',\n",
       "  'abstract': 'The recent advent of methods for high-throughput single-cell molecular profiling has catalyzed a growing sense in the scientific community that the time is ripe to complete the 150-year-old effort to identify all cell types in the human body. The Human Cell Atlas Project is an international collaborative effort that aims to define all human cell types in terms of distinctive molecular profiles (such as gene expression profiles) and to connect this information with classical cellular descriptions (such as location and morphology). An open comprehensive reference map of the molecular state of cells in healthy human tissues would propel the systematic study of physiological states, developmental trajectories, regulatory circuitry and interactions of cells, and also provide a framework for understanding cellular dysregulation in human disease. Here we describe the idea, its potential utility, early proofs-of-concept, and some design considerations for the Human Cell Atlas, including a commitment to open data, code, and community.',\n",
       "  'doi': '10.7554/eLife.27041',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Combinatorial genetics in liver repopulation and carcinogenesis with a in vivo CRISPR activation platform.',\n",
       "  'authors': ['Kirk J Wangensteen',\n",
       "   'Yue J Wang',\n",
       "   'Zhixun Dou',\n",
       "   'Amber W Wang',\n",
       "   'Elham Mosleh-Shirazi',\n",
       "   'Max A Horlbeck',\n",
       "   'Luke A Gilbert',\n",
       "   'Jonathan S Weissman',\n",
       "   'Shelley L Berger',\n",
       "   'Klaus H Kaestner'],\n",
       "  'publication_date': '02/11/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '29091290',\n",
       "  'abstract': 'Clustered regularly interspaced short palindromic repeats (CRISPR)/CRISPR-associated 9 activation (CRISPRa) systems have enabled genetic screens in cultured cell lines to discover and characterize drivers and inhibitors of cancer cell growth. We adapted this system for use in vivo to assess whether modulating endogenous gene expression levels can result in functional outcomes in the native environment of the liver. We engineered the catalytically dead CRISPR-associated 9 (dCas9)-positive mouse, cyclization recombination-inducible (Cre) CRISPRa system for cell type-specific gene activation in vivo. We tested the capacity for genetic screening in live animals by applying CRISPRa in a clinically relevant model of liver injury and repopulation. We targeted promoters of interest in regenerating hepatocytes using multiple single guide RNAs (gRNAs), and employed high-throughput sequencing to assess enrichment of gRNA sequences during liver repopulation and to link specific gRNAs to the initiation of carcinogenesis. All components of the CRISPRa system were expressed in a cell type-specific manner and activated endogenous gene expression in vivo. Multiple gRNA cassettes targeting a proto-oncogene were significantly enriched following liver repopulation, indicative of enhanced division of cells expressing the proto-oncogene. Furthermore, hepatocellular carcinomas developed containing gRNAs that activated this oncogene, indicative of cancer initiation events. Also, we employed our system for combinatorial cancer genetics in vivo as we found that while clonal hepatocellular carcinomas were dependent on the presence of the oncogene-inducing gRNAs, they were depleted for multiple gRNAs activating tumor suppressors.\\nThe in vivo CRISPRa platform developed here allows for parallel and combinatorial genetic screens in live animals; this approach enables screening for drivers and suppressors of cell replication and tumor initiation.',\n",
       "  'doi': '10.1002/hep.29626',\n",
       "  'journal': 'Hepatology'},\n",
       " {'title': 'Inducible and multiplex gene regulation using CRISPR-Cpf1-based transcription factors.',\n",
       "  'authors': ['Y Esther Tak',\n",
       "   'Benjamin P Kleinstiver',\n",
       "   'James K Nuñez',\n",
       "   'Jonathan Y Hsu',\n",
       "   'Joy E Horng',\n",
       "   'Jingyi Gong',\n",
       "   'Jonathan S Weissman',\n",
       "   'J Keith Joung'],\n",
       "  'publication_date': '31/10/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '29083402',\n",
       "  'abstract': 'Targeted and inducible regulation of mammalian gene expression is a broadly important capability. We engineered drug-inducible catalytically inactive Cpf1 nuclease fused to transcriptional activation domains to tune the expression of endogenous genes in human cells. Leveraging the multiplex capability of the Cpf1 platform, we demonstrate both synergistic and combinatorial gene expression in human cells. Our work should enable the development of multiplex gene perturbation library screens for understanding complex cellular phenotypes.',\n",
       "  'doi': '10.1038/nmeth.4483',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Combined CRISPRi/a-Based Chemical Genetic Screens Reveal that Rigosertib Is a Microtubule-Destabilizing Agent.',\n",
       "  'authors': ['Marco Jost',\n",
       "   'Yuwen Chen',\n",
       "   'Luke A Gilbert',\n",
       "   'Max A Horlbeck',\n",
       "   'Lenno Krenning',\n",
       "   'Grégory Menchon',\n",
       "   'Ankit Rai',\n",
       "   'Min Y Cho',\n",
       "   'Jacob J Stern',\n",
       "   'Andrea E Prota',\n",
       "   'Martin Kampmann',\n",
       "   'Anna Akhmanova',\n",
       "   'Michel O Steinmetz',\n",
       "   'Marvin E Tanenbaum',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '07/10/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '28985505',\n",
       "  'abstract': \"Chemical libraries paired with phenotypic screens can now readily identify compounds with therapeutic potential. A central limitation to exploiting these compounds, however, has been in identifying their relevant cellular targets. Here, we present a two-tiered CRISPR-mediated chemical-genetic strategy for target identification: combined genome-wide knockdown and overexpression screening as well as focused, comparative chemical-genetic profiling. Application of these strategies to rigosertib, a drug in phase 3 clinical trials for high-risk myelodysplastic syndrome whose molecular target had remained controversial, pointed singularly to microtubules as rigosertib's target. We showed that rigosertib indeed directly binds to and destabilizes microtubules using cell biological, in\\xa0vitro, and structural approaches. Finally, expression of tubulin with a structure-guided mutation in the rigosertib-binding pocket conferred resistance to rigosertib, establishing that rigosertib kills cancer cells by destabilizing microtubules. These results demonstrate the power of our chemical-genetic screening strategies for pinpointing the physiologically relevant targets of chemical agents.\",\n",
       "  'doi': '10.1016/j.molcel.2017.09.012',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'CAT-tailing as a fail-safe mechanism for efficient degradation of stalled nascent polypeptides.',\n",
       "  'authors': ['Kamena K Kostova',\n",
       "   'Kelsey L Hickey',\n",
       "   'Beatriz A Osuna',\n",
       "   'Jeffrey A Hussmann',\n",
       "   'Adam Frost',\n",
       "   'David E Weinberg',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '29/07/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '28751611',\n",
       "  'abstract': 'Ribosome stalling leads to recruitment of the ribosome quality control complex (RQC), which targets the partially synthesized polypeptide for proteasomal degradation through the action of the ubiquitin ligase Ltn1p. A second core RQC component, Rqc2p, modifies the nascent polypeptide by adding a carboxyl-terminal alanine and threonine (CAT) tail through a noncanonical elongation reaction. Here we examined the role of CAT-tailing in nascent-chain degradation in budding yeast. We found that Ltn1p efficiently accessed only nascent-chain lysines immediately proximal to the ribosome exit tunnel. For substrates without Ltn1p-accessible lysines, CAT-tailing enabled degradation by exposing lysines sequestered in the ribosome exit tunnel. Thus, CAT-tails do not serve as a degron, but rather provide a fail-safe mechanism that expands the range of RQC-degradable substrates.',\n",
       "  'doi': '10.1126/science.aam7787',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Suppression of B-cell development genes is key to glucocorticoid efficacy in treatment of acute lymphoblastic leukemia.',\n",
       "  'authors': ['Karina A Kruth',\n",
       "   'Mimi Fang',\n",
       "   'Dawne N Shelton',\n",
       "   'Ossama Abu-Halawa',\n",
       "   'Ryan Mahling',\n",
       "   'Hongxing Yang',\n",
       "   'Jonathan S Weissman',\n",
       "   'Mignon L Loh',\n",
       "   'Markus Müschen',\n",
       "   'Sarah K Tasian',\n",
       "   'Michael C Bassik',\n",
       "   'Martin Kampmann',\n",
       "   'Miles A Pufall'],\n",
       "  'publication_date': '21/04/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '28424165',\n",
       "  'abstract': 'Glucocorticoids (GCs), including dexamethasone (dex), are a central component of combination chemotherapy for childhood B-cell precursor acute lymphoblastic leukemia (B-ALL). GCs work by activating the GC receptor (GR), a ligand-induced transcription factor, which in turn regulates genes that induce leukemic cell death. Which GR-regulated genes are required for GC cytotoxicity, which pathways affect their regulation, and how resistance arises are not well understood. Here, we systematically integrate the transcriptional response of B-ALL to GCs with a next-generation short hairpin RNA screen to identify GC-regulated \"effector\" genes that contribute to cell death, as well as genes that affect the sensitivity of B-ALL cells to dex. This analysis reveals a pervasive role for GCs in suppression of B-cell development genes that is linked to therapeutic response. Inhibition of phosphatidylinositol 3-kinase δ (PI3Kδ), a linchpin in the pre-B-cell receptor and interleukin 7 receptor signaling pathways critical to B-cell development (with CAL-101 [idelalisib]), interrupts a double-negative feedback loop, enhancing GC-regulated transcription to synergistically kill even highly resistant B-ALL with diverse genetic backgrounds. This work not only identifies numerous opportunities for enhanced lymphoid-specific combination chemotherapies that have the potential to overcome treatment resistance, but is also a valuable resource for understanding GC biology and the mechanistic details of GR-regulated transcription.',\n",
       "  'doi': '10.1182/blood-2017-02-766204',\n",
       "  'journal': 'Blood'},\n",
       " {'title': 'Operon mRNAs are organized into ORF-centric structures that predict translation efficiency.',\n",
       "  'authors': ['David H Burkhardt',\n",
       "   'Silvi Rouskin',\n",
       "   'Yan Zhang',\n",
       "   'Gene-Wei Li',\n",
       "   'Jonathan S Weissman',\n",
       "   'Carol A Gross'],\n",
       "  'publication_date': '01/02/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '28139975',\n",
       "  'abstract': 'Bacterial mRNAs are organized into operons consisting of discrete open reading frames (ORFs) in a single polycistronic mRNA. Individual ORFs on the mRNA are differentially translated, with rates varying as much as 100-fold. The signals controlling differential translation are poorly understood. Our genome-wide mRNA secondary structure analysis indicated that operonic mRNAs are comprised of ORF-wide units of secondary structure that vary across ORF boundaries such that adjacent ORFs on the same mRNA molecule are structurally distinct. ORF translation rate is strongly correlated with its mRNA structure in vivo, and correlation persists, albeit in a reduced form, with its structure when translation is inhibited and with that of in vitro refolded mRNA. These data suggest that intrinsic ORF mRNA structure encodes a rough blueprint for translation efficiency. This structure is then amplified by translation, in a self-reinforcing loop, to provide the structure that ultimately specifies the translation of each ORF.',\n",
       "  'doi': '10.7554/eLife.22037',\n",
       "  'journal': 'eLife'},\n",
       " {'title': \"Translation from unconventional 5' start sites drives tumour initiation.\",\n",
       "  'authors': ['Ataman Sendoel',\n",
       "   'Joshua G Dunn',\n",
       "   'Edwin H Rodriguez',\n",
       "   'Shruti Naik',\n",
       "   'Nicholas C Gomez',\n",
       "   'Brian Hurwitz',\n",
       "   'John Levorse',\n",
       "   'Brian D Dill',\n",
       "   'Daniel Schramek',\n",
       "   'Henrik Molina',\n",
       "   'Jonathan S Weissman',\n",
       "   'Elaine Fuchs'],\n",
       "  'publication_date': '13/01/17',\n",
       "  'publication_year': '2017',\n",
       "  'pubmed_id': '28077873',\n",
       "  'abstract': \"We are just beginning to understand how translational control affects tumour initiation and malignancy. Here we use an epidermis-specific, in vivo ribosome profiling strategy to investigate the translational landscape during the transition from normal homeostasis to malignancy. Using a mouse model of inducible SOX2, which is broadly expressed in oncogenic RAS-associated cancers, we show that despite widespread reductions in translation and protein synthesis, certain oncogenic mRNAs are spared. During tumour initiation, the translational apparatus is redirected towards unconventional upstream initiation sites, enhancing the translational efficiency of oncogenic mRNAs. An in vivo RNA interference screen of translational regulators revealed that depletion of conventional eIF2 complexes has adverse effects on normal but not oncogenic growth. Conversely, the alternative initiation factor eIF2A is essential for cancer progression, during which it mediates initiation at these upstream sites, differentially skewing translation and protein expression. Our findings unveil a role for the translation of 5' untranslated regions in cancer, and expose new targets for therapeutic intervention.\",\n",
       "  'doi': '10.1038/nature21036',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Model-guided optogenetic study of PKA signaling in budding yeast.',\n",
       "  'authors': ['Jacob Stewart-Ornstein',\n",
       "   'Susan Chen',\n",
       "   'Rajat Bhatnagar',\n",
       "   'Jonathan S Weissman',\n",
       "   'Hana El-Samad'],\n",
       "  'publication_date': '31/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '28035051',\n",
       "  'abstract': 'In eukaryotes, protein kinase A (PKA) is a master regulator of cell proliferation and survival. The activity of PKA is subject to elaborate control and exhibits complex time dynamics. To probe the quantitative attributes of PKA dynamics in the yeast Saccharomyces cerevisiae, we developed an optogenetic strategy that uses a photoactivatable adenylate cyclase to achieve real-time regulation of cAMP and the PKA pathway. We capitalize on the precise and rapid control afforded by this optogenetic tool, together with quantitative computational modeling, to study the properties of feedback in the PKA signaling network and dissect the nonintuitive dynamic effects that ensue from perturbing its components. Our analyses reveal that negative feedback channeled through the Ras1/2 GTPase is delayed, pinpointing its time scale and its contribution to the dynamic features of the cAMP/PKA signaling network.',\n",
       "  'doi': '10.1091/mbc.E16-06-0354',\n",
       "  'journal': 'Molecular biology of the cell'},\n",
       " {'title': 'A Multiplexed Single-Cell CRISPR Screening Platform Enables Systematic Dissection of the Unfolded Protein Response.',\n",
       "  'authors': ['Britt Adamson',\n",
       "   'Thomas M Norman',\n",
       "   'Marco Jost',\n",
       "   'Min Y Cho',\n",
       "   'James K Nuñez',\n",
       "   'Yuwen Chen',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Luke A Gilbert',\n",
       "   'Max A Horlbeck',\n",
       "   'Marco Y Hein',\n",
       "   'Ryan A Pak',\n",
       "   'Andrew N Gray',\n",
       "   'Carol A Gross',\n",
       "   'Atray Dixit',\n",
       "   'Oren Parnas',\n",
       "   'Aviv Regev',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '17/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27984733',\n",
       "  'abstract': 'Functional genomics efforts face tradeoffs between number of perturbations examined and complexity of phenotypes measured. We bridge this gap with Perturb-seq, which combines droplet-based single-cell RNA-seq with a strategy for barcoding CRISPR-mediated perturbations, allowing many perturbations to be profiled in pooled format. We applied Perturb-seq to dissect the mammalian unfolded protein response (UPR) using single and combinatorial CRISPR perturbations. Two genome-scale CRISPR interference (CRISPRi) screens identified genes whose repression perturbs ER homeostasis. Subjecting ∼100 hits to Perturb-seq enabled high-precision functional clustering of genes. Single-cell analyses decoupled the three UPR branches, revealed bifurcated UPR branch activation among cells\\xa0subject to the same perturbation, and uncovered differential activation of the branches across hits, including an isolated feedback loop between the translocon and IRE1α. These studies provide insight into how the three sensors of ER homeostasis monitor distinct types of stress and highlight the ability of Perturb-seq to dissect complex cellular responses.',\n",
       "  'doi': '10.1016/j.cell.2016.11.048',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Perturb-Seq: Dissecting Molecular Circuits with Scalable Single-Cell RNA Profiling of Pooled Genetic Screens.',\n",
       "  'authors': ['Atray Dixit',\n",
       "   'Oren Parnas',\n",
       "   'Biyu Li',\n",
       "   'Jenny Chen',\n",
       "   'Charles P Fulco',\n",
       "   'Livnat Jerby-Arnon',\n",
       "   'Nemanja D Marjanovic',\n",
       "   'Danielle Dionne',\n",
       "   'Tyler Burks',\n",
       "   'Raktima Raychowdhury',\n",
       "   'Britt Adamson',\n",
       "   'Thomas M Norman',\n",
       "   'Eric S Lander',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nir Friedman',\n",
       "   'Aviv Regev'],\n",
       "  'publication_date': '17/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27984732',\n",
       "  'abstract': 'Genetic screens help infer gene function in mammalian cells, but it has remained difficult to assay complex phenotypes-such as transcriptional profiles-at scale. Here, we develop Perturb-seq, combining single-cell RNA sequencing (RNA-seq) and clustered regularly interspaced short palindromic repeats (CRISPR)-based perturbations to perform many such assays in a pool. We demonstrate Perturb-seq by analyzing 200,000 cells in immune cells and cell lines, focusing on transcription factors regulating the response of dendritic cells to lipopolysaccharide (LPS). Perturb-seq accurately identifies individual gene targets, gene signatures, and cell states affected by individual perturbations and\\xa0their genetic interactions. We posit new functions for regulators of differentiation, the anti-viral response, and mitochondrial function during immune activation. By decomposing many high content measurements into the effects of perturbations, their interactions, and diverse cell metadata, Perturb-seq dramatically increases the scope of pooled genomic assays.',\n",
       "  'doi': '10.1016/j.cell.2016.11.038',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'CRISPRi-based genome-scale identification of functional long noncoding RNA loci in human cells.',\n",
       "  'authors': ['S John Liu',\n",
       "   'Max A Horlbeck',\n",
       "   'Seung Woo Cho',\n",
       "   'Harjus S Birk',\n",
       "   'Martina Malatesta',\n",
       "   'Daniel He',\n",
       "   'Frank J Attenello',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Min Y Cho',\n",
       "   'Yuwen Chen',\n",
       "   'Mohammad A Mandegar',\n",
       "   'Michael P Olvera',\n",
       "   'Luke A Gilbert',\n",
       "   'Bruce R Conklin',\n",
       "   'Howard Y Chang',\n",
       "   'Jonathan S Weissman',\n",
       "   'Daniel A Lim'],\n",
       "  'publication_date': '17/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27980086',\n",
       "  'abstract': 'The human genome produces thousands of long noncoding RNAs (lncRNAs)-transcripts >200 nucleotides long that do not encode proteins. Although critical roles in normal biology and disease have been revealed for a subset of lncRNAs, the function of the vast majority remains untested. We developed a CRISPR interference (CRISPRi) platform targeting 16,401 lncRNA loci in seven diverse cell lines, including six transformed cell lines and human induced pluripotent stem cells (iPSCs). Large-scale screening identified 499 lncRNA loci required for robust cellular growth, of which 89% showed growth-modifying function exclusively in one cell type. We further found that lncRNA knockdown can perturb complex transcriptional networks in a cell type-specific manner. These data underscore the functional importance and cell type specificity of many lncRNAs.',\n",
       "  'doi': '10.1126/science.aah7111',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'The mTOR Complex Controls HIV Latency.',\n",
       "  'authors': ['Emilie Besnard',\n",
       "   'Shweta Hakre',\n",
       "   'Martin Kampmann',\n",
       "   'Hyung W Lim',\n",
       "   'Nina N Hosmane',\n",
       "   'Alyssa Martin',\n",
       "   'Michael C Bassik',\n",
       "   'Erik Verschueren',\n",
       "   'Emilie Battivelli',\n",
       "   'Jonathan Chan',\n",
       "   'J Peter Svensson',\n",
       "   'Andrea Gramatica',\n",
       "   'Ryan J Conrad',\n",
       "   'Melanie Ott',\n",
       "   'Warner C Greene',\n",
       "   'Nevan J Krogan',\n",
       "   'Robert F Siliciano',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eric Verdin'],\n",
       "  'publication_date': '16/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27978436',\n",
       "  'abstract': 'A population of CD4\\xa0T lymphocytes harboring latent HIV genomes can persist in patients on antiretroviral therapy, posing a barrier to HIV eradication. To examine cellular complexes controlling HIV latency, we conducted a genome-wide screen with a pooled ultracomplex shRNA library and in\\xa0vitro system modeling HIV latency and identified the mTOR complex as a modulator of HIV latency. Knockdown of mTOR complex subunits or pharmacological inhibition of mTOR activity suppresses reversal of latency in various HIV-1 latency models and HIV-infected patient cells. mTOR inhibitors suppress HIV transcription both through the viral transactivator Tat and via Tat-independent mechanisms. This inhibition occurs at least in part via blocking the phosphorylation of CDK9, a p-TEFb complex member that serves as a cofactor for Tat-mediated transcription. The control of HIV latency by mTOR signaling identifies a pathway that may have significant therapeutic opportunities.',\n",
       "  'doi': '10.1016/j.chom.2016.11.001',\n",
       "  'journal': 'Cell host & microbe'},\n",
       " {'title': 'The SND proteins constitute an alternative targeting route to the endoplasmic reticulum.',\n",
       "  'authors': ['Naama Aviram',\n",
       "   'Tslil Ast',\n",
       "   'Elizabeth A Costa',\n",
       "   'Eric C Arakel',\n",
       "   'Silvia G Chuartzman',\n",
       "   'Calvin H Jan',\n",
       "   'Sarah Haßdenteufel',\n",
       "   'Johanna Dudek',\n",
       "   'Martin Jung',\n",
       "   'Stefan Schorr',\n",
       "   'Richard Zimmermann',\n",
       "   'Blanche Schwappach',\n",
       "   'Jonathan S Weissman',\n",
       "   'Maya Schuldiner'],\n",
       "  'publication_date': '03/12/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27905431',\n",
       "  'abstract': 'In eukaryotes, up to one-third of cellular proteins are targeted to the endoplasmic reticulum, where they undergo folding, processing, sorting and trafficking to subsequent endomembrane compartments. Targeting to the endoplasmic reticulum has been shown to occur co-translationally by the signal recognition particle (SRP) pathway or post-translationally by the mammalian transmembrane recognition complex of 40\\u2009kDa (TRC40) and homologous yeast guided entry of tail-anchored proteins (GET) pathways. Despite the range of proteins that can be catered for by these two pathways, many proteins are still known to be independent of both SRP and GET, so there seems to be a critical need for an additional dedicated pathway for endoplasmic reticulum relay. We set out to uncover additional targeting proteins using unbiased high-content screening approaches. To this end, we performed a systematic visual screen using the yeast Saccharomyces cerevisiae, and uncovered three uncharacterized proteins whose loss affected targeting. We suggest that these proteins work together and demonstrate that they function in parallel with SRP and GET to target a broad range of substrates to the endoplasmic reticulum. The three proteins, which we name Snd1, Snd2 and Snd3 (for SRP-independent targeting), can synthetically compensate for the loss of both the SRP and GET pathways, and act as a backup targeting system. This explains why it has previously been difficult to demonstrate complete loss of targeting for some substrates. Our discovery thus puts in place an essential piece of the endoplasmic reticulum targeting puzzle, highlighting how the targeting apparatus of the eukaryotic cell is robust, interlinked and flexible.',\n",
       "  'doi': '10.1038/nature20169',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Plastid: nucleotide-resolution analysis of next-generation sequencing and genomics data.',\n",
       "  'authors': ['Joshua G Dunn', 'Jonathan S Weissman'],\n",
       "  'publication_date': '24/11/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27875984',\n",
       "  'abstract': \"Next-generation sequencing (NGS) informs many biological questions with unprecedented depth and nucleotide resolution. These assays have created a need for analytical tools that enable users to manipulate data nucleotide-by-nucleotide robustly and easily. Furthermore, because many NGS assays encode information jointly within multiple properties of read alignments - for example, in ribosome profiling, the locations of ribosomes are jointly encoded in alignment coordinates and length - analytical tools are often required to extract the biological meaning from the alignments before analysis. Many assay-specific pipelines exist for this purpose, but there remains a need for user-friendly, generalized, nucleotide-resolution tools that are not limited to specific experimental regimes or analytical workflows.\\nPlastid is a Python library designed specifically for nucleotide-resolution analysis of genomics and NGS data. As such, Plastid is designed to extract assay-specific information from read alignments while retaining generality and extensibility to novel NGS assays. Plastid represents NGS and other biological data as arrays of values associated with genomic or transcriptomic positions, and contains configurable tools to convert data from a variety of sources to such arrays. Plastid also includes numerous tools to manipulate even discontinuous genomic features, such as spliced transcripts, with nucleotide precision. Plastid automatically handles conversion between genomic and feature-centric coordinates, accounting for splicing and strand, freeing users of burdensome accounting. Finally, Plastid's data models use consistent and familiar biological idioms, enabling even beginners to develop sophisticated analytical workflows with minimal effort.\\nPlastid is a versatile toolkit that has been used to analyze data from multiple NGS assays, including RNA-seq, ribosome profiling, and DMS-seq. It forms the genomic engine of our ORF annotation tool, ORF-RATER, and is readily adapted to novel NGS assays. Examples, tutorials, and extensive documentation can be found at https://plastid.readthedocs.io .\",\n",
       "  'doi': '10.1186/s12864-016-3278-x',\n",
       "  'journal': 'BMC genomics'},\n",
       " {'title': 'DMS-MaPseq for genome-wide or targeted RNA structure probing in vivo.',\n",
       "  'authors': ['Meghan Zubradt',\n",
       "   'Paromita Gupta',\n",
       "   'Sitara Persad',\n",
       "   'Alan M Lambowitz',\n",
       "   'Jonathan S Weissman',\n",
       "   'Silvi Rouskin'],\n",
       "  'publication_date': '08/11/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27819661',\n",
       "  'abstract': \"Coupling of structure-specific in vivo chemical modification to next-generation sequencing is transforming RNA secondary structure studies in living cells. The dominant strategy for detecting in vivo chemical modifications uses reverse transcriptase truncation products, which introduce biases and necessitate population-average assessments of RNA structure. Here we present dimethyl sulfate (DMS) mutational profiling with sequencing (DMS-MaPseq), which encodes DMS modifications as mismatches using a thermostable group II intron reverse transcriptase. DMS-MaPseq yields a high signal-to-noise ratio, can report multiple structural features per molecule, and allows both genome-wide studies and focused in vivo investigations of even low-abundance RNAs. We apply DMS-MaPseq for the first analysis of RNA structure within an animal tissue and to identify a functional structure involved in noncanonical translation initiation. Additionally, we use DMS-MaPseq to compare the in vivo structure of pre-mRNAs with their mature isoforms. These applications illustrate DMS-MaPseq's capacity to dramatically expand in vivo analysis of RNA structure.\",\n",
       "  'doi': '10.1038/nmeth.4057',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Dynamic translation regulation in Caulobacter cell cycle control.',\n",
       "  'authors': ['Jared M Schrader',\n",
       "   'Gene-Wei Li',\n",
       "   'W Seth Childers',\n",
       "   'Adam M Perez',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lucy Shapiro',\n",
       "   'Harley H McAdams'],\n",
       "  'publication_date': '03/11/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27791168',\n",
       "  'abstract': 'Progression of the Caulobacter cell cycle requires temporal and spatial control of gene expression, culminating in an asymmetric cell division yielding distinct daughter cells. To explore the contribution of translational control, RNA-seq and ribosome profiling were used to assay global transcription and translation levels of individual genes at six times over the cell cycle. Translational efficiency (TE) was used as a metric for the relative rate of protein production from each mRNA. TE profiles with similar cell cycle patterns were found across multiple clusters of genes, including those in operons or in subsets of operons. Collections of genes associated with central cell cycle functional modules (e.g., biosynthesis of stalk, flagellum, or chemotaxis machinery) have consistent but different TE temporal patterns, independent of their operon organization. Differential translation of operon-encoded genes facilitates precise cell cycle-timing for the dynamic assembly of multiprotein complexes, such as the flagellum and the stalk and the correct positioning of regulatory proteins to specific cell poles. The cell cycle-regulatory pathways that produce specific temporal TE patterns are separate from-but highly coordinated with-the transcriptional cell cycle circuitry, suggesting that the scheduling of translational regulation is organized by the same cyclical regulatory circuit that directs the transcriptional control of the Caulobacter cell cycle.',\n",
       "  'doi': '10.1073/pnas.1614795113',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Compact and highly active next-generation libraries for CRISPR-mediated gene repression and activation.',\n",
       "  'authors': ['Max A Horlbeck',\n",
       "   'Luke A Gilbert',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Britt Adamson',\n",
       "   'Ryan A Pak',\n",
       "   'Yuwen Chen',\n",
       "   'Alexander P Fields',\n",
       "   'Chong Yon Park',\n",
       "   'Jacob E Corn',\n",
       "   'Martin Kampmann',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '04/11/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27661255',\n",
       "  'abstract': 'We recently found that nucleosomes directly block access of CRISPR/Cas9 to DNA (Horlbeck et al., 2016). Here, we build on this observation with a comprehensive algorithm that incorporates chromatin, position, and sequence features to accurately predict highly effective single guide RNAs (sgRNAs) for targeting nuclease-dead Cas9-mediated transcriptional repression (CRISPRi) and activation (CRISPRa). We use this algorithm to design next-generation genome-scale CRISPRi and CRISPRa libraries targeting human and mouse genomes. A CRISPRi screen for essential genes in K562 cells demonstrates that the large majority of sgRNAs are highly active. We also find CRISPRi does not exhibit any detectable non-specific toxicity recently observed with CRISPR nuclease approaches. Precision-recall analysis shows that we detect over 90% of essential genes with minimal false positives using a compact 5 sgRNA/gene library. Our results establish CRISPRi and CRISPRa as premier tools for loss- or gain-of-function studies and provide a general strategy for identifying Cas9 target sites.',\n",
       "  'doi': '10.7554/eLife.19760',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Combined chemical-genetic approach identifies cytosolic HSP70 dependence in rhabdomyosarcoma.',\n",
       "  'authors': ['Amit J Sabnis',\n",
       "   'Christopher J Guerriero',\n",
       "   'Victor Olivas',\n",
       "   'Anin Sayana',\n",
       "   'Jonathan Shue',\n",
       "   'Jennifer Flanagan',\n",
       "   'Saurabh Asthana',\n",
       "   'Adrienne W Paton',\n",
       "   'James C Paton',\n",
       "   'Jason E Gestwicki',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Peter Wipf',\n",
       "   'Jeffrey L Brodsky',\n",
       "   'Trever G Bivona'],\n",
       "  'publication_date': '28/07/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27450084',\n",
       "  'abstract': 'Cytosolic and organelle-based heat-shock protein (HSP) chaperones ensure proper folding and function of nascent and injured polypeptides to support cell growth. Under conditions of cellular stress, including oncogenic transformation, proteostasis components maintain homeostasis and prevent apoptosis. Although this cancer-relevant function has provided a rationale for therapeutically targeting proteostasis regulators (e.g., HSP90), cancer-subtype dependencies upon particular proteostasis components are relatively undefined. Here, we show that human rhabdomyosarcoma (RMS) cells, but not several other cancer cell types, depend upon heat-shock protein 70 kDA (HSP70) for survival. HSP70-targeted therapy (but not chemotherapeutic agents) promoted apoptosis in RMS cells by triggering an unfolded protein response (UPR) that induced PRKR-like endoplasmic reticulum kinase (PERK)-eukaryotic translation initiation factor α (eIF2α)-CEBP homologous protein (CHOP) signaling and CHOP-mediated cell death. Intriguingly, inhibition of only cytosolic HSP70 induced the UPR, suggesting that the essential activity of HSP70 in RMS cells lies at the endoplasmic reticulum-cytosol interface. We also found that increased CHOP mRNA in clinical specimens was a biomarker for poor outcomes in chemotherapy-treated RMS patients. The data suggest that, like human epidermal growth factor receptor 2 (HER2) amplification in breast cancer, increased CHOP in RMS is a biomarker of decreased response to chemotherapy but enhanced response to targeted therapy. Our findings identify the cytosolic HSP70-UPR axis as an unexpected regulator of RMS pathogenesis, revealing HSP70-targeted therapy as a promising strategy to engage CHOP-mediated apoptosis and improve RMS treatment. Our study highlights the utility of dissecting cancer subtype-specific dependencies on proteostasis networks to uncover unanticipated cancer vulnerabilities.',\n",
       "  'doi': '10.1073/pnas.1603883113',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Ligand-binding domains of nuclear receptors facilitate tight control of split CRISPR activity.',\n",
       "  'authors': ['Duy P Nguyen',\n",
       "   'Yuichiro Miyaoka',\n",
       "   'Luke A Gilbert',\n",
       "   'Steven J Mayerl',\n",
       "   'Brian H Lee',\n",
       "   'Jonathan S Weissman',\n",
       "   'Bruce R Conklin',\n",
       "   'James A Wells'],\n",
       "  'publication_date': '02/07/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27363581',\n",
       "  'abstract': 'Cas9-based RNA-guided nuclease (RGN) has emerged to be a versatile method for genome editing due to the ease of construction of RGN reagents to target specific genomic sequences. The ability to control the activity of Cas9 with a high temporal resolution will facilitate tight regulation of genome editing processes for studying the dynamics of transcriptional regulation or epigenetic modifications in complex biological systems. Here we show that fusing ligand-binding domains of nuclear receptors to split Cas9 protein fragments can provide chemical control over split Cas9 activity. The method has allowed us to control Cas9 activity in a tunable manner with no significant background, which has been challenging for other inducible Cas9 constructs. We anticipate that our design will provide opportunities through the use of different ligand-binding domains to enable multiplexed genome regulation of endogenous genes in distinct loci through simultaneous chemical regulation of orthogonal Cas9 variants.',\n",
       "  'doi': '10.1038/ncomms12009',\n",
       "  'journal': 'Nature communications'},\n",
       " {'title': 'Htm1p-Pdi1p is a folding-sensitive mannosidase that marks N-glycoproteins for ER-associated protein degradation.',\n",
       "  'authors': ['Yi-Chang Liu',\n",
       "   'Danica Galonić Fujimori',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '01/07/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27357682',\n",
       "  'abstract': 'Our understanding of how the endoplasmic reticulum (ER)-associated protein degradation (ERAD) machinery efficiently targets terminally misfolded proteins while avoiding the misidentification of nascent polypeptides and correctly folded proteins is limited. For luminal N-glycoproteins, demannosylation of their N-glycan to expose a terminal α1,6-linked mannose is necessary for their degradation via ERAD, but whether this modification is specific to misfolded proteins is unknown. Here we report that the complex of the mannosidase Htm1p and the protein disulfide isomerase Pdi1p (Htm1p-Pdi1p) acts as a folding-sensitive mannosidase for catalyzing this first committed step in Saccharomyces cerevisiae We reconstitute this step in vitro with Htm1p-Pdi1p and model glycoprotein substrates whose structural states we can manipulate. We find that Htm1p-Pdi1p is a glycoprotein-specific mannosidase that preferentially targets nonnative glycoproteins trapped in partially structured states. As such, Htm1p-Pdi1p is suited to act as a licensing factor that monitors folding in the ER lumen and preferentially commits glycoproteins trapped in partially structured states for degradation.',\n",
       "  'doi': '10.1073/pnas.1608795113',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Versatile in vivo regulation of tumor phenotypes by dCas9-mediated transcriptional perturbation.',\n",
       "  'authors': ['Christian J Braun',\n",
       "   'Peter M Bruno',\n",
       "   'Max A Horlbeck',\n",
       "   'Luke A Gilbert',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael T Hemann'],\n",
       "  'publication_date': '22/06/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27325776',\n",
       "  'abstract': 'Targeted transcriptional regulation is a powerful tool to study genetic mediators of cellular behavior. Here, we show that catalytically dead Cas9 (dCas9) targeted to genomic regions upstream or downstream of the transcription start site allows for specific and sustainable gene-expression level alterations in tumor cells in vitro and in syngeneic immune-competent mouse models. We used this approach for a high-coverage pooled gene-activation screen in vivo and discovered previously unidentified modulators of tumor growth and therapeutic response. Moreover, by using dCas9 linked to an activation domain, we can either enhance or suppress target gene expression simply by changing the genetic location of dCas9 binding relative to the transcription start site. We demonstrate that these directed changes in gene-transcription levels occur with minimal off-target effects. Our findings highlight the use of dCas9-mediated transcriptional regulation as a versatile tool to reproducibly interrogate tumor phenotypes in vivo.',\n",
       "  'doi': '10.1073/pnas.1600582113',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'A scalable strategy for high-throughput GFP tagging of endogenous human proteins.',\n",
       "  'authors': ['Manuel D Leonetti',\n",
       "   'Sayaka Sekine',\n",
       "   'Daichi Kamiyama',\n",
       "   'Jonathan S Weissman',\n",
       "   'Bo Huang'],\n",
       "  'publication_date': '09/06/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27274053',\n",
       "  'abstract': 'A central challenge of the postgenomic era is to comprehensively characterize the cellular role of the ∼20,000 proteins encoded in the human genome. To systematically study protein function in a native cellular background, libraries of human cell lines expressing proteins tagged with a functional sequence at their endogenous loci would be very valuable. Here, using electroporation of Cas9 nuclease/single-guide RNA ribonucleoproteins and taking advantage of a split-GFP system, we describe a scalable method for the robust, scarless, and specific tagging of endogenous human genes with GFP. Our approach requires no molecular cloning and allows a large number of cell lines to be processed in parallel. We demonstrate the scalability of our method by targeting 48 human genes and show that the resulting GFP fluorescence correlates with protein expression levels. We next present how our protocols can be easily adapted for the tagging of a given target with GFP repeats, critically enabling the study of low-abundance proteins. Finally, we show that our GFP tagging approach allows the biochemical isolation of native protein complexes for proteomic studies. Taken together, our results pave the way for the large-scale generation of endogenously tagged human cell lines for the proteome-wide analysis of protein localization and interaction networks in a native cellular context.',\n",
       "  'doi': '10.1073/pnas.1606731113',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'A Comprehensive, CRISPR-based Functional Analysis of Essential Genes in Bacteria.',\n",
       "  'authors': ['Jason M Peters',\n",
       "   'Alexandre Colavin',\n",
       "   'Handuo Shi',\n",
       "   'Tomasz L Czarny',\n",
       "   'Matthew H Larson',\n",
       "   'Spencer Wong',\n",
       "   'John S Hawkins',\n",
       "   'Candy H S Lu',\n",
       "   'Byoung-Mo Koo',\n",
       "   'Elizabeth Marta',\n",
       "   'Anthony L Shiver',\n",
       "   'Evan H Whitehead',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eric D Brown',\n",
       "   'Lei S Qi',\n",
       "   'Kerwyn Casey Huang',\n",
       "   'Carol A Gross'],\n",
       "  'publication_date': '31/05/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27238023',\n",
       "  'abstract': 'Essential gene functions underpin the core reactions required for cell viability, but their contributions and\\xa0relationships are poorly studied in\\xa0vivo. Using CRISPR interference, we created knockdowns of every essential gene in Bacillus subtilis and probed their phenotypes. Our high-confidence essential gene network, established using chemical genomics, showed extensive interconnections among distantly related processes and identified modes of action for uncharacterized antibiotics. Importantly, mild knockdown of essential gene functions significantly reduced stationary-phase survival without affecting maximal growth rate, suggesting that essential protein levels are set to maximize outgrowth from stationary phase. Finally, high-throughput microscopy indicated that cell morphology is relatively insensitive to mild knockdown but profoundly affected by depletion of gene function, revealing intimate connections between cell growth and shape. Our results provide a framework for systematic investigation of essential gene functions in\\xa0vivo broadly applicable to diverse microorganisms and amenable to comparative analysis. ',\n",
       "  'doi': '10.1016/j.cell.2016.05.003',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Single-cell analysis of long non-coding RNAs in the developing human neocortex.',\n",
       "  'authors': ['Siyuan John Liu',\n",
       "   'Tomasz J Nowakowski',\n",
       "   'Alex A Pollen',\n",
       "   'Jan H Lui',\n",
       "   'Max A Horlbeck',\n",
       "   'Frank J Attenello',\n",
       "   'Daniel He',\n",
       "   'Jonathan S Weissman',\n",
       "   'Arnold R Kriegstein',\n",
       "   'Aaron A Diaz',\n",
       "   'Daniel A Lim'],\n",
       "  'publication_date': '16/04/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '27081004',\n",
       "  'abstract': 'Long non-coding RNAs (lncRNAs) comprise a diverse class of transcripts that can regulate molecular and cellular processes in brain development and disease. LncRNAs exhibit cell type- and tissue-specific expression, but little is known about the expression and function of lncRNAs in the developing human brain. Furthermore, it has been unclear whether lncRNAs are highly expressed in subsets of cells within tissues, despite appearing lowly expressed in bulk populations.\\nWe use strand-specific RNA-seq to deeply profile lncRNAs from polyadenylated and total RNA obtained from human neocortex at different stages of development, and we apply this reference to analyze the transcriptomes of single cells. While lncRNAs are generally detected at low levels in bulk tissues, single-cell transcriptomics of hundreds of neocortex cells reveal that many lncRNAs are abundantly expressed in individual cells and are cell type-specific. Notably, LOC646329 is a lncRNA enriched in single radial glia cells but is detected at low abundance in tissues. CRISPRi knockdown of LOC646329 indicates that this lncRNA regulates cell proliferation.\\nThe discrete and abundant expression of lncRNAs among individual cells has important implications for both their biological function and utility for distinguishing neural cell types.',\n",
       "  'doi': '10.1186/s13059-016-0932-1',\n",
       "  'journal': 'Genome biology'},\n",
       " {'title': 'Versatile protein tagging in cells with split fluorescent protein.',\n",
       "  'authors': ['Daichi Kamiyama',\n",
       "   'Sayaka Sekine',\n",
       "   'Benjamin Barsi-Rhyne',\n",
       "   'Jeffrey Hu',\n",
       "   'Baohui Chen',\n",
       "   'Luke A Gilbert',\n",
       "   'Hiroaki Ishikawa',\n",
       "   'Manuel D Leonetti',\n",
       "   'Wallace F Marshall',\n",
       "   'Jonathan S Weissman',\n",
       "   'Bo Huang'],\n",
       "  'publication_date': '19/03/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '26988139',\n",
       "  'abstract': 'In addition to the popular method of fluorescent protein fusion, live cell protein imaging has now seen more and more application of epitope tags. The small size of these tags may reduce functional perturbation and enable signal amplification. To address their background issue, we adapt self-complementing split fluorescent proteins as epitope tags for live cell protein labelling. The two tags, GFP11 and sfCherry11 are derived from the eleventh β-strand of super-folder GFP and sfCherry, respectively. The small size of FP11-tags enables a cost-effective and scalable way to insert them into endogenous genomic loci via CRISPR-mediated homology-directed repair. Tandem arrangement FP11-tags allows proportional enhancement of fluorescence signal in tracking intraflagellar transport particles, or reduction of photobleaching for live microtubule imaging. Finally, we show the utility of tandem GFP11-tag in scaffolding protein oligomerization. These experiments illustrate the versatility of FP11-tag as a labelling tool as well as a multimerization-control tool for both imaging and non-imaging applications. ',\n",
       "  'doi': '10.1038/ncomms11046',\n",
       "  'journal': 'Nature communications'},\n",
       " {'title': 'Nucleosomes impede Cas9 access to DNA in vivo and in vitro.',\n",
       "  'authors': ['Max A Horlbeck',\n",
       "   'Lea B Witkowsky',\n",
       "   'Benjamin Guglielmi',\n",
       "   'Joseph M Replogle',\n",
       "   'Luke A Gilbert',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Sharon E Torigoe',\n",
       "   'Robert Tjian',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '18/03/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '26987018',\n",
       "  'abstract': 'The prokaryotic CRISPR (clustered regularly interspaced palindromic repeats)-associated protein, Cas9, has been widely adopted as a tool for editing, imaging, and regulating eukaryotic genomes. However, our understanding of how to select single-guide RNAs (sgRNAs) that mediate efficient Cas9 activity is incomplete, as we lack insight into how chromatin impacts Cas9 targeting. To address this gap, we analyzed large-scale genetic screens performed in human cell lines using either nuclease-active or nuclease-dead Cas9 (dCas9). We observed that highly active sgRNAs for Cas9 and dCas9 were found almost exclusively in regions of low nucleosome occupancy. In vitro experiments demonstrated that nucleosomes in fact directly impede Cas9 binding and cleavage, while chromatin remodeling can restore Cas9 access. Our results reveal a critical role of eukaryotic chromatin in dictating the targeting specificity of this transplanted bacterial enzyme, and provide rules for selecting Cas9 target sites distinct from and complementary to those based on sequence properties.',\n",
       "  'doi': '10.7554/eLife.12677',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'CRISPR Interference Efficiently Induces Specific and Reversible Gene Silencing in Human iPSCs.',\n",
       "  'authors': ['Mohammad A Mandegar',\n",
       "   'Nathaniel Huebsch',\n",
       "   'Ekaterina B Frolov',\n",
       "   'Edward Shin',\n",
       "   'Annie Truong',\n",
       "   'Michael P Olvera',\n",
       "   'Amanda H Chan',\n",
       "   'Yuichiro Miyaoka',\n",
       "   'Kristin Holmes',\n",
       "   'C Ian Spencer',\n",
       "   'Luke M Judge',\n",
       "   'David E Gordon',\n",
       "   'Tilde V Eskildsen',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Max A Horlbeck',\n",
       "   'Luke A Gilbert',\n",
       "   'Nevan J Krogan',\n",
       "   'Søren P Sheikh',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lei S Qi',\n",
       "   'Po-Lin So',\n",
       "   'Bruce R Conklin'],\n",
       "  'publication_date': '15/03/16',\n",
       "  'publication_year': '2016',\n",
       "  'pubmed_id': '26971820',\n",
       "  'abstract': 'Developing technologies for efficient and scalable disruption of gene expression will provide powerful tools for studying gene function, developmental pathways, and disease mechanisms. Here, we develop clustered regularly interspaced short palindromic repeat interference (CRISPRi) to repress gene expression in human induced pluripotent stem cells (iPSCs). CRISPRi, in which a doxycycline-inducible deactivated Cas9 is fused to a KRAB repression domain, can specifically and reversibly inhibit gene expression in iPSCs and iPSC-derived cardiac progenitors, cardiomyocytes, and T lymphocytes. This gene repression system is tunable and has the potential to silence single alleles. Compared with CRISPR nuclease (CRISPRn), CRISPRi gene repression is more efficient and homogenous across cell populations. The CRISPRi system in iPSCs provides a powerful platform to perform genome-scale screens in a wide range of iPSC-derived cell types, dissect developmental pathways, and model disease.',\n",
       "  'doi': '10.1016/j.stem.2016.01.022',\n",
       "  'journal': 'Cell stem cell'},\n",
       " {'title': 'A Regression-Based Analysis of Ribosome-Profiling Data Reveals a Conserved Complexity to Mammalian Translation.',\n",
       "  'authors': ['Alexander P Fields',\n",
       "   'Edwin H Rodriguez',\n",
       "   'Marko Jovanovic',\n",
       "   'Noam Stern-Ginossar',\n",
       "   'Brian J Haas',\n",
       "   'Philipp Mertins',\n",
       "   'Raktima Raychowdhury',\n",
       "   'Nir Hacohen',\n",
       "   'Steven A Carr',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Aviv Regev',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '08/12/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26638175',\n",
       "  'abstract': 'A fundamental goal of genomics is to identify the complete set of expressed proteins. Automated annotation strategies rely on assumptions about protein-coding sequences (CDSs), e.g., they are conserved, do not overlap, and exceed a minimum length. However, an increasing number of newly discovered proteins violate these rules. Here we present an experimental and analytical framework, based on ribosome profiling and linear regression, for systematic identification and quantification of translation. Application of this approach to lipopolysaccharide-stimulated mouse dendritic cells and HCMV-infected human fibroblasts identifies thousands of novel CDSs, including micropeptides and variants of known proteins, that bear the hallmarks of canonical translation and exhibit translation levels and dynamics comparable to that of annotated CDSs. Remarkably, many translation events are identified in both mouse and human cells even when the peptide sequence is not conserved. Our work thus reveals an unexpected complexity to mammalian translation suited to provide both conserved regulatory or protein-based functions.',\n",
       "  'doi': '10.1016/j.molcel.2015.11.013',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': '[KIL-d] Protein Element Confers Antiviral Activity via Catastrophic Viral Mutagenesis.',\n",
       "  'authors': ['Genjiro Suzuki', 'Jonathan S Weissman', 'Motomasa Tanaka'],\n",
       "  'publication_date': '23/11/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26590718',\n",
       "  'abstract': 'Eukaryotic cells are targeted by pathogenic viruses and have developed cell defense mechanisms against viral infection. In yeast, the cellular extrachromosomal genetic element [KIL-d] alters killer activity of M double-stranded RNA killer virus and confers cell resistance against the killer virus. However, its underlying mechanism and the molecular nature of [KIL-d] are unknown. Here, we demonstrate that [KIL-d] is a proteinaceous prion-like aggregate with non-Mendelian cytoplasmic transmission. Deep sequencing analyses revealed that [KIL-d] selectively increases the rate of de novo mutation in the killer toxin gene of the viral genome, producing yeast harboring a defective mutant killer virus with a selective growth advantage over those with WT killer virus. These results suggest that a prion-like [KIL-d] element reprograms the viral replication machinery to induce mutagenesis and genomic inactivation via the long-hypothesized mechanism of \"error catastrophe.\" The findings also support a role for prion-like protein aggregates in cellular defense and adaptation. ',\n",
       "  'doi': '10.1016/j.molcel.2015.10.020',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Targeting the AAA ATPase p97 as an Approach to Treat Cancer through Disruption of Protein Homeostasis.',\n",
       "  'authors': ['Daniel J Anderson',\n",
       "   'Ronan Le Moigne',\n",
       "   'Stevan Djakovic',\n",
       "   'Brajesh Kumar',\n",
       "   'Julie Rice',\n",
       "   'Steve Wong',\n",
       "   'Jinhai Wang',\n",
       "   'Bing Yao',\n",
       "   'Eduardo Valle',\n",
       "   'Szerenke Kiss von Soly',\n",
       "   'Antonett Madriaga',\n",
       "   'Ferdie Soriano',\n",
       "   'Mary-Kamala Menon',\n",
       "   'Zhi Yong Wu',\n",
       "   'Martin Kampmann',\n",
       "   'Yuwen Chen',\n",
       "   'Jonathan S Weissman',\n",
       "   'Blake T Aftab',\n",
       "   'F Michael Yakes',\n",
       "   'Laura Shawver',\n",
       "   'Han-Jie Zhou',\n",
       "   'David Wustrow',\n",
       "   'Mark Rolfe'],\n",
       "  'publication_date': '12/11/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26555175',\n",
       "  'abstract': 'p97 is a AAA-ATPase with multiple cellular functions, one of which is critical regulation of protein homeostasis pathways. We describe the characterization of CB-5083, a potent, selective, and orally bioavailable inhibitor of p97. Treatment of tumor cells with CB-5083 leads to accumulation of poly-ubiquitinated proteins, retention of endoplasmic reticulum-associated degradation (ERAD) substrates, and generation of irresolvable proteotoxic stress, leading to activation of the apoptotic arm of the unfolded protein response. In xenograft models, CB-5083 causes modulation of key p97-related pathways, induces apoptosis, and has antitumor activity in a broad range of both hematological and solid tumor models. Molecular determinants of CB-5083 activity include expression of genes in the ERAD pathway, providing a potential strategy for patient selection. ',\n",
       "  'doi': '10.1016/j.ccell.2015.10.002',\n",
       "  'journal': 'Cancer cell'},\n",
       " {'title': 'Ribosome profiling reveals the what, when, where and how of protein synthesis.',\n",
       "  'authors': ['Gloria A Brar', 'Jonathan S Weissman'],\n",
       "  'publication_date': '16/10/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26465719',\n",
       "  'abstract': 'Ribosome profiling, which involves the deep sequencing of ribosome-protected mRNA fragments, is a powerful tool for globally monitoring translation in vivo. The method has facilitated discovery of the regulation of gene expression underlying diverse and complex biological processes, of important aspects of the mechanism of protein synthesis, and even of new proteins, by providing a systematic approach for experimental annotation of coding regions. Here, we introduce the methodology of ribosome profiling and discuss examples in which this approach has been a key factor in guiding biological discovery, including its prominent role in identifying thousands of novel translated short open reading frames and alternative translation products. ',\n",
       "  'doi': '10.1038/nrm4069',\n",
       "  'journal': 'Nature reviews. Molecular cell biology'},\n",
       " {'title': 'Paradoxical resistance of multiple myeloma to proteasome inhibitors by decreased levels of 19S proteasomal subunits.',\n",
       "  'authors': ['Diego Acosta-Alvear',\n",
       "   'Min Y Cho',\n",
       "   'Thomas Wild',\n",
       "   'Tonia J Buchholz',\n",
       "   'Alana G Lerner',\n",
       "   'Olga Simakova',\n",
       "   'Jamie Hahn',\n",
       "   'Neha Korde',\n",
       "   'Ola Landgren',\n",
       "   'Irina Maric',\n",
       "   'Chunaram Choudhary',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Martin Kampmann'],\n",
       "  'publication_date': '04/09/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26327694',\n",
       "  'abstract': 'Hallmarks of cancer, including rapid growth and aneuploidy, can result in non-oncogene addiction to the proteostasis network that can be exploited clinically. The defining example is the exquisite sensitivity of multiple myeloma (MM) to 20S proteasome inhibitors, such as carfilzomib. However, MM patients invariably acquire resistance to these drugs. Using a next-generation shRNA platform, we found that proteostasis factors, including chaperones and stress-response regulators, controlled the response to carfilzomib. Paradoxically, 19S proteasome regulator knockdown induced resistance to carfilzomib in MM and non-MM cells. 19S subunit knockdown did not affect the activity of the 20S subunits targeted by carfilzomib nor their inhibition by the drug, suggesting an alternative mechanism, such as the selective accumulation of protective factors. In MM patients, lower 19S levels predicted a diminished response to carfilzomib-based therapies. Together, our findings suggest that an understanding of network rewiring can inform development of new combination therapies to overcome drug resistance. ',\n",
       "  'doi': '10.7554/eLife.08153',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Regulation of mRNA translation during mitosis.',\n",
       "  'authors': ['Marvin E Tanenbaum',\n",
       "   'Noam Stern-Ginossar',\n",
       "   'Jonathan S Weissman',\n",
       "   'Ronald D Vale'],\n",
       "  'publication_date': '26/08/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26305499',\n",
       "  'abstract': 'Passage through mitosis is driven by precisely-timed changes in transcriptional regulation and protein degradation. However, the importance of translational regulation during mitosis remains poorly understood. Here, using ribosome profiling, we find both a global translational repression and identified ~200 mRNAs that undergo specific translational regulation at mitotic entry. In contrast, few changes in mRNA abundance are observed, indicating that regulation of translation is the primary mechanism of modulating protein expression during mitosis. Interestingly, 91% of the mRNAs that undergo gene-specific regulation in mitosis are translationally repressed, rather than activated. One of the most pronounced translationally-repressed genes is Emi1, an inhibitor of the anaphase promoting complex (APC) which is degraded during mitosis. We show that full APC activation requires translational repression of Emi1 in addition to its degradation. These results identify gene-specific translational repression as a means of controlling the mitotic proteome, which may complement post-translational mechanisms for inactivating protein function.',\n",
       "  'doi': '10.7554/eLife.07957',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Next-generation libraries for robust RNA interference-based genome-wide screens.',\n",
       "  'authors': ['Martin Kampmann',\n",
       "   'Max A Horlbeck',\n",
       "   'Yuwen Chen',\n",
       "   'Jordan C Tsai',\n",
       "   'Michael C Bassik',\n",
       "   'Luke A Gilbert',\n",
       "   'Jacqueline E Villalta',\n",
       "   'S Chul Kwon',\n",
       "   'Hyeshik Chang',\n",
       "   'V Narry Kim',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '17/06/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26080438',\n",
       "  'abstract': 'Genetic screening based on loss-of-function phenotypes is a powerful discovery tool in biology. Although the recent development of clustered regularly interspaced short palindromic repeats (CRISPR)-based screening approaches in mammalian cell culture has enormous potential, RNA interference (RNAi)-based screening remains the method of choice in several biological contexts. We previously demonstrated that ultracomplex pooled short-hairpin RNA (shRNA) libraries can largely overcome the problem of RNAi off-target effects in genome-wide screens. Here, we systematically optimize several aspects of our shRNA library, including the promoter and microRNA context for shRNA expression, selection of guide strands, and features relevant for postscreen sample preparation for deep sequencing. We present next-generation high-complexity libraries targeting human and mouse protein-coding genes, which we grouped into 12 sublibraries based on biological function. A pilot screen suggests that our next-generation RNAi library performs comparably to current CRISPR interference (CRISPRi)-based approaches and can yield complementary results with high sensitivity and high specificity. ',\n",
       "  'doi': '10.1073/pnas.1508821112',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'LOCAL TRANSLATION. Response to Comment on \"Principles of ER cotranslational translocation revealed by proximity-specific ribosome profiling\".',\n",
       "  'authors': ['Calvin H Jan', 'Christopher C Williams', 'Jonathan S Weissman'],\n",
       "  'publication_date': '13/06/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '26068842',\n",
       "  'abstract': 'Reid and Nicchitta propose that most cellular translation is carried out by a noncycling pool of endoplasmic reticulum (ER)-associated ribosomes. However, proximity-specific ribosome profiling data place an upper bound of about 7 to 16% on the fraction of cytosolic protein translation carried out by ribosomes accessible to ER-tethered biotin ligases. Moreover, yeast pulse-labeling experiments argue against there being a static population of ER-associated ribosomes. ',\n",
       "  'doi': '10.1126/science.aaa8299',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Pharmacological dimerization and activation of the exchange factor eIF2B antagonizes the integrated stress response.',\n",
       "  'authors': ['Carmela Sidrauski',\n",
       "   'Jordan C Tsai',\n",
       "   'Martin Kampmann',\n",
       "   'Brian R Hearn',\n",
       "   'Punitha Vedantham',\n",
       "   'Priyadarshini Jaishankar',\n",
       "   'Masaaki Sokabe',\n",
       "   'Aaron S Mendez',\n",
       "   'Billy W Newton',\n",
       "   'Edward L Tang',\n",
       "   'Erik Verschueren',\n",
       "   'Jeffrey R Johnson',\n",
       "   'Nevan J Krogan',\n",
       "   'Christopher S Fraser',\n",
       "   'Jonathan S Weissman',\n",
       "   'Adam R Renslo',\n",
       "   'Peter Walter'],\n",
       "  'publication_date': '16/04/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25875391',\n",
       "  'abstract': 'The general translation initiation factor eIF2 is a major translational control point. Multiple signaling pathways in the integrated stress response phosphorylate eIF2 serine-51, inhibiting nucleotide exchange by eIF2B. ISRIB, a potent drug-like small molecule, renders cells insensitive to eIF2α phosphorylation and enhances cognitive function in rodents by blocking long-term depression. ISRIB was identified in a phenotypic cell-based screen, and its mechanism of action remained unknown. We now report that ISRIB is an activator of eIF2B. Our reporter-based shRNA screen revealed an eIF2B requirement for ISRIB activity. Our results define ISRIB as a symmetric molecule, show ISRIB-mediated stabilization of activated eIF2B dimers, and suggest that eIF2B4 (δ-subunit) contributes to the ISRIB binding site. We also developed new ISRIB analogs, improving its EC50 to 600 pM in cell culture. By modulating eIF2B function, ISRIB promises to be an invaluable tool in proof-of-principle studies aiming to ameliorate cognitive defects resulting from neurodegenerative diseases. ',\n",
       "  'doi': '10.7554/eLife.07314',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Corrigendum: functional genomics platform for pooled screening and generation of mammalian genetic interaction maps.',\n",
       "  'authors': ['Martin Kampmann', 'Michael C Bassik', 'Jonathan S Weissman'],\n",
       "  'publication_date': '27/03/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25811902',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/nprot0415-644d',\n",
       "  'journal': 'Nature protocols'},\n",
       " {'title': 'Biotechnology. A prudent path forward for genomic engineering and germline gene modification.',\n",
       "  'authors': ['David Baltimore',\n",
       "   'Paul Berg',\n",
       "   'Michael Botchan',\n",
       "   'Dana Carroll',\n",
       "   'R Alta Charo',\n",
       "   'George Church',\n",
       "   'Jacob E Corn',\n",
       "   'George Q Daley',\n",
       "   'Jennifer A Doudna',\n",
       "   'Marsha Fenner',\n",
       "   'Henry T Greely',\n",
       "   'Martin Jinek',\n",
       "   'G Steven Martin',\n",
       "   'Edward Penhoet',\n",
       "   'Jennifer Puck',\n",
       "   'Samuel H Sternberg',\n",
       "   'Jonathan S Weissman',\n",
       "   'Keith R Yamamoto'],\n",
       "  'publication_date': '21/03/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25791083',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1126/science.aab1028',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Immunogenetics. Dynamic profiling of the protein life cycle in response to pathogens.',\n",
       "  'authors': ['Marko Jovanovic',\n",
       "   'Michael S Rooney',\n",
       "   'Philipp Mertins',\n",
       "   'Dariusz Przybylski',\n",
       "   'Nicolas Chevrier',\n",
       "   'Rahul Satija',\n",
       "   'Edwin H Rodriguez',\n",
       "   'Alexander P Fields',\n",
       "   'Schraga Schwartz',\n",
       "   'Raktima Raychowdhury',\n",
       "   'Maxwell R Mumbach',\n",
       "   'Thomas Eisenhaure',\n",
       "   'Michal Rabani',\n",
       "   'Dave Gennert',\n",
       "   'Diana Lu',\n",
       "   'Toni Delorey',\n",
       "   'Jonathan S Weissman',\n",
       "   'Steven A Carr',\n",
       "   'Nir Hacohen',\n",
       "   'Aviv Regev'],\n",
       "  'publication_date': '07/03/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25745177',\n",
       "  'abstract': 'Protein expression is regulated by the production and degradation of messenger RNAs (mRNAs) and proteins, but their specific relationships remain unknown. We combine measurements of protein production and degradation and mRNA dynamics so as to build a quantitative genomic model of the differential regulation of gene expression in lipopolysaccharide-stimulated mouse dendritic cells. Changes in mRNA abundance play a dominant role in determining most dynamic fold changes in protein levels. Conversely, the preexisting proteome of proteins performing basic cellular functions is remodeled primarily through changes in protein production or degradation, accounting for more than half of the absolute change in protein molecules in the cell. Thus, the proteome is regulated by transcriptional induction for newly activated cellular functions and by protein life-cycle changes for remodeling of preexisting functions. ',\n",
       "  'doi': '10.1126/science.1259038',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Validation of the Hsp70-Bag3 protein-protein interaction as a potential therapeutic target in cancer.',\n",
       "  'authors': ['Xiaokai Li',\n",
       "   'Teresa Colvin',\n",
       "   'Jennifer N Rauch',\n",
       "   'Diego Acosta-Alvear',\n",
       "   'Martin Kampmann',\n",
       "   'Bryan Dunyak',\n",
       "   'Byron Hann',\n",
       "   'Blake T Aftab',\n",
       "   'Megan Murnane',\n",
       "   'Min Cho',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael Y Sherman',\n",
       "   'Jason E Gestwicki'],\n",
       "  'publication_date': '08/01/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25564440',\n",
       "  'abstract': 'Hsp70 is a stress-inducible molecular chaperone that is required for cancer development at several steps. Targeting the active site of Hsp70 has proven relatively challenging, driving interest in alternative approaches. Hsp70 collaborates with the Bcl2-associated athanogene 3 (Bag3) to promote cell survival through multiple pathways, including FoxM1. Therefore, inhibitors of the Hsp70-Bag3 protein-protein interaction (PPI) may provide a noncanonical way to target this chaperone. We report that JG-98, an allosteric inhibitor of this PPI, indeed has antiproliferative activity (EC50 values between 0.3 and 4 μmol/L) across cancer cell lines from multiple origins. JG-98 destabilized FoxM1 and relieved suppression of downstream effectors, including p21 and p27. On the basis of these findings, JG-98 was evaluated in mice for pharmacokinetics, tolerability, and activity in two xenograft models. The results suggested that the Hsp70-Bag3 interaction may be a promising, new target for anticancer therapy.',\n",
       "  'doi': '10.1158/1535-7163.MCT-14-0650',\n",
       "  'journal': 'Molecular cancer therapeutics'},\n",
       " {'title': 'Protein synthesis. Rqc2p and 60S ribosomal subunits mediate mRNA-independent elongation of nascent chains.',\n",
       "  'authors': ['Peter S Shen',\n",
       "   'Joseph Park',\n",
       "   'Yidan Qin',\n",
       "   'Xueming Li',\n",
       "   'Krishna Parsawar',\n",
       "   'Matthew H Larson',\n",
       "   'James Cox',\n",
       "   'Yifan Cheng',\n",
       "   'Alan M Lambowitz',\n",
       "   'Jonathan S Weissman',\n",
       "   'Onn Brandman',\n",
       "   'Adam Frost'],\n",
       "  'publication_date': '03/01/15',\n",
       "  'publication_year': '2015',\n",
       "  'pubmed_id': '25554787',\n",
       "  'abstract': 'In Eukarya, stalled translation induces 40S dissociation and recruitment of the ribosome quality control complex (RQC) to the 60S subunit, which mediates nascent chain degradation. Here we report cryo-electron microscopy structures revealing that the RQC components Rqc2p (YPL009C/Tae2) and Ltn1p (YMR247C/Rkr1) bind to the 60S subunit at sites exposed after 40S dissociation, placing the Ltn1p RING (Really Interesting New Gene) domain near the exit channel and Rqc2p over the P-site transfer RNA (tRNA). We further demonstrate that Rqc2p recruits alanine- and threonine-charged tRNA to the A site and directs the elongation of nascent chains independently of mRNA or 40S subunits. Our work uncovers an unexpected mechanism of protein synthesis, in which a protein--not an mRNA--determines tRNA recruitment and the tagging of nascent chains with carboxy-terminal Ala and Thr extensions (\"CAT tails\").',\n",
       "  'doi': '10.1126/science.1259724',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Causal signals between codon bias, mRNA structure, and the efficiency of translation and elongation.',\n",
       "  'authors': ['Cristina Pop',\n",
       "   'Silvi Rouskin',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Lu Han',\n",
       "   'Eric M Phizicky',\n",
       "   'Jonathan S Weissman',\n",
       "   'Daphne Koller'],\n",
       "  'publication_date': '30/12/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25538139',\n",
       "  'abstract': 'Ribosome profiling data report on the distribution of translating ribosomes, at steady-state, with codon-level resolution. We present a robust method to extract codon translation rates and protein synthesis rates from these data, and identify causal features associated with elongation and translation efficiency in physiological conditions in yeast. We show that neither elongation rate nor translational efficiency is improved by experimental manipulation of the abundance or body sequence of the rare AGG tRNA. Deletion of three of the four copies of the heavily used ACA tRNA shows a modest efficiency decrease that could be explained by other rate-reducing signals at gene start. This suggests that correlation between codon bias and efficiency arises as selection for codons to utilize translation machinery efficiently in highly translated genes. We also show a correlation between efficiency and RNA structure calculated both computationally and from recent structure probing data, as well as the Kozak initiation motif, which may comprise a mechanism to regulate initiation. ',\n",
       "  'doi': '10.15252/msb.20145524',\n",
       "  'journal': 'Molecular systems biology'},\n",
       " {'title': 'Engineering complex synthetic transcriptional programs with CRISPR RNA scaffolds.',\n",
       "  'authors': ['Jesse G Zalatan',\n",
       "   'Michael E Lee',\n",
       "   'Ricardo Almeida',\n",
       "   'Luke A Gilbert',\n",
       "   'Evan H Whitehead',\n",
       "   'Marie La Russa',\n",
       "   'Jordan C Tsai',\n",
       "   'Jonathan S Weissman',\n",
       "   'John E Dueber',\n",
       "   'Lei S Qi',\n",
       "   'Wendell A Lim'],\n",
       "  'publication_date': '24/12/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25533786',\n",
       "  'abstract': 'Eukaryotic cells execute complex transcriptional programs in which specific loci throughout the genome are regulated in distinct ways by targeted regulatory assemblies. We have applied this principle to generate synthetic CRISPR-based transcriptional programs in yeast and human cells. By extending guide RNAs to include effector protein recruitment sites, we construct modular scaffold RNAs that encode both target locus and regulatory action. Sets of scaffold RNAs can be used to generate synthetic multigene transcriptional programs in which some genes are activated and others are repressed. We apply this approach to flexibly redirect flux through a complex branched metabolic pathway in yeast. Moreover, these programs can be executed by inducing expression of the dCas9 protein, which acts as a single master regulatory control point. CRISPR-associated RNA scaffolds provide a powerful way to construct synthetic gene expression programs for a wide range of applications, including rewiring cell fates or engineering metabolic pathways. ',\n",
       "  'doi': '10.1016/j.cell.2014.11.052',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Principles of ER cotranslational translocation revealed by proximity-specific ribosome profiling.',\n",
       "  'authors': ['Calvin H Jan', 'Christopher C Williams', 'Jonathan S Weissman'],\n",
       "  'publication_date': '08/11/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25378630',\n",
       "  'abstract': 'Localized protein synthesis is a fundamental mechanism for creating distinct subcellular environments. Here we developed a generalizable proximity-specific ribosome profiling strategy that enables global analysis of translation in defined subcellular locations. We applied this approach to the endoplasmic reticulum (ER) in yeast and mammals. We observed the large majority of secretory proteins to be cotranslationally translocated, including substrates capable of posttranslational insertion in vitro. Distinct translocon complexes engaged nascent chains at different points during synthesis. Whereas most proteins engaged the ER immediately after or even before signal sequence (SS) emergence, a class of Sec66-dependent proteins entered with a looped SS conformation. Finally, we observed rapid ribosome exchange into the cytosol after translation termination. These data provide insights into how distinct translocation mechanisms act in concert to promote efficient cotranslational recruitment. ',\n",
       "  'doi': '10.1126/science.1257521',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Targeting and plasticity of mitochondrial proteins revealed by proximity-specific ribosome profiling.',\n",
       "  'authors': ['Christopher C Williams', 'Calvin H Jan', 'Jonathan S Weissman'],\n",
       "  'publication_date': '08/11/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25378625',\n",
       "  'abstract': 'Nearly all mitochondrial proteins are nuclear-encoded and are targeted to their mitochondrial destination from the cytosol. Here, we used proximity-specific ribosome profiling to comprehensively measure translation at the mitochondrial surface in yeast. Most inner-membrane proteins were cotranslationally targeted to mitochondria, reminiscent of proteins entering the endoplasmic reticulum (ER). Comparison between mitochondrial and ER localization demonstrated that the vast majority of proteins were targeted to a specific organelle. A prominent exception was the fumarate reductase Osm1, known to reside in mitochondria. We identified a conserved ER isoform of Osm1, which contributes to the oxidative protein-folding capacity of the organelle. This dual localization was enabled by alternative translation initiation sites encoding distinct targeting signals. These findings highlight the exquisite in vivo specificity of organellar targeting mechanisms. ',\n",
       "  'doi': '10.1126/science.1257522',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'The general mode of translation inhibition by macrolide antibiotics.',\n",
       "  'authors': ['Krishna Kannan',\n",
       "   'Pinal Kanabar',\n",
       "   'David Schryer',\n",
       "   'Tanja Florin',\n",
       "   'Eugene Oh',\n",
       "   'Neil Bahroos',\n",
       "   'Tanel Tenson',\n",
       "   'Jonathan S Weissman',\n",
       "   'Alexander S Mankin'],\n",
       "  'publication_date': '29/10/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25349425',\n",
       "  'abstract': \"Macrolides are clinically important antibiotics thought to inhibit bacterial growth by impeding the passage of newly synthesized polypeptides through the nascent peptide exit tunnel of the bacterial ribosome. Recent data challenged this view by showing that macrolide antibiotics can differentially affect synthesis of individual proteins. To understand the general mechanism of macrolide action, we used genome-wide ribosome profiling and analyzed the redistribution of ribosomes translating highly expressed genes in bacterial cells treated with high concentrations of macrolide antibiotics. The metagene analysis indicated that inhibition of early rounds of translation, which would be characteristic of the conventional view of macrolide action, occurs only at a limited number of genes. Translation of most genes proceeds past the 5'-proximal codons and can be arrested at more distal codons when the ribosome encounters specific short sequence motifs. The problematic sequence motifs are confined to the nascent peptide residues in the peptidyl transferase center but not to the peptide segment that contacts the antibiotic molecule in the exit tunnel. Therefore, it appears that the general mode of macrolide action involves selective inhibition of peptide bond formation between specific combinations of donor and acceptor substrates. Additional factors operating in the living cell but not functioning during in vitro protein synthesis may modulate site-specific action of macrolide antibiotics. \",\n",
       "  'doi': '10.1073/pnas.1417334111',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Genetic influences on translation in yeast.',\n",
       "  'authors': ['Frank W Albert',\n",
       "   'Dale Muzzey',\n",
       "   'Jonathan S Weissman',\n",
       "   'Leonid Kruglyak'],\n",
       "  'publication_date': '24/10/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25340754',\n",
       "  'abstract': 'Heritable differences in gene expression between individuals are an important source of phenotypic variation. The question of how closely the effects of genetic variation on protein levels mirror those on mRNA levels remains open. Here, we addressed this question by using ribosome profiling to examine how genetic differences between two strains of the yeast S. cerevisiae affect translation. Strain differences in translation were observed for hundreds of genes. Allele specific measurements in the diploid hybrid between the two strains revealed roughly half as many cis-acting effects on translation as were observed for mRNA levels. In both the parents and the hybrid, most effects on translation were of small magnitude, such that the direction of an mRNA difference was typically reflected in a concordant footprint difference. The relative importance of cis and trans acting variation on footprint levels was similar to that for mRNA levels. There was a tendency for translation to cause larger footprint differences than expected given the respective mRNA differences. This is in contrast to translational differences between yeast species that have been reported to more often oppose than reinforce mRNA differences. Finally, we catalogued instances of premature translation termination in the two yeast strains and also found several instances where erroneous reference gene annotations lead to apparent nonsense mutations that in fact reside outside of the translated gene body. Overall, genetic influences on translation subtly modulate gene expression differences, and translation does not create strong discrepancies between genetic influences on mRNA and protein levels. ',\n",
       "  'doi': '10.1371/journal.pgen.1004692',\n",
       "  'journal': 'PLoS genetics'},\n",
       " {'title': 'A protein-tagging system for signal amplification in gene expression and fluorescence imaging.',\n",
       "  'authors': ['Marvin E Tanenbaum',\n",
       "   'Luke A Gilbert',\n",
       "   'Lei S Qi',\n",
       "   'Jonathan S Weissman',\n",
       "   'Ronald D Vale'],\n",
       "  'publication_date': '14/10/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25307933',\n",
       "  'abstract': 'Signals in many biological processes can be amplified by recruiting multiple copies of regulatory proteins to a site of action. Harnessing this principle, we have developed a protein scaffold, a repeating peptide array termed SunTag, which can recruit multiple copies of an antibody-fusion protein. We show that the SunTag can recruit up to 24 copies of GFP, thereby enabling long-term imaging of single protein molecules in living cells. We also use the SunTag to create a potent synthetic transcription factor by recruiting multiple copies of a transcriptional activation domain to a nuclease-deficient CRISPR/Cas9 protein and demonstrate strong activation of endogenous gene expression and re-engineered cell behavior with this system. Thus, the SunTag provides a versatile platform for multimerizing proteins on a target protein scaffold and is likely to have many applications in imaging and controlling biological outputs. ',\n",
       "  'doi': '10.1016/j.cell.2014.09.039',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Genome-Scale CRISPR-Mediated Control of Gene Repression and Activation.',\n",
       "  'authors': ['Luke A Gilbert',\n",
       "   'Max A Horlbeck',\n",
       "   'Britt Adamson',\n",
       "   'Jacqueline E Villalta',\n",
       "   'Yuwen Chen',\n",
       "   'Evan H Whitehead',\n",
       "   'Carla Guimaraes',\n",
       "   'Barbara Panning',\n",
       "   'Hidde L Ploegh',\n",
       "   'Michael C Bassik',\n",
       "   'Lei S Qi',\n",
       "   'Martin Kampmann',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '14/10/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25307932',\n",
       "  'abstract': 'While the catalog of mammalian transcripts and their expression levels in different cell types and disease states is rapidly expanding, our understanding of transcript function lags behind. We present a robust technology enabling systematic investigation of the cellular consequences of repressing or inducing individual transcripts. We identify rules for specific targeting of transcriptional repressors (CRISPRi), typically achieving 90%-99% knockdown with minimal off-target effects, and activators (CRISPRa) to endogenous genes via endonuclease-deficient Cas9. Together they enable modulation of gene expression over a ∼1,000-fold range. Using these rules, we construct genome-scale CRISPRi and CRISPRa libraries, each of which we validate with two pooled screens. Growth-based screens identify essential genes, tumor suppressors, and regulators of differentiation. Screens for sensitivity to a cholera-diphtheria toxin provide broad insights into the mechanisms of pathogen entry, retrotranslocation and toxicity. Our results establish CRISPRi and CRISPRa as powerful tools that provide rich and complementary information for mapping complex pathways. ',\n",
       "  'doi': '10.1016/j.cell.2014.09.029',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Unraveling the mechanism of cell death induced by chemical fibrils.',\n",
       "  'authors': ['Olivier Julien',\n",
       "   'Martin Kampmann',\n",
       "   'Michael C Bassik',\n",
       "   'Julie A Zorn',\n",
       "   'Vincent J Venditto',\n",
       "   'Kazutaka Shimbo',\n",
       "   'Nicholas J Agard',\n",
       "   'Kenichi Shimada',\n",
       "   'Arnold L Rheingold',\n",
       "   'Brent R Stockwell',\n",
       "   'Jonathan S Weissman',\n",
       "   'James A Wells'],\n",
       "  'publication_date': '30/09/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25262416',\n",
       "  'abstract': \"We previously discovered a small-molecule inducer of cell death, named 1541, that noncovalently self-assembles into chemical fibrils ('chemi-fibrils') and activates procaspase-3 in vitro. We report here that 1541-induced cell death is caused by the fibrillar rather than the soluble form of the drug. A short hairpin RNA screen reveals that knockdown of genes involved in endocytosis, vesicle trafficking and lysosomal acidification causes partial 1541 resistance. We confirm the role of these pathways using pharmacological inhibitors. Microscopy shows that the fluorescent chemi-fibrils accumulate in punctae inside cells that partially colocalize with lysosomes. Notably, the chemi-fibrils bind and induce liposome leakage in vitro, suggesting they may do the same in cells. The chemi-fibrils induce extensive proteolysis including caspase substrates, yet modulatory profiling reveals that chemi-fibrils form a distinct class from existing inducers of cell death. The chemi-fibrils share similarities with proteinaceous fibrils and may provide insight into their mechanism of cellular toxicity. \",\n",
       "  'doi': '10.1038/nchembio.1639',\n",
       "  'journal': 'Nature chemical biology'},\n",
       " {'title': 'Ribosome profiling reveals pervasive translation outside of annotated protein-coding genes.',\n",
       "  'authors': ['Nicholas T Ingolia',\n",
       "   'Gloria A Brar',\n",
       "   'Noam Stern-Ginossar',\n",
       "   'Michael S Harris',\n",
       "   'Gaëlle J S Talhouarne',\n",
       "   'Sarah E Jackson',\n",
       "   'Mark R Wills',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/08/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25159147',\n",
       "  'abstract': \"Ribosome profiling suggests that ribosomes occupy many regions of the transcriptome thought to be noncoding, including 5' UTRs and long noncoding RNAs (lncRNAs). Apparent ribosome footprints outside of protein-coding regions raise the possibility of artifacts unrelated to translation, particularly when they occupy multiple, overlapping open reading frames (ORFs). Here, we show hallmarks of translation in these footprints: copurification with the large ribosomal subunit, response to drugs targeting elongation, trinucleotide periodicity, and initiation at early AUGs. We develop a metric for distinguishing between 80S footprints and nonribosomal sources using footprint size distributions, which validates the vast majority of footprints outside of coding regions. We present evidence for polypeptide production beyond annotated genes, including the induction of immune responses following human cytomegalovirus (HCMV) infection. Translation is pervasive on cytosolic transcripts outside of conserved reading frames, and direct detection of this expanded universe of translated products enables efforts at understanding how cells manage and exploit its consequences.\",\n",
       "  'doi': '10.1016/j.celrep.2014.07.045',\n",
       "  'journal': 'Cell reports'},\n",
       " {'title': 'Misfolded proteins induce aggregation of the lectin Yos9.',\n",
       "  'authors': ['Melanie H Smith', 'Edwin H Rodriguez', 'Jonathan S Weissman'],\n",
       "  'publication_date': '03/08/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25086047',\n",
       "  'abstract': 'A substantial fraction of nascent proteins delivered into the endoplasmic reticulum (ER) never reach their native conformations. Eukaryotes use a series of complementary pathways to efficiently recognize and dispose of these terminally misfolded proteins. In this process, collectively termed ER-associated degradation (ERAD), misfolded proteins are retrotranslocated to the cytosol, polyubiquitinated, and degraded by the proteasome. Although there has been great progress in identifying ERAD components, how these factors accurately identify substrates remains poorly understood. The targeting of misfolded glycoproteins in the ER lumen for ERAD requires the lectin Yos9, which recognizes the glycan species found on terminally misfolded proteins. In a role that remains poorly characterized, Yos9 also binds the protein component of ERAD substrates. Here, we identified a 45-kDa domain of Yos9, consisting of residues 22-421, that is proteolytically stable, highly structured, and able to fully support ERAD in vivo. In vitro binding studies show that Yos9(22-421) exhibits sequence-specific recognition of linear peptides from the ERAD substrate, carboxypeptidase Y G255R (CPY*), and binds a model unfolded peptide ΔEspP and protein Δ131Δ in solution. Binding of Yos9 to these substrates results in their cooperative aggregation. Although the physiological consequences of this substrate-induced aggregation remain to be seen, it has the potential to play a role in the regulation of ERAD. ',\n",
       "  'doi': '10.1074/jbc.M114.583344',\n",
       "  'journal': 'The Journal of biological chemistry'},\n",
       " {'title': 'The coding and noncoding architecture of the Caulobacter crescentus genome.',\n",
       "  'authors': ['Jared M Schrader',\n",
       "   'Bo Zhou',\n",
       "   'Gene-Wei Li',\n",
       "   'Keren Lasker',\n",
       "   'W Seth Childers',\n",
       "   'Brandon Williams',\n",
       "   'Tao Long',\n",
       "   'Sean Crosson',\n",
       "   'Harley H McAdams',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lucy Shapiro'],\n",
       "  'publication_date': '01/08/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '25078267',\n",
       "  'abstract': \"Caulobacter crescentus undergoes an asymmetric cell division controlled by a genetic circuit that cycles in space and time. We provide a universal strategy for defining the coding potential of bacterial genomes by applying ribosome profiling, RNA-seq, global 5'-RACE, and liquid chromatography coupled with tandem mass spectrometry (LC-MS) data to the 4-megabase C. crescentus genome. We mapped transcript units at single base-pair resolution using RNA-seq together with global 5'-RACE. Additionally, using ribosome profiling and LC-MS, we mapped translation start sites and coding regions with near complete coverage. We found most start codons lacked corresponding Shine-Dalgarno sites although ribosomes were observed to pause at internal Shine-Dalgarno sites within the coding DNA sequence (CDS). These data suggest a more prevalent use of the Shine-Dalgarno sequence for ribosome pausing rather than translation initiation in C. crescentus. Overall 19% of the transcribed and translated genomic elements were newly identified or significantly improved by this approach, providing a valuable genomic resource to elucidate the complete C. crescentus genetic circuitry that controls asymmetric cell division. \",\n",
       "  'doi': '10.1371/journal.pgen.1004463',\n",
       "  'journal': 'PLoS genetics'},\n",
       " {'title': 'Functional genomics platform for pooled screening and generation of mammalian genetic interaction maps.',\n",
       "  'authors': ['Martin Kampmann', 'Michael C Bassik', 'Jonathan S Weissman'],\n",
       "  'publication_date': '06/07/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24992097',\n",
       "  'abstract': \"Systematic genetic interaction maps in microorganisms are powerful tools for identifying functional relationships between genes and for defining the function of uncharacterized genes. We have recently implemented this strategy in mammalian cells as a two-stage approach. First, genes of interest are robustly identified in a pooled genome-wide screen using complex shRNA libraries. Second, phenotypes for all pairwise combinations of 'hit' genes are measured in a double-shRNA screen and used to construct a genetic interaction map. Our protocol allows for rapid pooled screening under various conditions without a requirement for robotics, in contrast to arrayed approaches. Each round of screening can be implemented in ∼2 weeks, with additional time for analysis and generation of reagents. We discuss considerations for screen design, and we present complete experimental procedures, as well as a full computational analysis suite for the identification of hits in pooled screens and generation of genetic interaction maps. Although the protocol outlined here was developed for our original shRNA-based approach, it can be applied more generally, including to CRISPR-based approaches. \",\n",
       "  'doi': '10.1038/nprot.2014.103',\n",
       "  'journal': 'Nature protocols'},\n",
       " {'title': 'A high-coverage shRNA screen identifies TMEM129 as an E3 ligase involved in ER-associated protein degradation.',\n",
       "  'authors': ['Michael L van de Weijer',\n",
       "   'Michael C Bassik',\n",
       "   'Rutger D Luteijn',\n",
       "   'Cornelia M Voorburg',\n",
       "   'Mirjam A M Lohuis',\n",
       "   'Elisabeth Kremmer',\n",
       "   'Rob C Hoeben',\n",
       "   'Emily M LeProust',\n",
       "   'Siyuan Chen',\n",
       "   'Hanneke Hoelen',\n",
       "   'Maaike E Ressing',\n",
       "   'Weronika Patena',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael T McManus',\n",
       "   'Emmanuel J H J Wiertz',\n",
       "   'Robert Jan Lebbink'],\n",
       "  'publication_date': '09/05/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24807418',\n",
       "  'abstract': 'Misfolded ER proteins are retrotranslocated into the cytosol for degradation via the ubiquitin-proteasome system. The human cytomegalovirus protein US11 exploits this ER-associated protein degradation (ERAD) pathway to downregulate HLA class I molecules in virus-infected cells, thereby evading elimination by cytotoxic T-lymphocytes. US11-mediated degradation of HLA class I has been instrumental in the identification of key components of mammalian ERAD, including Derlin-1, p97, VIMP and SEL1L. Despite this, the process governing retrotranslocation of the substrate is still poorly understood. Here using a high-coverage genome-wide shRNA library, we identify the uncharacterized protein TMEM129 and the ubiquitin-conjugating E2 enzyme UBE2J2 to be essential for US11-mediated HLA class I downregulation. TMEM129 is an unconventional C4C4-type RING finger E3 ubiquitin ligase that resides within a complex containing various other ERAD components, including Derlin-1, Derlin-2, VIMP and p97, indicating that TMEM129 is an integral part of the ER-resident dislocation complex mediating US11-induced HLA class I degradation. ',\n",
       "  'doi': '10.1038/ncomms4832',\n",
       "  'journal': 'Nature communications'},\n",
       " {'title': 'A pause sequence enriched at translation start sites drives transcription dynamics in vivo.',\n",
       "  'authors': ['Matthew H Larson',\n",
       "   'Rachel A Mooney',\n",
       "   'Jason M Peters',\n",
       "   'Tricia Windgassen',\n",
       "   'Dhananjaya Nayak',\n",
       "   'Carol A Gross',\n",
       "   'Steven M Block',\n",
       "   'William J Greenleaf',\n",
       "   'Robert Landick',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '03/05/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24789973',\n",
       "  'abstract': 'Transcription by RNA polymerase (RNAP) is interrupted by pauses that play diverse regulatory roles. Although individual pauses have been studied in vitro, the determinants of pauses in vivo and their distribution throughout the bacterial genome remain unknown. Using nascent transcript sequencing, we identified a 16-nucleotide consensus pause sequence in Escherichia coli that accounts for known regulatory pause sites as well as ~20,000 new in vivo pause sites. In vitro single-molecule and ensemble analyses demonstrate that these pauses result from RNAP-nucleic acid interactions that inhibit next-nucleotide addition. The consensus sequence also leads to pausing by RNAPs from diverse lineages and is enriched at translation start sites in both E. coli and Bacillus subtilis. Our results thus reveal a conserved mechanism unifying known and newly identified pause events. ',\n",
       "  'doi': '10.1126/science.1251871',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Quantifying absolute protein synthesis rates reveals principles underlying allocation of cellular resources.',\n",
       "  'authors': ['Gene-Wei Li',\n",
       "   'David Burkhardt',\n",
       "   'Carol Gross',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '29/04/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24766808',\n",
       "  'abstract': 'Quantitative views of cellular functions require precise measures of rates of biomolecule production, especially proteins-the direct effectors of biological processes. Here, we present a genome-wide approach, based on ribosome profiling, for measuring absolute protein synthesis rates. The resultant E. coli data set transforms our understanding of the extent to which protein synthesis is precisely controlled to optimize function and efficiency. Members of multiprotein complexes are made in precise proportion to their stoichiometry, whereas components of functional modules are produced differentially according to their hierarchical role. Estimates of absolute protein abundance also reveal principles for optimizing design. These include how the level of different types of transcription factors is optimized for rapid response and how a metabolic pathway (methionine biosynthesis) balances production cost with activity requirements. Our studies reveal how general principles, important both for understanding natural systems and for synthesizing new ones, emerge from quantitative analyses of protein synthesis.',\n",
       "  'doi': '10.1016/j.cell.2014.02.033',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Extensive and coordinated control of allele-specific expression by both transcription and translation in Candida albicans.',\n",
       "  'authors': ['Dale Muzzey', 'Gavin Sherlock', 'Jonathan S Weissman'],\n",
       "  'publication_date': '16/04/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24732588',\n",
       "  'abstract': 'Though sequence differences between alleles are often limited to a few polymorphisms, these differences can cause large and widespread allelic variation at the expression level. Such allele-specific expression (ASE) has been extensively explored at the level of transcription but not translation. Here we measured ASE in the diploid yeast Candida albicans at both the transcriptional and translational levels using RNA-seq and ribosome profiling, respectively. Since C. albicans is an obligate diploid, our analysis isolates ASE arising from cis elements in a natural, nonhybrid organism, where allelic effects reflect evolutionary forces. Importantly, we find that ASE arising from translation is of a similar magnitude as transcriptional ASE, both in terms of the number of genes affected and the magnitude of the bias. We further observe coordination between ASE at the levels of transcription and translation for single genes. Specifically, reinforcing relationships--where transcription and translation favor the same allele--are more frequent than expected by chance, consistent with selective pressure tuning ASE at multiple regulatory steps. Finally, we parameterize alleles based on a range of properties and find that SNP location and predicted mRNA-structure stability are associated with translational ASE in cis. Since this analysis probes more than 4000 allelic pairs spanning a broad range of variations, our data provide a genome-wide view into the relative impact of cis elements that regulate translation.',\n",
       "  'doi': '10.1101/gr.166322.113',\n",
       "  'journal': 'Genome research'},\n",
       " {'title': 'Uniform nomenclature for the mitochondrial contact site and cristae organizing system.',\n",
       "  'authors': ['Nikolaus Pfanner',\n",
       "   'Martin van der Laan',\n",
       "   'Paolo Amati',\n",
       "   'Roderick A Capaldi',\n",
       "   'Amy A Caudy',\n",
       "   'Agnieszka Chacinska',\n",
       "   'Manjula Darshi',\n",
       "   'Markus Deckers',\n",
       "   'Suzanne Hoppins',\n",
       "   'Tateo Icho',\n",
       "   'Stefan Jakobs',\n",
       "   'Jianguo Ji',\n",
       "   'Vera Kozjak-Pavlovic',\n",
       "   'Chris Meisinger',\n",
       "   'Paul R Odgren',\n",
       "   'Sang Ki Park',\n",
       "   'Peter Rehling',\n",
       "   'Andreas S Reichert',\n",
       "   'M Saeed Sheikh',\n",
       "   'Susan S Taylor',\n",
       "   'Nobuo Tsuchida',\n",
       "   'Alexander M van der Bliek',\n",
       "   'Ida J van der Klei',\n",
       "   'Jonathan S Weissman',\n",
       "   'Benedikt Westermann',\n",
       "   'Jiping Zha',\n",
       "   'Walter Neupert',\n",
       "   'Jodi Nunnari'],\n",
       "  'publication_date': '02/04/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24687277',\n",
       "  'abstract': 'The mitochondrial inner membrane contains a large protein complex that functions in inner membrane organization and formation of membrane contact sites. The complex was variably named the mitochondrial contact site complex, mitochondrial inner membrane organizing system, mitochondrial organizing structure, or Mitofilin/Fcj1 complex. To facilitate future studies, we propose to unify the nomenclature and term the complex \"mitochondrial contact site and cristae organizing system\" and its subunits Mic10 to Mic60. ',\n",
       "  'doi': '10.1083/jcb.201401006',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'Differential scales of protein quality control.',\n",
       "  'authors': ['Suzanne Wolff', 'Jonathan S Weissman', 'Andrew Dillin'],\n",
       "  'publication_date': '01/04/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24679526',\n",
       "  'abstract': \"Proteins are notorious for their unpleasant behavior-continually at risk of misfolding, collecting damage, aggregating, and causing toxicity and disease. To counter these challenges, cells have evolved elaborate chaperone and quality control networks that can resolve damage at the level of the protein, organelle, cell, or tissue. On the smallest scale, the integrity of individual proteins is monitored during their synthesis. On a larger scale, cells use compartmentalized defenses and networks of communication, capable sometimes of signaling between cells, to respond to changes in the proteome's health. Together, these layered defenses help protect cells from damaged proteins. \",\n",
       "  'doi': '10.1016/j.cell.2014.03.007',\n",
       "  'journal': 'Cell'},\n",
       " {'title': \"KSHV 2.0: a comprehensive annotation of the Kaposi's sarcoma-associated herpesvirus genome using next-generation sequencing reveals novel genomic and functional features.\",\n",
       "  'authors': ['Carolina Arias',\n",
       "   'Ben Weisburd',\n",
       "   'Noam Stern-Ginossar',\n",
       "   'Alexandre Mercier',\n",
       "   'Alexis S Madrid',\n",
       "   'Priya Bellare',\n",
       "   'Meghan Holdorf',\n",
       "   'Jonathan S Weissman',\n",
       "   'Don Ganem'],\n",
       "  'publication_date': '24/01/14',\n",
       "  'publication_year': '2014',\n",
       "  'pubmed_id': '24453964',\n",
       "  'abstract': \"Productive herpesvirus infection requires a profound, time-controlled remodeling of the viral transcriptome and proteome. To gain insights into the genomic architecture and gene expression control in Kaposi's sarcoma-associated herpesvirus (KSHV), we performed a systematic genome-wide survey of viral transcriptional and translational activity throughout the lytic cycle. Using mRNA-sequencing and ribosome profiling, we found that transcripts encoding lytic genes are promptly bound by ribosomes upon lytic reactivation, suggesting their regulation is mainly transcriptional. Our approach also uncovered new genomic features such as ribosome occupancy of viral non-coding RNAs, numerous upstream and small open reading frames (ORFs), and unusual strategies to expand the virus coding repertoire that include alternative splicing, dynamic viral mRNA editing, and the use of alternative translation initiation codons. Furthermore, we provide a refined and expanded annotation of transcription start sites, polyadenylation sites, splice junctions, and initiation/termination codons of known and new viral features in the KSHV genomic space which we have termed KSHV 2.0. Our results represent a comprehensive genome-scale image of gene regulation during lytic KSHV infection that substantially expands our understanding of the genomic architecture and coding capacity of the virus. \",\n",
       "  'doi': '10.1371/journal.ppat.1003847',\n",
       "  'journal': 'PLoS pathogens'},\n",
       " {'title': 'Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system.',\n",
       "  'authors': ['Baohui Chen',\n",
       "   'Luke A Gilbert',\n",
       "   'Beth A Cimini',\n",
       "   'Joerg Schnitzbauer',\n",
       "   'Wei Zhang',\n",
       "   'Gene-Wei Li',\n",
       "   'Jason Park',\n",
       "   'Elizabeth H Blackburn',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lei S Qi',\n",
       "   'Bo Huang'],\n",
       "  'publication_date': '24/12/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24360272',\n",
       "  'abstract': 'The spatiotemporal organization and dynamics of chromatin play critical roles in regulating genome function. However, visualizing specific, endogenous genomic loci remains challenging in living cells. Here, we demonstrate such an imaging technique by repurposing the bacterial CRISPR/Cas system.\\xa0Using an EGFP-tagged endonuclease-deficient Cas9 protein and a structurally optimized small guide (sg) RNA, we show robust imaging of repetitive elements in telomeres and coding genes in living cells. Furthermore, an array of sgRNAs tiling along the target locus enables the visualization of nonrepetitive genomic sequences. Using this method, we have studied telomere dynamics during elongation or disruption, the subnuclear localization of the MUC4 loci, the cohesion of replicated MUC4 loci on sister chromatids, and their dynamic behaviors during mitosis. This CRISPR imaging tool has potential to significantly improve the capacity to study the conformation and dynamics of native chromosomes in living human cells. ',\n",
       "  'doi': '10.1016/j.cell.2013.12.001',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Genome-wide probing of RNA structure reveals active unfolding of mRNA structures in vivo.',\n",
       "  'authors': ['Silvi Rouskin',\n",
       "   'Meghan Zubradt',\n",
       "   'Stefan Washietl',\n",
       "   'Manolis Kellis',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '18/12/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24336214',\n",
       "  'abstract': \"RNA has a dual role as an informational molecule and a direct effector of biological tasks. The latter function is enabled by RNA's ability to adopt complex secondary and tertiary folds and thus has motivated extensive computational and experimental efforts for determining RNA structures. Existing approaches for evaluating RNA structure have been largely limited to in vitro systems, yet the thermodynamic forces which drive RNA folding in vitro may not be sufficient to predict stable RNA structures in vivo. Indeed, the presence of RNA-binding proteins and ATP-dependent helicases can influence which structures are present inside cells. Here we present an approach for globally monitoring RNA structure in native conditions in vivo with single-nucleotide precision. This method is based on in vivo modification with dimethyl sulphate (DMS), which reacts with unpaired adenine and cytosine residues, followed by deep sequencing to monitor modifications. Our data from yeast and mammalian cells are in excellent agreement with known messenger RNA structures and with the high-resolution crystal structure of the Saccharomyces cerevisiae ribosome. Comparison between in vivo and in vitro data reveals that in rapidly dividing cells there are vastly fewer structured mRNA regions in vivo than in vitro. Even thermostable RNA structures are often denatured in cells, highlighting the importance of cellular processes in regulating RNA structure. Indeed, analysis of mRNA structure under ATP-depleted conditions in yeast shows that energy-dependent processes strongly contribute to the predominantly unfolded state of mRNAs inside cells. Our studies broadly enable the functional analysis of physiological RNA structures and reveal that, in contrast to the Anfinsen view of protein folding whereby the structure formed is the most thermodynamically favourable, thermodynamics have an incomplete role in determining mRNA structure in vivo. \",\n",
       "  'doi': '10.1038/nature12894',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Ribosome profiling reveals pervasive and regulated stop codon readthrough in Drosophila melanogaster.',\n",
       "  'authors': ['Joshua G Dunn',\n",
       "   'Catherine K Foo',\n",
       "   'Nicolette G Belletier',\n",
       "   'Elizabeth R Gavis',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '05/12/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24302569',\n",
       "  'abstract': 'Ribosomes can read through stop codons in a regulated manner, elongating rather than terminating the nascent peptide. Stop codon readthrough is essential to diverse viruses, and phylogenetically predicted to occur in a few hundred genes in Drosophila melanogaster, but the importance of regulated readthrough in eukaryotes remains largely unexplored. Here, we present a ribosome profiling assay (deep sequencing of ribosome-protected mRNA fragments) for Drosophila melanogaster, and provide the first genome-wide experimental analysis of readthrough. Readthrough is far more pervasive than expected: the vast majority of readthrough events evolved within D. melanogaster and were not predicted phylogenetically. The resulting C-terminal protein extensions show evidence of selection, contain functional subcellular localization signals, and their readthrough is regulated, arguing for their importance. We further demonstrate that readthrough occurs in yeast and humans. Readthrough thus provides general mechanisms both to regulate gene expression and function, and to add plasticity to the proteome during evolution. DOI: http://dx.doi.org/10.7554/eLife.01179.001. ',\n",
       "  'doi': '10.7554/eLife.01179',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Msn2 coordinates a stoichiometric gene expression program.',\n",
       "  'authors': ['Jacob Stewart-Ornstein',\n",
       "   'Christopher Nelson',\n",
       "   'Joe DeRisi',\n",
       "   'Jonathan S Weissman',\n",
       "   'Hana El-Samad'],\n",
       "  'publication_date': '12/11/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24210615',\n",
       "  'abstract': 'Many cellular processes operate in an \"analog\" regime in which the magnitude of the response is precisely tailored to the intensity of the stimulus. In order to maintain the coherence of such responses, the cell must provide for proportional expression of multiple target genes across a wide dynamic range of induction states. Our understanding of the strategies used to achieve graded gene regulation is limited.\\nIn this work, we document a relationship between stress-responsive gene expression and the transcription factor Msn2 that is graded over a large range of Msn2 concentrations. We use computational modeling and in vivo and in vitro analyses to dissect the roots of this relationship. Our studies reveal a simple and general strategy based on noncooperative low-affinity interactions between Msn2 and its cognate binding sites as well as competition over a large number of Msn2 binding sites in the genome relative to the number of Msn2 molecules.\\nIn addition to enabling precise tuning of gene expression to the state of the environment, this strategy ensures colinear activation of target genes, allowing for stoichiometric expression of large groups of genes without extensive promoter tuning. Furthermore, such a strategy enables precise modulation of the activity of any given promoter by addition of binding sites without altering the qualitative relationship between different genes in a regulon. This feature renders a given regulon highly \"evolvable.\"',\n",
       "  'doi': '10.1016/j.cub.2013.09.043',\n",
       "  'journal': 'Current biology'},\n",
       " {'title': 'Next-generation NAMPT inhibitors identified by sequential high-throughput phenotypic chemical and functional genomic screens.',\n",
       "  'authors': ['Christina J Matheny',\n",
       "   'Michael C Wei',\n",
       "   'Michael C Bassik',\n",
       "   'Alicia J Donnelly',\n",
       "   'Martin Kampmann',\n",
       "   'Masayuki Iwasaki',\n",
       "   'Obdulio Piloto',\n",
       "   'David E Solow-Cordero',\n",
       "   'Donna M Bouley',\n",
       "   'Rachel Rau',\n",
       "   'Patrick Brown',\n",
       "   'Michael T McManus',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael L Cleary'],\n",
       "  'publication_date': '05/11/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24183972',\n",
       "  'abstract': 'Phenotypic high-throughput chemical screens allow for discovery of small molecules that modulate complex phenotypes and provide lead compounds for novel therapies; however, identification of the mechanistically relevant targets remains a major experimental challenge. We report the application of sequential unbiased high-throughput chemical and ultracomplex small hairpin RNA (shRNA) screens to identify a distinctive class of inhibitors that target nicotinamide phosphoribosyl transferase (NAMPT), a rate-limiting enzyme in the biosynthesis of nicotinamide adenine dinucleotide, a crucial cofactor in many biochemical processes. The lead compound STF-118804 is a highly specific NAMPT inhibitor, improves survival in an orthotopic xenotransplant model of high-risk acute lymphoblastic leukemia, and targets leukemia stem cells. Tandem high-throughput screening using chemical and ultracomplex shRNA libraries, therefore, provides a rapid chemical genetics approach for seamless progression from small-molecule lead identification to target discovery and validation. ',\n",
       "  'doi': '10.1016/j.chembiol.2013.09.014',\n",
       "  'journal': 'Chemistry & biology'},\n",
       " {'title': 'Global cellular response to chemotherapy-induced apoptosis.',\n",
       "  'authors': ['Arun P Wiita',\n",
       "   'Etay Ziv',\n",
       "   'Paul J Wiita',\n",
       "   'Anatoly Urisman',\n",
       "   'Olivier Julien',\n",
       "   'Alma L Burlingame',\n",
       "   'Jonathan S Weissman',\n",
       "   'James A Wells'],\n",
       "  'publication_date': '31/10/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24171104',\n",
       "  'abstract': 'How cancer cells globally struggle with a chemotherapeutic insult before succumbing to apoptosis is largely unknown. Here we use an integrated systems-level examination of transcription, translation, and proteolysis to understand these events central to cancer treatment. As a model we study myeloma cells exposed to the proteasome inhibitor bortezomib, a first-line therapy. Despite robust transcriptional changes, unbiased quantitative proteomics detects production of only a few critical anti-apoptotic proteins against a background of general translation inhibition. Simultaneous ribosome profiling further reveals potential translational regulation of stress response genes. Once the apoptotic machinery is engaged, degradation by caspases is largely independent of upstream bortezomib effects. Moreover, previously uncharacterized non-caspase proteolytic events also participate in cellular deconstruction. Our systems-level data also support co-targeting the anti-apoptotic regulator HSF1 to promote cell death by bortezomib. This integrated approach offers unique, in-depth insight into apoptotic dynamics that may prove important to preclinical evaluation of any anti-cancer compound. DOI:http://dx.doi.org/10.7554/eLife.01236.001. ',\n",
       "  'doi': '10.7554/eLife.01236',\n",
       "  'journal': 'eLife'},\n",
       " {'title': 'Selective ribosome profiling as a tool for studying the interaction of chaperones and targeting factors with nascent polypeptide chains and ribosomes.',\n",
       "  'authors': ['Annemarie H Becker',\n",
       "   'Eugene Oh',\n",
       "   'Jonathan S Weissman',\n",
       "   'Günter Kramer',\n",
       "   'Bernd Bukau'],\n",
       "  'publication_date': '19/10/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24136347',\n",
       "  'abstract': 'A plethora of factors is involved in the maturation of newly synthesized proteins, including chaperones, membrane targeting factors and enzymes. Many factors act co-translationally through association with ribosome-nascent chain complexes (RNCs), but their target specificities and modes of action remain poorly understood. We developed selective ribosome profiling (SeRP) to identify substrate pools and points of RNC engagement of these factors. SeRP is based on sequencing mRNA fragments covered by translating ribosomes (general ribosome profiling (RP)), combined with a procedure to selectively isolate RNCs whose nascent polypeptides are associated with the factor of interest. Factor-RNC interactions are stabilized by cross-linking; the resulting factor-RNC adducts are nuclease-treated to generate monosomes, and then they are affinity purified. The ribosome-extracted mRNA footprints are converted to DNA libraries for deep sequencing. The protocol is specified for general RP and SeRP in bacteria. It was first applied to the chaperone trigger factor (TF) and is readily adaptable to other co-translationally acting factors, including eukaryotic factors. Factor-RNC purification and sequencing library preparation takes 7-8 d, and sequencing and data analysis can be completed in 5-6 d. ',\n",
       "  'doi': '10.1038/nprot.2013.133',\n",
       "  'journal': 'Nature protocols'},\n",
       " {'title': 'CRISPR interference (CRISPRi) for sequence-specific control of gene expression.',\n",
       "  'authors': ['Matthew H Larson',\n",
       "   'Luke A Gilbert',\n",
       "   'Xiaowo Wang',\n",
       "   'Wendell A Lim',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lei S Qi'],\n",
       "  'publication_date': '19/10/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24136345',\n",
       "  'abstract': 'Sequence-specific control of gene expression on a genome-wide scale is an important approach for understanding gene functions and for engineering genetic regulatory systems. We have recently described an RNA-based method, CRISPR interference (CRISPRi), for targeted silencing of transcription in bacteria and human cells. The CRISPRi system is derived from the Streptococcus pyogenes CRISPR (clustered regularly interspaced palindromic repeats) pathway, requiring only the coexpression of a catalytically inactive Cas9 protein and a customizable single guide RNA (sgRNA). The Cas9-sgRNA complex binds to DNA elements complementary to the sgRNA and causes a steric block that halts transcript elongation by RNA polymerase, resulting in the repression of the target gene. Here we provide a protocol for the design, construction and expression of customized sgRNAs for transcriptional repression of any gene of interest. We also provide details for testing the repression activity of CRISPRi using quantitative fluorescence assays and native elongating transcript sequencing. CRISPRi provides a simplified approach for rapid gene repression within 1-2 weeks. The method can also be adapted for high-throughput interrogation of genome-wide gene functions and genetic interactions, thus providing a complementary approach to RNA interference, which can be used in a wider variety of organisms. ',\n",
       "  'doi': '10.1038/nprot.2013.132',\n",
       "  'journal': 'Nature protocols'},\n",
       " {'title': 'A developmentally regulated translational control pathway establishes the meiotic chromosome segregation pattern.',\n",
       "  'authors': ['Luke E Berchowitz',\n",
       "   'Aaron S Gajadhar',\n",
       "   'Folkert J van Werven',\n",
       "   'Alexandra A De Rosa',\n",
       "   'Mariya L Samoylova',\n",
       "   'Gloria A Brar',\n",
       "   'Yifeng Xu',\n",
       "   'Che Xiao',\n",
       "   'Bruce Futcher',\n",
       "   'Jonathan S Weissman',\n",
       "   'Forest M White',\n",
       "   'Angelika Amon'],\n",
       "  'publication_date': '12/10/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24115771',\n",
       "  'abstract': \"Production of haploid gametes from diploid progenitor cells is mediated by a specialized cell division, meiosis, where two divisions, meiosis I and II, follow a single S phase. Errors in progression from meiosis I to meiosis II lead to aneuploid and polyploid gametes, but the regulatory mechanisms controlling this transition are poorly understood. Here, we demonstrate that the conserved kinase Ime2 regulates the timing and order of the meiotic divisions by controlling translation. Ime2 coordinates translational activation of a cluster of genes at the meiosis I-meiosis II transition, including the critical determinant of the meiotic chromosome segregation pattern CLB3. We further show that Ime2 mediates translational control through the meiosis-specific RNA-binding protein Rim4. Rim4 inhibits translation of CLB3 during meiosis I by interacting with the 5' untranslated region (UTR) of CLB3. At the onset of meiosis II, Ime2 kinase activity rises and triggers a decrease in Rim4 protein levels, thereby alleviating translational repression. Our results elucidate a novel developmentally regulated translational control pathway that establishes the meiotic chromosome segregation pattern. \",\n",
       "  'doi': '10.1101/gad.224253.113',\n",
       "  'journal': 'Genes & development'},\n",
       " {'title': 'Assembly of a phased diploid Candida albicans  genome facilitates allele-specific measurements  and provides a simple model for repeat and indel  structure.',\n",
       "  'authors': ['Dale Muzzey',\n",
       "   'Katja Schwartz',\n",
       "   'Jonathan S Weissman',\n",
       "   'Gavin Sherlock'],\n",
       "  'publication_date': '13/09/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '24025428',\n",
       "  'abstract': 'Candida albicans is a ubiquitous opportunistic fungal pathogen that afflicts immunocompromised human hosts. With rare and transient exceptions the yeast is diploid, yet despite its clinical relevance the respective sequences of its two homologous chromosomes have not been completely resolved.\\nWe construct a phased diploid genome assembly by deep sequencing a standard laboratory wild-type strain and a panel of strains homozygous for particular chromosomes. The assembly has 700-fold coverage on average,allowing extensive revision and expansion of the number of known SNPs and indels. This phased genome significantly enhances the sensitivity and specificity of allele-specific expression measurements by enabling pooling and cross-validation of signal across multiple polymorphic sites. Additionally, the diploid assembly reveals pervasive and unexpected patterns in allelic differences between homologous chromosomes. Firstly, we see striking clustering of indels, concentrated primarily in the repeat sequences in promoters. Secondly, both indels and their repeat-sequence substrate are enriched near replication origins. Finally, we reveal an intimate link between repeat sequences and indels, which argues that repeat length is under selective pressure for most eukaryotes. This connection is described by a concise one-parameter model that explains repeat-sequence abundance in C. albicans as a function of the indel rate,and provides a general framework to interpret repeat abundance in species ranging from bacteria to humans.\\nThe phased genome assembly and insights into repeat plasticity will be valuable for better understanding allele-specific phenomena and genome evolution.',\n",
       "  'doi': '10.1186/gb-2013-14-9-r97',\n",
       "  'journal': 'Genome biology'},\n",
       " {'title': 'CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes.',\n",
       "  'authors': ['Luke A Gilbert',\n",
       "   'Matthew H Larson',\n",
       "   'Leonardo Morsut',\n",
       "   'Zairan Liu',\n",
       "   'Gloria A Brar',\n",
       "   'Sandra E Torres',\n",
       "   'Noam Stern-Ginossar',\n",
       "   'Onn Brandman',\n",
       "   'Evan H Whitehead',\n",
       "   'Jennifer A Doudna',\n",
       "   'Wendell A Lim',\n",
       "   'Jonathan S Weissman',\n",
       "   'Lei S Qi'],\n",
       "  'publication_date': '16/07/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23849981',\n",
       "  'abstract': 'The genetic interrogation and reprogramming of cells requires methods for robust and precise targeting of genes for expression or repression. The CRISPR-associated catalytically inactive dCas9 protein offers a general platform for RNA-guided DNA targeting. Here, we show that fusion of dCas9 to effector domains with distinct regulatory functions enables stable and efficient transcriptional repression or activation in human and yeast cells, with the site of delivery determined solely by a coexpressed short guide (sg)RNA. Coupling of dCas9 to a transcriptional repressor domain can robustly silence expression of multiple endogenous genes. RNA-seq analysis indicates that CRISPR interference (CRISPRi)-mediated transcriptional repression is highly specific. Our results establish that the CRISPR system can be used as a modular and flexible DNA-binding platform for the recruitment of proteins to a target DNA sequence, revealing the potential of CRISPRi as a general tool for the precise regulation of gene expression in eukaryotic cells. ',\n",
       "  'doi': '10.1016/j.cell.2013.06.044',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Genome-wide annotation and quantitation of translation by ribosome profiling.',\n",
       "  'authors': ['Nicholas T Ingolia',\n",
       "   'Gloria A Brar',\n",
       "   'Silvia Rouskin',\n",
       "   'Anna M McGeachy',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '04/07/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23821443',\n",
       "  'abstract': 'Recent studies highlight the importance of translational control in determining protein abundance, underscoring the value of measuring gene expression at the level of translation. A protocol for genome-wide, quantitative analysis of in vivo translation by deep sequencing is presented here. This ribosome-profiling approach maps the exact positions of ribosomes on transcripts by nuclease footprinting. The nuclease-protected mRNA fragments are converted into a DNA library suitable for deep sequencing using a strategy that minimizes bias. The abundance of different footprint fragments in deep sequencing data reports on the amount of translation of a gene. Additionally, footprints reveal the exact regions of the transcriptome that are translated. To better define translated reading frames, an adaptation that reveals the sites of translation initiation by pre-treating cells with harringtonine to immobilize initiating ribosomes is described. The protocol described requires 5 to 7 days to generate a completed ribosome profiling sequencing library. Sequencing and data analysis requires an additional 4 to 5 days.',\n",
       "  'doi': '10.1002/0471142727.mb0418s103',\n",
       "  'journal': 'Current protocols in molecular biology'},\n",
       " {'title': 'Ribosome profiling provides evidence that large noncoding RNAs do not encode proteins.',\n",
       "  'authors': ['Mitchell Guttman',\n",
       "   'Pamela Russell',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Jonathan S Weissman',\n",
       "   'Eric S Lander'],\n",
       "  'publication_date': '03/07/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23810193',\n",
       "  'abstract': \"Large noncoding RNAs are emerging as an important component in cellular regulation. Considerable evidence indicates that these transcripts act directly as functional RNAs rather than through an encoded protein product. However, a recent study of ribosome occupancy reported that many large intergenic ncRNAs (lincRNAs) are bound by ribosomes, raising the possibility that they are translated into proteins. Here, we show that classical noncoding RNAs and 5' UTRs show the same ribosome occupancy as lincRNAs, demonstrating that ribosome occupancy alone is not sufficient to classify transcripts as coding or noncoding. Instead, we define a metric based on the known property of translation whereby translating ribosomes are released upon encountering a bona fide stop codon. We show that this metric accurately discriminates between protein-coding transcripts and all classes of known noncoding transcripts, including lincRNAs. Taken together, these results argue that the large majority of lincRNAs do not function through encoded proteins. \",\n",
       "  'doi': '10.1016/j.cell.2013.06.009',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Integrated platform for genome-wide screening and construction of high-density genetic interaction maps in mammalian cells.',\n",
       "  'authors': ['Martin Kampmann', 'Michael C Bassik', 'Jonathan S Weissman'],\n",
       "  'publication_date': '07/06/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23739767',\n",
       "  'abstract': 'A major challenge of the postgenomic era is to understand how human genes function together in normal and disease states. In microorganisms, high-density genetic interaction (GI) maps are a powerful tool to elucidate gene functions and pathways. We have developed an integrated methodology based on pooled shRNA screening in mammalian cells for genome-wide identification of genes with relevant phenotypes and systematic mapping of all GIs among them. We recently demonstrated the potential of this approach in an application to pathways controlling the susceptibility of human cells to the toxin ricin. Here we present the complete quantitative framework underlying our strategy, including experimental design, derivation of quantitative phenotypes from pooled screens, robust identification of hit genes using ultra-complex shRNA libraries, parallel measurement of tens of thousands of GIs from a single double-shRNA experiment, and construction of GI maps. We describe the general applicability of our strategy. Our pooled approach enables rapid screening of the same shRNA library in different cell lines and under different conditions to determine a range of different phenotypes. We illustrate this strategy here for single- and double-shRNA libraries. We compare the roles of genes for susceptibility to ricin and Shiga toxin in different human cell lines and reveal both toxin-specific and cell line-specific pathways. We also present GI maps based on growth and ricin-resistance phenotypes, and we demonstrate how such a comparative GI mapping strategy enables functional dissection of physical complexes and context-dependent pathways.',\n",
       "  'doi': '10.1073/pnas.1307002110',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'rRNA:mRNA pairing alters the length and the symmetry of mRNA-protected fragments in ribosome profiling experiments.',\n",
       "  'authors': [\"Patrick B F O'Connor\",\n",
       "   'Gene-Wei Li',\n",
       "   'Jonathan S Weissman',\n",
       "   'John F Atkins',\n",
       "   'Pavel V Baranov'],\n",
       "  'publication_date': '23/04/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23603333',\n",
       "  'abstract': \"Ribosome profiling is a new technique that allows monitoring locations of translating ribosomes on mRNA at a whole transcriptome level. A recent ribosome profiling study demonstrated that internal Shine-Dalgarno (SD) sequences have a major global effect on translation rates in bacteria: ribosomes pause at SD sites in mRNA. Therefore, it is important to understand how SD sites effect mRNA movement through the ribosome and generation of ribosome footprints.\\nHere, we provide evidence that in addition to pausing effect, internal SD sequences induce a caterpillar-like movement of mRNA through the ribosome cavity. Once an SD site binds to the ribosome, it remains attached to it while the ribosome decodes a few subsequent codons. This leads to asymmetric progressive elongation of ribosome footprints at the 3'-end. It is likely that internal SD sequences induce a pause not on a single, but on several adjacent codons. This finding is important for our understanding of mRNA movement through the ribosome and also should facilitate interpretation of ribosome profiling data.\",\n",
       "  'doi': '10.1093/bioinformatics/btt184',\n",
       "  'journal': 'Bioinformatics (Oxford, England)'},\n",
       " {'title': 'Aneuploid yeast strains exhibit defects in cell growth and passage through START.',\n",
       "  'authors': ['Rebecca R Thorburn',\n",
       "   'Christian Gonzalez',\n",
       "   'Gloria A Brar',\n",
       "   'Stefan Christen',\n",
       "   'Thomas M Carlile',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Uwe Sauer',\n",
       "   'Jonathan S Weissman',\n",
       "   'Angelika Amon'],\n",
       "  'publication_date': '08/03/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23468524',\n",
       "  'abstract': 'Aneuploidy, a chromosome content that is not a multiple of the haploid karyotype, is associated with reduced fitness in all organisms analyzed to date. In budding yeast aneuploidy causes cell proliferation defects, with many different aneuploid strains exhibiting a delay in G1, a cell cycle stage governed by extracellular cues, growth rate, and cell cycle events. Here we characterize this G1 delay. We show that 10 of 14 aneuploid yeast strains exhibit a growth defect during G1. Furthermore, 10 of 14 aneuploid strains display a cell cycle entry delay that correlates with the size of the additional chromosome. This cell cycle entry delay is due to a delayed accumulation of G1 cyclins that can be suppressed by supplying cells with high levels of a G1 cyclin. Our results indicate that aneuploidy frequently interferes with the ability of cells to grow and, as with many other cellular stresses, entry into the cell cycle.',\n",
       "  'doi': '10.1091/mbc.E12-07-0520',\n",
       "  'journal': 'Molecular biology of the cell'},\n",
       " {'title': 'Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression.',\n",
       "  'authors': ['Lei S Qi',\n",
       "   'Matthew H Larson',\n",
       "   'Luke A Gilbert',\n",
       "   'Jennifer A Doudna',\n",
       "   'Jonathan S Weissman',\n",
       "   'Adam P Arkin',\n",
       "   'Wendell A Lim'],\n",
       "  'publication_date': '05/03/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23452860',\n",
       "  'abstract': 'Targeted gene regulation on a genome-wide scale is\\xa0a powerful strategy for interrogating, perturbing, and engineering cellular systems. Here, we develop a method for controlling gene expression based on Cas9, an RNA-guided DNA endonuclease from a type II CRISPR system. We show that a catalytically dead Cas9 lacking endonuclease activity, when coexpressed with a guide RNA, generates a DNA recognition complex that can specifically interfere with transcriptional elongation, RNA polymerase binding, or transcription factor binding. This system, which we call CRISPR interference (CRISPRi), can efficiently repress expression of targeted genes in Escherichia coli, with no detectable off-target effects. CRISPRi can be used to repress multiple target genes simultaneously, and its effects are reversible. We also show evidence that the system can be adapted for gene repression in mammalian cells. This RNA-guided DNA recognition platform provides a simple approach for selectively perturbing gene expression on a genome-wide scale.',\n",
       "  'doi': '10.1016/j.cell.2013.02.022',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'A systematic mammalian genetic interaction map reveals pathways underlying ricin susceptibility.',\n",
       "  'authors': ['Michael C Bassik',\n",
       "   'Martin Kampmann',\n",
       "   'Robert Jan Lebbink',\n",
       "   'Shuyi Wang',\n",
       "   'Marco Y Hein',\n",
       "   'Ina Poser',\n",
       "   'Jimena Weibezahn',\n",
       "   'Max A Horlbeck',\n",
       "   'Siyuan Chen',\n",
       "   'Matthias Mann',\n",
       "   'Anthony A Hyman',\n",
       "   'Emily M Leproust',\n",
       "   'Michael T McManus',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '12/02/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23394947',\n",
       "  'abstract': 'Genetic interaction (GI) maps, comprising pairwise measures of how strongly the function of one gene depends on the presence of a second, have enabled the systematic exploration of gene function in microorganisms. Here, we present a two-stage strategy to construct high-density GI maps in mammalian cells. First, we use ultracomplex pooled shRNA libraries (25 shRNAs/gene) to identify high-confidence hit genes for a given phenotype and effective shRNAs. We then construct double-shRNA libraries from these to systematically measure GIs between hits. A GI map focused on ricin susceptibility broadly recapitulates known pathways and provides many unexpected insights. These include a noncanonical role for COPI, a previously uncharacterized protein complex affecting toxin clearance, a specialized role for the ribosomal protein RPS25, and functionally distinct mammalian TRAPP complexes. The ability to rapidly generate mammalian GI maps provides a potentially transformative tool for defining gene function and designing combination therapies based on synergistic pairs.',\n",
       "  'doi': '10.1016/j.cell.2013.01.030',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'The contribution of systematic approaches to characterizing the proteins and functions of the endoplasmic reticulum.',\n",
       "  'authors': ['Maya Schuldiner', 'Jonathan S Weissman'],\n",
       "  'publication_date': '30/01/13',\n",
       "  'publication_year': '2013',\n",
       "  'pubmed_id': '23359093',\n",
       "  'abstract': 'The endoplasmic reticulum (ER) is a complex organelle responsible for a range of functions including protein folding and secretion, lipid biosynthesis, and ion homeostasis. Despite its central and essential roles in eukaryotic cells during development, growth, and disease, many ER proteins are poorly characterized. Moreover, the range of biochemical reactions that occur within the ER membranes, let alone how these different activities are coordinated, is not yet defined. In recent years, focused studies on specific ER functions have been complemented by systematic approaches and innovative technologies for high-throughput analysis of the location, levels, and biological impact of given components. This article focuses on the recent progress of these efforts, largely pioneered in the budding yeast Saccharomyces cerevisiae, and also addresses how future systematic studies can be geared to uncover the \"dark matter\" of uncharted ER functions.',\n",
       "  'doi': '10.1101/cshperspect.a013284',\n",
       "  'journal': 'Cold Spring Harbor perspectives in biology'},\n",
       " {'title': 'Decoding human cytomegalovirus.',\n",
       "  'authors': ['Noam Stern-Ginossar',\n",
       "   'Ben Weisburd',\n",
       "   'Annette Michalski',\n",
       "   'Vu Thuy Khanh Le',\n",
       "   'Marco Y Hein',\n",
       "   'Sheng-Xiong Huang',\n",
       "   'Ming Ma',\n",
       "   'Ben Shen',\n",
       "   'Shu-Bing Qian',\n",
       "   'Hartmut Hengel',\n",
       "   'Matthias Mann',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/11/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '23180859',\n",
       "  'abstract': 'The human cytomegalovirus (HCMV) genome was sequenced 20 years ago. However, like those of other complex viruses, our understanding of its protein coding potential is far from complete. We used ribosome profiling and transcript analysis to experimentally define the HCMV translation products and follow their temporal expression. We identified hundreds of previously unidentified open reading frames and confirmed a fraction by means of mass spectrometry. We found that regulated use of alternative transcript start sites plays a broad role in enabling tight temporal control of HCMV protein expression and allowing multiple distinct polypeptides to be generated from a single genomic locus. Our results reveal an unanticipated complexity to the HCMV coding capacity and illustrate the role of regulated changes in transcript start sites in generating this complexity.',\n",
       "  'doi': '10.1126/science.1227919',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'A ribosome-bound quality control complex triggers degradation of nascent peptides and signals translation stress.',\n",
       "  'authors': ['Onn Brandman',\n",
       "   'Jacob Stewart-Ornstein',\n",
       "   'Daisy Wong',\n",
       "   'Adam Larson',\n",
       "   'Christopher C Williams',\n",
       "   'Gene-Wei Li',\n",
       "   'Sharleen Zhou',\n",
       "   'David King',\n",
       "   'Peter S Shen',\n",
       "   'Jimena Weibezahn',\n",
       "   'Joshua G Dunn',\n",
       "   'Silvi Rouskin',\n",
       "   'Toshifumi Inada',\n",
       "   'Adam Frost',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/11/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '23178123',\n",
       "  'abstract': 'The conserved transcriptional regulator heat shock factor 1 (Hsf1) is a key sensor of proteotoxic and other stress in the eukaryotic cytosol. We surveyed Hsf1 activity in a genome-wide loss-of-function library in Saccaromyces cerevisiae as well as ~78,000 double mutants and found Hsf1 activity to be modulated by highly diverse stresses. These included disruption of a ribosome-bound complex we named the Ribosome Quality Control Complex (RQC) comprising the Ltn1 E3 ubiquitin ligase, two highly conserved but poorly characterized proteins (Tae2 and Rqc1), and Cdc48 and its cofactors. Electron microscopy and biochemical analyses revealed that the RQC forms a stable complex with 60S ribosomal subunits containing stalled polypeptides and triggers their degradation. A negative feedback loop regulates the RQC, and Hsf1 senses an RQC-mediated translation-stress signal distinctly from other stresses. Our work reveals the range of stresses Hsf1 monitors and elucidates a conserved cotranslational protein quality control mechanism.',\n",
       "  'doi': '10.1016/j.cell.2012.10.044',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'The ribosome profiling strategy for monitoring translation in vivo by deep sequencing of ribosome-protected mRNA fragments.',\n",
       "  'authors': ['Nicholas T Ingolia',\n",
       "   'Gloria A Brar',\n",
       "   'Silvia Rouskin',\n",
       "   'Anna M McGeachy',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '28/07/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22836135',\n",
       "  'abstract': 'Recent studies highlight the importance of translational control in determining protein abundance, underscoring the value of measuring gene expression at the level of translation. We present a protocol for genome-wide, quantitative analysis of in vivo translation by deep sequencing. This ribosome profiling approach maps the exact positions of ribosomes on transcripts by nuclease footprinting. The nuclease-protected mRNA fragments are converted into a DNA library suitable for deep sequencing using a strategy that minimizes bias. The abundance of different footprint fragments in deep sequencing data reports on the amount of translation of a gene. In addition, footprints reveal the exact regions of the transcriptome that are translated. To better define translated reading frames, we describe an adaptation that reveals the sites of translation initiation by pretreating cells with harringtonine to immobilize initiating ribosomes. The protocol we describe requires 5-7 days to generate a completed ribosome profiling sequencing library. Sequencing and data analysis require a further 4-5 days.',\n",
       "  'doi': '10.1038/nprot.2012.086',\n",
       "  'journal': 'Nature protocols'},\n",
       " {'title': 'Proto-genes and de novo gene birth.',\n",
       "  'authors': ['Anne-Ruxandra Carvunis',\n",
       "   'Thomas Rolland',\n",
       "   'Ilan Wapinski',\n",
       "   'Michael A Calderwood',\n",
       "   'Muhammed A Yildirim',\n",
       "   'Nicolas Simonis',\n",
       "   'Benoit Charloteaux',\n",
       "   'César A Hidalgo',\n",
       "   'Justin Barbette',\n",
       "   'Balaji Santhanam',\n",
       "   'Gloria A Brar',\n",
       "   'Jonathan S Weissman',\n",
       "   'Aviv Regev',\n",
       "   'Nicolas Thierry-Mieg',\n",
       "   'Michael E Cusick',\n",
       "   'Marc Vidal'],\n",
       "  'publication_date': '23/06/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22722833',\n",
       "  'abstract': \"Novel protein-coding genes can arise either through re-organization of pre-existing genes or de novo. Processes involving re-organization of pre-existing genes, notably after gene duplication, have been extensively described. In contrast, de novo gene birth remains poorly understood, mainly because translation of sequences devoid of genes, or 'non-genic' sequences, is expected to produce insignificant polypeptides rather than proteins with specific biological functions. Here we formalize an evolutionary model according to which functional genes evolve de novo through transitory proto-genes generated by widespread translational activity in non-genic sequences. Testing this model at the genome scale in Saccharomyces cerevisiae, we detect translation of hundreds of short species-specific open reading frames (ORFs) located in non-genic sequences. These translation events seem to provide adaptive potential, as suggested by their differential regulation upon stress and by signatures of retention by natural selection. In line with our model, we establish that S. cerevisiae ORFs can be placed within an evolutionary continuum ranging from non-genic sequences to genes. We identify ~1,900 candidate proto-genes among S. cerevisiae ORFs and find that de novo gene birth from such a reservoir may be more prevalent than sporadic gene duplication. Our work illustrates that evolution exploits seemingly dispensable sequences to generate adaptive functional innovation.\",\n",
       "  'doi': '10.1038/nature11184',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Functional repurposing revealed by comparing S. pombe and S. cerevisiae genetic interactions.',\n",
       "  'authors': ['Adam Frost',\n",
       "   'Marc G Elgort',\n",
       "   'Onn Brandman',\n",
       "   'Clinton Ives',\n",
       "   'Sean R Collins',\n",
       "   'Lakshmi Miller-Vedam',\n",
       "   'Jimena Weibezahn',\n",
       "   'Marco Y Hein',\n",
       "   'Ina Poser',\n",
       "   'Matthias Mann',\n",
       "   'Anthony A Hyman',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '12/06/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22682253',\n",
       "  'abstract': 'We present a genetic interaction map of pairwise measures including ∼40% of nonessential S. pombe genes. By comparing interaction maps for fission and budding yeast, we confirmed widespread conservation of genetic relationships within and between complexes and pathways. However, we identified an important subset of orthologous complexes that have undergone functional \"repurposing\": the evolution of divergent functions and partnerships. We validated three functional repurposing events in S. pombe and mammalian cells and discovered that (1) two lumenal sensors of misfolded ER proteins, the kinase/nuclease Ire1 and the glucosyltransferase Gpt1, act together to mount an ER stress response; (2) ESCRT factors regulate spindle-pole-body duplication; and (3) a membrane-protein phosphatase and kinase complex, the STRIPAK complex, bridges the cis-Golgi, the centrosome, and the outer nuclear membrane to direct mitotic progression. Each discovery opens new areas of inquiry and-together-have implications for model organism-based research and the evolution of genetic systems.',\n",
       "  'doi': '10.1016/j.cell.2012.04.028',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Special issue: the molecular and cellular mechanisms of amyloidosis.',\n",
       "  'authors': ['Sheena E Radford', 'Jonathan S Weissman'],\n",
       "  'publication_date': '06/06/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22664198',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1016/j.jmb.2012.05.042',\n",
       "  'journal': 'Journal of molecular biology'},\n",
       " {'title': 'Native elongating transcript sequencing (NET-seq).',\n",
       "  'authors': ['L Stirling Churchman', 'Jonathan S Weissman'],\n",
       "  'publication_date': '04/04/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22470065',\n",
       "  'abstract': \"Advances in sequencing technology have led to the development of many high-resolution methodologies that observe genomic activity and gene expression. This unit describes such an approach, native elongating transcript sequencing (NET-seq), which reveals the density of RNA polymerase across the Saccharomyces cerevisiae genome with single-nucleotide resolution. A procedure for capturing nascent RNA transcripts directly from live cells through their association with the DNA-RNA-RNAP ternary complex is described. A protocol to create DNA libraries from the nascent RNA, allowing the identity and abundance of the 3' end of purified transcripts to be revealed by next generation sequencing, is also provided. By deep sequencing this DNA library, a quantitative measure of RNAP density with single-nucleotide precision is obtained. The quantitative nature of the NET-seq dataset relies on the high efficiency of many steps in the protocol. The steps that are most critical are described with suggestions for monitoring their success.\",\n",
       "  'doi': '10.1002/0471142727.mb0414s98',\n",
       "  'journal': 'Current protocols in molecular biology'},\n",
       " {'title': 'The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria.',\n",
       "  'authors': ['Gene-Wei Li', 'Eugene Oh', 'Jonathan S Weissman'],\n",
       "  'publication_date': '30/03/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22456704',\n",
       "  'abstract': 'Protein synthesis by ribosomes takes place on a linear substrate but at non-uniform speeds. Transient pausing of ribosomes can affect a variety of co-translational processes, including protein targeting and folding. These pauses are influenced by the sequence of the messenger RNA. Thus, redundancy in the genetic code allows the same protein to be translated at different rates. However, our knowledge of both the position and the mechanism of translational pausing in vivo is highly limited. Here we present a genome-wide analysis of translational pausing in bacteria by ribosome profiling--deep sequencing of ribosome-protected mRNA fragments. This approach enables the high-resolution measurement of ribosome density profiles along most transcripts at unperturbed, endogenous expression levels. Unexpectedly, we found that codons decoded by rare transfer RNAs do not lead to slow translation under nutrient-rich conditions. Instead, Shine-Dalgarno-(SD)-like features within coding sequences cause pervasive translational pausing. Using an orthogonal ribosome possessing an altered anti-SD sequence, we show that pausing is due to hybridization between the mRNA and 16S ribosomal RNA of the translating ribosome. In protein-coding sequences, internal SD sequences are disfavoured, which leads to biased usage, avoiding codons and codon pairs that resemble canonical SD sites. Our results indicate that internal SD-like sequences are a major determinant of translation rates and a global driving force for the coding of bacterial genomes.',\n",
       "  'doi': '10.1038/nature10965',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'The translational landscape of mTOR signalling steers cancer initiation and metastasis.',\n",
       "  'authors': ['Andrew C Hsieh',\n",
       "   'Yi Liu',\n",
       "   'Merritt P Edlind',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Matthew R Janes',\n",
       "   'Annie Sher',\n",
       "   'Evan Y Shi',\n",
       "   'Craig R Stumpf',\n",
       "   'Carly Christensen',\n",
       "   'Michael J Bonham',\n",
       "   'Shunyou Wang',\n",
       "   'Pingda Ren',\n",
       "   'Michael Martin',\n",
       "   'Katti Jessen',\n",
       "   'Morris E Feldman',\n",
       "   'Jonathan S Weissman',\n",
       "   'Kevan M Shokat',\n",
       "   'Christian Rommel',\n",
       "   'Davide Ruggero'],\n",
       "  'publication_date': '01/03/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22367541',\n",
       "  'abstract': \"The mammalian target of rapamycin (mTOR) kinase is a master regulator of protein synthesis that couples nutrient sensing to cell growth and cancer. However, the downstream translationally regulated nodes of gene expression that may direct cancer development are poorly characterized. Using ribosome profiling, we uncover specialized translation of the prostate cancer genome by oncogenic mTOR signalling, revealing a remarkably specific repertoire of genes involved in cell proliferation, metabolism and invasion. We extend these findings by functionally characterizing a class of translationally controlled pro-invasion messenger RNAs that we show direct prostate cancer invasion and metastasis downstream of oncogenic mTOR signalling. Furthermore, we develop a clinically relevant ATP site inhibitor of mTOR, INK128, which reprograms this gene expression signature with therapeutic benefit for prostate cancer metastasis, for which there is presently no cure. Together, these findings extend our understanding of how the 'cancerous' translation machinery steers specific cancer cell behaviours, including metastasis, and may be therapeutically targeted.\",\n",
       "  'doi': '10.1038/nature10912',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Cellular noise regulons underlie fluctuations in Saccharomyces cerevisiae.',\n",
       "  'authors': ['Jacob Stewart-Ornstein',\n",
       "   'Jonathan S Weissman',\n",
       "   'Hana El-Samad'],\n",
       "  'publication_date': '01/03/12',\n",
       "  'publication_year': '2012',\n",
       "  'pubmed_id': '22365828',\n",
       "  'abstract': 'Stochasticity is a hallmark of cellular processes, and different classes of genes show large differences in their cell-to-cell variability (noise). To decipher the sources and consequences of this noise, we systematically measured pairwise correlations between large numbers of genes, including those with high variability. We find that there is substantial pathway variability shared across similarly regulated genes. This induces quantitative correlations in the expression of functionally related genes such as those involved in the Msn2/4 stress response pathway, amino-acid\\xa0biosynthesis, and mitochondrial maintenance. Bioinformatic analyses and genetic perturbations suggest that fluctuations in PKA and Tor signaling contribute to pathway-specific variability. Our results argue that a limited number of well-delineated \"noise regulons\" operate across a yeast cell and that such coordinated fluctuations enable a stochastic but coherent induction of functionally related genes. Finally, we show that pathway noise is a quantitative tool for exploring pathway features and regulatory relationships in un-stimulated systems.',\n",
       "  'doi': '10.1016/j.molcel.2011.11.035',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'High-resolution view of the yeast meiotic program revealed by ribosome profiling.',\n",
       "  'authors': ['Gloria A Brar',\n",
       "   'Moran Yassour',\n",
       "   'Nir Friedman',\n",
       "   'Aviv Regev',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '24/12/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22194413',\n",
       "  'abstract': \"Meiosis is a complex developmental process that generates haploid cells from diploid progenitors. We measured messenger RNA (mRNA) abundance and protein production through the yeast meiotic sporulation program and found strong, stage-specific expression for most genes, achieved through control of both mRNA levels and translational efficiency. Monitoring of protein production timing revealed uncharacterized recombination factors and extensive organellar remodeling. Meiotic translation is also shifted toward noncanonical sites, including short open reading frames (ORFs) on unannnotated transcripts and upstream regions of known transcripts (uORFs). Ribosome occupancy at near-cognate uORFs was associated with more efficient ORF translation; by contrast, some AUG uORFs, often exposed by regulated 5' leader extensions, acted competitively. This work reveals pervasive translational control in meiosis and helps to illuminate the molecular basis of the broad restructuring of meiotic cells.\",\n",
       "  'doi': '10.1126/science.1215110',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Genetics: Noise rules.',\n",
       "  'authors': ['Hana El-Samad', 'Jonathan S Weissman'],\n",
       "  'publication_date': '14/12/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22158239',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/480188a',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Selective ribosome profiling reveals the cotranslational chaperone action of trigger factor in vivo.',\n",
       "  'authors': ['Eugene Oh',\n",
       "   'Annemarie H Becker',\n",
       "   'Arzu Sandikci',\n",
       "   'Damon Huber',\n",
       "   'Rachna Chaba',\n",
       "   'Felix Gloge',\n",
       "   'Robert J Nichols',\n",
       "   'Athanasios Typas',\n",
       "   'Carol A Gross',\n",
       "   'Günter Kramer',\n",
       "   'Jonathan S Weissman',\n",
       "   'Bernd Bukau'],\n",
       "  'publication_date': '14/12/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22153074',\n",
       "  'abstract': 'As nascent polypeptides exit ribosomes, they are engaged by a series of processing, targeting, and folding factors. Here, we present a selective ribosome profiling strategy that enables global monitoring of when these factors engage polypeptides in the complex cellular environment. Studies of the Escherichia coli chaperone trigger factor (TF) reveal that, though TF can interact with many polypeptides, β-barrel outer-membrane proteins are the most prominent substrates. Loss of TF leads to broad outer-membrane defects and premature, cotranslational protein translocation. Whereas in vitro studies suggested that TF is prebound to ribosomes waiting for polypeptides to emerge from the exit channel, we find that in vivo TF engages ribosomes only after ~100 amino acids are translated. Moreover, excess TF interferes with cotranslational removal of the N-terminal formyl methionine. Our studies support a triaging model in which proper protein biogenesis relies on the fine-tuned, sequential engagement of processing, targeting, and folding factors.',\n",
       "  'doi': '10.1016/j.cell.2011.10.044',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Road to ruin: targeting proteins for degradation in the endoplasmic reticulum.',\n",
       "  'authors': ['Melanie H Smith', 'Hidde L Ploegh', 'Jonathan S Weissman'],\n",
       "  'publication_date': '26/11/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22116878',\n",
       "  'abstract': 'Some nascent proteins that fold within the endoplasmic reticulum (ER) never reach their native state. Misfolded proteins are removed from the folding machinery, dislocated from the ER into the cytosol, and degraded in a series of pathways collectively referred to as ER-associated degradation (ERAD). Distinct ERAD pathways centered on different E3 ubiquitin ligases survey the range of potential substrates. We now know many of the components of the ERAD machinery and pathways used to detect substrates and target them for degradation. Much less is known about the features used to identify terminally misfolded conformations and the broader role of these pathways in regulating protein half-lives.',\n",
       "  'doi': '10.1126/science.1209235',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Protein kinase Ypk1 phosphorylates regulatory proteins Orm1 and Orm2 to control sphingolipid homeostasis in Saccharomyces cerevisiae.',\n",
       "  'authors': ['Françoise M Roelants',\n",
       "   'David K Breslow',\n",
       "   'Alexander Muir',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jeremy Thorner'],\n",
       "  'publication_date': '15/11/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22080611',\n",
       "  'abstract': 'The Orm family proteins are conserved integral membrane proteins of the endoplasmic reticulum that are key homeostatic regulators of sphingolipid biosynthesis. Orm proteins bind to and inhibit serine:palmitoyl-coenzyme A transferase, the first enzyme in sphingolipid biosynthesis. In Saccharomyces cerevisiae, Orm1 and Orm2 are inactivated by phosphorylation in response to compromised sphingolipid synthesis (e.g., upon addition of inhibitor myriocin), thereby restoring sphingolipid production. We show here that protein kinase Ypk1, one of an essential pair of protein kinases, is responsible for this regulatory modification. Myriocin-induced hyperphosphorylation of Orm1 and Orm2 does not occur in ypk1 cells, and immunopurified Ypk1 phosphorylates Orm1 and Orm2 robustly in vitro exclusively on three residues that are known myriocin-induced sites. Furthermore, the temperature-sensitive growth of ypk1(ts) ypk2 cells is substantially ameliorated by deletion of ORM genes, confirming that a primary physiological role of Ypk1-mediated phosphorylation is to negatively regulate Orm function. Ypk1 immunoprecipitated from myriocin-treated cells displays a higher specific activity for Orm phosphorylation than Ypk1 from untreated cells. To identify the mechanism underlying Ypk1 activation, we systematically tested several candidate factors and found that the target of rapamycin complex 2 (TORC2) kinase plays a key role. In agreement with prior evidence that a TORC2-dependent site in Ypk1(T662) is necessary for cells to exhibit a wild-type level of myriocin resistance, a Ypk1(T662A) mutant displays only weak Orm phosphorylation in vivo and only weak activation in vitro in response to sphingolipid depletion. Additionally, sphingolipid depletion increases phosphorylation of Ypk1 at T662. Thus, Ypk1 is both a sensor and effector of sphingolipid level, and reduction in sphingolipids stimulates Ypk1, at least in part, via TORC2-dependent phosphorylation.',\n",
       "  'doi': '10.1073/pnas.1116948108',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes.',\n",
       "  'authors': ['Nicholas T Ingolia', 'Liana F Lareau', 'Jonathan S Weissman'],\n",
       "  'publication_date': '08/11/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '22056041',\n",
       "  'abstract': 'The ability to sequence genomes has far outstripped approaches for deciphering the information they encode. Here we present a suite of techniques, based on ribosome profiling (the deep sequencing of ribosome-protected mRNA fragments), to provide genome-wide maps of protein synthesis as well as a pulse-chase strategy for determining rates of translation elongation. We exploit the propensity of harringtonine to cause ribosomes to accumulate at sites of translation initiation together with a machine learning algorithm to define protein products systematically. Analysis of translation in mouse embryonic stem cells reveals thousands of strong pause sites and unannotated translation products. These include amino-terminal extensions and truncations and upstream open reading frames with regulatory potential, initiated at both AUG and non-AUG codons, whose translation changes after differentiation. We also define a class of short, polycistronic ribosome-associated coding RNAs (sprcRNAs) that encode small proteins. Our studies reveal an unanticipated complexity to mammalian proteomes.',\n",
       "  'doi': '10.1016/j.cell.2011.10.002',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'A mitochondrial-focused genetic interaction map reveals a scaffold-like complex required for inner membrane organization in mitochondria.',\n",
       "  'authors': ['Suzanne Hoppins',\n",
       "   'Sean R Collins',\n",
       "   'Ann Cassidy-Stone',\n",
       "   'Eric Hummel',\n",
       "   'Rachel M Devay',\n",
       "   'Laura L Lackner',\n",
       "   'Benedikt Westermann',\n",
       "   'Maya Schuldiner',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jodi Nunnari'],\n",
       "  'publication_date': '12/10/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21987634',\n",
       "  'abstract': 'To broadly explore mitochondrial structure and function as well as the communication of mitochondria with other cellular pathways, we constructed a quantitative, high-density genetic interaction map (the MITO-MAP) in Saccharomyces cerevisiae. The MITO-MAP provides a comprehensive view of mitochondrial function including insights into the activity of uncharacterized mitochondrial proteins and the functional connection between mitochondria and the ER. The MITO-MAP also reveals a large inner membrane-associated complex, which we term MitOS for mitochondrial organizing structure, comprised of Fcj1/Mitofilin, a conserved inner membrane protein, and five additional components. MitOS physically and functionally interacts with both outer and inner membrane components and localizes to extended structures that wrap around the inner membrane. We show that MitOS acts in concert with ATP synthase dimers to organize the inner membrane and promote normal mitochondrial morphology. We propose that MitOS acts as a conserved mitochondrial skeletal structure that differentiates regions of the inner membrane to establish the normal internal architecture of mitochondria.',\n",
       "  'doi': '10.1083/jcb.201107053',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'Ganglion-specific splicing of TRPV1 underlies infrared sensation in vampire bats.',\n",
       "  'authors': ['Elena O Gracheva',\n",
       "   'Julio F Cordero-Morales',\n",
       "   'José A González-Carcacía',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Carlo Manno',\n",
       "   'Carla I Aranguren',\n",
       "   'Jonathan S Weissman',\n",
       "   'David Julius'],\n",
       "  'publication_date': '05/08/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21814281',\n",
       "  'abstract': \"Vampire bats (Desmodus rotundus) are obligate blood feeders that have evolved specialized systems to suit their sanguinary lifestyle. Chief among such adaptations is the ability to detect infrared radiation as a means of locating hotspots on warm-blooded prey. Among vertebrates, only vampire bats, boas, pythons and pit vipers are capable of detecting infrared radiation. In each case, infrared signals are detected by trigeminal nerve fibres that innervate specialized pit organs on the animal's face. Thus, vampire bats and snakes have taken thermosensation to the extreme by developing specialized systems for detecting infrared radiation. As such, these creatures provide a window into the molecular and genetic mechanisms underlying evolutionary tuning of thermoreceptors in a species-specific or cell-type-specific manner. Previously, we have shown that snakes co-opt a non-heat-sensitive channel, vertebrate TRPA1 (transient receptor potential cation channel A1), to produce an infrared detector. Here we show that vampire bats tune a channel that is already heat-sensitive, TRPV1, by lowering its thermal activation threshold to about 30\\u2009°C. This is achieved through alternative splicing of TRPV1 transcripts to produce a channel with a truncated carboxy-terminal cytoplasmic domain. These splicing events occur exclusively in trigeminal ganglia, and not in dorsal root ganglia, thereby maintaining a role for TRPV1 as a detector of noxious heat in somatic afferents. This reflects a unique organization of the bat Trpv1 gene that we show to be characteristic of Laurasiatheria mammals (cows, dogs and moles), supporting a close phylogenetic relationship with bats. These findings reveal a novel molecular mechanism for physiological tuning of thermosensory nerve fibres.\",\n",
       "  'doi': '10.1038/nature10245',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'H3K4 trimethylation by Set1 promotes efficient termination by the Nrd1-Nab3-Sen1 pathway.',\n",
       "  'authors': ['Nihal Terzi',\n",
       "   'L Stirling Churchman',\n",
       "   'Lidia Vasiljeva',\n",
       "   'Jonathan Weissman',\n",
       "   'Stephen Buratowski'],\n",
       "  'publication_date': '29/06/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21709022',\n",
       "  'abstract': 'In Saccharomyces cerevisiae, the Nrd1-Nab3-Sen1 pathway mediates the termination of snoRNAs and cryptic unstable transcripts (CUTs). Both Nrd1 and the Set1 histone H3K4 methyltransferase complex interact with RNA polymerase II (Pol II) during early elongation, leading us to test whether these two processes are functionally linked. The deletion of SET1 exacerbates the growth rate and termination defects of nrd1 mutants. Set1 is important for the appropriate recruitment of Nrd1. Additionally, Set1 modulates histone acetylation levels in the promoter-proximal region via the Rpd3L deacetylase and NuA3 acetyltransferase complexes, both of which contain PHD finger proteins that bind methylated H3K4. Increased levels of histone acetylation reduce the efficiency of Nrd1-dependent termination. We speculate that Set1 promotes proper early termination by the Nrd1-Nab3-Sen1 complex by affecting the kinetics of Pol II transcription in early elongation.',\n",
       "  'doi': '10.1128/MCB.05590-11',\n",
       "  'journal': 'Molecular and cellular biology'},\n",
       " {'title': 'Amyloid structure: conformational diversity and consequences.',\n",
       "  'authors': ['Brandon H Toyama', 'Jonathan S Weissman'],\n",
       "  'publication_date': '05/04/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21456964',\n",
       "  'abstract': 'Many, perhaps most, proteins, are capable of forming self-propagating, β-sheet (amyloid) aggregates. Amyloid-like aggregates are found in a wide range of diseases and underlie prion-based inheritance. Despite intense interest in amyloids, structural details have only recently begun to be revealed as advances in biophysical approaches, such as hydrogen-deuterium exchange, X-ray crystallography, solid-state nuclear magnetic resonance (SSNMR), and cryoelectron microscopy (cryoEM), have enabled high-resolution insights into their molecular organization. Initial studies found that despite the highly divergent primary structure of different amyloid-forming proteins, amyloids from different sources share many structural similarities. With higher-resolution information, however, it has become clear that, on the molecular level, amyloids comprise a wide diversity of structures. Particularly surprising has been the finding that identical polypeptides can fold into multiple, distinct amyloid conformations and that this structural diversity can lead to distinct heritable prion states or strains.',\n",
       "  'doi': '10.1146/annurev-biochem-090908-120656',\n",
       "  'journal': 'Annual review of biochemistry'},\n",
       " {'title': 'Strain conformation, primary structure and the propagation of the yeast prion [PSI+].',\n",
       "  'authors': ['Katherine J Verges',\n",
       "   'Melanie H Smith',\n",
       "   'Brandon H Toyama',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '23/03/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21423194',\n",
       "  'abstract': 'Prion proteins can adopt multiple infectious strain conformations. Here we investigate how the sequence of a prion protein affects its capacity to propagate specific conformations by exploiting our ability to create two distinct infectious conformations of the yeast [PSI(+)] prion protein Sup35, termed Sc4 and Sc37. PNM2, a G58D point mutant of Sup35 that was originally identified for its dominant interference with prion propagation, leads to rapid, recessive loss of Sc4 but does not interfere with propagation of Sc37. PNM2 destabilizes the amyloid core of Sc37 and causes compensatory effects that slow prion growth but aid prion division and result in robust propagation of Sc37. By contrast, PNM2 does not affect the structure or chaperone-mediated division of Sc4 but interferes with its delivery to daughter cells. Thus, effective delivery of infectious particles during cell division is a crucial and conformation-dependent step in prion inheritance.',\n",
       "  'doi': '10.1038/nsmb.2030',\n",
       "  'journal': 'Nature structural & molecular biology'},\n",
       " {'title': 'Radically different amyloid conformations dictate the seeding specificity of a chimeric Sup35 prion.',\n",
       "  'authors': ['Catherine K Foo',\n",
       "   'Yumiko Ohhashi',\n",
       "   'Mark J S Kelly',\n",
       "   'Motomasa Tanaka',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '22/02/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21333653',\n",
       "  'abstract': 'A remarkable feature of prion biology is that the same prion protein can misfold into more than one infectious conformation, and these conformations in turn lead to distinct heritable prion strains with different phenotypes. The yeast prion [PSI(+)] is a powerful system for studying how changes in strain conformation affect cross-species transmission. We have previously established that a chimera of the Saccharomyces cerevisiae (SC) and Candida albicans (CA) Sup35 prion domains can cross the SC/CA species barrier in a strain-dependent manner. In vitro, the conversion of the monomeric chimera into the prion (amyloid) form can be seeded by either SC or CA Sup35 amyloid fibers, resulting in two strains: Chim[SC] and Chim[CA]. These strains have a \"molecular memory\" of their originating species in that Chim[SC] preferentially seeds the conversion of SC Sup35, and vice versa. To investigate how this species specificity is conformationally encoded, we used amide exchange and limited proteolysis to probe the structures of these two strains. We found that the amyloid cores of Chim[SC] and Chim[CA] are predominantly confined to the SC-derived and CA-derived residues, respectively. In addition, the chimera is able to propagate the Chim[CA] conformation even when the SC residues comprising the Chim[SC] core were deleted. Thus, the two strains have non-overlapping and modular amyloid cores that determine whether SC or CA residues are presented on the growing face of the prion seed. These observations establish how conformations determine the specificity of prion transmission and demonstrate a remarkable plasticity to amyloid misfolding.',\n",
       "  'doi': '10.1016/j.jmb.2011.02.025',\n",
       "  'journal': 'Journal of molecular biology'},\n",
       " {'title': 'Nascent transcript sequencing visualizes transcription at nucleotide resolution.',\n",
       "  'authors': ['L Stirling Churchman', 'Jonathan S Weissman'],\n",
       "  'publication_date': '21/01/11',\n",
       "  'publication_year': '2011',\n",
       "  'pubmed_id': '21248844',\n",
       "  'abstract': \"Recent studies of transcription have revealed a level of complexity not previously appreciated even a few years ago, both in the intricate use of post-initiation control and the mass production of rapidly degraded transcripts. Dissection of these pathways requires strategies for precisely following transcripts as they are being produced. Here we present an approach (native elongating transcript sequencing, NET-seq), based on deep sequencing of 3' ends of nascent transcripts associated with RNA polymerase, to monitor transcription at nucleotide resolution. Application of NET-seq in Saccharomyces cerevisiae reveals that although promoters are generally capable of divergent transcription, the Rpd3S deacetylation complex enforces strong directionality to most promoters by suppressing antisense transcript initiation. Our studies also reveal pervasive polymerase pausing and backtracking throughout the body of transcripts. Average pause density shows prominent peaks at each of the first four nucleosomes, with the peak location occurring in good agreement with in vitro biophysical measurements. Thus, nucleosome-induced pausing represents a major barrier to transcriptional elongation in vivo.\",\n",
       "  'doi': '10.1038/nature09652',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Systematic screen of Schizosaccharomyces pombe deletion collection uncovers parallel evolution of the phosphate signal transduction pathway in yeasts.',\n",
       "  'authors': ['Theresa C Henry',\n",
       "   'Juliette E Power',\n",
       "   'Christine L Kerwin',\n",
       "   'Aishat Mohammed',\n",
       "   'Jonathan S Weissman',\n",
       "   'Dale M Cameron',\n",
       "   'Dennis D Wykoff'],\n",
       "  'publication_date': '21/12/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '21169418',\n",
       "  'abstract': 'The phosphate signal transduction (PHO) pathway, which regulates genes in response to phosphate starvation, is well defined in Saccharomyces cerevisiae. We asked whether the PHO pathway was the same in the distantly related fission yeast Schizosaccharomyces pombe. We screened a deletion collection for mutants aberrant in phosphatase activity, which is primarily a consequence of pho1(+) transcription. We identified a novel zinc finger-containing protein (encoded by spbc27b12.11c(+)), which we have named pho7(+), that is essential for pho1(+) transcriptional induction during phosphate starvation. Few of the S. cerevisiae genes involved in the PHO pathway appear to be involved in the regulation of the phosphate starvation response in S. pombe. Only the most upstream genes in the PHO pathway in S. cerevisiae (ADO1, DDP1, and PPN1) share a similar role in both yeasts. Because ADO1 and DDP1 regulate ATP and IP(7) levels, we hypothesize that the ancestor of these yeasts must have sensed similar metabolites in response to phosphate starvation but have evolved distinct mechanisms in parallel to sense these metabolites and induce phosphate starvation genes.',\n",
       "  'doi': '10.1128/EC.00216-10',\n",
       "  'journal': 'Eukaryotic cell'},\n",
       " {'title': 'The epistemology of cell biology.',\n",
       "  'authors': ['Jonathan S Weissman'],\n",
       "  'publication_date': '17/11/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '21079033',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1091/mbc.E10-04-0370',\n",
       "  'journal': 'Molecular biology of the cell'},\n",
       " {'title': 'Membranes in balance: mechanisms of sphingolipid homeostasis.',\n",
       "  'authors': ['David K Breslow', 'Jonathan S Weissman'],\n",
       "  'publication_date': '23/10/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20965421',\n",
       "  'abstract': 'Sphingolipids and their metabolites play key cellular roles both as structural components of membranes and as signaling molecules that mediate responses to physiologic cues and stresses. Despite progress during the last two decades in defining the enzymatic machinery responsible for synthesizing and degrading sphingolipids, comparatively little is known about how these enzymes are regulated to ensure sphingolipid homeostasis. Here, we review new insights into how cells sense and control sphingolipid biosynthesis and transport. We also discuss emerging evidence that sphingolipid metabolism is closely coordinated with that of sterols and glycerolipids and with other processes that occur in the secretory pathway. An improved understanding of sphingolipid homeostasis promises to shed light on basic processes in cell biology and disease, including how cells establish and maintain the complex membrane composition and architecture that is a defining feature of eukaryotic cell biology.',\n",
       "  'doi': '10.1016/j.molcel.2010.10.005',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Mammalian microRNAs predominantly act to decrease target mRNA levels.',\n",
       "  'authors': ['Huili Guo',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Jonathan S Weissman',\n",
       "   'David P Bartel'],\n",
       "  'publication_date': '13/08/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20703300',\n",
       "  'abstract': 'MicroRNAs (miRNAs) are endogenous approximately 22-nucleotide RNAs that mediate important gene-regulatory events by pairing to the mRNAs of protein-coding genes to direct their repression. Repression of these regulatory targets leads to decreased translational efficiency and/or decreased mRNA levels, but the relative contributions of these two outcomes have been largely unknown, particularly for endogenous targets expressed at low-to-moderate levels. Here, we use ribosome profiling to measure the overall effects on protein production and compare these to simultaneously measured effects on mRNA levels. For both ectopic and endogenous miRNA regulatory interactions, lowered mRNA levels account for most (>/=84%) of the decreased protein production. These results show that changes in mRNA levels closely reflect the impact of miRNAs on gene expression and indicate that destabilization of target mRNAs is the predominant reason for reduced protein output.',\n",
       "  'doi': '10.1038/nature09267',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Automated identification of pathways from quantitative genetic interaction data.',\n",
       "  'authors': ['Alexis Battle',\n",
       "   'Martin C Jonikas',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Daphne Koller'],\n",
       "  'publication_date': '10/06/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20531408',\n",
       "  'abstract': 'High-throughput quantitative genetic interaction (GI) measurements provide detailed information regarding the structure of the underlying biological pathways by reporting on functional dependencies between genes. However, the analytical tools for fully exploiting such information lag behind the ability to collect these data. We present a novel Bayesian learning method that uses quantitative phenotypes of double knockout organisms to automatically reconstruct detailed pathway structures. We applied our method to a recent data set that measures GIs for endoplasmic reticulum (ER) genes, using the unfolded protein response as a quantitative phenotype. The results provided reconstructions of known functional pathways including N-linked glycosylation and ER-associated protein degradation. It also contained novel relationships, such as the placement of SGT2 in the tail-anchored biogenesis pathway, a finding that we experimentally validated. Our approach should be readily applicable to the next generation of quantitative GI data sets, as assays become available for additional phenotypes and eventually higher-level organisms.',\n",
       "  'doi': '10.1038/msb.2010.27',\n",
       "  'journal': 'Molecular systems biology'},\n",
       " {'title': 'A general lack of compensation for gene dosage in yeast.',\n",
       "  'authors': ['Michael Springer', 'Jonathan S Weissman', 'Marc W Kirschner'],\n",
       "  'publication_date': '13/05/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20461075',\n",
       "  'abstract': 'Gene copy number variation has been discovered in humans, between related species, and in different cancer tissues, but it is unclear how much of this genomic-level variation leads to changes in the level of protein abundance. To address this, we eliminated one of the two genomic copies of 730 different genes in Saccharomyces cerevisiae and asked how often a 50% reduction in gene dosage leads to a 50% reduction in protein level. For at least 80% of genes tested, and under several environmental conditions, it does: protein levels in the heterozygous strain are close to 50% of wild type. For <5% of the genes tested, the protein levels in the heterozygote are maintained at nearly wild-type levels. These experiments show that protein levels are not, in general, directly monitored and adjusted to a desired level. Combined with fitness data, this implies that proteins are expressed at levels higher than necessary for survival.',\n",
       "  'doi': '10.1038/msb.2010.19',\n",
       "  'journal': 'Molecular systems biology'},\n",
       " {'title': 'J domain co-chaperone specificity defines the role of BiP during protein translocation.',\n",
       "  'authors': ['Shruthi S Vembar',\n",
       "   'Martin C Jonikas',\n",
       "   'Linda M Hendershot',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jeffrey L Brodsky'],\n",
       "  'publication_date': '01/05/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20430885',\n",
       "  'abstract': 'Hsp70 chaperones can potentially interact with one of several J domain-containing Hsp40 co-chaperones to regulate distinct cellular processes. However, features within Hsp70s that determine Hsp40 specificity are undefined. To investigate this question, we introduced mutations into the ER-lumenal Hsp70, BiP/Kar2p, and found that an R217A substitution in the J domain-interacting surface of BiP compromised the physical and functional interaction with Sec63p, an Hsp40 required for ER translocation. In contrast, interaction with Jem1p, an Hsp40 required for ER-associated degradation, was unaffected. Moreover, yeast expressing R217A BiP exhibited defects in translocation but not in ER-associated degradation. Finally, the genetic interactions of the R217A BiP mutant were found to correlate with those of known translocation mutants. Together, our results indicate that residues within the Hsp70 J domain-interacting surface help confer Hsp40 specificity, in turn influencing distinct chaperone-mediated cellular activities.',\n",
       "  'doi': '10.1074/jbc.M110.102186',\n",
       "  'journal': 'The Journal of biological chemistry'},\n",
       " {'title': 'Molecular basis of infrared detection by snakes.',\n",
       "  'authors': ['Elena O Gracheva',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Yvonne M Kelly',\n",
       "   'Julio F Cordero-Morales',\n",
       "   'Gunther Hollopeter',\n",
       "   'Alexander T Chesler',\n",
       "   'Elda E Sánchez',\n",
       "   'John C Perez',\n",
       "   'Jonathan S Weissman',\n",
       "   'David Julius'],\n",
       "  'publication_date': '17/03/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20228791',\n",
       "  'abstract': \"Snakes possess a unique sensory system for detecting infrared radiation, enabling them to generate a 'thermal image' of predators or prey. Infrared signals are initially received by the pit organ, a highly specialized facial structure that is innervated by nerve fibres of the somatosensory system. How this organ detects and transduces infrared signals into nerve impulses is not known. Here we use an unbiased transcriptional profiling approach to identify TRPA1 channels as infrared receptors on sensory nerve fibres that innervate the pit organ. TRPA1 orthologues from pit-bearing snakes (vipers, pythons and boas) are the most heat-sensitive vertebrate ion channels thus far identified, consistent with their role as primary transducers of infrared stimuli. Thus, snakes detect infrared signals through a mechanism involving radiant heating of the pit organ, rather than photochemical transduction. These findings illustrate the broad evolutionary tuning of transient receptor potential (TRP) channels as thermosensors in the vertebrate nervous system.\",\n",
       "  'doi': '10.1038/nature08943',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Orm family proteins mediate sphingolipid homeostasis.',\n",
       "  'authors': ['David K Breslow',\n",
       "   'Sean R Collins',\n",
       "   'Bernd Bodenmiller',\n",
       "   'Ruedi Aebersold',\n",
       "   'Kai Simons',\n",
       "   'Andrej Shevchenko',\n",
       "   'Christer S Ejsing',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '26/02/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20182505',\n",
       "  'abstract': 'Despite the essential roles of sphingolipids both as structural components of membranes and critical signalling molecules, we have a limited understanding of how cells sense and regulate their levels. Here we reveal the function in sphingolipid metabolism of the ORM genes (known as ORMDL genes in humans)-a conserved gene family that includes ORMDL3, which has recently been identified as a potential risk factor for childhood asthma. Starting from an unbiased functional genomic approach in Saccharomyces cerevisiae, we identify Orm proteins as negative regulators of sphingolipid synthesis that form a conserved complex with serine palmitoyltransferase, the first and rate-limiting enzyme in sphingolipid production. We also define a regulatory pathway in which phosphorylation of Orm proteins relieves their inhibitory activity when sphingolipid production is disrupted. Changes in ORM gene expression or mutations to their phosphorylation sites cause dysregulation of sphingolipid metabolism. Our work identifies the Orm proteins as critical mediators of sphingolipid homeostasis and raises the possibility that sphingolipid misregulation contributes to the development of childhood asthma.',\n",
       "  'doi': '10.1038/nature08787',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Differences in prion strain conformations result from non-native interactions in a nucleus.',\n",
       "  'authors': ['Yumiko Ohhashi',\n",
       "   'Kazuki Ito',\n",
       "   'Brandon H Toyama',\n",
       "   'Jonathan S Weissman',\n",
       "   'Motomasa Tanaka'],\n",
       "  'publication_date': '19/01/10',\n",
       "  'publication_year': '2010',\n",
       "  'pubmed_id': '20081853',\n",
       "  'abstract': \"Aggregation-prone proteins often misfold into multiple distinct amyloid conformations that dictate different physiological impacts. Although amyloid formation is triggered by a transient nucleus, the mechanism by which an initial nucleus is formed and allows the protein to form a specific amyloid conformation has been unclear. Here we show that, before fiber formation, the prion domain (Sup35NM, consisting of residues 1-254) of yeast prion Sup35, the [PSI(+)] protein determinant, forms oligomers in a temperature-dependent, reversible manner. Mutational and biophysical analyses revealed that 'non-native' aromatic interactions outside the amyloid core drive oligomer formation by bringing together different Sup35NM monomers, which specifically leads to the formation of highly infectious strain conformations with more limited amyloid cores. Thus, transient non-native interactions in the initial nucleus are pivotal in determining the diversity of amyloid conformations and resulting prion strain phenotypes.\",\n",
       "  'doi': '10.1038/nchembio.306',\n",
       "  'journal': 'Nature chemical biology'},\n",
       " {'title': 'From information to knowledge:  new technologies for defining gene  function.',\n",
       "  'authors': ['Sean R Collins', 'Jonathan S Weissman', 'Nevan J Krogan'],\n",
       "  'publication_date': '03/12/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19953683',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/nmeth1009-721',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Journal club. A biochemist looks at how DNA sequencing can reveal more than just sequences.',\n",
       "  'authors': ['Jonathan Weissman'],\n",
       "  'publication_date': '30/10/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19865125',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/4611177e',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Regulated Ire1-dependent decay of messenger RNAs in mammalian cells.',\n",
       "  'authors': ['Julie Hollien',\n",
       "   'Jonathan H Lin',\n",
       "   'Han Li',\n",
       "   'Nicole Stevens',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '05/08/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19651891',\n",
       "  'abstract': 'Maintenance of endoplasmic reticulum (ER) function is achieved in part through Ire1 (inositol-requiring enzyme 1), a transmembrane protein activated by protein misfolding in the ER. The cytoplasmic nuclease domain of Ire1 cleaves the messenger RNA (mRNA) encoding XBP-1 (X-box-binding protein 1), enabling splicing and production of this active transcription factor. We recently showed that Ire1 activation independently induces the rapid turnover of mRNAs encoding membrane and secreted proteins in Drosophila melanogaster cells through a pathway we call regulated Ire1-dependent decay (RIDD). In this study, we show that mouse fibroblasts expressing wild-type Ire1 but not an Ire1 variant lacking nuclease activity also degrade mRNAs in response to ER stress. Using a second variant of Ire1 that is activated by a small adenosine triphosphate analogue, we show that although XBP-1 splicing can be artificially induced in the absence of ER stress, RIDD appears to require both Ire1 activity and ER stress. Our data suggest that cells use a multitiered mechanism by which different conditions in the ER lead to distinct outputs from Ire1.',\n",
       "  'doi': '10.1083/jcb.200903014',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'An ER-mitochondria tethering complex revealed by a synthetic biology screen.',\n",
       "  'authors': ['Benoît Kornmann',\n",
       "   'Erin Currie',\n",
       "   'Sean R Collins',\n",
       "   'Maya Schuldiner',\n",
       "   'Jodi Nunnari',\n",
       "   'Jonathan S Weissman',\n",
       "   'Peter Walter'],\n",
       "  'publication_date': '27/06/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19556461',\n",
       "  'abstract': 'Communication between organelles is an important feature of all eukaryotic cells. To uncover components involved in mitochondria/endoplasmic reticulum (ER) junctions, we screened for mutants that could be complemented by a synthetic protein designed to artificially tether the two organelles. We identified the Mmm1/Mdm10/Mdm12/Mdm34 complex as a molecular tether between ER and mitochondria. The tethering complex was composed of proteins resident of both ER and mitochondria. With the use of genome-wide mapping of genetic interactions, we showed that the components of the tethering complex were functionally connected to phospholipid biosynthesis and calcium-signaling genes. In mutant cells, phospholipid biosynthesis was impaired. The tethering complex localized to discrete foci, suggesting that discrete sites of close apposition between ER and mitochondria facilitate interorganelle calcium and phospholipid exchange.',\n",
       "  'doi': '10.1126/science.1175088',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Rapid creation and quantitative monitoring of high coverage shRNA libraries.',\n",
       "  'authors': ['Michael C Bassik',\n",
       "   'Robert Jan Lebbink',\n",
       "   'L Stirling Churchman',\n",
       "   'Nicholas T Ingolia',\n",
       "   'Weronika Patena',\n",
       "   'Emily M LeProust',\n",
       "   'Maya Schuldiner',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael T McManus'],\n",
       "  'publication_date': '19/05/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19448642',\n",
       "  'abstract': 'Short hairpin RNA libraries are limited by low efficacy of many shRNAs and by off-target effects, which give rise to false negatives and false positives, respectively. Here we present a strategy for rapidly creating expanded shRNA pools (approximately 30 shRNAs per gene) that are analyzed by deep sequencing (EXPAND). This approach enables identification of multiple effective target-specific shRNAs from a complex pool, allowing a rigorous statistical evaluation of true hits.',\n",
       "  'doi': '10.1038/nmeth.1330',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Analysis of Dom34 and its function in no-go decay.',\n",
       "  'authors': ['Dario O Passos',\n",
       "   'Meenakshi K Doma',\n",
       "   'Christopher J Shoemaker',\n",
       "   'Denise Muhlrad',\n",
       "   'Rachel Green',\n",
       "   'Jonathan Weissman',\n",
       "   'Julie Hollien',\n",
       "   'Roy Parker'],\n",
       "  'publication_date': '08/05/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19420139',\n",
       "  'abstract': 'Eukaryotic mRNAs are subject to quality control mechanisms that degrade defective mRNAs. In yeast, mRNAs with stalls in translation elongation are targeted for endonucleolytic cleavage by No-Go decay (NGD). The cleavage triggered by No-Go decay is dependent on Dom34p and Hbs1p, and Dom34 has been proposed to be the endonuclease responsible for mRNA cleavage. We created several Dom34 mutants and examined their effects on NGD in yeast. We identified mutations in several loops of the Dom34 structure that affect NGD. In contrast, mutations inactivating the proposed nuclease domain do not affect NGD in vivo. Moreover, we observed that overexpression of the Rps30a protein, a high copy suppressor of dom34Delta cold sensitivity, can restore some mRNA cleavage in a dom34Delta strain. These results identify important functional regions of Dom34 and suggest that the proposed endonuclease activity of Dom34 is not required for mRNA cleavage in NGD. We also provide evidence that the process of NGD is conserved in insect cells. On the basis of these results and the process of translation termination, we suggest a multistep model for the process of NGD.',\n",
       "  'doi': '10.1091/mbc.e09-01-0028',\n",
       "  'journal': 'Molecular biology of the cell'},\n",
       " {'title': 'Comprehensive characterization of genes required for protein folding in the endoplasmic reticulum.',\n",
       "  'authors': ['Martin C Jonikas',\n",
       "   'Sean R Collins',\n",
       "   'Vladimir Denic',\n",
       "   'Eugene Oh',\n",
       "   'Erin M Quan',\n",
       "   'Volker Schmid',\n",
       "   'Jimena Weibezahn',\n",
       "   'Blanche Schwappach',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Maya Schuldiner'],\n",
       "  'publication_date': '28/03/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19325107',\n",
       "  'abstract': \"Protein folding in the endoplasmic reticulum is a complex process whose malfunction is implicated in disease and aging. By using the cell's endogenous sensor (the unfolded protein response), we identified several hundred yeast genes with roles in endoplasmic reticulum folding and systematically characterized their functional interdependencies by measuring unfolded protein response levels in double mutants. This strategy revealed multiple conserved factors critical for endoplasmic reticulum folding, including an intimate dependence on the later secretory pathway, a previously uncharacterized six-protein transmembrane complex, and a co-chaperone complex that delivers tail-anchored proteins to their membrane insertion machinery. The use of a quantitative reporter in a comprehensive screen followed by systematic analysis of genetic dependencies should be broadly applicable to functional dissection of complex cellular processes from yeast to human.\",\n",
       "  'doi': '10.1126/science.1167983',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Functional organization of the S. cerevisiae phosphorylation network.',\n",
       "  'authors': ['Dorothea Fiedler',\n",
       "   'Hannes Braberg',\n",
       "   'Monika Mehta',\n",
       "   'Gal Chechik',\n",
       "   'Gerard Cagney',\n",
       "   'Paromita Mukherjee',\n",
       "   'Andrea C Silva',\n",
       "   'Michael Shales',\n",
       "   'Sean R Collins',\n",
       "   'Sake van Wageningen',\n",
       "   'Patrick Kemmeren',\n",
       "   'Frank C P Holstege',\n",
       "   'Jonathan S Weissman',\n",
       "   'Michael-Christopher Keogh',\n",
       "   'Daphne Koller',\n",
       "   'Kevan M Shokat',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '10/03/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19269370',\n",
       "  'abstract': 'Reversible protein phosphorylation is a signaling mechanism involved in all cellular processes. To create a systems view of the signaling apparatus in budding yeast, we generated an epistatic miniarray profile (E-MAP) comprised of 100,000 pairwise, quantitative genetic interactions, including virtually all protein and small-molecule kinases and phosphatases as well as key cellular regulators. Quantitative genetic interaction mapping reveals factors working in compensatory pathways (negative genetic interactions) or those operating in linear pathways (positive genetic interactions). We found an enrichment of positive genetic interactions between kinases, phosphatases, and their substrates. In addition, we assembled a higher-order map from sets of three genes that display strong interactions with one another: triplets enriched for functional connectivity. The resulting network view provides insights into signaling pathway regulation and reveals a link between the cell-cycle kinase, Cak1, the Fus3 MAP kinase, and a pathway that regulates chromatin integrity during transcription by RNA polymerase II.',\n",
       "  'doi': '10.1016/j.cell.2008.12.039',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling.',\n",
       "  'authors': ['Nicholas T Ingolia',\n",
       "   'Sina Ghaemmaghami',\n",
       "   'John R S Newman',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '14/02/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19213877',\n",
       "  'abstract': 'Techniques for systematically monitoring protein translation have lagged far behind methods for measuring messenger RNA (mRNA) levels. Here, we present a ribosome-profiling strategy that is based on the deep sequencing of ribosome-protected mRNA fragments and enables genome-wide investigation of translation with subcodon resolution. We used this technique to monitor translation in budding yeast under both rich and starvation conditions. These studies defined the protein sequences being translated and found extensive translational control in both determining absolute protein abundance and responding to environmental stress. We also observed distinct phases during translation that involve a large decrease in ribosome density going from early to late peptide elongation as well as widespread regulated initiation at non-adenine-uracil-guanine (AUG) codons. Ribosome profiling is readily adaptable to other organisms, making high-precision investigation of protein translation experimentally accessible.',\n",
       "  'doi': '10.1126/science.1168978',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'High-throughput, quantitative analyses of genetic interactions in E. coli.',\n",
       "  'authors': ['Athanasios Typas',\n",
       "   'Robert J Nichols',\n",
       "   'Deborah A Siegele',\n",
       "   'Michael Shales',\n",
       "   'Sean R Collins',\n",
       "   'Bentley Lim',\n",
       "   'Hannes Braberg',\n",
       "   'Natsuko Yamamoto',\n",
       "   'Rikiya Takeuchi',\n",
       "   'Barry L Wanner',\n",
       "   'Hirotada Mori',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nevan J Krogan',\n",
       "   'Carol A Gross'],\n",
       "  'publication_date': '23/01/09',\n",
       "  'publication_year': '2009',\n",
       "  'pubmed_id': '19160513',\n",
       "  'abstract': 'Large-scale genetic interaction studies provide the basis for defining gene function and pathway architecture. Recent advances in the ability to generate double mutants en masse in Saccharomyces cerevisiae have dramatically accelerated the acquisition of genetic interaction information and the biological inferences that follow. Here we describe a method based on F factor-driven conjugation, which allows for high-throughput generation of double mutants in Escherichia coli. This method, termed genetic interaction analysis technology for E. coli (GIANT-coli), permits us to systematically generate and array double-mutant cells on solid media in high-density arrays. We show that colony size provides a robust and quantitative output of cellular fitness and that GIANT-coli can recapitulate known synthetic interactions and identify previously unidentified negative (synthetic sickness or lethality) and positive (suppressive or epistatic) relationships. Finally, we describe a complementary strategy for genome-wide suppressor-mutant identification. Together, these methods permit rapid, large-scale genetic interaction studies in E. coli.',\n",
       "  'doi': '10.1038/nmeth.1240',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Defining the glycan destruction signal for endoplasmic reticulum-associated degradation.',\n",
       "  'authors': ['Erin M Quan',\n",
       "   'Yukiko Kamiya',\n",
       "   'Daiki Kamiya',\n",
       "   'Vladimir Denic',\n",
       "   'Jimena Weibezahn',\n",
       "   'Koichi Kato',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '30/12/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '19111666',\n",
       "  'abstract': \"The endoplasmic reticulum (ER) must target potentially toxic misfolded proteins for retrotranslocation and proteasomal degradation while avoiding destruction of productive folding intermediates. For luminal proteins, this discrimination typically depends not only on the folding status of a polypeptide, but also on its glycosylation state. Two putative sugar binding proteins, Htm1p and Yos9p, are required for degradation of misfolded glycoproteins, but the nature of the glycan degradation signal and how such signals are generated and decoded remains unclear. Here we characterize Yos9p's oligosaccharide-binding specificity and find that it recognizes glycans containing terminal alpha1,6-linked mannose residues. We also provide evidence in vivo that a terminal alpha1,6-linked mannose-containing oligosaccharide is required for degradation and that Htm1p acts upstream of Yos9p to mediate the generation of such sugars. This strategy of marking potential substrates by Htm1p and decoding the signal by Yos9p is well suited to provide a proofreading mechanism that enhances substrate specificity.\",\n",
       "  'doi': '10.1016/j.molcel.2008.11.017',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'A genetic interaction map of RNA-processing factors reveals links between Sem1/Dss1-containing complexes and mRNA export and splicing.',\n",
       "  'authors': ['Gwendolyn M Wilmes',\n",
       "   'Megan Bergkessel',\n",
       "   'Sourav Bandyopadhyay',\n",
       "   'Michael Shales',\n",
       "   'Hannes Braberg',\n",
       "   'Gerard Cagney',\n",
       "   'Sean R Collins',\n",
       "   'Gregg B Whitworth',\n",
       "   'Tracy L Kress',\n",
       "   'Jonathan S Weissman',\n",
       "   'Trey Ideker',\n",
       "   'Christine Guthrie',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '09/12/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '19061648',\n",
       "  'abstract': 'We used a quantitative, high-density genetic interaction map, or E-MAP (Epistatic MiniArray Profile), to interrogate the relationships within and between RNA-processing pathways. Due to their complexity and the essential roles of many of the components, these pathways have been difficult to functionally dissect. Here, we report the results for 107,155 individual interactions involving 552 mutations, 166 of which are hypomorphic alleles of essential genes. Our data enabled the discovery of links between components of the mRNA export and splicing machineries and Sem1/Dss1, a component of the 19S proteasome. In particular, we demonstrate that Sem1 has a proteasome-independent role in mRNA export as a functional component of the Sac3-Thp1 complex. Sem1 also interacts with Csn12, a component of the COP9 signalosome. Finally, we show that Csn12 plays a role in pre-mRNA splicing, which is independent of other signalosome components. Thus, Sem1 is involved in three separate and functionally distinct complexes.',\n",
       "  'doi': '10.1016/j.molcel.2008.11.012',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'In vivo monitoring of the prion replication cycle reveals a critical role for Sis1 in delivering substrates to Hsp104.',\n",
       "  'authors': ['Kimberly A Tipton',\n",
       "   'Katherine J Verges',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '26/11/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '19026788',\n",
       "  'abstract': 'Prions in Saccharomyces cerevisiae are inherited ordered aggregates reliant upon the disaggregase Hsp104 for stable maintenance. The function of other factors in the natural prion cycle is unclear. We constructed yeast-bacterial chimeric chaperones to resolve the roles of Hsp104 domains, and by extension chaperones that interact with these domains, in prion propagation. Our results show that, as with amorphous aggregate dissolution, the Hsp70/40 system recruits prion substrates to Hsp104 via its top ring. By adapting our chimera to couple to an inactive protease \"trap,\" we monitored the reaction products of prion propagation in vivo. We find that prion maintenance is accompanied by translocation of prion proteins through Hsp104 hexamers and that both processes critically rely upon the Hsp40 Sis1. Our data suggest that yeast prion replication is a natural extension of chaperone activity in dissolving amorphous aggregates, distinguished from its ancestral reaction by the ordered, self-propagating structure of the substrate.',\n",
       "  'doi': '10.1016/j.molcel.2008.11.003',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Activity motifs reveal principles of timing in  transcriptional control of the yeast metabolic network.',\n",
       "  'authors': ['Gal Chechik',\n",
       "   'Eugene Oh',\n",
       "   'Oliver Rando',\n",
       "   'Jonathan Weissman',\n",
       "   'Aviv Regev',\n",
       "   'Daphne Koller'],\n",
       "  'publication_date': '28/10/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18953355',\n",
       "  'abstract': \"Significant insight about biological networks arises from the study of network motifs--overly abundant network subgraphs--but such wiring patterns do not specify when and how potential routes within a cellular network are used. To address this limitation, we introduce activity motifs, which capture patterns in the dynamic use of a network. Using this framework to analyze transcription in Saccharomyces cerevisiae metabolism, we find that cells use different timing activity motifs to optimize transcription timing in response to changing conditions: forward activation to produce metabolic compounds efficiently, backward shutoff to rapidly stop production of a detrimental product and synchronized activation for co-production of metabolites required for the same reaction. Measuring protein abundance over a time course reveals that mRNA timing motifs also occur at the protein level. Timing motifs significantly overlap with binding activity motifs, where genes in a linear chain have ordered binding affinity to a transcription factor, suggesting a mechanism for ordered transcription. Finely timed transcriptional regulation is therefore abundant in yeast metabolism, optimizing the organism's adaptation to new environmental conditions.\",\n",
       "  'doi': '10.1038/nbt.1499',\n",
       "  'journal': 'Nature biotechnology'},\n",
       " {'title': 'Conservation and rewiring of functional modules revealed by an epistasis map in fission yeast.',\n",
       "  'authors': ['Assen Roguev',\n",
       "   'Sourav Bandyopadhyay',\n",
       "   'Martin Zofall',\n",
       "   'Ke Zhang',\n",
       "   'Tamas Fischer',\n",
       "   'Sean R Collins',\n",
       "   'Hongjing Qu',\n",
       "   'Michael Shales',\n",
       "   'Han-Oh Park',\n",
       "   'Jacqueline Hayles',\n",
       "   'Kwang-Lae Hoe',\n",
       "   'Dong-Uk Kim',\n",
       "   'Trey Ideker',\n",
       "   'Shiv I Grewal',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '27/09/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18818364',\n",
       "  'abstract': 'An epistasis map (E-MAP) was constructed in the fission yeast, Schizosaccharomyces pombe, by systematically measuring the phenotypes associated with pairs of mutations. This high-density, quantitative genetic interaction map focused on various aspects of chromosome function, including transcription regulation and DNA repair/replication. The E-MAP uncovered a previously unidentified component of the RNA interference (RNAi) machinery (rsh1) and linked the RNAi pathway to several other biological processes. Comparison of the S. pombe E-MAP to an analogous genetic map from the budding yeast revealed that, whereas negative interactions were conserved between genes involved in similar biological processes, positive interactions and overall genetic profiles between pairs of genes coding for physically associated proteins were even more conserved. Hence, conservation occurs at the level of the functional module (protein complex), but the genetic cross talk between modules can differ substantially.',\n",
       "  'doi': '10.1126/science.1162609',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'Systems biology: Reverse engineering the cell.',\n",
       "  'authors': ['Nicholas T Ingolia', 'Jonathan S Weissman'],\n",
       "  'publication_date': '30/08/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18756243',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/4541059a',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'The GET complex mediates insertion of tail-anchored proteins into the ER membrane.',\n",
       "  'authors': ['Maya Schuldiner',\n",
       "   'Jutta Metz',\n",
       "   'Volker Schmid',\n",
       "   'Vladimir Denic',\n",
       "   'Magdalena Rakwalska',\n",
       "   'Hans Dieter Schmitt',\n",
       "   'Blanche Schwappach',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '30/08/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18724936',\n",
       "  'abstract': 'Tail-anchored (TA) proteins, defined by the presence of a single C-terminal transmembrane domain (TMD), play critical roles throughout the secretory pathway and in mitochondria, yet the machinery responsible for their proper membrane insertion remains poorly characterized. Here we show that Get3, the yeast homolog of the TA-interacting factor Asna1/Trc40, specifically recognizes TMDs of TA proteins destined for the secretory pathway. Get3 recognition represents a key decision step, whose loss can lead to misinsertion of TA proteins into mitochondria. Get3-TA protein complexes are recruited for endoplasmic reticulum (ER) membrane insertion by the Get1/Get2 receptor. In vivo, the absence of Get1/Get2 leads to cytosolic aggregation of Get3-TA complexes and broad defects in TA protein biogenesis. In vitro reconstitution demonstrates that the Get proteins directly mediate insertion of newly synthesized TA proteins into ER membranes. Thus, the GET complex represents a critical mechanism for ensuring efficient and accurate targeting of TA proteins.',\n",
       "  'doi': '10.1016/j.cell.2008.06.025',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Construction, verification and experimental use of two epitope-tagged collections of budding yeast strains.',\n",
       "  'authors': ['Russell Howson',\n",
       "   'Won-Ki Huh',\n",
       "   'Sina Ghaemmaghami',\n",
       "   'James V Falvo',\n",
       "   'Kiowa Bower',\n",
       "   'Archana Belle',\n",
       "   'Noah Dephoure',\n",
       "   'Dennis D Wykoff',\n",
       "   'Jonathan S Weissman',\n",
       "   \"Erin K O'Shea\"],\n",
       "  'publication_date': '17/07/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18629296',\n",
       "  'abstract': 'A major challenge in the post-genomic era is the development of experimental approaches to monitor the properties of proteins on a proteome-wide level. It would be particularly useful to systematically assay protein subcellular localization, post-translational modifications and protein-protein interactions, both at steady state and in response to environmental stimuli. Development of new reagents and methods will enhance our ability to do so efficiently and systematically. Here we describe the construction of two collections of budding yeast strains that facilitate proteome-wide measurements of protein properties. These collections consist of strains with an epitope tag integrated at the C-terminus of essentially every open reading frame (ORF), one with the tandem affinity purification (TAP) tag, and one with the green fluorescent protein (GFP) tag. We show that in both of these collections we have accurately tagged a high proportion of all ORFs (approximately 75% of the proteome) by confirming expression of the fusion proteins. Furthermore, we demonstrate the use of the TAP collection in performing high-throughput immunoprecipitation experiments. Building on these collections and the methods described in this paper, we hope that the yeast community will expand both the quantity and type of proteome level data available.',\n",
       "  'doi': '10.1002/cfg.449',\n",
       "  'journal': 'Comparative and functional genomics'},\n",
       " {'title': 'A comprehensive strategy enabling high-resolution functional analysis of the yeast genome.',\n",
       "  'authors': ['David K Breslow',\n",
       "   'Dale M Cameron',\n",
       "   'Sean R Collins',\n",
       "   'Maya Schuldiner',\n",
       "   'Jacob Stewart-Ornstein',\n",
       "   'Heather W Newman',\n",
       "   'Sigurd Braun',\n",
       "   'Hiten D Madhani',\n",
       "   'Nevan J Krogan',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '16/07/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18622397',\n",
       "  'abstract': 'Functional genomic studies in Saccharomyces cerevisiae have contributed enormously to our understanding of cellular processes. Their full potential, however, has been hampered by the limited availability of reagents to systematically study essential genes and the inability to quantify the small effects of most gene deletions on growth. Here we describe the construction of a library of hypomorphic alleles of essential genes and a high-throughput growth competition assay to measure fitness with unprecedented sensitivity. These tools dramatically increase the breadth and precision with which quantitative genetic analysis can be performed in yeast. We illustrate the value of these approaches by using genetic interactions to reveal new relationships between chromatin-modifying factors and to create a functional map of the proteasome. Finally, by measuring the fitness of strains in the yeast deletion library, we addressed an enigma regarding the apparent prevalence of gene dispensability and found that most genes do contribute to growth.',\n",
       "  'doi': '10.1038/nmeth.1234',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'Small-molecule aggregates inhibit amyloid polymerization.',\n",
       "  'authors': ['Brian Y Feng',\n",
       "   'Brandon H Toyama',\n",
       "   'Holger Wille',\n",
       "   'David W Colby',\n",
       "   'Sean R Collins',\n",
       "   'Barnaby C H May',\n",
       "   'Stanley B Prusiner',\n",
       "   'Jonathan Weissman',\n",
       "   'Brian K Shoichet'],\n",
       "  'publication_date': '29/01/08',\n",
       "  'publication_year': '2008',\n",
       "  'pubmed_id': '18223646',\n",
       "  'abstract': 'Many amyloid inhibitors resemble molecules that form chemical aggregates, which are known to inhibit many proteins. Eight known chemical aggregators inhibited amyloid formation of the yeast and mouse prion proteins Sup35 and recMoPrP in a manner characteristic of colloidal inhibition. Similarly, three known anti-amyloid molecules inhibited beta-lactamase in a detergent-dependent manner, which suggests that they too form colloidal aggregates. The colloids localized to preformed fibers and prevented new fiber formation in electron micrographs. They also blocked infection of yeast cells with Sup35 prions, which suggests that colloidal inhibition may be relevant in more biological milieus.',\n",
       "  'doi': '10.1038/nchembio.65',\n",
       "  'journal': 'Nature chemical biology'},\n",
       " {'title': 'Identification of yeast proteins necessary for cell-surface function of a potassium channel.',\n",
       "  'authors': ['Friederike A Haass',\n",
       "   'Martin Jonikas',\n",
       "   'Peter Walter',\n",
       "   'Jonathan S Weissman',\n",
       "   'Yuh-Nung Jan',\n",
       "   'Lily Y Jan',\n",
       "   'Maya Schuldiner'],\n",
       "  'publication_date': '09/11/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17989219',\n",
       "  'abstract': 'Inwardly rectifying potassium (Kir) channels form gates in the cell membrane that regulate the flow of K(+) ions into and out of the cell, thereby influencing the membrane potential and electrical signaling of many cell types, including neurons and cardiomyocytes. Kir-channel function depends on other cellular proteins that aid in the folding of channel subunits, assembly into tetrameric complexes, trafficking of quality-controlled channels to the plasma membrane, and regulation of channel activity at the cell surface. We used the yeast Saccharomyces cerevisiae as a model system to identify proteins necessary for the functional expression of a mammalian Kir channel at the cell surface. A screen of 376 yeast strains, each lacking one nonessential protein localized to the early secretory pathway, identified seven deletion strains in which functional expression of the Kir channel at the plasma membrane was impaired. Six deletions were of genes with known functions in trafficking and lipid biosynthesis (sur4Delta, csg2Delta, erv14Delta, emp24Delta, erv25Delta, and bst1Delta), and one deletion was of an uncharacterized gene (yil039wDelta). We provide genetic and functional evidence that Yil039wp, a conserved, phosphoesterase domain-containing protein, which we named \"trafficking of Emp24p/Erv25p-dependent cargo disrupted 1\" (Ted1p), acts together with Emp24p/Erv25p in cargo exit from the endoplasmic reticulum (ER). The seven yeast proteins identified in our screen likely impact Kir-channel functional expression at the level of vesicle budding from the ER and/or the local lipid environment at the plasma membrane.',\n",
       "  'doi': '10.1073/pnas.0708765104',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'High-throughput genetic interaction mapping in the fission yeast Schizosaccharomyces pombe.',\n",
       "  'authors': ['Assen Roguev',\n",
       "   'Marianna Wiren',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '26/09/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17893680',\n",
       "  'abstract': \"Epistasis analysis, which reports on the extent to which the function of one gene depends on the presence of a second, is a powerful tool for studying the functional organization of the cell. Systematic genome-wide studies of epistasis, however, have been limited, with the majority of data being collected in the budding yeast, Saccharomyces cerevisiae. Here we present two 'pombe epistasis mapper' strategies, PEM-1 and PEM-2, which allow for high-throughput double mutant generation in the fission yeast, S. pombe. These approaches take advantage of a previously undescribed, recessive, cycloheximide-resistance mutation. Both systems can be used for genome-wide screens or for the generation of high-density, quantitative epistatic miniarray profiles (E-MAPs). Since S. cerevisiae and S. pombe are evolutionary distant, this methodology will provide insight into conserved biological pathways that are present in S. pombe, but not S. cerevisiae, and will enable a comprehensive analysis of the conservation of genetic interaction networks.\",\n",
       "  'doi': '10.1038/nmeth1098',\n",
       "  'journal': 'Nature methods'},\n",
       " {'title': 'The structural basis of yeast prion strain variants.',\n",
       "  'authors': ['Brandon H Toyama',\n",
       "   'Mark J S Kelly',\n",
       "   'John D Gross',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '04/09/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17767153',\n",
       "  'abstract': \"Among the many surprises to arise from studies of prion biology, perhaps the most unexpected is the strain phenomenon whereby a single protein can misfold into structurally distinct, infectious states that cause distinguishable phenotypes. Similarly, proteins can adopt a spectrum of conformations in non-infectious diseases of protein folding; some are toxic and others are well tolerated. However, our understanding of the structural differences underlying prion strains and how these differences alter their physiological impact remains limited. Here we use a combination of solution NMR, amide hydrogen/deuterium (H/D) exchange and mutagenesis to study the structural differences between two strain conformations, termed Sc4 and Sc37 (ref. 5), of the yeast Sup35 prion. We find that these two strains have an overlapping amyloid core spanning most of the Gln/Asn-rich first 40 amino acids that is highly protected from H/D exchange and very sensitive to mutation. These features indicate that the cores are composed of tightly packed beta-sheets possibly resembling 'steric zipper' structures revealed by X-ray crystallography of Sup35-derived peptides. The stable structure is greatly expanded in the Sc37 conformation to encompass the first 70 amino acids, revealing why this strain shows increased fibre stability and decreased ability to undergo chaperone-mediated replication. Our findings establish that prion strains involve large-scale conformational differences and provide a structural basis for understanding a broad range of functional studies, including how conformational changes alter the physiological impact of prion strains.\",\n",
       "  'doi': '10.1038/nature06108',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'A molecular caliper mechanism for determining very long-chain fatty acid length.',\n",
       "  'authors': ['Vladimir Denic', 'Jonathan S Weissman'],\n",
       "  'publication_date': '28/08/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17719544',\n",
       "  'abstract': 'Very long-chain fatty acids (VLCFAs) are essential lipids whose functional diversity is enabled by variation in their chain length. The full VLCFA biosynthetic machinery and how this machinery generates structural diversity remain elusive. Proteoliposomes reconstituted here from purified membrane components-an elongase protein (Elop), a novel dehydratase, and two reductases-catalyzed repeated rounds of two-carbon addition that elongated shorter FAs into VLCFAs whose length was dictated by the specific Elop homolog present. Mutational analysis revealed that the Elop active site faces the cytosol, whereas VLCFA length is determined by a lysine near the luminal end of an Elop transmembrane helix. By stepping the lysine residue along one face of the helix toward the cytosol, we engineered novel synthases with correspondingly shorter VLCFA outputs. Thus the distance between the active site and the lysine residue determines chain length. Our results uncover a mutationally adjustable, caliper-like mechanism that generates the repertoire of cellular VLCFAs.',\n",
       "  'doi': '10.1016/j.cell.2007.06.031',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Mec1/Tel1 phosphorylation of the INO80 chromatin remodeling complex influences DNA damage checkpoint responses.',\n",
       "  'authors': ['Ashby J Morrison',\n",
       "   'Jung-Ae Kim',\n",
       "   'Maria D Person',\n",
       "   'Jessica Highland',\n",
       "   'Jing Xiao',\n",
       "   'Tammy S Wehr',\n",
       "   'Sean Hensley',\n",
       "   'Yunhe Bao',\n",
       "   'Jianjun Shen',\n",
       "   'Sean R Collins',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jeff Delrow',\n",
       "   'Nevan J Krogan',\n",
       "   'James E Haber',\n",
       "   'Xuetong Shen'],\n",
       "  'publication_date': '19/08/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17693258',\n",
       "  'abstract': \"The yeast Mec1/Tel1 kinases, ATM/ATR in mammals, coordinate the DNA damage response by phosphorylating proteins involved in DNA repair and checkpoint pathways. Recently, ATP-dependent chromatin remodeling complexes, such as the INO80 complex, have also been implicated in DNA damage responses, although regulatory mechanisms that direct their function remain unknown. Here, we show that the Ies4 subunit of the INO80 complex is phosphorylated by the Mec1/Tel1 kinases during exposure to DNA-damaging agents. Mutation of Ies4's phosphorylation sites does not significantly affect DNA repair processes, but does influence DNA damage checkpoint responses. Additionally, ies4 phosphorylation mutants are linked to the function of checkpoint regulators, such as the replication checkpoint factors Tof1 and Rad53. These findings establish a chromatin remodeling complex as a functional component in the Mec1/Tel1 DNA damage signaling pathway that modulates checkpoint responses and suggest that posttranslational modification of chromatin remodeling complexes regulates their involvement in distinct processes.\",\n",
       "  'doi': '10.1016/j.cell.2007.06.010',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Backup without redundancy: genetic interactions reveal the cost of duplicate gene loss.',\n",
       "  'authors': ['Jan Ihmels',\n",
       "   'Sean R Collins',\n",
       "   'Maya Schuldiner',\n",
       "   'Nevan J Krogan',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '29/03/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17389874',\n",
       "  'abstract': \"Many genes can be deleted with little phenotypic consequences. By what mechanism and to what extent the presence of duplicate genes in the genome contributes to this robustness against deletions has been the subject of considerable interest. Here, we exploit the availability of high-density genetic interaction maps to provide direct support for the role of backup compensation, where functionally overlapping duplicates cover for the loss of their paralog. However, we find that the overall contribution of duplicates to robustness against null mutations is low ( approximately 25%). The ability to directly identify buffering paralogs allowed us to further study their properties, and how they differ from non-buffering duplicates. Using environmental sensitivity profiles as well as quantitative genetic interaction spectra as high-resolution phenotypes, we establish that even duplicate pairs with compensation capacity exhibit rich and typically non-overlapping deletion phenotypes, and are thus unable to comprehensively cover against loss of their paralog. Our findings reconcile the fact that duplicates can compensate for each other's loss under a limited number of conditions with the evolutionary instability of genes whose loss is not associated with a phenotypic penalty.\",\n",
       "  'doi': '10.1038/msb4100127',\n",
       "  'journal': 'Molecular systems biology'},\n",
       " {'title': 'CCR4/NOT complex associates with the proteasome and regulates histone methylation.',\n",
       "  'authors': ['R Nicholas Laribee',\n",
       "   'Yoichiro Shibata',\n",
       "   'Douglas P Mersman',\n",
       "   'Sean R Collins',\n",
       "   'Patrick Kemmeren',\n",
       "   'Assen Roguev',\n",
       "   'Jonathan S Weissman',\n",
       "   'Scott D Briggs',\n",
       "   'Nevan J Krogan',\n",
       "   'Brian D Strahl'],\n",
       "  'publication_date': '29/03/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17389396',\n",
       "  'abstract': 'The proteasome regulates histone lysine methylation and gene transcription, but how it does so is poorly understood. To better understand this process, we used the epistatic miniarray profile (E-MAP) approach to identify factors that genetically interact with proteasomal subunits. In addition to members of the Set1 complex that mediate histone H3 lysine 4 methylation (H3K4me), we found that deleting members of the CCR4/NOT mRNA processing complex exhibit synthetic phenotypes when combined with proteasome mutants. Further biochemical analyses revealed physical associations between CCR4/NOT and the proteasome in vivo. Consistent with the genetic and biochemical interactions linking CCR4/NOT with proteasome and Set1-mediated methylation, we find that loss of Not4 decreases global and gene-specific H3K4 trimethylation (H3K4me3) and decreases 19S proteasome recruitment to the PMA1 gene. Similar to proteasome regulation of histone methylation, loss of CCR4/NOT members does not affect ubiquitinated H2B. Mapping of Not4 identified the RING finger domain as essential for H3K4me3, suggesting a role for ubiquitin in this process. Consistent with this idea, loss of the Not4-interacting protein Ubc4, a known ubiquitin-conjugating enzyme, decreases H3K4me3. These studies implicate CCR4/NOT in the regulation of H3K4me3 through a ubiquitin-dependent pathway that likely involves the proteasome.',\n",
       "  'doi': '10.1073/pnas.0607996104',\n",
       "  'journal': 'PNAS'},\n",
       " {'title': 'Functional dissection of protein complexes involved in yeast chromosome biology using a genetic interaction map.',\n",
       "  'authors': ['Sean R Collins',\n",
       "   'Kyle M Miller',\n",
       "   'Nancy L Maas',\n",
       "   'Assen Roguev',\n",
       "   'Jeffrey Fillingham',\n",
       "   'Clement S Chu',\n",
       "   'Maya Schuldiner',\n",
       "   'Marinella Gebbia',\n",
       "   'Judith Recht',\n",
       "   'Michael Shales',\n",
       "   'Huiming Ding',\n",
       "   'Hong Xu',\n",
       "   'Junhong Han',\n",
       "   'Kristin Ingvarsdottir',\n",
       "   'Benjamin Cheng',\n",
       "   'Brenda Andrews',\n",
       "   'Charles Boone',\n",
       "   'Shelley L Berger',\n",
       "   'Phil Hieter',\n",
       "   'Zhiguo Zhang',\n",
       "   'Grant W Brown',\n",
       "   'C James Ingles',\n",
       "   'Andrew Emili',\n",
       "   'C David Allis',\n",
       "   'David P Toczyski',\n",
       "   'Jonathan S Weissman',\n",
       "   'Jack F Greenblatt',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '23/02/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17314980',\n",
       "  'abstract': 'Defining the functional relationships between proteins is critical for understanding virtually all aspects of cell biology. Large-scale identification of protein complexes has provided one important step towards this goal; however, even knowledge of the stoichiometry, affinity and lifetime of every protein-protein interaction would not reveal the functional relationships between and within such complexes. Genetic interactions can provide functional information that is largely invisible to protein-protein interaction data sets. Here we present an epistatic miniarray profile (E-MAP) consisting of quantitative pairwise measurements of the genetic interactions between 743 Saccharomyces cerevisiae genes involved in various aspects of chromosome biology (including DNA replication/repair, chromatid segregation and transcriptional regulation). This E-MAP reveals that physical interactions fall into two well-represented classes distinguished by whether or not the individual proteins act coherently to carry out a common function. Thus, genetic interaction data make it possible to dissect functionally multi-protein complexes, including Mediator, and to organize distinct protein complexes into pathways. In one pathway defined here, we show that Rtt109 is the founding member of a novel class of histone acetyltransferases responsible for Asf1-dependent acetylation of histone H3 on lysine 56. This modification, in turn, enables a ubiquitin ligase complex containing the cullin Rtt101 to ensure genomic integrity during DNA replication.',\n",
       "  'doi': '10.1038/nature05649',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Toward a comprehensive atlas of the physical interactome of Saccharomyces cerevisiae.',\n",
       "  'authors': ['Sean R Collins',\n",
       "   'Patrick Kemmeren',\n",
       "   'Xue-Chu Zhao',\n",
       "   'Jack F Greenblatt',\n",
       "   'Forrest Spencer',\n",
       "   'Frank C P Holstege',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '04/01/07',\n",
       "  'publication_year': '2007',\n",
       "  'pubmed_id': '17200106',\n",
       "  'abstract': 'Defining protein complexes is critical to virtually all aspects of cell biology. Two recent affinity purification/mass spectrometry studies in Saccharomyces cerevisiae have vastly increased the available protein interaction data. The practical utility of such high throughput interaction sets, however, is substantially decreased by the presence of false positives. Here we created a novel probabilistic metric that takes advantage of the high density of these data, including both the presence and absence of individual associations, to provide a measure of the relative confidence of each potential protein-protein interaction. This analysis largely overcomes the noise inherent in high throughput immunoprecipitation experiments. For example, of the 12,122 binary interactions in the general repository of interaction data (BioGRID) derived from these two studies, we marked 7504 as being of substantially lower confidence. Additionally, applying our metric and a stringent cutoff we identified a set of 9074 interactions (including 4456 that were not among the 12,122 interactions) with accuracy comparable to that of conventional small scale methodologies. Finally we organized proteins into coherent multisubunit complexes using hierarchical clustering. This work thus provides a highly accurate physical interaction map of yeast in a format that is readily accessible to the biological community.',\n",
       "  'doi': '10.1074/mcp.M600381-MCP200',\n",
       "  'journal': 'Molecular & cellular proteomics'},\n",
       " {'title': 'Systems biology: many things from one.',\n",
       "  'authors': ['John R S Newman', 'Jonathan S Weissman'],\n",
       "  'publication_date': '24/11/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '17122769',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1038/nature05407',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'An efficient protein transformation protocol for introducing prions into yeast.',\n",
       "  'authors': ['Motomasa Tanaka', 'Jonathan S Weissman'],\n",
       "  'publication_date': '19/10/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '17046659',\n",
       "  'abstract': 'Although a range of robust techniques exists for transforming organisms with nucleic acids, approaches for introducing proteins into cells are far less developed. Here we describe a facile and highly efficient protein transformation protocol suitable for introducing prion particles, produced in vitro from pure protein or purified from an in vivo source, into yeast. Prion particles composed of amyloid forms of fragments of Sup35p, the protein determinant of the yeast prion state [PSI(+)], lead to dose-dependent de novo induction of [PSI(+)] with efficiencies approaching 100% at high protein concentrations. We also describe a procedure for generating distinct, self-propagating amyloid conformations of a prionogenic Sup35p fragment termed Sup-NM. Remarkably, infection of yeast with different Sup-NM amyloid conformations leads to distinct [PSI(+)] prion strains, establishing that the heritable differences in prion strain differences result directly from self-propagating differences in the conformations of the infectious protein. This protein transformation protocol can be readily adapted to the analysis of other yeast prion states, as well as to test the infectious (prion) nature of protein extracts from less well-characterized epigenetic traits. More generally, the protein transformation procedure makes it possible to bridge in vitro and in vivo studies, thus greatly facilitating efforts to explain the structural and mechanistic basis of prion inheritance.',\n",
       "  'doi': '10.1016/S0076-6879(06)12012-1',\n",
       "  'journal': 'Methods in enzymology'},\n",
       " {'title': 'A luminal surveillance complex that selects misfolded glycoproteins for ER-associated degradation.',\n",
       "  'authors': ['Vladimir Denic', 'Erin M Quan', 'Jonathan S Weissman'],\n",
       "  'publication_date': '29/07/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16873065',\n",
       "  'abstract': 'How the ER-associated degradation (ERAD) machinery accurately identifies terminally misfolded proteins is poorly understood. For luminal ERAD substrates, this recognition depends on their folding and glycosylation status as well as on the conserved ER lectin Yos9p. Here we show that Yos9p is part of a stable complex that organizes key components of ERAD machinery on both sides of the ER membrane, including the transmembrane ubiquitin ligase Hrd1p. We further demonstrate that Yos9p, together with Kar2p and Hrd3p, forms a luminal surveillance complex that both recruits nonnative proteins to the core ERAD machinery and assists a distinct sugar-dependent step necessary to commit substrates for degradation. When Hrd1p is uncoupled from the Yos9p surveillance complex, degradation can occur independently of the requirement for glycosylation. Thus, Yos9p/Kar2p/Hrd3p acts as a gatekeeper, ensuring correct identification of terminally misfolded proteins by recruiting misfolded forms to the ERAD machinery, contributing to the interrogation of substrate sugar status, and preventing glycosylation-independent degradation.',\n",
       "  'doi': '10.1016/j.cell.2006.05.045',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'A strategy for extracting and analyzing large-scale quantitative epistatic interaction data.',\n",
       "  'authors': ['Sean R Collins',\n",
       "   'Maya Schuldiner',\n",
       "   'Nevan J Krogan',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '25/07/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16859555',\n",
       "  'abstract': 'Recently, approaches have been developed for high-throughput identification of synthetic sick/lethal gene pairs. However, these are only a specific example of the broader phenomenon of epistasis, wherein the presence of one mutation modulates the phenotype of another. We present analysis techniques for generating high-confidence quantitative epistasis scores from measurements made using synthetic genetic array and epistatic miniarray profile (E-MAP) technology, as well as several tools for higher-level analysis of the resulting data that are greatly enhanced by the quantitative score and detection of alleviating interactions.',\n",
       "  'doi': '10.1186/gb-2006-7-7-r63',\n",
       "  'journal': 'Genome biology'},\n",
       " {'title': 'Decay of endoplasmic reticulum-localized mRNAs during the unfolded protein response.',\n",
       "  'authors': ['Julie Hollien', 'Jonathan S Weissman'],\n",
       "  'publication_date': '11/07/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16825573',\n",
       "  'abstract': 'The unfolded protein response (UPR) allows the endoplasmic reticulum (ER) to recover from the accumulation of misfolded proteins, in part by increasing its folding capacity. Inositol-requiring enzyme-1 (IRE1) promotes this remodeling by detecting misfolded ER proteins and activating a transcription factor, X-box-binding protein 1, through endonucleolytic cleavage of its messenger RNA (mRNA). Here, we report that IRE1 independently mediates the rapid degradation of a specific subset of mRNAs, based both on their localization to the ER membrane and on the amino acid sequence they encode. This response is well suited to complement other UPR mechanisms because it could selectively halt production of proteins that challenge the ER and clear the translocation and folding machinery for the subsequent remodeling process.',\n",
       "  'doi': '10.1126/science.1129631',\n",
       "  'journal': 'Science'},\n",
       " {'title': 'The physical basis of how prion conformations determine strain phenotypes.',\n",
       "  'authors': ['Motomasa Tanaka',\n",
       "   'Sean R Collins',\n",
       "   'Brandon H Toyama',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '01/07/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16810177',\n",
       "  'abstract': 'A principle that has emerged from studies of protein aggregation is that proteins typically can misfold into a range of different aggregated forms. Moreover, the phenotypic and pathological consequences of protein aggregation depend critically on the specific misfolded form. A striking example of this is the prion strain phenomenon, in which prion particles composed of the same protein cause distinct heritable states. Accumulating evidence from yeast prions such as [PSI+] and mammalian prions argues that differences in the prion conformation underlie prion strain variants. Nonetheless, it remains poorly understood why changes in the conformation of misfolded proteins alter their physiological effects. Here we present and experimentally validate an analytical model describing how [PSI+] strain phenotypes arise from the dynamic interaction among the effects of prion dilution, competition for a limited pool of soluble protein, and conformation-dependent differences in prion growth and division rates. Analysis of three distinct prion conformations of yeast Sup35 (the [PSI+] protein determinant) and their in vivo phenotypes reveals that the Sup35 amyloid causing the strongest phenotype surprisingly shows the slowest growth. This slow growth, however, is more than compensated for by an increased brittleness that promotes prion division. The propensity of aggregates to undergo breakage, thereby generating new seeds, probably represents a key determinant of their physiological impact for both infectious (prion) and non-infectious amyloids.',\n",
       "  'doi': '10.1038/nature04922',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Single-cell proteomic analysis of S. cerevisiae reveals the architecture of biological noise.',\n",
       "  'authors': ['John R S Newman',\n",
       "   'Sina Ghaemmaghami',\n",
       "   'Jan Ihmels',\n",
       "   'David K Breslow',\n",
       "   'Matthew Noble',\n",
       "   'Joseph L DeRisi',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '16/05/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16699522',\n",
       "  'abstract': \"A major goal of biology is to provide a quantitative description of cellular behaviour. This task, however, has been hampered by the difficulty in measuring protein abundances and their variation. Here we present a strategy that pairs high-throughput flow cytometry and a library of GFP-tagged yeast strains to monitor rapidly and precisely protein levels at single-cell resolution. Bulk protein abundance measurements of >2,500 proteins in rich and minimal media provide a detailed view of the cellular response to these conditions, and capture many changes not observed by DNA microarray analyses. Our single-cell data argue that noise in protein expression is dominated by the stochastic production/destruction of messenger RNAs. Beyond this global trend, there are dramatic protein-specific differences in noise that are strongly correlated with a protein's mode of transcription and its function. For example, proteins that respond to environmental changes are noisy whereas those involved in protein synthesis are quiet. Thus, these studies reveal a remarkable structure to biological noise and suggest that protein noise levels have been selected to reflect the costs and potential benefits of this variation.\",\n",
       "  'doi': '10.1038/nature04785',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Molecular chaperones and protein quality control.',\n",
       "  'authors': ['Bernd Bukau', 'Jonathan Weissman', 'Arthur Horwich'],\n",
       "  'publication_date': '09/05/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16678092',\n",
       "  'abstract': 'In living cells, both newly made and preexisting polypeptide chains are at constant risk for misfolding and aggregation. In accordance with the wide diversity of misfolded forms, elaborate quality-control strategies have evolved to counter these inevitable mishaps. Recent reports describe the removal of aggregates from the cytosol; reveal mechanisms for protein quality control in the endoplasmic reticulum; and provide new insight into two classes of molecular chaperones, the Hsp70 system and the AAA+ (Hsp100) unfoldases.',\n",
       "  'doi': '10.1016/j.cell.2006.04.014',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Global landscape of protein complexes in the yeast Saccharomyces cerevisiae.',\n",
       "  'authors': ['Nevan J Krogan',\n",
       "   'Gerard Cagney',\n",
       "   'Haiyuan Yu',\n",
       "   'Gouqing Zhong',\n",
       "   'Xinghua Guo',\n",
       "   'Alexandr Ignatchenko',\n",
       "   'Joyce Li',\n",
       "   'Shuye Pu',\n",
       "   'Nira Datta',\n",
       "   'Aaron P Tikuisis',\n",
       "   'Thanuja Punna',\n",
       "   'José M Peregrín-Alvarez',\n",
       "   'Michael Shales',\n",
       "   'Xin Zhang',\n",
       "   'Michael Davey',\n",
       "   'Mark D Robinson',\n",
       "   'Alberto Paccanaro',\n",
       "   'James E Bray',\n",
       "   'Anthony Sheung',\n",
       "   'Bryan Beattie',\n",
       "   'Dawn P Richards',\n",
       "   'Veronica Canadien',\n",
       "   'Atanas Lalev',\n",
       "   'Frank Mena',\n",
       "   'Peter Wong',\n",
       "   'Andrei Starostine',\n",
       "   'Myra M Canete',\n",
       "   'James Vlasblom',\n",
       "   'Samuel Wu',\n",
       "   'Chris Orsi',\n",
       "   'Sean R Collins',\n",
       "   'Shamanta Chandran',\n",
       "   'Robin Haw',\n",
       "   'Jennifer J Rilstone',\n",
       "   'Kiran Gandi',\n",
       "   'Natalie J Thompson',\n",
       "   'Gabe Musso',\n",
       "   'Peter St Onge',\n",
       "   'Shaun Ghanny',\n",
       "   'Mandy H Y Lam',\n",
       "   'Gareth Butland',\n",
       "   'Amin M Altaf-Ul',\n",
       "   'Shigehiko Kanaya',\n",
       "   'Ali Shilatifard',\n",
       "   \"Erin O'Shea\",\n",
       "   'Jonathan S Weissman',\n",
       "   'C James Ingles',\n",
       "   'Timothy R Hughes',\n",
       "   'John Parkinson',\n",
       "   'Mark Gerstein',\n",
       "   'Shoshana J Wodak',\n",
       "   'Andrew Emili',\n",
       "   'Jack F Greenblatt'],\n",
       "  'publication_date': '24/03/06',\n",
       "  'publication_year': '2006',\n",
       "  'pubmed_id': '16554755',\n",
       "  'abstract': 'Identification of protein-protein interactions often provides insight into protein function, and many cellular processes are performed by stable protein complexes. We used tandem affinity purification to process 4,562 different tagged proteins of the yeast Saccharomyces cerevisiae. Each preparation was analysed by both matrix-assisted laser desorption/ionization-time of flight mass spectrometry and liquid chromatography tandem mass spectrometry to increase coverage and accuracy. Machine learning was used to integrate the mass spectrometry scores and assign probabilities to the protein-protein interactions. Among 4,087 different proteins identified with high confidence by mass spectrometry from 2,357 successful purifications, our core data set (median precision of 0.69) comprises 7,123 protein-protein interactions involving 2,708 proteins. A Markov clustering algorithm organized these interactions into 547 protein complexes averaging 4.9 subunits per complex, about half of them absent from the MIPS database, as well as 429 additional interactions between pairs of complexes. The data (all of which are available online) will help future studies on individual proteins as well as functional genomics and systems biology.',\n",
       "  'doi': '10.1038/nature04670',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Domain architecture of protein-disulfide isomerase facilitates its dual role as an oxidase and an isomerase in Ero1p-mediated disulfide formation.',\n",
       "  'authors': ['Mohini S Kulp',\n",
       "   'Eva-Maria Frickel',\n",
       "   'Lars Ellgaard',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '22/12/05',\n",
       "  'publication_year': '2005',\n",
       "  'pubmed_id': '16368681',\n",
       "  'abstract': \"Native disulfide bond formation in eukaryotes is dependent on protein-disulfide isomerase (PDI) and its homologs, which contain varying combinations of catalytically active and inactive thioredoxin domains. However, the specific contribution of PDI to the formation of new disulfides versus reduction/rearrangement of non-native disulfides is poorly understood. We analyzed the role of individual PDI domains in disulfide bond formation in a reaction driven by their natural oxidant, Ero1p. We found that Ero1p oxidizes the isolated PDI catalytic thioredoxin domains, A and A' at the same rate. In contrast, we found that in the context of full-length PDI, there is an asymmetry in the rate of oxidation of the two active sites. This asymmetry is the result of a dual effect: an enhanced rate of oxidation of the second catalytic (A') domain and the substrate-mediated inhibition of oxidation of the first catalytic (A) domain. The specific order of thioredoxin domains in PDI is important in establishing the asymmetry in the rate of oxidation of the two active sites thus allowing A and A', two thioredoxin domains that are similar in sequence and structure, to serve opposing functional roles as a disulfide isomerase and disulfide oxidase, respectively. These findings reveal how native disulfide folding is accomplished in the endoplasmic reticulum and provide a context for understanding the proliferation of PDI homologs with combinatorial arrangements of thioredoxin domains.\",\n",
       "  'doi': '10.1074/jbc.M511764200',\n",
       "  'journal': 'The Journal of biological chemistry'},\n",
       " {'title': 'Cotranscriptional set2 methylation of histone H3 lysine 36 recruits a repressive Rpd3 complex.',\n",
       "  'authors': ['Michael-Christopher Keogh',\n",
       "   'Siavash K Kurdistani',\n",
       "   'Stephanie A Morris',\n",
       "   'Seong Hoon Ahn',\n",
       "   'Vladimir Podolny',\n",
       "   'Sean R Collins',\n",
       "   'Maya Schuldiner',\n",
       "   'Kayu Chin',\n",
       "   'Thanuja Punna',\n",
       "   'Natalie J Thompson',\n",
       "   'Charles Boone',\n",
       "   'Andrew Emili',\n",
       "   'Jonathan S Weissman',\n",
       "   'Timothy R Hughes',\n",
       "   'Brian D Strahl',\n",
       "   'Michael Grunstein',\n",
       "   'Jack F Greenblatt',\n",
       "   'Stephen Buratowski',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '16/11/05',\n",
       "  'publication_year': '2005',\n",
       "  'pubmed_id': '16286008',\n",
       "  'abstract': 'The yeast histone deacetylase Rpd3 can be recruited to promoters to repress transcription initiation. Biochemical, genetic, and gene-expression analyses show that Rpd3 exists in two distinct complexes. The smaller complex, Rpd3C(S), shares Sin3 and Ume1 with Rpd3C(L) but contains the unique subunits Rco1 and Eaf3. Rpd3C(S) mutants exhibit phenotypes remarkably similar to those of Set2, a histone methyltransferase associated with elongating RNA polymerase II. Chromatin immunoprecipitation and biochemical experiments indicate that the chromodomain of Eaf3 recruits Rpd3C(S) to nucleosomes methylated by Set2 on histone H3 lysine 36, leading to deacetylation of transcribed regions. This pathway apparently acts to negatively regulate transcription because deleting the genes for Set2 or Rpd3C(S) bypasses the requirement for the positive elongation factor Bur1/Bur2.',\n",
       "  'doi': '10.1016/j.cell.2005.10.025',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Exploration of the function and organization of the yeast early secretory pathway through an epistatic miniarray profile.',\n",
       "  'authors': ['Maya Schuldiner',\n",
       "   'Sean R Collins',\n",
       "   'Natalie J Thompson',\n",
       "   'Vladimir Denic',\n",
       "   'Arunashree Bhamidipati',\n",
       "   'Thanuja Punna',\n",
       "   'Jan Ihmels',\n",
       "   'Brenda Andrews',\n",
       "   'Charles Boone',\n",
       "   'Jack F Greenblatt',\n",
       "   'Jonathan S Weissman',\n",
       "   'Nevan J Krogan'],\n",
       "  'publication_date': '05/11/05',\n",
       "  'publication_year': '2005',\n",
       "  'pubmed_id': '16269340',\n",
       "  'abstract': 'We present a strategy for generating and analyzing comprehensive genetic-interaction maps, termed E-MAPs (epistatic miniarray profiles), comprising quantitative measures of aggravating or alleviating interactions between gene pairs. Crucial to the interpretation of E-MAPs is their high-density nature made possible by focusing on logically connected gene subsets and including essential genes. Described here is the analysis of an E-MAP of genes acting in the yeast early secretory pathway. Hierarchical clustering, together with novel analytical strategies and experimental verification, revealed or clarified the role of many proteins involved in extensively studied processes such as sphingolipid metabolism and retention of HDEL proteins. At a broader level, analysis of the E-MAP delineated pathway organization and components of physical complexes and illustrated the interconnection between the various secretory processes. Extension of this strategy to other logically connected gene subsets in yeast and higher eukaryotes should provide critical insights into the functional/organizational principles of biological systems.',\n",
       "  'doi': '10.1016/j.cell.2005.08.031',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'Exploration of the topological requirements of ERAD identifies Yos9p as a lectin sensor of misfolded glycoproteins in the ER lumen.',\n",
       "  'authors': ['Arunashree Bhamidipati',\n",
       "   'Vladimir Denic',\n",
       "   'Erin M Quan',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '20/09/05',\n",
       "  'publication_year': '2005',\n",
       "  'pubmed_id': '16168370',\n",
       "  'abstract': \"ER-associated degradation (ERAD) of glycoproteins depends on dual recognition of protein misfolding and remodeling of the substrate's N-linked glycans. After recognition, substrates are retrotranslocated to the cytosol for proteasomal degradation. To explore the directionality of this process, we fused a highly stable protein, DHFR, to the N or C terminus of the soluble ERAD substrate CPY* in yeast. Degradation of the C-terminal CPY*-DHFR fusion is markedly slowed and is accompanied by DHFR release in the ER lumen. Thus, folded lumenal domains can impede protein retrotranslocation. The ER lumenal protein Yos9p is required for both release of DHFR and degradation of multiple ERAD substrates. Yos9p forms a complex with substrates and has a sugar binding pocket that is essential for its ERAD function. Nonetheless, substrate recognition persists even when the sugar binding site is mutated or CPY* is unglycosylated. These and other considerations suggest that Yos9p plays a critical role in the bipartite recognition of terminally misfolded glycoproteins.\",\n",
       "  'doi': '10.1016/j.molcel.2005.07.027',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Mechanism of cross-species prion transmission: an infectious conformation compatible with two highly divergent yeast prion proteins.',\n",
       "  'authors': ['Motomasa Tanaka',\n",
       "   'Peter Chien',\n",
       "   'Koji Yonekura',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '12/04/05',\n",
       "  'publication_year': '2005',\n",
       "  'pubmed_id': '15820678',\n",
       "  'abstract': 'Efficiency of interspecies prion transmission decreases as the primary structures of the infectious proteins diverge. Yet, a single prion protein can misfold into multiple infectious conformations, and such differences in \"strain conformation\" also alter infection specificity. Here, we explored the relationship between prion strains and species barriers by creating distinct synthetic prion forms of the yeast prion protein Sup35. We identified a strain conformation of Sup35 that allows transmission from the S. cerevisiae (Sc) Sup35 to the highly divergent C. albicans (Ca) Sup35 both in vivo and in vitro. Remarkably, cross-species transmission leads to a novel Ca strain that in turn can infect the Sc protein. Structural studies reveal strain-specific conformational differences in regions of the prion domain that are involved in intermolecular contacts. Our findings support a model whereby strain conformation is the critical determinant of cross-species prion transmission while primary structure affects transmission specificity by altering the spectrum of preferred amyloid conformations.',\n",
       "  'doi': '10.1016/j.cell.2005.03.008',\n",
       "  'journal': 'Cell'},\n",
       " {'title': '2004 Irving Sigal Young Investigator Award.',\n",
       "  'authors': ['Jonathan S Weissman', \"Erin K O'Shea\"],\n",
       "  'publication_date': '24/11/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '15557272',\n",
       "  'abstract': None,\n",
       "  'doi': '10.1110/ps.041134604',\n",
       "  'journal': 'Protein science : a publication of the Protein Society'},\n",
       " {'title': 'Mechanism of prion propagation: amyloid growth occurs by monomer addition.',\n",
       "  'authors': ['Sean R Collins',\n",
       "   'Adam Douglass',\n",
       "   'Ronald D Vale',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '24/09/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '15383837',\n",
       "  'abstract': 'Abundant nonfibrillar oligomeric intermediates are a common feature of amyloid formation, and these oligomers, rather than the final fibers, have been suggested to be the toxic species in some amyloid diseases. Whether such oligomers are critical intermediates for fiber assembly or form in an alternate, potentially separable pathway, however, remains unclear. Here we study the polymerization of the amyloidogenic yeast prion protein Sup35. Rapid polymerization occurs in the absence of observable intermediates, and both targeted kinetic and direct single-molecule fluorescence measurements indicate that fibers grow by monomer addition. A three-step model (nucleation, monomer addition, and fiber fragmentation) accurately accounts for the distinctive kinetic features of amyloid formation, including weak concentration dependence, acceleration by agitation, and sigmoidal shape of the polymerization time course. Thus, amyloid growth can occur by monomer addition in a reaction distinct from and competitive with formation of potentially toxic oligomeric intermediates.',\n",
       "  'doi': '10.1371/journal.pbio.0020321',\n",
       "  'journal': 'PLoS biology'},\n",
       " {'title': 'Emerging principles of conformation-based prion inheritance.',\n",
       "  'authors': ['Peter Chien', 'Jonathan S Weissman', 'Angela H DePace'],\n",
       "  'publication_date': '11/06/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '15189155',\n",
       "  'abstract': 'The prion hypothesis proposes that proteins can act as infectious agents. Originally formulated to explain transmissible spongiform encephalopathies (TSEs), the prion hypothesis has been extended with the finding that several non-Mendelian traits in fungi are due to heritable changes in protein conformation, which may in some cases be beneficial. Although much remains to be learned about the specific role of cellular cofactors, mechanistic parallels between the mammalian and yeast prion phenomena point to universal features of conformation-based infection and inheritance involving propagation of ordered beta-sheet-rich protein aggregates commonly referred to as amyloid. Here we focus on two such features and discuss recent efforts to explain them in terms of the physical properties of amyloid-like aggregates. The first is prion strains, wherein chemically identical infectious particles cause distinct phenotypes. The second is barriers that often prohibit prion transmission between different species. There is increasing evidence suggesting that both of these can be manifestations of the same phenomenon: the ability of a protein to misfold into multiple self-propagating conformations. Even single mutations can change the spectrum of favored misfolded conformations. In turn, changes in amyloid conformation can shift the specificity of propagation and alter strain phenotypes. This model helps explain many common and otherwise puzzling features of prion inheritance as well as aspects of noninfectious diseases involving toxic misfolded proteins.',\n",
       "  'doi': '10.1146/annurev.biochem.72.121801.161837',\n",
       "  'journal': 'Annual review of biochemistry'},\n",
       " {'title': 'Dissection and design of yeast prions.',\n",
       "  'authors': ['Lev Z Osherovich',\n",
       "   'Brian S Cox',\n",
       "   'Mick F Tuite',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '27/03/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '15045026',\n",
       "  'abstract': 'Many proteins can misfold into beta-sheet-rich, self-seeding polymers (amyloids). Prions are exceptional among such aggregates in that they are also infectious. In fungi, prions are not pathogenic but rather act as epigenetic regulators of cell physiology, providing a powerful model for studying the mechanism of prion replication. We used prion-forming domains from two budding yeast proteins (Sup35p and New1p) to examine the requirements for prion formation and inheritance. In both proteins, a glutamine/asparagine-rich (Q/N-rich) tract mediates sequence-specific aggregation, while an adjacent motif, the oligopeptide repeat, is required for the replication and stable inheritance of these aggregates. Our findings help to explain why although Q/N-rich proteins are relatively common, few form heritable aggregates: prion inheritance requires both an aggregation sequence responsible for self-seeded growth and an element that permits chaperone-dependent replication of the aggregate. Using this knowledge, we have designed novel artificial prions by fusing the replication element of Sup35p to aggregation-prone sequences from other proteins, including pathogenically expanded polyglutamine.',\n",
       "  'doi': '10.1371/journal.pbio.0020086',\n",
       "  'journal': 'PLoS biology'},\n",
       " {'title': 'Conformational variations in an infectious protein determine prion strain differences.',\n",
       "  'authors': ['Motomasa Tanaka',\n",
       "   'Peter Chien',\n",
       "   'Nariman Naber',\n",
       "   'Roger Cooke',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '19/03/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '15029196',\n",
       "  'abstract': \"A remarkable feature of prion biology is the strain phenomenon wherein prion particles apparently composed of the same protein lead to phenotypically distinct transmissible states. To reconcile the existence of strains with the 'protein-only' hypothesis of prion transmission, it has been proposed that a single protein can misfold into multiple distinct infectious forms, one for each different strain. Several studies have found correlations between strain phenotypes and conformations of prion particles; however, whether such differences cause or are simply a secondary manifestation of prion strains remains unclear, largely due to the difficulty of creating infectious material from pure protein. Here we report a high-efficiency protocol for infecting yeast with the [PSI+] prion using amyloids composed of a recombinant Sup35 fragment (Sup-NM). Using thermal stability and electron paramagnetic resonance spectroscopy, we demonstrate that Sup-NM amyloids formed at different temperatures adopt distinct, stably propagating conformations. Infection of yeast with these different amyloid conformations leads to different [PSI+] strains. These results establish that Sup-NM adopts an infectious conformation before entering the cell--fulfilling a key prediction of the prion hypothesis--and directly demonstrate that differences in the conformation of the infectious protein determine prion strain variation.\",\n",
       "  'doi': '10.1038/nature02392',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Oxidative protein folding in eukaryotes: mechanisms and consequences.',\n",
       "  'authors': ['Benjamin P Tu', 'Jonathan S Weissman'],\n",
       "  'publication_date': '06/02/04',\n",
       "  'publication_year': '2004',\n",
       "  'pubmed_id': '14757749',\n",
       "  'abstract': 'The endoplasmic reticulum (ER) provides an environment that is highly optimized for oxidative protein folding. Rather than relying on small molecule oxidants like glutathione, it is now clear that disulfide formation is driven by a protein relay involving Ero1, a novel conserved FAD-dependent enzyme, and protein disulfide isomerase (PDI); Ero1 is oxidized by molecular oxygen and in turn acts as a specific oxidant of PDI, which then directly oxidizes disulfide bonds in folding proteins. While providing a robust driving force for disulfide formation, the use of molecular oxygen as the terminal electron acceptor can lead to oxidative stress through the production of reactive oxygen species and oxidized glutathione. How Ero1p distinguishes between the many different PDI-related proteins and how the cell minimizes the effects of oxidative damage from Ero1 remain important open questions.',\n",
       "  'doi': '10.1083/jcb.200311055',\n",
       "  'journal': 'The Journal of cell biology'},\n",
       " {'title': 'Global analysis of protein expression in yeast.',\n",
       "  'authors': ['Sina Ghaemmaghami',\n",
       "   'Won-Ki Huh',\n",
       "   'Kiowa Bower',\n",
       "   'Russell W Howson',\n",
       "   'Archana Belle',\n",
       "   'Noah Dephoure',\n",
       "   \"Erin K O'Shea\",\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '17/10/03',\n",
       "  'publication_year': '2003',\n",
       "  'pubmed_id': '14562106',\n",
       "  'abstract': 'The availability of complete genomic sequences and technologies that allow comprehensive analysis of global expression profiles of messenger RNA have greatly expanded our ability to monitor the internal state of a cell. Yet biological systems ultimately need to be explained in terms of the activity, regulation and modification of proteins--and the ubiquitous occurrence of post-transcriptional regulation makes mRNA an imperfect proxy for such information. To facilitate global protein analyses, we have created a Saccharomyces cerevisiae fusion library where each open reading frame is tagged with a high-affinity epitope and expressed from its natural chromosomal location. Through immunodetection of the common tag, we obtain a census of proteins expressed during log-phase growth and measurements of their absolute levels. We find that about 80% of the proteome is expressed during normal growth conditions, and, using additional sequence information, we systematically identify misannotated genes. The abundance of proteins ranges from fewer than 50 to more than 10(6) molecules per cell. Many of these molecules, including essential proteins and most transcription factors, are present at levels that are not readily detectable by other proteomic techniques nor predictable by mRNA levels or codon bias measurements.',\n",
       "  'doi': '10.1038/nature02046',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Global analysis of protein localization in budding yeast.',\n",
       "  'authors': ['Won-Ki Huh',\n",
       "   'James V Falvo',\n",
       "   'Luke C Gerke',\n",
       "   'Adam S Carroll',\n",
       "   'Russell W Howson',\n",
       "   'Jonathan S Weissman',\n",
       "   \"Erin K O'Shea\"],\n",
       "  'publication_date': '17/10/03',\n",
       "  'publication_year': '2003',\n",
       "  'pubmed_id': '14562095',\n",
       "  'abstract': 'A fundamental goal of cell biology is to define the functions of proteins in the context of compartments that organize them in the cellular environment. Here we describe the construction and analysis of a collection of yeast strains expressing full-length, chromosomally tagged green fluorescent protein fusion proteins. We classify these proteins, representing 75% of the yeast proteome, into 22 distinct subcellular localization categories, and provide localization information for 70% of previously unlocalized proteins. Analysis of this high-resolution, high-coverage localization data set in the context of transcriptional, genetic, and protein-protein interaction data helps reveal the logic of transcriptional co-regulation, and provides a comprehensive view of interactions within and between organelles in eukaryotic cells.',\n",
       "  'doi': '10.1038/nature02026',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Generation of prion transmission barriers by mutational control of amyloid conformations.',\n",
       "  'authors': ['Peter Chien',\n",
       "   'Angela H DePace',\n",
       "   'Sean R Collins',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '22/08/03',\n",
       "  'publication_year': '2003',\n",
       "  'pubmed_id': '12931190',\n",
       "  'abstract': \"Self-propagating beta-sheet-rich protein aggregates are implicated in a wide range of protein-misfolding phenomena, including amyloid diseases and prion-based inheritance. Two properties have emerged as common features of amyloids. Amyloid formation is ubiquitous: many unrelated proteins form such aggregates and even a single polypeptide can misfold into multiple forms--a process that is thought to underlie prion strain variation. Despite this promiscuity, amyloid propagation can be highly sequence specific: amyloid fibres often fail to catalyse the aggregation of other amyloidogenic proteins. In prions, this specificity leads to barriers that limit transmission between species. Using the yeast prion [PSI+], we show in vitro that point mutations in Sup35p, the protein determinant of [PSI+], alter the range of 'infectious' conformations, which in turn changes amyloid seeding specificity. We generate a new transmission barrier in vivo by using these mutations to specifically disfavour subsets of prion strains. The ability of mutations to alter the conformations of amyloid states without preventing amyloid formation altogether provides a general mechanism for the generation of prion transmission barriers and may help to explain how mutations alter toxicity in conformational diseases.\",\n",
       "  'doi': '10.1038/nature01894',\n",
       "  'journal': 'Nature'},\n",
       " {'title': 'Directed evolution of substrate-optimized GroEL/S chaperonins.',\n",
       "  'authors': ['Jue D Wang',\n",
       "   'Christophe Herman',\n",
       "   'Kimberly A Tipton',\n",
       "   'Carol A Gross',\n",
       "   'Jonathan S Weissman'],\n",
       "  'publication_date': '01/01/03',\n",
       "  'publication_year': '2003',\n",
       "  'pubmed_id': '12507429',\n",
       "  'abstract': 'GroEL/S chaperonin ring complexes fold many unrelated proteins. To understand the basis and extent of the chaperonin substrate spectrum, we used rounds of selection and DNA shuffling to obtain GroEL/S variants that dramatically enhanced folding of a single substrate-green fluorescent protein (GFP). Changes in the substrate-optimized chaperonins increase the polarity of the folding cavity and alter the ATPase cycle. These findings reveal a surprising plasticity of GroEL/S, which can be exploited to aid folding of recombinant proteins. Our studies also reveal a conflict between specialization and generalization of chaperonins as increased GFP folding comes at the expense of the ability of GroEL/S to fold its natural substrates. This conflict and the nature of the ring structure may help explain the evolution of cellular chaperone systems.',\n",
       "  'doi': '10.1016/s0092-8674(02)01198-4',\n",
       "  'journal': 'Cell'},\n",
       " {'title': 'The FAD- and O(2)-dependent reaction cycle of Ero1-mediated oxidative protein folding in the endoplasmic reticulum.',\n",
       "  'authors': ['Benjamin P Tu', 'Jonathan S Weissman'],\n",
       "  'publication_date': '28/11/02',\n",
       "  'publication_year': '2002',\n",
       "  'pubmed_id': '12453408',\n",
       "  'abstract': \"The endoplasmic reticulum (ER) supports disulfide formation through an essential protein relay involving Ero1p and protein disulfide isomerase (PDI). We find that in addition to having a tightly associated flavin adenine dinucleotide (FAD) moiety, yeast Ero1p is highly responsive to small changes in physiological levels of free FAD. This sensitivity underlies the dependence of oxidative protein folding on cellular FAD levels. FAD is synthesized in the cytosol but can readily enter the ER lumen and promote Ero1p-catalyzed oxidation. Ero1p then uses molecular oxygen as its preferred terminal electron acceptor. Thus Ero1p directly couples disulfide formation to the consumption of molecular oxygen, but its activity is modulated by free lumenal FAD levels, potentially linking disulfide formation to a cell's nutritional or metabolic status.\",\n",
       "  'doi': '10.1016/s1097-2765(02)00696-2',\n",
       "  'journal': 'Molecular cell'},\n",
       " {'title': 'Origins and kinetic consequences of diversity in Sup35 yeast prion fibers.',\n",
       "  'authors': ['Angela H DePace', 'Jonathan S Weissman'],\n",
       "  'publication_date': '09/04/02',\n",
       "  'publication_year': '2002',\n",
       "  'pubmed_id': '11938354',\n",
       "  'abstract': 'A remarkable feature of prions is that infectious particles composed of the same prion protein can give rise to different phenotypes. This strain phenomenon suggests that a single prion protein can adopt multiple infectious conformations. Here we use a novel single fiber growth assay to examine the heterogeneity of amyloid fibers formed by the yeast Sup35 prion protein. Sup35 spontaneously forms multiple, distinct and faithfully propagating fiber types, which differ dramatically both in their degrees of polarity and overall growth rates. Both in terms of the number of distinct self-propagating fiber types, as well as the ability of these differences to dictate the rate of prion growth, this diversity is well suited to account for the range of prion strain phenotypes observed in vivo.',\n",
       "  'doi': '10.1038/nsb786',\n",
       "  'journal': 'Nature structural biology'},\n",
       " {'title': 'The utility of prions.',\n",
       "  'authors': ['Lev Z Osherovich', 'Jonathan S Weissman'],\n",
       "  'publication_date': '08/02/02',\n",
       "  'publication_year': '2002',\n",
       "  'pubmed_id': '11832240',\n",
       "  'abstract': 'Infectious, self-propagating protein aggregates (prions) as well as structurally related amyloid fibrils have traditionally been associated with neurodegenerative diseases in mammals. However, recent work in fungi indicates that prions are not simply aberrations of protein folding, but are in fact widespread, conserved, and in certain cases, apparently beneficial. Analysis of prion behavior in yeast has led to insights into the mechanisms of prion appearance and propagation as well as the effect of prions on cellular physiology and perhaps evolution. The prion-forming proteins of Saccharomyces cerevisiae are members of a larger class of Gln/Asn-rich proteins that is abundantly represented in the genomes of higher eukaryotes, raising the prospect of genetically programmed prion-like behavior in other organisms.',\n",
       "  'doi': '10.1016/s1534-5807(02)00118-1',\n",
       "  'journal': 'Developmental cell'}]"
      ]
     },
     "execution_count": 14,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pubs_records"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "b33a32b6-dc27-47c2-84d3-c10ee261175a",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "438632"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "open('_data/publications.yml', 'w').write(yaml.dump(pubs_records, default_flow_style=False, sort_keys=False))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ce98febf-fc16-4529-b8d6-2630646bb4fe",
   "metadata": {
    "tags": []
   },
   "source": [
    "### Trim down lab photos to max size"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "e6707396-53be-4d3c-a270-0077d0dae0d3",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      ".DS_Store\n"
     ]
    }
   ],
   "source": [
    "import os\n",
    "for img_path in os.listdir('assets/img/people/'):\n",
    "    try:\n",
    "        image = Image.open('assets/img/people/'+img_path)\n",
    "        if image.size[0] > 400 or image.size[1] > 400:\n",
    "            image.thumbnail((400, 400))\n",
    "            # convert image to jpg if png\n",
    "            if image.format == 'PNG':\n",
    "                image = image.convert('RGB')\n",
    "                img_path = img_path.split('.')[0]+'.jpg'\n",
    "            image.save('assets/img/people/'+img_path)\n",
    "    except:\n",
    "        print(img_path)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "43333216-0575-46cb-8e4e-5621ed900139",
   "metadata": {},
   "outputs": [],
   "source": [
    "# id must be both the publication first/last, and the name of the image. Name is just the display name."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d4665bcf",
   "metadata": {},
   "source": [
    "## Update people page"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "006f6993-ac2b-4298-ad0f-ff149387735f",
   "metadata": {},
   "outputs": [],
   "source": [
    "url = 'https://docs.google.com/spreadsheets/d/1-Eju9h1XovEBoBv0DGpxh92GYsZ8bzGkxRdRgUb7hvg'\n",
    "people = pd.read_csv(url+'/export?format=csv')\n",
    "# strip strings\n",
    "df_obj = people.select_dtypes(['object'])\n",
    "people[df_obj.columns] = df_obj.apply(lambda x: x.str.strip())"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "1898b1f0-9b78-48c5-bf38-ac1db5fb4158",
   "metadata": {},
   "outputs": [],
   "source": [
    "with open('_data/people.yml', 'r') as f:\n",
    "    old_people_data = pd.json_normalize(yaml.safe_load(f))"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "4be671d7-6384-4c94-80c9-9840bfd3cfe5",
   "metadata": {
    "tags": []
   },
   "source": [
    "#### New IDs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "02ddcce2-a14f-43f0-b507-2d1d86843a18",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>pos</th>\n",
       "      <th>email</th>\n",
       "      <th>id</th>\n",
       "      <th>alum</th>\n",
       "      <th>now</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Tessa Dinnie</td>\n",
       "      <td>Administrative Lab Manager</td>\n",
       "      <td>tdinnie@wi.mit.edu</td>\n",
       "      <td>TessaDinnie</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Irina Cashen</td>\n",
       "      <td>Administrative Assistant</td>\n",
       "      <td>icashen@wi.mit.edu</td>\n",
       "      <td>IrinaCashen</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>Yijia Chen</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>ychen@wi.mit.edu</td>\n",
       "      <td>YijiaChen</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>Max Hyman</td>\n",
       "      <td>Technician</td>\n",
       "      <td>mhyman@wi.mit.edu</td>\n",
       "      <td>MaxHyman</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>Xinjie Pan</td>\n",
       "      <td>Visiting Scientist</td>\n",
       "      <td>xpan@wi.mit.edu</td>\n",
       "      <td>XinjiePan</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>Gokul Gowri</td>\n",
       "      <td>Visiting Scientist</td>\n",
       "      <td>gokulg@mit.edu</td>\n",
       "      <td>GokulGowri</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "            name                         pos               email           id  \\\n",
       "2   Tessa Dinnie  Administrative Lab Manager  tdinnie@wi.mit.edu  TessaDinnie   \n",
       "3   Irina Cashen    Administrative Assistant  icashen@wi.mit.edu  IrinaCashen   \n",
       "35    Yijia Chen                Grad student    ychen@wi.mit.edu    YijiaChen   \n",
       "36     Max Hyman                  Technician   mhyman@wi.mit.edu     MaxHyman   \n",
       "61    Xinjie Pan          Visiting Scientist     xpan@wi.mit.edu    XinjiePan   \n",
       "63   Gokul Gowri          Visiting Scientist      gokulg@mit.edu   GokulGowri   \n",
       "\n",
       "     alum  now  \n",
       "2   False  NaN  \n",
       "3   False  NaN  \n",
       "35  False  NaN  \n",
       "36  False  NaN  \n",
       "61  False  NaN  \n",
       "63  False  NaN  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "people[ ~people.id.isin(old_people_data.id)]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d71717f6-ab76-408f-baad-2e248907784e",
   "metadata": {},
   "source": [
    "#### Updates to existing IDs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "368d3214-2842-403f-adfb-521f50a19f7c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>pos</th>\n",
       "      <th>email</th>\n",
       "      <th>id</th>\n",
       "      <th>alum</th>\n",
       "      <th>now</th>\n",
       "      <th>update</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Edwin Neumann</td>\n",
       "      <td>Post-doc</td>\n",
       "      <td>enn@wi.mit.edu</td>\n",
       "      <td>EdwinNeumann</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>Edwin Neumann</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>enn@wi.mit.edu</td>\n",
       "      <td>EdwinNeumann</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>Ever Marinelli</td>\n",
       "      <td>Undergrad</td>\n",
       "      <td>emarinelli@wi.mit.edu</td>\n",
       "      <td>EverMarinelli</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>Ever Marinelli</td>\n",
       "      <td>Undergrad</td>\n",
       "      <td>emarinelli@wi.mit.edu</td>\n",
       "      <td>EverMarinelli</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>Gayathri Muthukumar</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>gmkumar@wi.mit.edu</td>\n",
       "      <td>GayathriMuthukumar</td>\n",
       "      <td>True</td>\n",
       "      <td>Postdoctoral Fellow | Bertozzi Lab, ChEM-H Ins...</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Gayathri Muthukumar</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>gmkumar@wi.mit.edu</td>\n",
       "      <td>GayathriMuthukumar</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>Jiahe Ai</td>\n",
       "      <td>Undergrad</td>\n",
       "      <td>jaipynnic@wi.mit.edu</td>\n",
       "      <td>JiaheAi</td>\n",
       "      <td>True</td>\n",
       "      <td>PhD student  |  Cancer Science Institute of Si...</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>Jiahe Ai</td>\n",
       "      <td>Undergrad</td>\n",
       "      <td>jaipynnic@wi.mit.edu</td>\n",
       "      <td>JiaheAi</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>Reuben Saunders</td>\n",
       "      <td>Visiting Scientist</td>\n",
       "      <td>reubens@wi.mit.edu</td>\n",
       "      <td>ReubenSaunders</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Reuben Saunders</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>reubens@wi.mit.edu</td>\n",
       "      <td>ReubenSaunders</td>\n",
       "      <td>True</td>\n",
       "      <td>Junior Fellow | Harvard Society of Fellows</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Richard She</td>\n",
       "      <td>Post-doc</td>\n",
       "      <td>rshe@wi.mit.edu</td>\n",
       "      <td>RichardShe</td>\n",
       "      <td>True</td>\n",
       "      <td>Assistant Professor  | Nanyang Technological U...</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Richard She</td>\n",
       "      <td>Post-doc</td>\n",
       "      <td>rshe@wi.mit.edu</td>\n",
       "      <td>RichardShe</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Yassine Damergi</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>ydamergi@wi.mit.edu</td>\n",
       "      <td>YassineDamergi</td>\n",
       "      <td>True</td>\n",
       "      <td>NaN</td>\n",
       "      <td>current</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>Yassine Damergi</td>\n",
       "      <td>Grad student</td>\n",
       "      <td>ydamergi@wi.mit.edu</td>\n",
       "      <td>YassineDamergi</td>\n",
       "      <td>False</td>\n",
       "      <td>NaN</td>\n",
       "      <td>previous</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                   name                 pos                  email  \\\n",
       "26        Edwin Neumann            Post-doc         enn@wi.mit.edu   \n",
       "31        Edwin Neumann        Grad student         enn@wi.mit.edu   \n",
       "51       Ever Marinelli           Undergrad  emarinelli@wi.mit.edu   \n",
       "48       Ever Marinelli           Undergrad  emarinelli@wi.mit.edu   \n",
       "30  Gayathri Muthukumar        Grad student     gmkumar@wi.mit.edu   \n",
       "28  Gayathri Muthukumar        Grad student     gmkumar@wi.mit.edu   \n",
       "52             Jiahe Ai           Undergrad   jaipynnic@wi.mit.edu   \n",
       "49             Jiahe Ai           Undergrad   jaipynnic@wi.mit.edu   \n",
       "62      Reuben Saunders  Visiting Scientist     reubens@wi.mit.edu   \n",
       "25      Reuben Saunders        Grad student     reubens@wi.mit.edu   \n",
       "7           Richard She            Post-doc        rshe@wi.mit.edu   \n",
       "5           Richard She            Post-doc        rshe@wi.mit.edu   \n",
       "33      Yassine Damergi        Grad student    ydamergi@wi.mit.edu   \n",
       "32      Yassine Damergi        Grad student    ydamergi@wi.mit.edu   \n",
       "\n",
       "                    id   alum  \\\n",
       "26        EdwinNeumann  False   \n",
       "31        EdwinNeumann  False   \n",
       "51       EverMarinelli   True   \n",
       "48       EverMarinelli  False   \n",
       "30  GayathriMuthukumar   True   \n",
       "28  GayathriMuthukumar  False   \n",
       "52             JiaheAi   True   \n",
       "49             JiaheAi  False   \n",
       "62      ReubenSaunders  False   \n",
       "25      ReubenSaunders   True   \n",
       "7           RichardShe   True   \n",
       "5           RichardShe  False   \n",
       "33      YassineDamergi   True   \n",
       "32      YassineDamergi  False   \n",
       "\n",
       "                                                  now    update  \n",
       "26                                                NaN   current  \n",
       "31                                                NaN  previous  \n",
       "51                                                NaN   current  \n",
       "48                                                NaN  previous  \n",
       "30  Postdoctoral Fellow | Bertozzi Lab, ChEM-H Ins...   current  \n",
       "28                                                NaN  previous  \n",
       "52  PhD student  |  Cancer Science Institute of Si...   current  \n",
       "49                                                NaN  previous  \n",
       "62                                                NaN   current  \n",
       "25         Junior Fellow | Harvard Society of Fellows  previous  \n",
       "7   Assistant Professor  | Nanyang Technological U...   current  \n",
       "5                                                 NaN  previous  \n",
       "33                                                NaN   current  \n",
       "32                                                NaN  previous  "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "people['update'] = 'current'\n",
    "old_people_data['update'] = 'previous'\n",
    "\n",
    "pd.concat([people[people.id.isin(old_people_data.id)],old_people_data[old_people_data.id.isin(people.id)]]).drop_duplicates(subset=['name', 'pos', 'email', 'alum', 'now'], keep=False).sort_values(by='id')\n"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d8591ba1-b21f-43ac-95d2-bfc585fccfef",
   "metadata": {},
   "source": [
    "#### Removal of IDs:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "f1233203-5d3a-455d-8ced-5faa1da00d73",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>name</th>\n",
       "      <th>pos</th>\n",
       "      <th>email</th>\n",
       "      <th>id</th>\n",
       "      <th>alum</th>\n",
       "      <th>now</th>\n",
       "      <th>update</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [name, pos, email, id, alum, now, update]\n",
       "Index: []"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "old_people_data[ ~old_people_data.id.isin(people.id)]"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6cddae88-f41d-453c-a0ac-d4c5a6456f65",
   "metadata": {
    "tags": []
   },
   "source": [
    "#### Write updated sheet to yaml"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "922ec1a1-b130-4595-a994-0ae9fe3d4f64",
   "metadata": {},
   "outputs": [],
   "source": [
    "people = people.drop('update', axis=1)\n",
    "people_records = [{k:v for k,v in m.items() if pd.notnull(v)} for m in people.to_dict(orient='records')]\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 23,
   "id": "e43cf38b-3bf5-489e-912b-c081151b2fb9",
   "metadata": {},
   "outputs": [],
   "source": [
    "class MyDumper(yaml.SafeDumper):\n",
    "    # HACK: insert blank lines between top-level objects\n",
    "    # inspired by https://stackoverflow.com/a/44284819/3786245\n",
    "    def write_line_break(self, data=None):\n",
    "        super().write_line_break(data)\n",
    "\n",
    "        if len(self.indents) == 1:\n",
    "            super().write_line_break()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "b72f7d29-d480-4646-8afb-f7bb3f49a9aa",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "21594"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "open('_data/people.yml', 'w').write(yaml.dump(people_records, Dumper=MyDumper, default_flow_style=False, sort_keys=False))\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "12a67ae7-88ca-40c7-b671-41379c655ea1",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "py-env",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.11.4"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
